# **ACTA2 -- Familial Thoracic Aortic Aneurysm and Dissection**

**Review of source material:**

**ClinGen:**

https://search.clinicalgenome.org/kb/gene-validity/8249

**ClinGen Evidence for Haploinsufficiency**
Variants in ACTA2 have been observed in individuals with several potentially related phenotypes: thoracic aortic aneurysms and dissections (TAAD); Moyamoya disease; and multisystemic smooth muscle dysfunction syndrome (see OMIM 102620 for additional information). **All reported variants in ACTA2 to date are missense variants. Additional information is needed to determine the mechanism by which variants in ACTA2 result in disease, though a few publications (PMIDs: 17994018 (see below for additional details), GeneReviews NBK1120) suggest that variants result in TAAD by a dominant negative mechanism.**

PMID 17994018 discusses that mutations in ACTA2 gene result in Thoracic aortic aneurysms and dissections (TAAD) . Authors describe a family with autosomal dominant TAAD and a missense mutation in ACTA2. Sequencing of the ACTA2 gene in 97 unrealted TAAD families identified an additional 14 families with mutations. All mutations segregated with TAAD and were absent in 192 controls. Structural analyses and immunofluorescence of ACTA2 filaments in aorticsmooth muscle cells dervived from patients with heterozygous ACTA2 mutations showed that the mutations perturb ACTA2 filament assembly and stability. Furthermore , aortic tissue from ACTA2 mutation patients showed typical findings of medial degredation of the aorta. Based on the findings, authors suggest ACTA2 mutations cause a dominant negative pathogenesis. To date only heterozygous missese mutations have been described in the ACTA2 gene (PMID 26034244,19409525,25207230). PMID 26153720 authors showed the the R258C ACTA2 mutation is less stable and more susceptible to severing by cofilin cleavage when compared to WT. Also smooth muscle myosin moves R258C filaments slower than WTand the slowing is exacerbated by smooth muscle tropomyosin. Overall many of the observed defects are not due to direct interaction with mutant but allosterically effects multiple regions of the monomer. Additional PMID:11053242

**Literature review:**

Guo et al. (2007) PMID 17994018 showed that missense mutations in ACTA2 are responsible for 14% of inherited ascending thoracic aortic aneurysms and dissections. Structural analyses and immunofluorescence of actin filaments in SMCs derived from individuals heterozygous for ACTA2 mutations illustrated that **these mutations interfere with actin filament assembly and are predicted to decrease SMC contraction.** 

The penetrance of TAAD in individuals with ACTA2 mutations was low (0.48) and did not increase with age, differing from the pattern for other identified loci and genes for familial TAAD, which have a higher, age-related penetrance. 

*Omim https://www.omim.org/entry/102620*

"...Several different missense mutations, including
two recurrent mutations found in unrelated
families, have been identified. In the family in
which the disease was mapped, the mutation in
ACTA2 (R149C) segregates invariantly with a
skin rash caused by dermal capillary and small
artery occlusion referred to as livedo reticularis.
Other features associated in a subset of families with ACTA2 mutations include iris flocculi,
PDA, and BAV...The majority of affected individuals
presented with acute type A dissections or type
B dissections, and 16 of 24 deaths occured due
to type A dissections.

**the missense ACTA2 mutations
identified in familial TAAD are predicted
to alter the dynamics of actin assembly into
filaments, by either disrupting the actin-actin
interaction sites or interfering with ATP
hydrolysis.** Analysis of SMCs explanted from
patients heterozygous for ACTA2 mutations
demonstrated reduced ACTA2-containing
fibers and therefore confirmed that ACTA2
missense mutations disrupt actin fiber assembly
or stability..."

*Milewicz D et al. 2008 PMID: 18544034*

"**All missense mutations in ACTA2 reported here affect highly conserved amino acids** and result in a deleterious alteration of the 2nd subdomain surface, a domain important for the conformational change of actin upon the exchange of ADP for ATP, in its turn important in filament assembly dynamics [20]. A dominant negative mechanism with reduced assembly and incomplete and disorganized actin filament assembly was previously suggested for ACTA2 mutations [10]. The current study demonstrates for the first time that TAAD may also result from premature truncating mutations in ACTA2. However, both nonsense mutations are predicted to escape nonsense mediated decay and could as such still exert a dominant negative effect. This confirms the hypothesis formulated by Guo et al, who suggest that mutations resulting in true null alleles are inherited only with a recessive transmission pattern"

*Renard M et al. 2013 May 10 (PMID:21937134)*

Syndromic HTAD. ACTA2 missense pathogenic variants that specifically disrupt the arginine 179 residue cause multisystem smooth-muscle dysfunction syndrome in which dysfunction of smooth muscle cells leads to severe and highly penetrant vascular diseases, pulmonary hypertension, and loss of proper smooth muscle cell contraction in other organs [Guo et al 2009, Milewicz et al 2010, Munot et al 2012]. 

Nonsyndromic HTAD. ACTA2 pathogenic variants, the most frequent cause of nonsyndromic familial HTAD, account for 12%-21% of cases [Guo et al 2007, Morisaki et al 2009, Disabella et al 2011, Renard et al 2013]. 

Thoracic aortic aneurysms are typically fusiform and initially involve the aortic root, extending into the ascending aorta and aortic arch. Descending and abdominal aortic aneurysms are less common. Penetrance for TAAD is reduced: the lifetime risk of an aortic event (aortic dissection or aneurysm repair) is 76% at age 85 years. 

TAAD cosegregating with premature coronary artery disease, ischemic stroke, and moyamoya disease have been observed in families with ACTA2 pathogenic variants, more frequently in patients with mutation of the R258, R118, and R149 residues [Guo et al 2009]. 

*Gene reviews https://www.ncbi.nlm.nih.gov/books/NBK1120/*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 
incomplete penetrance

**Allelic requirement:**

Monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The majority of pathogenic variants are missense variants. There is a report of two nonsense variants but they are predicted to escape NMD. The molecular mechanism is not completely understood but it is suggested disease results from a dominant negative mechanism. One proposed mechanism is an **altered gene product** alters the dynamics of actin assembly into filaments, by either disrupting the actin-actin interaction sites or interfering with ATP hydrolysis. Specific subsets of ACTA2 variants appear to be associated with different presentations for example TAAD cosegregating with premature coronary artery disease, ischemic stroke, and moyamoya disease has been observed more frequently in patients with mutations of the R258, R118, and R149 residues. ACTA2 missense variants that specifically disrupt the arginine 179 residue cause multisystem smooth-muscle dysfunction syndrome.

**List variant classes in this gene proven to cause this disease:**

- Missense
- Stop_gained predicted to escape NMD

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant predicted to escape NMD
- Splice donor variant
- Splice donor variant predicted to escape NMD
- Frameshift predicted to escape NMD
- Stop_lost
- In frame deletion
- In frame insertion# **ACTC1 -- Hypertrophic cardiomyopathy**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:143

The ACTC1 gene has been associated with autosomal dominant hypertrophic cardiomyopathy (HCM) in at least 6 probands in 4 publications. Four unique variants (missense) with convincing evidence of pathogenicity have been reported in humans, including de novo inheritance with maternity and paternity confirmed in two cases and segregation with disease in 26 additional family members. ACTC1 was first associated with this disease in humans in 1999 (Mogensen et al, PMID 10330430). More evidence is available in the literature, but the maximum score for genetic evidence (12 pts.) has been reached. The mechanism for disease is unknown. The ACTC1 gene was significantly enriched for missense variants in Walsh et al, 2016 (PMID 27532257). Overall, the gene was found to have an Odds Ratio of 8.59 (5.06-14.5) for HCM. This gene-disease association is supported by expression studies, in vitro functional assays, and an animal model. In summary, ACTC1 is
definitively associated with autosomal dominant HCM. This has been repeatedly demonstrated in both the research and clinical diagnostic settings and has been upheld over time. This classification was approved by the ClinGen Hypertrophic Cardiomyopathy Expert Panel on September 5, 2017.

**Heterozygous missense mutations and in-frame codon deletions of ACTC1 are the major types of pathogenic variants found in patients with apical hypertrophic cardiomyopathy and left ventricular non compaction, congenital heart defects and arrhythmia (PMID:17611253, 26061005).** Thus far, there are only a few reports of mutations that could be interpreted to support haploinsufficiency of ACTC1 \[PMID 1794298, 24503780\]; however, the evidence for this remains incomplete. Large deletions or
duplications have not been described.

**Literature review:**

In a cohort of 368 unrelated patients with HCM (sporadic or familial), 3 unique missense mutations were identified. 2 of these mutations were adjacent to regions of actin-actin and actin-myosin interaction. Authors suggested the mutations may affect actin-myosin interaction and force generation.

*Olson et al. 2000 (PMID:10966831)*

Walsh et al found an excess of non-truncating variants associated with HCM. There was no excess of truncating variants in comparison to reference dataset (ExAC).

*Walsh et al, 2016 (PMID 27532257)*
<https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=ACTC1&icc=HCM>

"...myofibrils isolated from the hearts of *ACTC* E99K transgenic (TG) mice have enhanced Ca^2+^ sensitivity of force. We also measured work and heat output by papillary muscles from *ACTC* E99K mice and from their non-TG littermates, allowing a direct comparison of the efficiency of contraction. These measurements show that the papillary muscle of the *ACTC* E99K HCM mouse is hypercontractile and that hypercontractility is associated with lower efficiency."

*Song W et al. 2013 (PMID:23604709)*

"In a large 3-generation family with hypertrophic cardiomyopathy [Mogensen et al. (1999)] PMID 10330430 identified heterozygosity for a missense mutation in the ACTC1 gene that was located near 2 missense mutations previously identified as causing an inherited form of dilated cardiomyopathy (CMD1R). The variant was a 253G-T transversion in exon 5 of the ACTC gene resulting in an ala295-to-ser substitution. The ala at position 295 is conserved in 19 different species. **The expression of the actin mutation in this family gave the impression of a highly penetrant disease with diverse phenotypes and variable age of onset.** Only 1 individual of 13 family members carrying the mutant allele was nonpenetrant, and morbidity was low, as only 3 of the 13 carrying the mutant allele had symptoms of the disease. **The authors stated that ACTC1 was the first sarcomeric gene described in which mutations are responsible for 2 different cardiomyopathies, and hypothesized that ACTC1 mutations affecting sarcomere contraction lead to HCM and that mutations affecting force transmission from the sarcomere to the surrounding syncytium lead to dilated cardiomyopathy."**

*OMIM <https://www.omim.org/entry/102540>*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

**Allelic requirement:**

Monoallelic_aut

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is not definitively known but may involve impact on sarcomere force generation due to **altered gene product structure**. Heterozygous missense mutations and in-frame deletions are the major types of pathogenic variants found. There are only a few reports of mutations that could be interpreted to support haploinsufficiency of ACTC1 \[PMID 1794298, 24503780\]; however, the evidence for this remains incomplete. Large deletions or duplications have not been described. The variant identified in the original family appeared to be highly penetrant but with a variable age of onset. 

**Additional information related to ACMG evidence types**

**PM1** 
Walsh et al propose adaptation of ACMG/AMP guidelines for rule PM1 and HCM, relating to the relative frequencies of non-truncating variants in case cohorts and population controls. 
PM1_strong - EF >0.95
PM1_moderate - EF between 0.90 and 0.95 
PM1_supporting - EF between 0.80 and 0.90

ACTC1 etiological fraction across the whole gene is 0.884 (0.826–0.941) so would enable PM1_supporting to be applied

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
AF>0.1% (het) 
AF>3.16% (hom) 
Due to the rarity of our disorders this threshold was reduced and this rule is activated for all variants with a filtering allele frequency
cardioclassifier

**BS1** (MAF too high for disease) 0.02%
Assumptions
• Disease prevalence: 1/200 individuals (1/400 chromosomes)
• Penetrance: 30%
• Maximum pathogenic variant contribution: 2% based on MYBPC3 variant p.Arg502Trp (Walsh et al. 20175:6,000 probands)
• Note that the FAF (95% poisson) is available for each variant in ExAC (http://exac.broadinstitute.org/).

**PM2** A filtering allele frequency (FAF) <0.004% activates this rule
CAUTION: Population databases may contain presymptomatic individuals for diseases with reduced
penetrance/variable onset.

*Kelly MA et al 2018 PMID: 29300372* 

*Whiffin N et al 2018 PMID: 29369293*

**List variant classes in this gene proven to cause this disease:**

Missense

In frame deletion

**List other variant classes predicted to lead to the same functional consequence**

Frameshift predicted to escape NMD

Stop_gained predicted to escape NMD

Splice donor variant

Splice donor variant predicted to escape NMD

Splice acceptor variant predicted to escape NMD

Stop_lost

In frame_insertion

**APC -- Familial adenomatous polyposis**

**Review of source material:**

**ClinGen:**

https://search.clinicalgenome.org/kb/genes/HGNC:583

**Genetic evidence:**

-   Autosomal dominant disorder

-   Case level data with evidence of probands with predicted or proven
    null variants or some evidence of gene impact - *Gayther SA et al.
    1994 Jan (PMID:816205)*

 

**Experimental evidence:**

-   Function - *Smith KJ et al. 1993 Apr 1 (PMID:8385345)*

-   Functional alteration - *Inomata M et al. 1996 May 1 (PMID:8616874);
    Baeg GH et al. 1995 Nov 15 (PMID:8521819)*

-   Models and Rescue *- Fodde R et al. 1994 Sep 13 (PMID:8090754);
    Oshima M et al. 1995 May 9 (PMID:7753829)*

**Conclusion:**

> ·   Convincing evidence with replication in \>2 publications over time
> (\>3yrs)

 

**Outcome:**

> ·   Gene-disease relationship -- DEFINITIVE

**Literature Review**

Mutations predicted to result in protein truncation, that is,
    nonsense mutations, short deletion/insertion mutations associated
    with a frame shift, mutations involving nucleotides ±1 and ±2 within
    splicing junctions, and large genomic rearrangements, are highly
    likely to impair the *APC *function and are definitely classified as
    causing FAP without additional information. These alterations
    contribute to almost all genetic variations identified through
    the *APC *gene molecular analysis (94%)" 
    *Grandval et al PMID 24599579*
    
Most truncating mutations are fully penetrant but may be
    associated with a differing severity of colorectal polyposis and
    differing risks of the extra-colorectal manifestations. Mutations in
    the central region of *APC* (codons 1290--1400) are associated with
    the most severe polyposis phenotype. Two codons (1061 and 1309) are
    mutational hotspots and account for 11% and 17% of all germline
    mutations, respectively. CHRPE is associated with mutations between
    codons 457 and 1444, while jaw osteomas and fibromatosis (desmoids
    tumors) are more prevalent in patients with mutations occurring
    after codon 1400" *Jass, PMID: 18541388*

-   "*APC* plays a central role in the Wnt-signalling pathway,
    especially in regards to the degradation of β-catenin within the
    cell cytoplasm. If *APC* is mutated, the β-catenin-Tcf complex is
    not suppressed and leads to constitutive activation of several genes
    and oncogenes controlling cell growth and division. Mutations
    in *APC* affect the ability of the cell to maintain normal growth
    and function, which results in cell overgrowth/adenoma formation.

About 25% of people with FAP do not have any family history of disease
and harbour a de novo mutation in *APC* without any clinical or genetic
evidence of FAP in the family. One study suggests that a 5 bp deletion
of codon 1309 (c.3927_3931del) is over-represented in patients with a
suspected de novo mutations (29%) and in proven de novo mutation
carriers (45%) supporting the view of codon 1309 as a hot-spot for
mutations.

New methods that can screen genomic loci at great depths are revealing
that patients that were thought to be *APC* mutation negative have
pathogenic germline heterozygous *APC* mutations, *APC* promoter
mutations, deep intronic mutations, complex genomic rearrangements,
somatic mutations or *APC* mutation mosaicism." *Talseth-Palmer PMID:
28331556*

"According to the previous published reports, the majority of
pathogenic *APC* germline mutations belong to three categories,
nonsense/frameshift mutations, splice sites mutations and deep intronic
deletions. Nonsense/frameshift mutations splice sites mutations and deep
intronic deletions of the *APC* gene lead to large genomic
rearrangements, resulting in the formation of truncated APC proteins.
Although, it has been found that in few cases, point mutations or
missense variants within the coding sequence also result in the
formation of alternative transcripts due to aberrant splicing. It has
been reported that 2% of all germline *APC* gene mutations are large
genomic deletions. Moreover, in HGMD, 1000 different *APC* germline
mutations have been reported till date (Zhao-Zhang et al, PMID: 27391059

\'**Pathogenic variants.** At least 700 germline pathogenic variants
have been found in families with an *APC*-associated polyposis condition
\[Béroud et al 2000\]. Pathogenic variants almost always cause a
premature truncation of the APC protein, usually through single
amino-acid substitutions or frameshifts. While pathogenic variants have
been found scattered throughout the gene, they are predominantly located
in the 5\' end of the gene. The most common germline *APC* pathogenic
variant is c.3927_3931delAAAGA.'

Gene reviews

https://www.ncbi.nlm.nih.gov/books/NBK1345/ 

\'The APC gene encodes a multidomain protein that plays a major role in
tumor suppression by antagonizing the WNT signaling pathway.
Inappropriate activation of this pathway through loss of APC function
contributes to cancer progression, as in familial adenomatous polyposis
APC also has a role in cell migration, adhesion, chromosome segregation,
spindle assembly, apoptosis, and neuronal differentiation (Hanson and
Miller, 2005).\'

Omim https://www.omim.org/entry/611731 

 

The majority of these germline mutations in FAP patients were recognized
in the 5'-half of the APC gene. Nonsense (28%) or truncating frameshift
(67%) mutations are responsible for almost 95% of all APC germline
mutations in FAP disease \[17\]

https://pubmed.ncbi.nlm.nih.gov/30414835/

**[Pilot application of harmonised terms:]{.ul}**

**Disease associated variant consequences:**

Altered gene product structure

Dose Change: dose reduction: Absent gene product

Decreased gene product

**Allelic requirement:**

Monoallelic_aut

**Inheritance:**

Autosomal dominant

**Narrative summary of molecular mechanisms:**

Almost all mutations truncate the protein or take the form of allelic
loss. Mechanism is likely loss of function leading to reduction/absence
of gene product. Loss of APC protein causes aberrant constitutive
activation of the Wnt pathway and thus accumulation of cytosolic
beta-catenin and a lack of normal apoptosis. The majority of germline
mutations in FAP patients fall in the 5'-half of the APC gene. Nonsense
(28%) or truncating frameshift (67%) mutations are responsible for
almost 95% of all APC germline mutations in FAP disease. All targets of
the APC protein are not yet known but it may also regulate expression of
genes such as the oncogenes c-Myc and cyclin D1.

**List variant classes in this gene proven to cause this disease**

Splice region variant

Spice acceptor variant

Splice donor variant

Frameshift variant

Stop gained

Stop gained predicted to undergo NMD

Missense

In frame insertion

In frame deletion

**[Potential novel variant classes based on predicted functional
consequence:]{.ul}**

Start lost
Spice acceptor variant predicted to escape NMD

Splice donor variant predicted to escape NMD

Frameshift variant predicted to escape NMD
Spice acceptor variant predicted to undergo NMD

Splice donor variant predicted to undergo NMD

Frameshift variant predicted to undergo NMD
Stop gained predicted to escape NMD
Stop lost
Gain of upstream Start \[uORF\]

Gain of upstream Start \[oORF\]

Stop lost \[oORF\]


Frameshift \[oORF\]# **APOB -- Familial Hypercholesterolemia**

**Review of source material:**

**ClinGen:**

https://search.clinicalgenome.org/kb/genes/HGNC:603

The APOB gene has been reported in association with hypercholesterolemia (autosomal dominant) and hypobetalipoproteinemia (autosomal recessive). The relationship between APOB and hypercholesterolemia (autosomal dominant) was evaluated using the ClinGen Clinical Validity Framework. Variants in APOB were first reported in humans with hypercholesterolemia as early as 1989 (Soria et al., PMID: 2563166). **At least 7 variants (missense) in at least 13 probands in 6 publications been reported in humans** (PMIDs: 24498611, 24234650, 15135245, 22408029, 2563166, 7627691). Variants in this gene segregated with disease in at least 41 family members. This gene-disease relationship has been studied in at least 1 case-control study at the single variant level with an odds ratio of 78 (95% CI 16-388, p=0.0001) (PMID: 9603795). **The mechanism for disease involves mainly heterozygous missense variants resulting in defective apo B100 on LDL particles that fails to bind to LDLR (PMID: 29219151).** The gene-disease association is also supported by in vitro studies and animal models. In summary, APOB is definitively associated with autosomal dominant hypercholesterolemia. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This classification was approved by the ClinGen General Gene Curation EP on 11/14/2018.
Gene Clinical Validity Standard Operating Procedures (SOP) - Version 7

**Literature review:**

"Apolipoprotein B (apoB) is the major apolipoprotein on lipoprotein molecules, especially LDL-C, and functions as a ligand to the LDL-receptor. The gene is located on chromosome 2p and spans more than 43 kb. The gene comprises 29 exons and is transcribed and translated into a protein of 4563 amino acids [14]. While truncation mutations in the APOB gene cause hypobetalipoproteinemia, **mutations causing hypercholesterolaemia are due to missense mutations** that result in ligand-defective apoB protein. The LDL-C particles made from this allele are therefore not able to bind to the LDL-receptor and thus accumulate in the blood [15]. A single mutation of the APOB gene (p.Arg3527Gln) accounts for approximately 6–10% of all FH cases in European population, and it is located in exon 26 of APOB gene [16]. Other APOB mutations in other regions of the gene such as p.Arg50Trp, p.Arg1164Thr and p.Gln4494del were also recently found to cause FH..."

*Sharifi M et al 2017 PMID: 28405938*

"Familial hypercholesterolemia-2 (FHCL2; 144010) is caused by mutation in APOB causing decreased LDLR (606945) binding affinity, so-called familial ligand-defective apolipoprotein B. The first mutation of this sort was described by Soria et al. (1989); see 107730.0009. A second was described by Pullinger et al. (1995)..."

*Omim https://www.omim.org/entry/107730?search=APOB*

"...More than five pathogenic variants have been reported to be associated with FH, two of which are repeatedly found to be significant:

- p.Arg3527Gln, found mainly in people with European ancestry

- p.Arg3527Trp, which tends to be found mostly within Asian populations [Calandra et al 2011]

Pathogenic variants in this locus account for approximately 1%-5% of all persons with FH. APOB is generally involved in aiding the binding of LDL-C to its receptor on the cell surface. APOB pathogenic variants alter the ability of protein to effectively bind LDL-C to LDLR, causing fewer LDL-C particles to be removed from the blood."

"FH caused by a heterozygous APOB pathogenic variant is reported to be less severe than FH caused by a heterozygous pathogenic variant in LDLR or PCSK9 [Hopkins et al 2011]...

Penetrance for FH can be incomplete in persons with a heterozygous APOB pathogenic variant [Fahed & Nemer 2011]...

Homozygous or heterozygous loss-of-function pathogenic variants in APOB cause familial hypobetalipoproteinemia (OMIM 615558) characterized by extremely low levels of LDL-C [Burnett et al 2012]."

*Gene reviews https://www.ncbi.nlm.nih.gov/books/NBK174884*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 
incomplete penetrance

**Allelic requirement:**

Monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The mechanism appears to be missense mutations resulting in an **Altered gene product** which leads to defective APOB protein causing LDL-C particles to accumulate in the blood. Penetrance is incomplete and there are recurrent mutations identified in different populations. A single mutation of the APOB gene (p.Arg3527Gln) accounts for approximately 6–10% of all FH cases in European population, and it is located in exon 26 of APOB gene. 

**List variant classes in this gene proven to cause this disease:**

- Missense

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant predicted to escape NMD
- Splice donor variant 
- Splice donor variant predicted to escape NMD
- Frameshift predicted to escape NMD
- Stop_gained predicted to escape NMD
- Stop_lost
- In frame deletion
- In frame insertion# **ATP7B -- Wilson Disease**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:870

The relationship between ATP7B and Wilson disease (autosomal recessive) was evaluated using the ClinGen Clinical Validity Framework as of March 26th, 2019. Variants in ATP7B were first reported in humans with this disease as early as 1993 (Bull et al., PMID 8298639). Wilson disease is a disorder of copper metabolism characterized by an accumulation of copper in many organs, particularly the liver and brain. **At least 700 unique germline variants have been identified (missense, nonsense, splice site, frameshift)** (reviewed in Członkowska et al., 2018; PMID 30190489). This gene-disease relationship is well-known and therefore a significant amount of case-level data, segregation data and experimental data is available in the literature, therefore the maximum score for both genetic evidence and experimental evidence has been reached. Note, this curation effort may not be exhaustive of all literature related to this gene-disease relationship. **The mutational mechanism for disease is loss of function leading to reduced (or absent) copper-transporter activity (Huster et al., 2012; PMID 22240481).** This gene-disease association is supported by expression studies, in vitro functional assays, and animal models. In summary, ATP7B is definitively associated with autosomal recessive Wilson disease. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This classification was approved by the ClinGen General Gene Curation Expert Panel on March 27, 2019.

**Literature review:**

More than 700 mutations have been described according to The Human Gene Mutation Database16,17 and patients can be homozygous for one disease-causing mutation or carry two different disease-causing mutations as compound heterozygotes. Mutations can affect almost all 21 exons and are frequently missense and nonsense. The missense mutation H1069Q in exon 14 is very common. About 50–80% of WD patients from Central, Eastern, and Northern Europe carry at least one allele with the H1069Q mutation2. In Southern Europe, other mutations are common, such as the missense mutation M645R in mainland Spain. The R778L in exon 8 is found more frequently in South-eastern Asia where the mutation has an allele frequency of 14 to 49%2.

*Członkowska et al., 2018; PMID 30190489*

The product of ATP7B is copper-transporting ATPase 2, an intracellular transmembrane copper transporter that is key in incorporating copper into ceruloplasmin and in moving copper out of the hepatocyte into bile. The protein is a P-type ATPase, characterized by cation channel and phosphorylation domains containing a highly conserved Asp-Lys-Thr-Gly-Thr (DKTGT) motif, in which the aspartate residue forms a phosphorylated intermediate during the transport cycle.

The gene is expressed mainly in liver and kidney.

Abnormal gene product. Tissue damage occurs after excessive copper accumulation resulting from lack of copper transport from the liver. Even when no transporter function is present, accumulation of copper occurs over several years.

More than 800 pathogenic variants have been identified (see Wilson Disease Mutation Database [Kenney & Cox 2007], including nonsense, missense, frameshift, and splice site variants as well as large deletions.

The most common pathogenic variant in populations of European origin is an amino acid substitution in a highly conserved motif close to the ATP-binding region (p.His1069Gln) [Tanzi et al 1993]. This pathogenic variant occurs at a frequency of 26%-70% in various populations and is associated with neurologic or hepatic disease and a mean onset age of about 20 years [Houwen et al 1995, Thomas et al 1995, Maier-Dobersberger et al 1997, Shah et al 1997].

The most common pathogenic variant in the Asian population is an amino acid substitution in exon 8, p.Arg778Leu [Thomas et al 1995], found at a high frequency in all Chinese [Gu et al 2003] and ethnically related populations studied.

Pathogenic variants in the promoter region are rare [Cullen et al 2003] except in Sardinia, where a 15-bp deletion in the 1-kb promoter region (c.-441_-427del15) predominates [Loudianos et al 1999].

*Gene reviews https://www.ncbi.nlm.nih.gov/books/NBK1512/*

"...Three Sardinian patients were found to
be heterozygous for a 15 nucleotide deletion in
the promoter of ATP7B, 441/427del, previously shown to reduce the activity of the
ATP7B promoter. We did not identify any other
mutations within the promoter and 50 UTR of
ATP7B in 164 WND chromosomes likely to be
causative of the disease. This suggests that mutations in the regulatory elements of ATP7B that
result in WND are rare in patients of white European ancestry, with the exception of Sardinia.
These observations are important for facilitating
the molecular diagnosis of WND."

*Cullen LM et al 2003 (PMID: 14616767)*

"...robust genotype–phenotype correlation in the case of missense mutations has not been found so far, although complete loss of protein expression due to nonsense mutations is expected to result in a more severe phenotype

...mutations in ATP7B have various effects altering protein expression levels, catalytic and transport activity, as well as intracellular localization (Figure 6B). Mutants with a partial preserved transport function can result in later onset of disease or have variable manifestation if their stability and localization is modulated by the metabolic state of cells"

*Huster et al., 2012; PMID 22240481*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal recessive

(optional) modifiers: 

**Allelic requirement:**

biallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The product of ATP7B is copper-transporting ATPase 2, an intracellular transmembrane copper transporter that is key in incorporating copper into ceruloplasmin and in moving copper out of the hepatocyte into bile. The mechanism of disease appears to be loss of function of ATP7B resulting in **decreased/absent or altered gene product** which leads to lack of copper transport from the liver through various mechanisms inlcuding defective catalytic activity, transport and intracellular localisation. Patients can be homozygous for one disease-causing mutation or carry two different disease-causing mutations as compound heterozygotes. Mutations can affect almost all 21 exons and are frequently missense and nonsense. The missense mutation H1069Q in exon 14 is very common. About 50–80% of WD patients from Central, Eastern, and Northern Europe carry at least one allele with the H1069Q mutation. In Southern Europe, other mutations are common, such as the missense mutation M645R in mainland Spain. The R778L in exon 8 is found more frequently in South-eastern Asia where the mutation has an allele frequency of 14 to 49%2.
In Sardinia there is a 15bp deletion in the 1kb promoter region that is pathogenic. Other pathogenic variants in the promoter region are rare. Robust genotype–phenotype correlation in the case of missense mutations has not been found so far, although complete loss of protein expression due to nonsense mutations is expected to result in a more severe phenotype.

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Splice region variant
- Missense

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- In frame deletion
- In frame duplication
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
**BMPR1A -- Juvenile polyposis syndrome, (MIM 174900)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:1076

**Literature Review:**

.. identified four BMPR1A **germline truncation pathogenic**
**variants** in four juvenile polyposis kindreds. Genomic sequencing of
BMPR1A in each of these juvenile polyposis kindreds disclosed these
truncation pathogenic variants in all affected kindred members but not
in normal individuals (figure 2). Individual Case Evidence: Segregation
Among Similarly Affected Family Members ?
NM_004329.2(BMPR1A):c.44_47delTGTT (p.Leu15Serfs) ?
NM_004329.2(BMPR1A):c.715C\>T (p.Gln239Ter) ?
NM_004329.2(BMPR1A):c.812G\>A (p.Trp271Ter) ?
NM_004329.2(BMPR1A):c.961del (p.Phe320_Leu321insTer)

***Howe et al, 2001, PMID 11381269***

...a large cohort study including a total of 77 Juvenile polyposis (JP)
cases. Germline BMPR1A gene pathogenic variants were identified in 16
cases (20.8%). These included **ten truncation (including nonsense and
deletion) variants and six missense** mutations.

***Howe et al, 2004, PMID 15235019***

...a large prospective, referral-based study of 603 patients with
moderate-load colorectal polyps. Twenty cases (3.3%) were identified
carrying a BMPR1A pathogenic variants including **twelve BMPR1A focal
deletion and/or truncation variants and eight misssense** variants. All
the BMPR1A pathogenic variants carriers were diagnosed with Juvenile
polyposis and/or other types of cancers (supplementary table).

***Ngeow et al, 2013, PMID 23399955***

Forty-three types of BMPR1A gene mutations were deposited in the Human
Mutation Database at the Institute of Medical Genetics in Cardiff. Most
often, these were nucleotide changes generating a stop codon
(**nonsense**) or leading to amino acid changes (**missense**). These
mutations are **distributed evenly in the entire gene** sequence. Five
other mutations were reported at the gene assembly: **two in intron 1,
and one in each of introns 3, 4 and 5.** **Small deletions**, most often
identified between codons 224 and 359, constitute a considerable
proportion of mutations in the BMPR1A gene.

***Cichy et al, 2014, PMID: 25097590***

Overall, **frameshift, nonsense, and missense** variants accounted for
the majority of pathogenic SMAD4 (72.9%) and BMPR1A (61.8%) alterations
in the ECS as well as the

LBSB group (SMAD4: 79.9%; BMPR1A: 70.8%; Supplementary Table 3). Only
**large genomic, i.e., single or multiexon deletions** in SMAD4 were
significantly overrepresented in the ECS compared with the LBSB group.
**Splice site variants were noted** in 4-6% of SMAD4 and 10-16% of
BMPAR1A

***Blatter et al, 2020, PMID: 32398773***

**OMIM:** https://www.omim.org/entry/174900 (and 601299

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1469/

*Mechanism action*

BMPR1A is not known to be a tumor suppressor gene, although few studies
have examined it in cancer. BMPR1A is a **type I cell surface receptor**
for the BMP pathway. Ligands, such as TGF-β or BMP, bind to a receptor
and **activate signaling pathways** leading to protein complexes that
migrate to the nucleus and bind directly to DNA sequences to regulate
**transcription** \[Heldin et al 1997\]. The downstream genes under the
control of these signaling pathways are still being actively
investigated.

*Pathogenic variants.*

Sixty pathogenic variants, including **insertions, deletions, and
missense, nonsense, and splice site alterations,** have been described
\[Calva-Cerqueira et al 2009\]. **Germline deletions or pathogenic
missense variants of the promoter** have also been described
\[Calva-Cerqueira et al 2010\]. **Large deletions** of BMPR1A may also
occur in up to 6% of individuals and be associated with additional or
more severe clinical features \[Aretz et al 2007, van Hattem et al 2008,
Calva-Cerqueira et al 2009\].

*Abnormal gene product.*

Abnormal BMPR1A proteins frequently result from pathogenic DNA variants
in **the protein kinase domain and occasionally by variants in the
cysteine-rich region of the extracellular domain**. No pathogenic
variants have been described in the transmembrane domain \[Howe et al
2004\]. In vitro studies have shown that proteins resulting from BMPR1A
pathogenic **missense** variants as seen in individuals with JPS are
**retained in the cytoplasm and do not traffic to the cell membrane**
like the wild-type protein \[Howe et al 2013\].

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

*Dose change* -decreased gene product level

*Altered gene product structure*

**Narrative summary of molecular mechanisms:**

The precise molecular mechanism is not known and BMPR1A is not known to
be a tumour suppressor gene. Variants cluster in the protein kinase
domain and occasionally in the cysteine-rich region of the extracellular
domain. Variant classes include small insertions, small (considerable
proportion of mutations ) and large deletions, missense, nonsense, and
splice site alterations. Germline deletions or pathogenic missense
variants of the promoter have also been described.

**List variant classes in this gene proven to cause this disease:**

Stop_gained

missense

Frameshift_variant

Splice_acceptor_variant

Splice_donor_variant

regulatory_region_variant (deletions/missense in promotor region)

**Potential novel variant classes based on predicted functional
consequence**

splice_region_variant

splice_acceptor_variant predicted to undergo NMD

splice_acceptor_variant predicted to escape NMD

splice_donor_variant predicted to undergo NMD

splice_donor_variant predicted to escape NMD

start_lost

frameshift_variant predicted to undergo NMD

frameshift_variant predicted to escape NMD

stop_gained predicted to undergo NMD

stop_gained predicted to escape NMD

stop_lost

inframe_insertion

inframe_deletion

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[uORF\]

Stop lost \[oORF\]

(Start lost \[uORF\])

(Frameshift \[uORF\])

(Frameshift \[oORF\])

(Stop gained \[uORF\])

**Not included**

**synonymous_variant**

**intron_variant**

**intergenic_variant**

**5_prime_UTR_variant**

**3_prime_UTR_variant**
**BRCA1 -- Breast-ovarian cancer, familial 1 (MIM 604370)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:1100

Loss of function mutations in BRCA1 (**nonsense, frameshift, splice site, and exonic deletions**) as well as **whole gene deletions** of BRCA1 have been associated with cancer development (Genereviews and
PMIDs: 21989022, 17661172, and 22762150). The penetrance associated with BRCA1 mutations is still an active area of study; however, patients with pathogenic BRCA1 mutations are thought to have an **increased lifetime
risk of developing breast cancer (50-80% in females, 1-2% in males), ovarian cancer (24-40%), prostate cancer (up to 30%), and pancreatic
cancer (1-7%)** (Genereviews Table 3).

No evidence triploidy

**Literature Review:**

**OMIM:** <https://www.omim.org/entry/604370>

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1247/

Pathogenic variants. More than **1800 pathogenic variants** have been
identified in BRCA1. While a small number of these variants have been
identified repeatedly in unrelated families, the vast majority have not
been reported in more than a few families.

**Reduced-penetrance variant**. p.Arg1699Gln is established to be a
reduced penetrance variant in BRCA1 \[Spurdle et al 2012\]. Data from
functional assays were ambiguous for deficiency across multiple assays.
Thus, this allele was determined to be associated with intermediate risk
for breast and ovarian cancer, highlighting challenges for risk modeling
and clinical management of individuals of this and other potential
moderate-risk variants.

Abnormal gene product. **Most BRCA1 pathogenic variants lead to frameshifts resulting in a missing or non-functional protein**. In
cancers from individuals with a BRCA1 germline pathogenic variant, the
normal allele is deleted or inactivated, resulting in somatic
inactivation of BRCA1. This strongly suggests that **BRCA1 is a
tumor-suppressor gene whose loss of function can result in genomic
instability,** resulting in a high susceptibility to malignant
transformation \[Smith et al 1992, Deng 2006\]. Additional evidence in
support of a tumor suppressor function is that overexpression of the
BRCA1 protein leads to growth suppression similar to that seen with the
classic tumor suppressors TP53 and the retinoblastoma gene (RB1) \[Holt
et al 1996\]. **Loss of function of BRCA1 results in defects in DNA
repair, defects in transcription, abnormal centrosome duplication,
defective G2/M cell-cycle checkpoint regulation, impaired spindle
checkpoint, and chromosome damage** \[Brodie & Deng 2001, Deng 2002,
Venkitaraman 2002\].

1,650 unique BRCA1 and 1,731 unique BRCA2 mutations were identified. The
unique mutations and number of families in which each mutation was
observed are listed in Supplementary Table 1. In each gene, the five
most common mutations (including founder mutations) accounted for 33% of
all mutations in BRCA1..

..most common mutation type was **frameshift followed by nonsense**. The
most common effect of BRCA1 and BRCA2 mutations was **premature
translation termination and most of the mutant mRNAs were predicted to
undergo nonsense-mediated mRNA decay (NMD)** (Anczukow, et al., 2008).
Despite having the same spectrum of mutations in BRCA1 and BRCA2, the
frequency distribution by mutation type, effect, or function differed
significantly (p\<0.05) between BRCA1 and BRCA2 mutation carriers for
many groups, as shown in Table 1. These observed differences are largely
because **genomic rearrangements and missense mutations account for a
much higher proportion of mutations in BRCA1** when compared with BRCA2

As expected, the most common mutations in the entire data set were the
**founder mutations** BRCA1 c.5266dup (5382insC), BRCA1 c.68_69del
(185delAG), and BRCA2 c.5946del (6174delT)..... The **large
rearrangement mutation** in BRCA1 c.548-?4185+?del (ex9-12del) appears
to be an important founder mutation in Mexico, with findings of a common
haplotype and an estimated age at 74 generations (\~1,500 years)

***Rebbeck et al, 2018, PMID: 29446198***

***...***dominantly **inherited 5\' UTR variant associated with
epigenetic BRCA1 silencing due to promoter hypermethylation** in two
families affected by breast and ovarian cancer....multiple women are
affected by grade 3 breast cancer or high-grade serous ovarian
cancer.....RNA sequencing revealed the allelic loss of BRCA1 expression
in both families and that this loss of expression segregated with the
heterozygous variant c.-107A\>T in the BRCA1 5\' UTR.

***Evans et al, 2018, PMID: 30075112***

variant in the 3\'UTR of BRCA1 is functional, leading to decreased BRCA1
expression, modest increased breast cancer risk, and most importantly,
presentation with stage IV breast cancer

***Dorairaj, 2013, PMID: 24915755***

Spliceogenic mutations of group B included 6 already analyzed (*BRCA1* c.212G\>A, c.213−11T\>G, and c.4484G\>T, and *BRCA2....*c.631G\>A, c.8754+3G\>C, and
c.9117G\>A) and 2 newly characterized (c.134+3_134+6delAAGT, c.4986+5G\>A in *BRCA1*).
Three mutations caused the skipping of an entire
exon: *BRCA1* c.4484G\>T (exon 14) ([Fig.
2A](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3579815/figure/pone-0057173-g002/?report=objectonly)), *BRCA2* c.631G\>A
(exon 7) ([Fig.
2B](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3579815/figure/pone-0057173-g002/?report=objectonly))
and *BRCA2* c.9117G\>A (exon 23) ([Fig.
2C](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3579815/figure/pone-0057173-g002/?report=objectonly)).
Conversely, partial intronic retention caused by the activation of
cryptic splice sites was observed for *BRCA1* c.213−11T\>G (59 bp at the
3′-end of intron 5) ([Fig.
2D](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3579815/figure/pone-0057173-g002/?report=objectonly)), *BRCA1*c.4986+5G\>A
(65 bp at the 5′-end of intron 16) ([Fig.
2E](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3579815/figure/pone-0057173-g002/?report=objectonly)),
and for *BRCA2* c.8754+3G\>C (46 bp at the 5′-end of intron 21) ([Fig.
2F](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3579815/figure/pone-0057173-g002/?report=objectonly)).
Finally, the *BRCA1* c.134+3_134+6delAAGT and c.212G\>A variants were
associated with a relevant increase, in comparison with normal controls,
of the Δexon3 and Δexon5q (missing 22 bp at the 3′-end of exon 5)
isoforms, respectively ([Fig.
2G--H](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3579815/figure/pone-0057173-g002/?report=objectonly)).
Both isoforms contain PTCs

***Colombo et al, 2013, PMID: 23451180***

BRCA1 c.5073 A \> T variant might play a pathogenic role in HOC syndrome
in this family (synonymous -- single case study)

***Minucci, 2018, PMID: 29760936***

The intronic BRCA1 c.5407-25T\>A variant causing partly skipping of exon
23-a likely pathogenic variant with reduced penetrance?

***Høberg-Vetti H, 2020,PMID: 32203205***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

Optional modifiers: incomplete penetrance

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is predominantly loss of function (*BRCA1* is a tumour suppressor gene) leading to genomic instability. Premature translation termination is the most common outcome with most (though not all) mutant
mRNAs predicted to undergo nonsense-mediated mRNA decay. Variants include large deletions/duplications, frameshift, in-frame deletions/insertions, missense, nonsense, splice site (including non-canonical, deeper intronic) and large scale structural rearrangement. Less frequently reported variants involve both 3'and 5'
UTRs, in the latter associated with promotor hypermethylation. There is
insufficient evidence for pathogenicity of synonymous variants. Genomic
rearrangements and missense mutations are more common in *BRCA1* when
compared with *BRCA2*

**List variant classes in this gene proven to cause this disease:**

splice_region_variant

splice_acceptor_variant

(splice_acceptor_variant predicted to undergo NMD)

splice_donor_variant

(splice_donor_variant predicted to undergo NMD)

(splice_acceptor_variant predicted to escape NMD)

(splice_donor_variant predicted to escape NMD)

Frameshift_variant

(frameshift_variant predicted to undergo NMD)

(frameshift_variant predicted to escape NMD)

Stop_gained

(stop_gained predicted to undergo NMD)

Inframe_deletion

Inframe_insertion

Missense

5_prime_UTR_variant

3_prime_UTR_variant

intron_variant

**Potential novel variant classes based on predicted functional
consequence**

start_lost

(stop_gained predicted to escape NMD)

stop_lost

(gain of upstream Start \[uORF\])

(gain of upstream Start \[oORF\])

(Stop lost \[uORF\])

(Stop lost \[oORF\])

(Start lost \[uORF\])

(Frameshift \[uORF\])

(Frameshift \[oORF\])

(Stop gained \[uORF\])
**BRCA2 -- Breast-ovarian cancer, familial 2 (MIM 612555)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:1101

Many **loss of function mutations** have been described; see OMIM gene
entry for details. Heterozygous mutations of BRCA2 cause susceptibility
to breast and ovarian cancer (see OMIM), a phenotype that is not
appropriate for our purposes. Homozygous mutations of BRCA2 cause
Fanconi anemia (see GeneReviews), which is an appropriate phenotype but
is an inappropriate mutational mechanism. No evidence of dosage
pathology

**Literature Review:**

**OMIM:** https://www.omim.org/entry/612555

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1247/

Finally, *Brca2* knockout mice can be partially rescued by crossing with
a *Tp53* knockout strain, suggesting that these genes interact with
the *TP53* -mediated DNA damage checkpoint \[[Brugarolas & Jacks
1997](https://www.ncbi.nlm.nih.gov/books/NBK1247/)\]. Therefore, the
available evidence indicates that ***BRCA2* is a \"caretaker,\"
like *TP53*, serving to
maintain [genomic] integrity**
\[[Zhang et al 1998]. It
is likely that *BRCA2* will eventually be implicated in a variety of
cellular processes, only some of which will be related to their role in
the etiology of breast and ovarian cancer.

Most *BRCA2* pathogenic
variants reported to date consist of **frameshift deletions, insertions,
or nonsense variants that predict premature truncation of protein transcription, consistent with the loss of function** that is expected with clinically significant
variants of tumor suppressor genes. Cells lacking BRCA2 are deficient in
the repair of double-strand DNA breaks, as reflected in a
hypersensitivity to ionizing radiation \[[Venkitaraman
2001](https://www.ncbi.nlm.nih.gov/books/NBK1247/)\].

Reduced-penetrance variant. There is evidence that p.Lys3326Ter is
associated with risk of developing breast and ovarian cancers
independent of other **pathogenic variants in BRCA2, although penetrance
is reduced**. This was demonstrated through a large case-control study
based on an international consortium of patients with cancer. Further
studies are needed to determine the biologic mechanism of action
responsible for these associations \[Meeks et al 2015\].

1,650 unique BRCA1 and 1,731 unique BRCA2 mutations were identified. The
unique mutations and number of families in which each mutation was
observed are listed in Supplementary Table 1. In each gene, the five
most common mutations (including founder mutations) accounted for 33% of
all mutations in BRCA1..

..most common mutation type was **frameshift followed by nonsense**. The
most common effect of BRCA1 and BRCA2 mutations was **premature
translation termination and most of the mutant mRNAs were predicted to
undergo nonsense-mediated mRNA decay (NMD)** (Anczukow, et al., 2008).
Despite having the same spectrum of mutations in BRCA1 and BRCA2, the
frequency distribution by mutation type, effect, or function differed
significantly (p\<0.05) between BRCA1 and BRCA2 mutation carriers for
many groups, as shown in Table 1. These observed differences are largely
**because genomic rearrangements and missense mutations account for a
much higher proportion of mutations in BRCA1 when compared with BRCA2**

As expected, the most common mutations in the entire data set were the
**founder mutations** BRCA1 c.5266dup (5382insC), BRCA1 c.68_69del
(185delAG), and **BRCA2 c.5946del (6174delT).....**

***Rebbeck et al, 2018, PMID: 29446198***

9/18 kindreds identified potentially deleterious sequence alterations in
the BRCA2 gene. All except 1, a **deletion of 3 nucleotides, involved
nucleotide deletions that altered the reading frame, leading to
truncation of the BRCA2 protein. No missense or nonsense mutations were
found.** The authors noted that the mutational profile of BRCA2 differs
from that of BRCA1: microinsertions and point mutations are about as
common in BRCA1 as **microdeletions, which predominate in BRCA2**

***Tavtigian et al, 1996,*** ***8589730***

Spliceogenic mutations of group B included 6 already analyzed (BRCA1
c.212G\>A, c.213−11T\>G, and c.4484G\>T, and **BRCA2.... c.631G\>A,
c.8754+3G\>C, and c.9117G\>A)** \[11\], \[18\], \[19\], \[21\], \[22\],
\[26\], \[44\]--\[50\] and 2 newly characterized (c.134+3_134+6delAAGT,
c.4986+5G\>A in BRCA1). Three mutations caused the skipping of an entire
exon: BRCA1 c.4484G\>T (exon 14) (Fig. 2A), BRCA2 c.631G\>A (exon 7)
(Fig. 2B) and BRCA2 c.9117G\>A (exon 23) (Fig. 2C). Conversely, partial
intronic retention caused by the activation of cryptic splice sites was
observed for BRCA1 c.213−11T\>G (59 bp at the 3′-end of intron 5) (Fig.
2D), BRCA1c.4986+5G\>A (65 bp at the 5′-end of intron 16) (Fig. 2E), and
**for BRCA2 c.8754+3G\>C (46 bp at the 5′-end of intron 21**) (Fig. 2F).
Finally, the BRCA1 c.134+3_134+6delAAGT and c.212G\>A variants were
associated with a relevant increase, in comparison with normal controls,
of the Δexon3 and Δexon5q (missing 22 bp at the 3′-end of exon 5)
isoforms, respectively (Fig. 2G--H). Both isoforms contain PTCs

***Colombo et al, 2013, PMID: 23451180***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

Optional modifiers: incomplete penetrance

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is predominantly loss of function (*BRCA2* is a tumour
suppressor gene) leading to genomic instability. Premature translation
termination is the most common outcome with most (though not all) mutant
mRNAs predicted to undergo nonsense-mediated mRNA decay. Variants
include large deletions/duplications, frameshift, in-frame
deletions/insertions, missense, nonsense, splice site (including
non-canonical, deeper intronic) and large scale structural
rearrangement. The mutational spectrum is similar to BRCA1 though
deletions are more common with BRCA2, with proportionatly less missense
and large scale genomic rearrangements. Less evidence identified for
variants in 3'and 5' UTRs,

**List variant classes in this gene proven to cause this disease:**

splice_region_variant

splice_acceptor_variant

(splice_acceptor_variant predicted to undergo NMD)

splice_donor_variant

(splice_donor_variant predicted to undergo NMD)

(splice_acceptor_variant predicted to escape NMD)

(splice_donor_variant predicted to escape NMD)

Frameshift_variant

(frameshift_variant predicted to undergo NMD)

(frameshift_variant predicted to escape NMD)

Stop_gained

(stop_gained predicted to undergo NMD)

Inframe_deletion

Missense

**Potential novel variant classes based on predicted functional
consequence**

start_lost

(stop_gained predicted to escape NMD)

stop_lost

Inframe_insertion

5_prime_UTR_variant

3_prime_UTR_variant

intron_variant

(gain of upstream Start \[uORF\])

(gain of upstream Start \[oORF\])

(Stop lost \[uORF\])

(Stop lost \[oORF\])

(Start lost \[uORF\])

(Frameshift \[uORF\])

(Frameshift \[oORF\])

(Stop gained \[uORF\])
**CACNA1S -- Malignant hyperthermia (MIM 601887)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:1397

Mutations also can be associated with malignant hyperthermia
susceptibility (MHS).

There is **no current evidence for haploinsufficiency** as a
disease-causing mechanism.

The **penetrance of MHS is unknown**. What is known is that up to 50% of individuals with MHS have undergone anesthesia uneventfully despite use of one of the agents known to trigger MH. (Tier 4)

**Literature Review:**

**OMIM:** https://www.omim.org/entry/601887

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1146/

CACNA1S gene cause less than 1 percent of all cases of malignant
hyperthermia susceptibility. Mutations in the RYR1 or CACNA1S gene cause
the RYR1 channel to open more easily and close more slowly in response
to certain drugs. As a result, abnormally large amounts of calcium ions
are released from storage within muscle cells..

...Note that due to the gain-of-function disease mechanism, genetic
heterogeneity, and variable expressivity of this disorder, data from
functional studies are critical in reaching a likely pathogenic or
pathogenic classification using ACMG criteria.

rare *CACNA1S* variant, Thr852Met, of unknown significance and with a
frequency of 0.02% in the general population

**Kraeva et al, 2017, PMID: 28326467**

Variants are much less common in the *CACNA1S* gene
that encodes the α1 subunit of the skeletal muscle voltage-gated calcium
channel (interacting with RYR1 in the process of excitation--contraction
coupling).**Bayesian statistics predicted CACNA1S variant p.Thr1009Lys
and RYR1 variants p.Ser1728Phe and p.Leu4824Pro are likely pathogenic**

**Sadhasivam et al, 2019, PMID: 31559918**

Our findings, supported by the Exome Variant Server CACNA1S allele
frequencies, suggest that other previously implicated MHS variants may
be benign. Caution is warranted regarding variants claimed to be
causative for MHS, especially when used for predictive individualized
medicine.....We conclude that some RYR1 and CACNA1S variants may have
been misclassified as pathogenic without adequate genetic (e.g.,
cosegregation) or functional Gonsalves et al. Page 9 Anesthesiology.
Author manuscript; available in PMC 2014 November 01. NIH-PA Author
Manuscript NIH-PA Author Manuscript NIH-PA Author Manuscript data. It is
important to stress that in addition to robust genetic analysis, there
is a critical need for a robust and non-invasive functional test for
MHS, which together with genetic data could allow accurate determination
of the prevalence and penetrance of this trait.

***Gonsalves et al, 2013, PMID 24195946***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

*Incomplete penetrance*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

*Altered gene product structure*

**Narrative summary of molecular mechanisms:**

Mechanism is predominantly thought to be gain-of-function disease (with genetic heterogeneity and variable expressivity), though some Loss of function variants in CACNA1S have been implicated in autosomal dominant disease

**List variant classes in this gene proven to cause this disease:**

Missense

Frameshift_variant

Splice_acceptor_variant

Splice_donor_variant

Inframe_deletion

**Potential novel variant classes based on predicted functional
consequence**

Splice_donor_variant

Splice acceptor variant predicted to escape NMD

Splice donor variant predicted to escape NMD

Frameshift variant predicted to escape NMD

Stop gained predicted to escape NMD

Stop lost

Inframe_insertion

Inframe_deletion

*?? If consider haploinsufficiency as a potential MOA*

Frameshift

Splice region variant

Stop_gained

start_lost

Splice acceptor predicted to escape NMD

Splice donor predicted to escape NMD

stop_gained predicted to escape NMD

stop_lost

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[oORF\]

Frameshift \[oORF\]
**COL3A1 -- Ehlers-Danlos syndrome, type 4 (MIM 130050)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:2201

The relationship between COL3A1 and Ehlers-Danlos syndrome, vascular
type (autosomal dominant) was evaluated using the ClinGen Clinical
Validity Framework as of February 25th, 2019. Variants in COL3A1 were
first reported in humans with this disease as early as 1988
(Superti-Furga et al., PMID 2834369). At least 600 unique variants are
found in the Ehlers Danlos Syndrome Variant Database and \~250 more in
ClinVar (Byers, 1999; PMID 20301667). This is a well-known gene-disease
relationship and there is a significant amount of case-level,
segregation and experimental data in the literature, therefore the
maximum points for genetic and experimental evidence has been reached.
The mechanism for disease is **mainly gain-of-function as most cases result from missense variants leading to the substitution of a crucial glycine in the repetitive Gly-X-Y sequence of the triple helix domain**, resulting in only 1 in 8 normal mature collagen homotrimers. **About 5% of cases are due to haploinsufficiency and are associated with milder phenotypes** (Frank et al., 2015; PMID 25758994). This gene-disease relationship is supported by in vitro assays, expression studies, and
animal models. In summary, COL3A1 is definitively associated with
autosomal dominant Ehlers Danlos Syndrome, vascular type. This has been
repeatedly demonstrated in both the research and clinical diagnostic
settings, and has been upheld over time. This classification was
approved by the ClinGen General GCEP on February 27, 2019 (SOP Version
6).

**Literature Review:**

**OMIM:** https://www.omim.org/entry/130050

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1494/

Pathogenic variants. More than 600 COL3A1 variants that result in a disease-causing phenotype have been identified.

The majority of identified pathogenic variants result in **single-amino acid substitutions for glycines in the Gly-X-Y repeat of the triple helical region of the type III procollagen molecule.** About a quarter of reported pathogenic variants occur at **splice sites, most resulting in exon skipping**. A smaller number of splice site variants lead to the use of **cryptic splice sites with partial-exon exclusion or intron
inclusion**. The vast majority of exon-skipping splice site variants
have been identified at the 5\' donor site, with very few found at the
3\' splice site.

Several **partial-gene deletions** have been reported as well. Less
common are variants that create **premature termination codons predicted
to result in COL3A1 haploinsufficiency** (\"null\" pathogenic variants)
\[Schwarze et al 2001, Leistritz et al 2011\]. (See Database of Human
Type I and Type III Collagen Mutations.)

Of note, at least two classes of COL3A1 variants are underrepresented
(in terms of predicted frequency) among individuals with clinical
features of vEDS: Substitutions of glycine in the triple helical domain
by alanine, Null variants. Thus, **some pathogenic variants in COL3A1
may not produce a typical vEDS clinical picture**. It is unclear if
individuals with these classes of pathogenic variants have attenuated or
subclinical phenotypes and present at later ages or if there is a
molecular explanation for the absence of certain pathogenic variant
types.

Normal gene product. COL3A1 encodes the proα1(III) chain of type III
procollagen, a major structural component of skin, blood vessels, and
hollow organs. The type III procollagen molecule is a homotrimer, with
constituent chains 1,466 amino acids in length.

**Abnormal gene product. Pathogenic variants in COL3A1 typically result
in a structural alteration** of type III collagen that leads to
intracellular storage and impaired secretion of collagen chains.
Production of half the normal amount of type III procollagen occurs in
a minority of individuals.

We identified the underlying causative COL3A1 mutation in 572 of the
index individuals (410 different mutations) (Supplementary Table S1
online). Among the families with mutations, 356 had alterations that led
to substitutions for glycine in the repeated Gly-X-Y triple motif of the triple helical domain. Another 164 had a mutation that disrupted a splice donor site (144) or splice acceptor site (20**). The remaining 52 had unique **insertions, duplications, deletions, or insertions/ deletions**. In 27 individuals, the described mutation led to loss of a
stable mRNA transcript from one allele, which was a **null** mutation.
Of the 410 unique mutation sites, 69 had more than one family with the
same mutation. Among these, four sites (c.1662+1G\>A, IVS24+1G\>A (32),
c.547G\>A, p.Gly183Ser, Gly16Ser in the triple helical domain (18),
c.755G\>T, p.Gly252Val, c.1347+1G\>A, IVS20+1G\>A (9), and Gly85Val

in the triple helical domain (8)) accounted for 30% of recurrent
mutations. There were 17 additional sites with more than 2 unrelated
families with the same mutation.

Single-nucleotide substitutions in glycine codons (GGN) can give rise to
codons for eight other amino acids and a single termination codon. The
observed distribution of substitutions that resulted from
single-nucleotide substitutions in glycine codons in the triple helical
domain differed significantly (P \< 0.01 χ2 test, in Excel) from that
which had been predicted (Supplementary Table S2 online). Substitutions
of glycine codons by those for small or neutral amino acids (alanine,
serine, and cysteine) were less frequent than expected.

*Pepin, MG et al, 2014, PMID: **24922459***

Biallelic variants of COL3A1 have been reported in two families with a
severe EDS phenotype and early death as well as structural brain
anomalies resembling GPR56-associated brain malformations (Jørgensen et
al., 2015; Plancke et al., 2009). Very recently, a **homozygous missense
variant in COL3A1** was identified in two affected siblings with
cobblestone-like lissencephaly but no EDS phenotype (Vandervore et al.,
2017). Here, we describe two families with biallelic variants in COL3A1:
The first family was found to have a **nonsense variant and a frameshift
variant in COL3A1** leading to skeletal, joint, and brain manifestations
in the index patient who is compound heterozygous for these variants. In
the second family with two affected siblings a **homozygous, recently
reported missense** variant in COL3A1 was identified. These patients did
not show any signs of EDS but cobblestone-like lissencephaly,
developmental delay, and epilepsy overlapping the phenotype associated
with GPR56 variants.

*Horn et al, 2017 PMID: **28742248***

Vascular Ehlers--Danlos syndrome (vEDS) is a rare and severe autosomal
dominant disorder caused by variants at the COL3A1gene. Clinical
characteristics and course of disease of 215 molecularly proven patients
(146 index cases and 69 relatives) were analysed. We found 126 distincts
variants that were divided into five groups: (1) **Glycine substitutions
(n = 71), (2) splice-site and in-frame insertions--deletions (n = 36),
(3) variants leading to haplo-insufficiency (n = 7), (4) non-glycine
missense variants within the triple helix (n = 4 variants), and (5)
non-glycine missense variants or in-frame insertions--deletions, in the
N- or C-terminal part of the protein (n = 8)**. Overall, our cohort
confirmed the severity of the disease with a median age at first
complication of 29 years (IQR 22--39), the most frequent being arterial
(48%) and digestive (24%) ruptures. Groups 2 and 1 were significantly
more severe than groups 3--5, with extreme median ages at first major
complication of 23--47 years. Patients of groups 3--5 had a less typical
phenotype and remarkably absence of digestive events. **The distribution
of glycine-replacing amino acids was strongly biased towards more
destabilizing residues of the collagen assembly**. Thus the natural
course of vEDS and the clinical phenotype of patients are influenced by
the type of COL3A1 variant. This study also confirms that patients with
**variants located in the C- and N-termini or leading to
haplo-insufficiency have milder course** of the disease and l

*Frank et al, 2015, PMID: 25758994*

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

Altered gene product structure

Decreased gene product level

Absent gene product

**Narrative summary of molecular mechanisms:**

Mechanism of action is predominantly due to an abnormal gene product - a structural alteration of type III collagen that leads to intracellular storage and impaired secretion of collagen chains. Common variants include single-amino acid substitutions for glycines in the Gly-X-Y repeat of the triple helical region of the type III procollagen molecule. Canonical and non-canonical splice variants (include though creating cryptic splice sites) are recognised. Production of half the normal amount of type III procollagen occurs in a minority of individuals and may results from partial gene deletions, stop gain and frameshift variants.

**List variant classes in this gene proven to cause this disease:**

Missense

Splice_acceptor_variant

Splice_donor_variant

Splice Region Variant

Stop_gained

Frameshift_variant

Inframe_deletion

Splice acceptor variant predicted to undergo NMD

Splice donor variant predicted to undergo NMD

Frameshift variant predicted to undergo NMD

**Potential novel variant classes based on predicted functional
consequence**

Splice acceptor variant predicted to escape NMD

Splice donor variant predicted to escape NMD

Frameshift variant predicted to escape NMD

start_lost

stop_gained predicted to escape NMD

stop_lost

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[oORF\]

Stop lost \[uORF\]

Frameshift \[oORF\]

Frameshift \[uORF\]

Inframe_insertion
# **DSC2 -- Arrhythmogenic Right Ventricular Cardiomyopathy**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:3036

The relationship between DSC2 and arrhythmogenic right ventricular
dysplasia (autosomal dominant) and arrhythmogenic right ventricular dysplasia with mild palmoplantar keratoderma with or without woolly hair (autosomal recessive) was evaluated using the ClinGen Clinical Validity Framework as of July, 2018. Variants in DSC2 were first reported in humans with this disease as early as 2006 (Syrris et al., PMID: 17033975). At least 13 variants (e.g. missense, nonsense, frameshift) have been reported in humans. Evidence supporting this gene-disease relationship includes case-level data and experimental data. Summary of Case Level Data: 8.5 points. Variants in this gene have been reported in at least 13 probands in 6 publications (PMIDs: 17963498, 21062920,
23863954, 17186466, 18957847, 17033975). This gene-disease relationship is supported by animal models, expression studies, and protein interactions. In summary, DSC2 is definitively associated with ARVD (AD) and ARVD with mild palmoplantar keratoderma with or without woolly hair (AR). This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This
classification was approved by the ClinGen Arrythmogenic Right
Ventricular Cardiomyopathy Gene Curation Expert Panel on September 14,2018 (SOP Version 6)

**ClinGen Haploinsufficiency comments**

DSC2 encodes the desmosomal cadherin, desmocollin 2. Sequence-level variants in DSC2 are associated with arrhythmogenic right ventricular cardiomyopathy 11 (AVRC) (OMIM: 610476), a genetically heterogeneous cardiovascular disorder characterized by fibrofatty replacement of the right ventricular myocardium, ventricular arrhythmia, and an increased risk of premature sudden cardiac death.

**A large number (n\>50) of DSC2 mutations have been reported in the literature, the majority of which are missense mutations.** Several mutations that may confer a loss-of-function have also been reported, including splice-site, intragenic deletions/duplications, and nonsense mutations, which have been observed in association with both autosomal dominant (PMID: 17186466 and 17033975) and autosomal recessive (PMID:20400443, 19863551, 24793512, 23863954) forms of AVRC. Incomplete penetrance for some DSC2 variants has also been observed.

Although more than two independent publications with loss-of-function-type variants provides evidence in support of DSC2 haploinsufficiency, due to the variable inheritance patterns, mutational spectrum, and lack of reports of whole gene deletions, the haploinsufficiency score is a 2.

**Literature review:**

More than 50 pathogenic variants have been described. In addition, **multiple instances of digenic inheritance have been identified with DSC2 variants along with other desmosomal gene mutations [Bhuiyan et al 2009, Groeneweg et al 2015] and compound heterozygosity [Lorenzon et al 2015] and homozygosity [Wong et al 2014].**

Desmocollin pathogenic isoforms that lack the last 37 amino acid residues of the carboxyl-terminal domain of DSC2a are unable to bind plakoglobin. It is unknown how the variants affect desmosome formation, but it is speculated that the result would be impaired desmosome structure or function.

*Gene reviews 
https://www.ncbi.nlm.nih.gov/books/NBK1131/#arvd.Molecular_Genetics*

**The molecular links between desmosome mutations and the pathological hallmarks of ACM --- cardiomyocyte loss, fibrosis, adipogenesis, inflammation and arrhythmogenesis --- remain poorly defined. Probable pathogenic mechanisms include loss of mechanical integrity at cell--cell junctions, altered signalling pathways at intercalated discs, disruption of ion channels and gap junctions, and aberrant protein trafficking.**

*Austin KM et al. 2019. (PMID: 31028357)*

**The ARVC variant database can be found at https://molgenis136.gcc.rug.nl/**

In 2019, Ye et al reevaluated ARVC variants using large population databases. "...more than 10% of variants previously
reported to cause ARVC were found unlikely to be associated with highly penetrant monogenic forms of ARVC."
There were some variants that were found in population databases but were nevertheless associated with serious cardiac phenotypes suggesting they could be disease-modifiers of ARVC. Updated classification of variants by gene is available in the supplementary data

*Ye JZ et al 2019 PMID: 31402444*

"This study assessed the origin of ARVC-associated P/LP desmosomal variants and has 2 main findings. First, most P/LP desmosomal variants in ARVC are nonunique (75.3%), that is, occurring in multiple families and inherited (98.6%). Second, most nonunique PKP2 variants share haplotypes. Taken together **these results suggest most ARVC-associated variants originate from common founders.**

The high percentage of inherited variants will inform genetic counseling and approaches to cascade screening. Specifically, anticipatory guidance given during pretest counseling should include the likelihood that a variant, if identified, has very likely been inherited. Thus, signs or symptoms in a relative should be considered with a high index of suspicion.

it is also worth emphasizing that **13 (4.0%) of the P/LP variants in this study were large deletions.** While these large deletions in ARVC have been previously reported16,17,35 our findings highlight the importance of using a genetic test capable of identifying deletions."

*van Lint FHM et al 2019 PMID: 31386562*

We analyzed the genes in (a) 57 patients who fulfilled the ARVD/C TFC (TFC+), (b) 28 patients with probable ARVD/C (1 major and 1 minor, or 3 minor criteria), and (c) 31 patients with 2 minor or 1 major criteria. In the TFC+ ARVD/C group, 23 patients (40%) had PKP2 mutations, 4 (7%) had DSG2 mutations, and 1 patient (2%) carried a mutation in DSC2, whereas **1 patient (2%) had a mutation in both DSG2 and DSC2.** Among the DSG2 and DSC2 mutation-positive TFC+ ARVD/C probands, 2 carried compound heterozygous mutations and 1 had digenic mutations.

*Bhuiyan et al 2009 PMID: 20031616*

In 2006, two heterozygous *DSC2* mutations---a deletion and an
insertion---were detected in four probands and three family members. The deletion of a single nucleotide (1430delC) was detected in exon 10 of *DSC2.* It leads to a frameshift and a premature termination codon at position 480 (M477fsX480). The authors speculate that this results in haploinsufficiency, however functional studies to support this association are lacking. The other mutation was an insertion of two bases in exon 17 (2687_2688insGA) which is predicted to generate a
termination codon 4 aa residues downstream (E896fsX900).

*Syrris et al., 2006 (PMID: 17033975)*

Beffagna et al Identified two heterozygous mutations (c.304G\>A
(p.E102K) and c.1034T\>C (p.I345T)) in two probands and in four family members. The two missense mutations in the N-terminal domain affect the normal localisation of DSC2.

*Beffagna et al. 2007.(PMID:17963498)*

A heterozygous splice-acceptor--site mutation was identified in intron 5 (c.631-2A→G) of the *DSC2* gene, which led to the use of a cryptic splice-acceptor site and the creation of a downstream premature termination codon. The truncated protein encoded by the mutant allele in this case may have a dominant negative effect disrupting cell-cell adhesion, but the markedly decreased levels of mRNA observed with the mutant *DSC2* allele suggest that nonsense-mediated mRNA decay and consequent haploinsufficiency likely contribute to the disease mechanism.

*Heuser A et al. 2006 (PMID: 17186466)*

E102K and I345T mutant DSC2 proteins delocalize from the plasma membrane to the cytosol [\[90\]] inferring trafficking defects, while the R203C and T275M mutants show impaired proteolytic processing [\[64\]]. A pair of NMD mutations, R375X and Q554S, are found in the EC3 and EC4 domains, respectively, as well as deletions due to frameshift mutations [\[54\]].
The EC5 domain contains a single pathogenic mutation,
I603T [\[91\]], as well as frameshifts and deletions that would cause NMD. 

*Al-jassar C et al. 2013. PMID: (23911551)*

**Homozygous DSC2 mutations have been associated with ACM forms with or without cutaneous manifestations.**

A homozygous missense mutation p.D179G was associated with biventricular ACM and no cutaneous features. The mutation involves an amino acid in the first extracellular protein domain. Incorporation of the mutant protein in intercalated discs may affect the normal cadherin interactions in the extracellular space, leading to adhesion impairment between cells.

*Lorenzon A et al. 2015. (PMID: 26310507)*

Homozygosity of a truncation mutation (p.Q554X) in the desmosomal
cadherin DSC2 causes an early onset arrhythmic cardiomyopathy in the North American Hutterite population. Immunohistochemistry of diseased myocardium and recombinant expression in cell culture showed that the extracellular truncated protein remains stable, partially processed and localized at the intercalated discs, suggesting a more complex mechanism with partial contribution in maintaining cell junction integrity rather than the expected complete loss of function attributable to non-functional alleles.

*Gerull B et al. 2013. (PMID: 23863954)*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

Autosomal recessive

Optional modifiers: incomplete penetrance; digenic

**Allelic requirement:**

Monoallelic_aut

Biallelic_aut

Optional modifiers: digenic

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism likely loss of function of *DSC2* due to **reduction/absence of gene product or altered gene product structure** due to a variety of mechanisms (e.g. null alleles, trafficking defects, impaired proteolytic processing, absence of or impaired protein-protein interactions). A number of DSC2 mutations have been reported in the literature, the majority of which are missense mutations. Although loss of function is a likely mechanism, there is remaining uncertainty about haploisufficiency as a mechanism given variable inheritance patterns, mutational spectrum and lack of reports of whole gene deletions.(ClinGen) Autosomal dominant inheritance with incomplete penetrance is the most common mode of transmission, although autosomal recessive mutations have been described causing ARVC with and without cutaneous features. Instances of digenic inheritance have been identified with DSC2 variants along with other desmosomal gene mutations.

**Additional information related to ACMG evidence types**

As per Cardioclassifier:

**PVS1** - null variant in a gene where Loss of Function (LoF) is a known mechanism of disease
Consider activating for DSG2 and ARVC as there is a significant burden of truncating variants in cases against controls from analysis of 7,855 cardiomyopathy cases and 60,706 controls 

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework). Adjusted for ICC from original ACMG guidelines
0.1% (het) 
3.16% (hom) 

**For BS1 and PM2:**

Estimated Prevalence: 1/1000
Max Allelic Contribution: 0.092	
Maximum population AF: 0.000092

*Whiffin N et al 2018 PMID: 29369293*

**List variant classes in this gene proven to cause this disease**

- Splice region variant
- ?cryptic splice site variant
- Spice acceptor variant
- Splice acceptor variant predicted to undergo NMD
- Splice donor variant
- Splice donor variant predicted to undergo NMD
- Frameshift variant
- Frameshift variant predicted to undergo NMD
- Stop gained
- Stop gained predicted to undergo NMD
- Missense
- In frame insertion
- In frame deletion

**List other variant classes predicted to lead to the same functional consequence:**

- Splice acceptor variant predicted to escape NMD
- Splice donor variant predicted to escape NMD
- Frameshift variant predicted to escape NMD
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
- ?larger deletions?
# **DSG2 -- Arrhythmogenic Right Ventricular Cardiomyopathy**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:3049

The relationship between DSG2 and arrhythmogenic right ventricular
cardiomyopathy (autosomal dominant) was evaluated using the ClinGen Clinical Validity Framework as of July, 2018. Variants in DSG2 were first reported in humans with this disease as early as 2006 (Pilchou et al., PMID: 16505173). Variation in DSG2 is a well-known cause of ARVC and accounts for 5%-26% of cases (McNAlly et al., 2005; PMID: 20301310). Since this gene-disease relationship is well-known, there is a significant amount of case-level data, segregation data and experimental data available in the literature, therefore the maximum score for both genetic evidence and experimental evidence has been reached. Note, this
curation effort may not be exhaustive of all literature related to this gene-disease relationship. This gene-disease relationship is supported by animal models, in vitro assays, expression assays, and protein interactions. In summary, DSG2 is definitively associated with autosomal dominant ARVC. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This classification was approved by the ClinGen Arrythmogenic Right Ventricular Cardiomyopathy Gene Curation Expert Panel on September
14, 2018 (SOP Version 6).

**ClinGen Haploinsufficiency phenotype comments:**

PMID 21397041 - Lahtinen et al. (2011) identified a proband with ARVC who had a novel 3059_3062delAGAG frameshift mutation leading to abolishing 99 carboxy-terminal amino acids of desmoglein-2. This mutation was found in four additional family members: two of whom met minor criteria for first-degree family members, one had minor repolarization abnormalities, and the fourth was unaffected. A screen for this mutation in the control samples revealed one carrier who then was found to have a conduction defect. Since the immunoreactive signal for desmoglein-2 was reduced by over 50%, a dominant-negative effect for the mutation was suggested as a possibility.

PMID 26296472 - Zhang et al. (2015) obtained DNA from post-mortem heart tissues, 25 patients had ARVC as a cause of death while the other 25 had sudden unexplained death. Two novel mutations in DSG2 were identified in each group. A frameshift insertion c.3075_3076insC (S1026Q fsX12) and a missense c.2032G>A (G678R) mutation was found in the ARVC group. In the samples from patients with sudden unexplained death, the missense mutations were c.2686G>A (E896K) and c.2573C>T (A858V). None of these variants was found in 96 control samples. No commentary as to possible mutation mechanism was included.

PMID 16773573 - Awad et al (2006) identified four patients with ARVD/C with mutations in DSG2. One proband had two mutations in trans, a missense c.143G>A (R48H) on his paternal allele and a c.915G>A change resulting in a premature termination codon, W305X, in exon 8 of his maternal allele. Of note, his 77-year-old mother was found to carry the same nonsense mutation, but was unaffected based upon extensive clinical assessments. His 24 year-old sister also carries the W305 mutation and is believed to be unaffected. His father who presumably passed on the R48H mutation died due to non-cardiac issues at age 74 and was never tested. The remaining three probands in this study had a single, heterozygous missense mutation each: c.134G>A (R45Q), c.1517G>A (C506Y), and c.2431G>T (G811C). Of note, the mother of the proband with R45Q and the mother of the proband with G811C were both found to be carriers of their respective familial mutation and were determined to be unaffected after extensive cardiac assessments. **Discussion regarding mutational mechanism included the possibility of both haploinsufficiency and dominant-negative as possible mechanisms.**

Desmoglein-2 (DSG2) is a member of the desmoglein family and is expressed in myocardium. DSG2 is an essential component of the desmosome so mutations of this gene disrupt the proper organization of desmosomal junctions. Mutations of DSG2 and other desmosomal genes have been demonstrated to be associated with arrhythmogenic right ventricular dilatation/cardiomyopathy (ARVD/C), but the phenotype is widely variable ranging from individuals who meet strict diagnostic criteria to those who are asymptomatic and identified in control populations. Some affected patients have compound heterozygous mutations. So variants in DSG2 might be susceptibility variants, rather than pathogenic variants. Also, digenic, homozygous, and compound heterozygous inheritance has been suggested.

**While a few protein truncating mutations have been described, the mechanism by which these mutations cause disease has not been clearly determined and no patients with complete gene deletions and ARVD/C have been reported.**

**Literature review:**

"More than 20 pathogenic variants have been described (see Table A, Locus Specific) In addition, **multiple instances of digenic inheritance have been identified with other desmosomal gene variants [Rigato et al 2013, Groeneweg et al 2015]. Biallelic inheritance has also been described in ARVC with compound heterozygous variants** [Awad et al 2006, Pilichou et al 2006, Bhuiyan et al 2009].

Much of the underlying pathogenesis of DSG2 pathogenic variants is still unknown; it is believed that loss of DSG2 compromises cell-to-cell adhesion between cardiomyocytes [Kant et al 2015]"

*Gene reviews https://www.ncbi.nlm.nih.gov/books/NBK1131/*

"The observed autosomal dominant inheritance of desmoglein 2 gene
(*DSG2*)--related AC cases suggests that reduction of the Dsg2 wild-type (WT) protein leads to a loss of cardiac function or that Dsg2 mutant (MT) protein exerts a dominant negative effect on the remaining Dsg2 WT protein. Two major types of nonexclusive pathomechanisms have been described to depend on Dsg2. As a cell--cell adhesion molecule, it provides adhesive force and thereby supports force transmission between contractile cardiomyocytes. Alternatively, Dsg2 regulates intracellular signaling. This function is expected to be linked to the cytoplasmic domain of Dsg2 and may involve associated signaling molecules, such as Pg and Pkp2/PKCα acting through downstream effectors implicating the Wnt or Hippo pathways."

*Kant S et al. 2015 (PMID:26085008)*

"In a series of 80 unrelated ARVC probands, 26 carried a mutation in DSP (16%), PKP2 (14%), and transforming growth factor-β3 (2.5%) genes; the remaining 54 were screened for DSG2 mutations by denaturing high-performance liquid chromatography and direct sequencing. **Nine heterozygous DSG2 mutations (5 missense, 2 insertion-deletions, 1 nonsense, and 1 splice site mutation) were detected in 8 probands (10%).**
**In 1 patient (patient 5), 2 different mutations (E331K and 1881 to 2A→G) were detected.** Family members were available for molecular analysis (Figure 3, family 172). Subject II,1 carried both mutations, subjects I,2 and III,2 carried the missense mutation, and subject III,1 carried the splice site mutation, thus demonstrating that the 2 mutations are in trans orientation. Although both mutations are potentially pathogenic, the possibility that the missense mutation is a rare polymorphism cannot be ruled out.

*Pilichou K et al 2006 PMID: 16505173*

**"Among the 27 *DSG2* mutation carriers, 18 (67%) carried a missense mutation, one (4%) a stop mutation, three (11%) a frameshift mutation, and five (18%) a splice site mutation. Most patients (17/18, 94%) with *DSG2* missense mutations carried a mutation located within the *DSG2* pro‐peptide cleavage site, a hot‐spot for *DSG2* mutations *(Awad M. 2008 (PMID: 18382419)***

Among the DSG2 mutation carriers, one patient was compound heterozygous (DSG2 p.Gly640AspfsX15+DSG2 p.Val295SerfsX6) and five were homozygous [p.Thr335Ala, p.Lys834ArgfsX3 (n=2), c.523+1G >A and p.Gly720Ter].

**DSG2 mutation carriers more often displayed biventricular involvement and left ventricular dysfunction at diagnosis...**

...Although DSG2 mutation carriers more frequently display more severe right and left ventricular involvement than PKP2 mutation carriers, their risk for VA is similar.

Although the main aim of our study was not to assess the role
of multiple mutations on prognosis, **unadjusted survival analysis
showed that a complex genetic status was associated with a
higher risk of HF-related death/transplantation.** This finding is
in accordance with previous studies in which multiple mutations
were associated with a worse prognosis, especially a higher risk
of arrhythmic events and SCD.17,27 In addition to the higher
arrhythmic risk which was previously reported, our results suggest
that multiple/homozygous mutations carriers are at higher risk of
developing end-stage HF.

The relatively small number of DSG2 patients followed and the
dominance of Caucasian subjects in our population could hinder
the generalization of these results to other populations."

*Hermida A et al. 2019  PMID: 30790397*

There is significant excess of truncating and non truncating variants in DSG2 in ARVC cases vs population controls.

*Walsh et al, 2016 (PMID 27532257)*

**"the amino-terminal propeptide and cadherin repeats I-III domains (residues 24 through 388) of DSG2 demonstrated an over-representation of missense mutations compared to controls** (16/21 vs. 2/20, P < 2.5×10−5, Figure 6B). These mutation hot spots localize to key regions of the desmosomal macromolecular complex as demonstrated in Figure 7." 

*Kapplinger JD et al 2011 PMID: 21636032*

**The ARVC variant database can be found at https://molgenis136.gcc.rug.nl/**

In 2019, Ye et al reevaluated ARVC variants using large population databases. "...more than 10% of variants previously
reported to cause ARVC were found unlikely to be associated with highly penetrant monogenic forms of ARVC."
There were some variants that were found in population databases but were nevertheless associated with serious cardiac phenotypes suggesting they could be disease-modifiers of ARVC. Updated classification of variants by gene is available in the supplementary data

*Ye JZ et al 2019 PMID: 31402444*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

Autosomal recessive

Optional modifiers:

Incomplete penetrance; Digenic (PKP2, DSP)

**Allelic requirement:**

Monoallelic_aut

Biallelic_aut

Optional modifiers:

Digenic (PKP2, DSP)

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism likely due to **reduction/absence of gene product or altered gene product structure** due to a variety of mechanisms.  Much of the underlying pathogenesis of DSG2 pathogenic variants is still unknown; it is believed that loss of DSG2 compromises cell-to-cell adhesion between cardiomyocytes. Both haploinsufficiency and dominant-negative have been proposed as possible mechanisms. Instances of digenic inheritance have been identified with other desmosomal gene variants and biallelic inheritance has also been described with compound heterozygous and homozygous variants.
Hermida et al found DSG2 mutation carriers display more severe disease than PKP2 mutation carriers, with an increased risk of biventricular involvement and evolution to end-stage HF. Also a complex genetic status had a worse prognosis with a higher risk of end stage HF and arrhythmic events. Most patients (17/18, 94%) with *DSG2* missense mutations carried a mutation located within the *DSG2* pro‐peptide cleavage site.

**Additional information related to ACMG evidence types**

As per Cardioclassifier:

**PVS1** - null variant in a gene where Loss of Function (LoF) is a known mechanism of disease
Consider activating for DSG2 and ARVC as there is a significant burden of truncating variants in cases against controls from analysis of 7,855 cardiomyopathy cases and 60,706 controls 

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework). Adjusted for ICC from original ACMG guidelines
0.1% (het) 
3.16% (hom) 

**For BS1 and PM2:**

Estimated Prevalence: 1/1000
Max Allelic Contribution: 0.092	
Maximum population AF: 0.000092

*Whiffin N et al 2018 PMID: 29369293*

**List variant classes in this gene proven to cause this disease**

- Splice acceptor variant
- Splice donor variant
- Frameshift variant
- Stop gained
- Stop gained variant predicted to undergo NMD
- Stop gained variant predicted to escape NMD
- Missense
- In frame insertion
- In frame deletion

**Potential novel variant classes based on predicted functional
consequence:**

- Splice acceptor variant predicted to undergo NMD
- Splice acceptor variant predicted to escape NMD
- Splice donor variant predicted to undergo NMD
- Splice donor variant predicted to escape NMD
- Frameshift variant predicted to undergo NMD
- Frameshift variant predicted to escape NMD
- start_lost
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
# **DSP -- Arrhythmogenic Right Ventricular Cardiomyopathy**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:3052

The DSP gene was the first ARVC-gene to be associated to the disease, the initial mutation description was done in Carvajal syndrome characterized by woolly hair, keratoderma and ARVC, it is transmitted in a autosomal recessive pattern, homozygous mutations in DSP were described in in the year 2000, PMID 11063735. This was followed by the description of a heterozygous mutation in DSP in an Italian family with ARVC and clear co-segregation of the variant with the disease, PMID 12373648. These findings have been replicated worldwide in several studies performed in different ethnicities, PMID 15941723, PMID 25765472, PMID 23954618, PMID 20864495, PMID 21397041, PMID 24938629.

The initial descriptions recognized also a high frequency of left
ventricular compromise in families with DSP mutations, PMID 16061754, PMID 28527814. A murine model was generated able to replicate the arrhythmia phenotype and Cx43 mislocalization, PMID 22240500. A transgenic mouse overexpressing a mutant DSP had increased cardiomyocyte apoptosis, cardiac fibrosis and lipid accumulation PMID 16917092.
Abnormal DSP protein expression in DSP mutation carriers has also been reported, PMID 23137101. The role of this gene in this particular disease has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time (in general,at least 3 years). No convincing evidence has emerged that contradicts the role of the gene in the specified disease. In summary, based on this overwhelming evidence, DSP is definitely associated with arrhythmogenic cardiomyopathy with woolly hair and keratoderma, maximum association score was achieved rapidly after the analysis of few main reports.

**ClinGen Haploinsufficiency phenotype comments:**

The desmosomal cardiomyopathies are often associated with a complex mode of inheritance and a highly variable pattern of protein expression. AR inheritance form mutations are often ?loss of function? mutations and AD forms are known to occur. However, there is no clear evidence that they are due to haploinsufficiency; dominant negative effect appears to be
the more likely mechanism of pathogenicity in these cases. Furthermore, whole gene deletions have not been reported in the literature. At most, haploinsuffiency may result in skin manifestations (hereditary palmoplantar keratoderma) or may contribute in a multifactorial manner to susceptibility to heart disease

**Literature review:**

DSP is the largest among the desmosome genes and encodes the 2872 amino acid protein, desmoplakin. The N-terminus of desmoplakin is required for localization to the desmosome and interaction with plakophilin and plakoglobin. The C-terminus of desmoplakin binds to the intermediate filaments (desmin)...
A DSP mutation was first identified in a homozygous manner as the cause of Carvajal disease, which shows dilated cardiomyopathy with wooly hair and keratoedema...

In 2002, a DSP mutation, S229R, was identified in a 18 year-old male who suffered cardiac arrest and was diagnosed with
ARVC. An extended clinical and genetic analysis of his family members from 4 generation confirmed that the mutation was the cause of ARVC. The residue 229 located in the N-terminus of desmoplakin is involved in binding with plakoglobin or plakophilin, and the mutation S229R would disrupt the normal binding with those proteins.

After the first report, other DSP mutations have been identified in ARVC patients. Although DSP is the largest among the desmosome genes, the number of reported mutations is small compared to other genes. Only 12 mutations have been reported in recent study with 439 families.

*Ohno S. 2016 PMID: (27761164)*

More than 80 pathogenic variants have been described (see Table A, Locus Specific). In addition, multiple instances of digenic inheritance have been identified with other desmosomal gene mutations [Rigato et al 2013, Groeneweg et al 2015].

It is speculated that abnormalities in desmoplakin lead to desmosomal instability. Defective desmosomes cannot sustain the constant mechanical stress in contracting cardiomyocytes, resulting in cardiac dysfunction and cell death [Yang et al 2006].

Data from a desmoplakin-deficient mouse model suggest that abnormal desmosomes lead to abnormal β-catenin signaling through Tcf-Lef1 transcription factors resulting in de-differentiation of myocytes into adipocytes [Garcia-Gras et al 2006]. Conditional deletion of Dsp using the Hcn4-Cre allele, which deleted Dsp in the cardiac conduction system, resulted in sinus node dysfunction, underscoring the importance of desmosomes for cardiac conduction system integrity [Mezzano et al 2016].

*Gene reviews https://www.ncbi.nlm.nih.gov/books/NBK1131/*

"an amino-terminal region of DSP encompassing three of the predicted alpha-helical bundles Z, Y, and X (residues 250 through 604) contained 8/17 (47%) of the DSP missense mutations found in ARVC cases compared to 1/28 (3.6%) of the DSP missense mutations found in controls (*P* \< 0.0008, [Figure 6A](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6311127/figure/F6/))."

*Kapplinger et al 2011 (PMID: 21636032)*

**our data support the notion that there are no differences in terms of arrhythmic risk between missense and non-missense DSP mutations in ARVC.**

We show that ARVC-associated DSP mutations correlate with a high
arrhythmic risk and that non-missense mutations are specifically
associated with left-dominant forms. **The presence of DSP truncating mutations should alert to the likely development of LV dysfunction.**

*Castelletti S et al. 2017 Dec 15 (PMID:28527814)*

Here, we found that DSP cardiomyopathy involves the LV in almost all cases and often without any apparent RV involvement. This finding was in clear contrast to PKP2 cardiomyopathy, which always involved the RV predominantly and most often in isolation. Not surprisingly, the ARVC task force criteria, validated primarily in PKP2-enriched cohorts, performed poorly for detection of clinically affected DSP probands and nonprobands.


**From in house atlas of variants in DSP**

21/43 truncating variants

Nonsense

Frameshift

Splice donor variant

22/43 non-truncating variants (mainly VUS)

Missense

https://www.cardiodb.org/acgv/acgv_gene.php?gene=MYH7

*Walsh R et al 2017 PMID:27532257*

In a mutation analysis of 66 probands with ARVD, [Yang et al.
(2006)](https://www.omim.org/entry/125647#31) identified 4 variants in DSP: V30M, Q90R, W233X, and R2834H ([125647.0012](https://www.omim.org/entry/125647#0012)). To establish a
cause and effect relationship between these DSP missense mutations and ARVD, they performed in vitro and in vivo analyses of the mutant proteins. **Unlike wildtype DSP, the N-terminal mutants (V30M and Q90R) failed to localize to the cell membrane in a desmosome-forming cell line and failed to bind to and coimmunoprecipitate junction plakoglobin.**

Multiple attempts to generate N-terminal DSP (V30M and Q90R)
cardiac-specific transgenes failed; analysis of embryos revealed
evidence of profound ventricular dilation, which likely resulted in embryonic lethality. [Yang et al. (2006) PMID 16917092](https://www.omim.org/entry/125647#31) were able to develop transgenic (Tg) mice with cardiac-restricted overexpression of the C-terminal mutant (R2834H) or wildtype DSP. Whereas mice overexpressing
wildtype DSP had no detectable histologic, morphologic, or functional cardiac changes, the R2834H-Tg mice had increased cardiomyocyte apoptosis, cardiac fibrosis, and lipid accumulation, along with ventricular enlargement and cardiac dysfunction in both ventricles.
These mice also displayed interruption of DSP-desmin interaction at intercalated discs and marked ultrastructural changes of these discs.
**The data suggested that DSP expression in cardiomyocytes is crucial for maintaining cardiac tissue integrity, and that DSP abnormalities result in ARVD by cardiomyocyte death, changes in lipid metabolism, and defects in cardiac development**

*Omim https://omim.org/entry/125647?search=dsp&highlight=dsp*

.... 4 patients with mutations in the DSP gene, 3 ARVD patients and 1 patient with Carvajal syndrome. The mutation carriers had abnormal DSP expression in both myocardial and epidermal tissue; disease mechanisms included haploinsufficiency, dominant-negative effects, or both.

*Rasmussen TB et al. 2013 Jul (PMID:23137101)*

"Mutations in the DSP gene encoding desmoplakin were first identified in an autosomal recessive form of arrhythmogenic cardiomyopathy. The present study makes a case for DSP cardiomyopathy being a distinct form of cardiomyopathy. DSP cardiomyopathy results in episodic inflammation, which precedes the development of fibrosis. Frequent PVCs occurring before LV systolic dysfunction or LV enlargement is also a key distinction from most cases of DCM. In DSP patients, late gadolinium enhancement was found in the LV subepicardium rather than mid-myocardial as is often seen with nonischemic cardiomyopathy. The correct diagnosis must include the identification of the DSP mutation. This paper is likely to be very impactful, as it suggests that some of the patients suspected of myocarditis and cardiac sarcoidosis may actually have DSP cardiomyopathy.

Mutations in this study were primarily truncating mutations (ie, frameshift, nonsense, or splice site
mutations). Truncating mutations in this and other
studies occur throughout the gene without evident
clustering. **There was no clear correlation between
truncating mutation location and clinical presentation.** These findings indicate a similar loss of function consequence of DSP truncating mutations through nonsense mediated RNA decay of mutant
transcripts, rather than effects from truncated proteins,
consistent with reduced DSP protein levels in skin and
myocardium from patients with DSP truncating mutations.18,30 

The specific mutation location may contribute
to the disease phenotype in the case of missense mutations (eg, mutations in the desmin versus plakophilin/
plakoglobin binding domains), but this study did not include enough missense mutation carriers to analyze this
relationship. Supporting this concept, a previous case
series of 10 patients with missense mutations versus
17 patients with truncating mutations found that the
former were less likely to have LV dysfunction.28 Also
suggesting the possibility of locus-specific effects for
missense variants is the evidence of mutation clustering in the plakophilin/plakoglobin-binding and desmin binding protein domains..

*Smith ED et al 2020 PMID: 32372669*

#### **Pilot application of harmonised terms**

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Allelic requirement:**

Monoallelic_aut

Optional modifiers:

Digenic (PKP2, DSG2)


**Inheritance:**

Autosomal dominant

Optional modifiers:

Incomplete penetrance

Digenic (PKP2, DSG2)

**Narrative summary of molecular mechanisms:**

The initial mutation description in DSP was done in Carvajal syndrome characterized by woolly hair, keratoderma and ARVC. This was followed by the description of a heterozygous mutation in DSP in an Italian family with ARVC and clear co-segregation of the variant with the disease. The disease mechanism is not clear and haploinsufficiency, dominant negative or both have all been proposed as mechanisms leading to **reduction/absence of gene product or altered gene product structure** in DSP. It is speculated that abnormalities in desmoplakin lead to desmosomal instability. Defective desmosomes cannot sustain the constant mechanical stress in contracting cardiomyocytes, resulting in cardiac dysfunction and cell death.
One study suggested that ARVC-associated DSP mutations correlate with a high arrhythmic risk and the presence of truncating mutations should alert to the likely development of LV dysfunction. Smith E et al 2020 have suggested that DSP mutations are associated with a distinct type of cardiomyopathy with a high prevalence of LV inflammation, fibrosis, and systolic dysfunction, and DSP cardiomyopathy should be considered in the differential diagnosis for myocarditis and sarcoidosis. **These authors suggest a loss of function consequence of DSP truncating mutations through nonsense mediated RNA decay of mutant transcripts,** rather than effects from truncated proteins, consistent with reduced DSP protein levels in skin and myocardium from patients with DSP. They did not find a clear correlation between truncating mutation location and clinical presentation, however the study did not include enough missense mutation carriers to analyze this for missense mutations. They suggest that as there seems to be mutation clustering in the plakophilin/plakoglobin-binding and desmin binding protein domains there could be a locus specific effect for missense variants. Although missense mutations are reported, pathogenic truncating mutations appear to be more common. One study reported that the amino-terminal region of DSP (residues 250 through 604) contained 8/17 (47%) of the DSP missense mutations found in ARVC cases compared to 1/28 (3.6%) found in controls.

There have been reports of digenic inheritance with other desomosomal mutations.

**List variant classes in this gene proven to cause this disease**

- Splice donor variant
- Splice donor variant predicted to undergo NMD
- Frameshift variant
- Frameshift variant predicted to undergo NMD
- Stop gained
- Stop gained predicted to undergo NMD
- Missense

**Potential novel variant classes based on predicted functional
consequence:**

- Start lost
- In frame insertion
- In frame deletion
- Spice acceptor variant
- Spice acceptor variant predicted to undergo NMD
- Spice acceptor variant predicted to escape NMD
- Frameshift variant predicted to escape NMD
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
# **DSP -- Arrhythmogenic Right Ventricular Cardiomyopathy**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:3052

The DSP gene was the first ARVC-gene to be associated to the disease, the initial mutation description was done in Carvajal syndrome characterized by woolly hair, keratoderma and ARVC, it is transmitted in a autosomal recessive pattern, homozygous mutations in DSP were described in in the year 2000, PMID 11063735. This was followed by the description of a heterozygous mutation in DSP in an Italian family with ARVC and clear co-segregation of the variant with the disease, PMID 12373648. These findings have been replicated worldwide in several studies performed in different ethnicities, PMID 15941723, PMID 25765472, PMID 23954618, PMID 20864495, PMID 21397041, PMID 24938629.

The initial descriptions recognized also a high frequency of left
ventricular compromise in families with DSP mutations, PMID 16061754, PMID 28527814. A murine model was generated able to replicate the arrhythmia phenotype and Cx43 mislocalization, PMID 22240500. A transgenic mouse overexpressing a mutant DSP had increased cardiomyocyte apoptosis, cardiac fibrosis and lipid accumulation PMID 16917092.
Abnormal DSP protein expression in DSP mutation carriers has also been reported, PMID 23137101. The role of this gene in this particular disease has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time (in general,at least 3 years). No convincing evidence has emerged that contradicts the role of the gene in the specified disease. In summary, based on this overwhelming evidence, DSP is definitely associated with arrhythmogenic cardiomyopathy with woolly hair and keratoderma, maximum association score was achieved rapidly after the analysis of few main reports.

**ClinGen Haploinsufficiency phenotype comments:**

The desmosomal cardiomyopathies are often associated with a complex mode of inheritance and a highly variable pattern of protein expression. AR inheritance form mutations are often ?loss of function? mutations and AD forms are known to occur. However, there is no clear evidence that they are due to haploinsufficiency; dominant negative effect appears to be
the more likely mechanism of pathogenicity in these cases. Furthermore, whole gene deletions have not been reported in the literature. At most, haploinsuffiency may result in skin manifestations (hereditary palmoplantar keratoderma) or may contribute in a multifactorial manner to susceptibility to heart disease

**Literature review:**

"DSP is the largest among the desmosome genes and encodes the 2872 amino acid protein, desmoplakin. The N-terminus of desmoplakin is required for localization to the desmosome and interaction with plakophilin and plakoglobin. The C-terminus of desmoplakin binds to the intermediate filaments (desmin)...
A DSP mutation was first identified in a homozygous manner as the cause of Carvajal disease, which shows dilated cardiomyopathy with wooly hair and keratoedema..."

*Ohno S. 2016 PMID: (27761164)*

"Here, we describe the **first recessive human mutation, 7901delG,** in the desmoplakin gene which causes a generalized striate keratoderma particularly affecting the palmoplantar epidermis, woolly hair and a dilated left ventricular cardiomyopathy. A number of the patients with this syndromic disorder suffer heart failure in their teenage years, resulting in early morbidity. **All tested affected members of three families from Ecuador were homozygous for this mutation which produces a premature stop codon leading to a truncated desmoplakin protein missing the C domain of the tail region**...In contrast to null DESMOPLAKIN: mice which die in early development, the truncated protein due to the homozygous 7901delG mutation in humans is not embryonic lethal. This suggests that the tail domain of desmoplakin is not required for establishing tissue architecture during development." 

*Norgett EE et al 2000 PMID 11063735*

Rasmussen et al. identified homozygosity for a 1-bp deletion in the DSP gene in a Turkish girl with Carvajal syndrome. Her unaffected first-cousin parents and 2 sibs were heterozygous for the mutation.

".... 4 patients with mutations in the DSP gene, 3 ARVD patients and 1 patient with Carvajal syndrome. The mutation carriers had abnormal DSP expression in both myocardial and epidermal tissue; disease mechanisms included haploinsufficiency, dominant-negative effects, or both."

*Rasmussen TB et al. 2013 Jul (PMID:23137101)*

"Desmoplakin consists of six spectrin repeats (SRs) at the
N-terminus, a central coiled-coiled rod domain, and three plakin
repeat regions at the C-terminus (Figure 3).4 The N-terminus interacts with plakophilins and plakoglobins, the rod domain forms
a dimer, and the C-terminus links desmosomes with intermediate
filaments. Alternative mRNA splicing creates two isoforms, DSP I
and DSP II, the latter of which is absent from the heart and simple epithelia.
The type and location of mutations determine the phenotype.
**Autosomal dominant Carvajal mutations occur within the SR6 region of DSP.4 Autosomal recessive Carvajal mutations occur within SR4, SR8, the variable region of the rod domain, and the C-terminal plakin repeats.4 Truncation mutations in the variable region result in loss of the C-terminus, generating an abbreviated DSP I.** Studies suggest that the resulting impaired interaction between desmoplakin and intermediate filaments manifests as the classic phenotype of Carvajal syndrome.23 DSP mutations affect cell-cell adhesion, signaling, differentiation, and morphogenesis.24 Thus, disrupting the interaction between DSP and the intermediate filament, desmin, causes failure of desmin to localize to intercalated disks and is an underlying mechanism of cardiomyopathy in Carvajal syndrome."

Cardiac involvement in Carvajal syndrome is primarily characterized by left ventricular cardiomyopathy in early childhood.20 The clinical presentation is typically heart failure rather than arrhythmia.
The major cardiac feature distinguishing Carvajal syndrome from Naxos disease is that the latter is characterized by right ventricular fibrofatty replacement of myocardial tissue with occasional left ventricular involvement. **In Carvajal syndrome, right ventricular involvement is less pronounced, and clinical presentation is typically heart failure rather than arrhythmia, which is typically seen in Naxos disease."**

*Sun Q et al 2020 PMID: 33275305*

"In a DNA sample from a girl who died at age 18 years with dilated cardiomyopathy, palmoplantar keratoderma, woolly hair, and tooth agenesis, Norgett et al. (2006) analyzed the candidate gene desmoplakin (DSP; 125647) and identified a heterozygous 30-bp insertion (125647.0015). The mutation was not present in her unaffected mother or 160 control chromosomes; no DNA was available from her deceased, similarly affected father. 

Chalabreysse et al. (2011) screened the DSP and plakoglobin (JUP; 173325) genes in the proband of a family with DCWHKTA and identified a heterozygous missense mutation in the DSP gene (S597L; 125647.0016); no mutations were found in the JUP gene. The proband's affected father was also heterozygous for the DSP missense mutation, but his affected older brother declined genetic testing. The mutation was not found in the unaffected mother, paternal grandparents, or 2 unaffected sibs. 

In a father and son with DCWHKTA, Boule et al. (2012) analyzed the desmosomal genes DSP, JUP, PKP2 (602861), DSG2 (125671), and DSC2 (125645), and identified heterozygosity for a missense mutation in the DSP gene (T564I; 125647.0017). No mutations were detected in the other genes, and the DSP mutation was not found in 600 control chromosomes. 

Boyden et al. (2016) analyzed exome data from a cohort of 496 kindreds with disorders of keratinization and identified 3 unrelated children with dilated cardiomyopathy, woolly hair, erythrokeratoderma, and tooth agenesis who were heterozygous for de novo tightly clustered missense mutations in the DSP gene: Q616P (125647.0021), H618P (125647.0022), and L622P (125647.0023)."

*Omim https://omim.org/entry/615821*

"Six AC pedigrees with 38 carriers of a dominant loss-of-function (nonsense or frameshift) mutation in DSP were evaluated by detailed clinical examination (cardiac, hair and skin) and molecular phenotyping.
All carriers with mutations affecting both major DSP isoforms (DSPI and II) were observed to have curly or wavy hair in the pedigrees examined, except for members of Family 6, where the position of the mutation only affected the cardiac-specific isoform DSPI. A mild palmoplantar keratoderma was also present
in many carriers. Sanger sequencing of cDNA from nonlesional carrier skin suggested degradation of the mutant allele. Immunohistochemistry of patient skin demonstrated mislocalization of DSP and other junctional proteins (plakoglobin, connexin 43) in the basal epidermis..."

"Our cutaneous findings are not completely unexpected. As
in Carvajal disease, the very severe, often lethal, paediatric cardiomyopathy is combined with the presence of woolly hair
and striate palmoplantar keratoderma.6 However, in those
families, the homozygous premature stop mutation is in the
third (and last) globular unit of the C-terminal tail domain of
DSP. Even though this mutation leads to a truncated protein
product and weakens the resilience of the interaction between
DSP and the keratin/desmin intermediate filaments in the
skin/heart of homozygous patients, their carrier parents
showed no sign of the cardiocutaneous disease. Interestingly,
other heterozygous cases, such as carriers of an N-terminal
in-frame duplication mutation do exhibit the Carvajal
phenotype.8 Hence, the consistent presence of this milder
cardiocutaneous phenotype in N-terminal head-domain mutations of our well-defined, relatively large, heterozygous cohort
is intriguing."

*Maruthappu T et al 2019 PMID: 30382575*

#### **Pilot application of harmonised terms**

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Allelic requirement:**

Monoallelic_aut
Biallelic_aut

Optional modifiers:

**Inheritance:**

Autosomal dominant
Autosomal recessive

Optional modifiers:

**Narrative summary of molecular mechanisms:**

Carvajal syndrome is characterized by woolly hair, keratoderma and ARVC. In 2000 three families from Ecuador were found to be homozygous for the mutation 7901delG in DSP which produces a premature stop codon leading to a truncated desmoplakin protein missing the C domain of the tail region. Since then bth AD and AR patterns of inheritance have been described. Cardiac features tend to develop in childhood and are characterised by left ventricular dilated cardiomyopathy with epicardial fibrosis, biventricular dilation. Clinical presentation is usually heart failure rather than arrhythmia. Both missense and truncating variants have been described. In a study in 2019 nearly all 38 patients with arrhythmogenic cardiomyopathy who were carriers of a dominant loss-of-function (nonsense or frameshift) mutation in DSP were found to have curly hair and palmoplantar keratoderma. The mechanism appears to be loss of function due to **reduced/absent gene product or altered gene product structure**. Sun Q et al suggested that impaired interaction between desmoplakin and intermediate filaments manifests as the classic phenotype of Carvajal syndrome.

**List variant classes in this gene proven to cause this disease**

- stop gained
- frameshift
- in frame insertion
- Missense

**Potential novel variant classes based on predicted functional
consequence:**

????

- stop_lost
- Start lost
- In frame deletion
- Splice donor
- Splice donor predicted to escape NMD
- Splice donor predicted to undergo NMD
- Spice acceptor variant
- Spice acceptor variant predicted to undergo NMD
- Spice acceptor variant predicted to escape NMD
- Frameshift variant predicted to escape NMD
- Frameshift variant predicted to undergo NMD
- stop_gained predicted to escape NMD
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
# **DSP -- Dilated Cardiomyopathy**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:3052

DSP was evaluated for autosomal dominant dilated cardiomyopathy (DCM). Human genetic evidence supporting this gene-disease relationship includes case-level data, segregation data, and case-control data. There are multiple families with DCM with truncating variants in DSP segregating in the family in the literature. Only case-level and segregation data was scored for DCM and there was no evidence suggesting the probands and/or family members met ARVC TFC. Case-control evidence shows that DSP variants are over-represented in cases versus controls (Mazzarotto et al, 2020, PMID 31983221). Additional human genetic evidence is available in the literature that was not included in the curation scoring as the maximum score for genetic evidence was reached. In addition, this gene-disease association is supported by experimental evidence, including expression data and animal models. DSP has been shown to be expressed in the heart, and also one of the intercalated disc components highly related to LV function, providing evidence of protein interaction (Kazerounian et al, 2002, PMID: 12366696; Ortega et al, 2017, PMID: 28934278). DSP KO mice exhibit a biventricular cardiomyopathy, although it is particularly arrhythmic with sudden death (Lyon et al, 2014, PMID: 24108106). In summary, there is strong evidence to support the relationship between DSP and autosomal dominant DCM. Of note, DSP has also been curated by the ARVC Gene Curation Expert Panel for arrhythmogenic cardiomyopathy with wooly hair and keratoderma (Strong, July 12, 2019), as there are also several families in which individuals meet arrhythmogenic right ventricular cardiomyopathy (ARVC) task force criteria (TFC). Experts in the DCM GCEP encourage clinicians to carefully evaluate for a possibility of an ACM phenotype and treat patients and families with DSP variants accordingly. This classification was approved by the ClinGen Dilated Cardiomyopathy Working Group on June 12, 2020.
Gene Clinical Validity Standard Operating Procedures (SOP) - Version 7

**Literature review:**

Protein-truncating variants in constitutive exons (Percent Spliced In>90%) of TTN and in DSP were significantly enriched in patients with DCM compared with healthy controls (TTN 11.3% DCM vs 0.4% healthy controls, P=6.2×10−27; DSP 1.4% vs 0.0%, P=4.2×10−3). 

*Mazzarotto F et al 2020 PMID: 31983221*

**From in house atlas of variants in DSP**

12/32 truncating variants

Nonsense

Frameshift

Splice donor variant

20/32 non-truncating variants (mainly VUS)

Missense (all VUS)

in frame insertion (likely pathogenic) 

https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=DSP&icc=DCM

*Walsh et al, 2016 (PMID 27532257)*
*Pugh TJ et al, 2014 (PMID: 24503780)*

Mutations in the DSP gene encoding desmoplakin were first identified in an autosomal recessive form of arrhythmogenic cardiomyopathy. The present study makes a case for DSP cardiomyopathy being a distinct form of cardiomyopathy. DSP cardiomyopathy results in episodic inflammation, which precedes the development of fibrosis. Frequent PVCs occurring before LV systolic dysfunction or LV enlargement is also a key distinction from most cases of DCM. In DSP patients, late gadolinium enhancement was found in the LV subepicardium rather than mid-myocardial as is often seen with nonischemic cardiomyopathy. The correct diagnosis must include the identification of the DSP mutation. This paper is likely to be very impactful, as it suggests that some of the patients suspected of myocarditis and cardiac sarcoidosis may actually have DSP cardiomyopathy.

Mutations in this study were primarily truncating mutations (ie, frameshift, nonsense, or splice site
mutations). Truncating mutations in this and other
studies occur throughout the gene without evident
clustering. There was no clear correlation between
truncating mutation location and clinical presentation. These findings indicate a similar loss of function consequence of DSP truncating mutations through nonsense mediated RNA decay of mutant
transcripts, rather than effects from truncated proteins,
consistent with reduced DSP protein levels in skin and
myocardium from patients with DSP truncating mutations.18,30 

The specific mutation location may contribute
to the disease phenotype in the case of missense mutations (eg, mutations in the desmin versus plakophilin/
plakoglobin binding domains), but this study did not include enough missense mutation carriers to analyze this
relationship. Supporting this concept, a previous case
series of 10 patients with missense mutations versus
17 patients with truncating mutations found that the
former were less likely to have LV dysfunction.28 Also
suggesting the possibility of locus-specific effects for
missense variants is the evidence of mutation clustering in the plakophilin/plakoglobin-binding and desmin binding protein domains..

*Smith ED et al 2020 PMID: 32372669*


#### **Pilot application of harmonised terms**

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Allelic requirement:**

Monoallelic_aut

Optional modifiers:

**Inheritance:**

Autosomal dominant

Optional modifiers:

Incomplete penetrance

**Narrative summary of molecular mechanisms:**

The disease mechanism is not clear and haploinsufficiency, dominant negative or both have all been proposed as mechanisms leading to **reduction/absence of gene product or altered gene product structure** in DSP. In DCM patients, protein truncating variants (including nonsense, frameshift, essential splice) in DSP are enriched when compared to controls. Non-truncating variants were not enriched. Truncating mutations appear to occur throughout the gene whereas missense mutations are more likely to cluster in either the plakophilin/plakoglobin-binding or desmin binding protein domains. Smith E et al have suggested that DSP mutations are associated with a distinct type of cardiomyopathy with a high prevalence of LV inflammation, fibrosis, and systolic dysfunction, and DSP cardiomyopathy should be considered in the differential diagnosis for myocarditis and sarcoidosis. These authors suggest a loss of function consequence of DSP truncating mutations through nonsense mediated RNA decay of mutant transcripts, rather than effects from truncated proteins, consistent with reduced DSP protein levels in skin and myocardium from patients with DSP truncating mutations.

**List variant classes in this gene proven to cause this disease**

- Splice donor variant
- Splice donor variant predicted to undergo NMD
- Frameshift variant
- Frameshift variant predicted to undergo NMD
- Stop gained
- Stop gained predicted to undergo NMD
- Missense

**Potential novel variant classes based on predicted functional
consequence:**

- Start lost
- In frame insertion
- In frame deletion
- Spice acceptor variant
- Spice acceptor variant predicted to undergo NMD
- Spice acceptor variant predicted to escape NMD
- Frameshift variant predicted to escape NMD
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
**[FBN1 - Marfan Syndrome]{.ul}**

**Review of source material:**

**ClinGen:**

https://search.clinicalgenome.org/kb/genes/HGNC:3603

"The first report indicating a relationship between the FBN1 gene and autosomal dominant Marfan syndrome was reported by Dietz et al in 1991 (PMID: 1852208). 

The nomenclature for Marfan syndrome is ascribed to Antoine Marfan, who described a individual with part of the phenotypic features associated with the greater syndrome in 1896 (reviewed in Gott 1998, PMID:9798380). Marfan syndrome a multisystemic disorder affecting mainly the connective tissue. Marfan comprises a broad phenotypic spectrum and severity and thus represents a continuum of disease (reviewed in Dietz 1991, Marfan GeneReviews, PMID: 20301510). Cardiovascular, ocular, and skeletal phenotypes represent the most common phenotypic manifestations among individuals diagnosed with Marfan syndrome. The cardiovascular
phenotypes, including dilatation of the aorta, are the areas that
represent the major morbidity and mortality of Marfan syndrome. 

**Over 1800 variants have been identified in FBN1 (Collod-Beroud et al.,2003 PMID: 12938084), including missense, nonsense, frameshift, splice site, small deletions, and large deletions, according to two databases that house variant information: (1) The UMD-FBN1 database (http://www.umd.be/FBN1/); and (2) the LOVD FBN1 database (https://databases.lovd.nl/shared/genes/FBN1).**

The mechanism for the gene-disease relationship is **protein loss of function,** as mutation in the FBN1 protein, fibrillin, results in the inability of the protein to be excreted from cells to help in the formation and stabilization of connective tissue (Sakai et al., 1986, PMID: 3536967). 

Evidence supporting this gene-disease relationship includes case-level data, segregation data, functional data, and model organisms. This gene-disease relationship has been studied for more than 20 years, therefore a significant amount of case-level data, segregation data, and experimental data is available and the maximum score for genetic evidence (12 points) and experimental evidence (6 points) has been reached. Note, this curation effort may not be exhaustive of all literature related to this gene-disease relationship. In particular, earlier compelling evidence suggestive of the gene-disease relationship, such as linkage data, may not be reflected in the current curation. 

In summary, FBN1 is definitively associated with autosomal dominant Marfan syndrome. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This classification was reviewed by the FBN1 VCEP with final approval by the ClinGen General GCEP on March 4, 2019 (SOP Version 6)."

ClinGen Haploinsufficiency Comments:

**In summary, Loss of function variants (exonic deletions and whole gene deletions) of the FBN1 gene have been described in individuals with Marfan phenotype.** Additional PMIDs: 21936929, 30479897, 17492313, 28842177

**[Literature review:]{.ul}**

"When information about transmission type (available for 398 mutations) is examined, there are a surprising number of de novo mutations as compared to transmitted mutations (188 de novo vs. 210 familial).

The global molecular analysis of the FBN1 mutations reveals two classes of mutations. The first, which represents more than one-third of the mutations (38.6%), corresponds to mutations predicted to result in shortened fibrillin-1 molecules; 61 nonsense mutations, 71 splicing errors, 23 insertions and duplications, 51 deletions, and 10 inframe deletions are in this class. They act as dominant negative but display a highly variable clinical phenotype, the severity of which is directly related to the quantitative expression of the mutant allele and to the
percentage of truncated or shortened proteins incorporated in the
microfibrils. The second class represents less than two-thirds (60.3%) of the mutations and corresponds to missense mutations, most among them are located in cbEGF-like modules (78%). They can be subclassified into:
1) mutations creating or substituting cysteine residues potentially implicated in disulfide bonding and consequently in the correct folding of the monomer; and 2) amino acids implicated in calcium binding and subsequently in interdomain linkage, rigidification of monomer, and in protease susceptibility; see www.hgvs.org/mutamen for updated recommendations.)"

*Collod-Beroud G et al 2003 PMID: 12938084*

**"Most of the mutations that cause Marfan syndrome change a single amino acid in the fibrillin-1 protein. The remaining FBN1 gene mutations result in an abnormal fibrillin-1 protein that cannot function properly.
FBN1 gene mutations that cause Marfan syndrome reduce the amount of fibrillin-1 produced by the cell, alter the structure or stability of fibrillin-1, or impair the transport of fibrillin-1 out of the cell.**

These mutations lead to a severe reduction in the amount of fibrillin-1 available to form microfibrils. Without enough microfibrils, excess TGF-β growth factors are activated and elasticity in many tissues is decreased, leading to overgrowth and instability of tissues and the signs and symptoms of Marfan syndrome.)"

*Genetics Home reference:
[https://ghr.nlm.nih.gov/gene/FBN1\#conditions]{.ul} *

"\...These data are consistent with a model that invokes
haploinsufficiency for WT fibrillin-1, rather than production of mutant
protein, as the primary determinant of failed microfibrillar assembly.)"

*Judge DP et al. 2004 PMID:15254584*

"In case of FBN1, **about 1,850 different mutations
that are widely spread all over the protein have been described so
far (http://www.umd.be/fbn1/), of which the majority is unique
to each family. The mutational spectrum encompasses nonsense,
frameshift, splice altering, insertion/deletion, and missense variations as well as whole-gene or multiexon deletions.** 

**It has been a longstanding matter of debate, however, whether these mutations cause marfanoid syndromes through haploinsufficiency, a dominant-negative mechanism or a combination of both** (Dietz et al., 1993; Judge et al., 2004; Matyas et al., 2007). Lately, particular the latter hypothesis is gaining momentum in the MFS field (Colovati et al., 2012; Franken et al., 2015). 
Despite significant progress in the understanding of the molecular defects underlying MFS, a limited number of convincing genotype–phenotype correlations has emerged. 

**Cysteine-destroying or cysteine-creating mutations
are commonly associated with ectopia lentis (Schrijver et al., 1999). Haploinsufficient mutations, on the other hand, more frequently correlate with aortic events at young ages as well as pectus carinatum, dural ectasia, and skin striae (Baudhuin et al., 2015; Franken et al., 2015). Additionally, the vast majority of the mutations in exons 24–32 are linked to either neonatal MFS or other severe marfanoid presentations (Tiecke et al., 2001).** 

Although MFS mostly inherits in an autosomal-dominant manner, rare cases with recessive homozygous or compound heterozygous FBN1 mutations have been described (Hogue et al., 2013). They exhibit a more severe phenotype compared with their heterozygous relatives, who are usually asymptomatic or only mildly affected. Taken together, the nature of the implicated FBN1 defect can, at least to some extent, explain phenotypic variability. Yet, also related individuals carrying an identical FBN1 mutation vary widely with respect to onset age, organ-system involvement, and disease severity."

*Verstraeten A et al. 2016 Jun (PMID:26919284)*

**[Pilot application of harmonised terms:]{.ul}**

**Disease associated variant consequences:**

Dose change: dose reduction: Decreased gene product level

Dose change: dose reduction: Absent gene product

Altered gene product structure

**Allelic requirement:**

Monoallelic_aut

**Inheritance:**

Autosomal dominant

Optional modifiers: 

**Narrative summary of molecular mechanisms:**

The mutational spectrum encompasses nonsense, frameshift, splice altering, insertion/deletion, and missense variations as well as whole-gene or multiexon deletions. Most of the mutations that cause Marfan syndrome (MFS) are missense mutations. The mechanism for the gene-disease relationship is loss of function **primarily due to altered gene product structure** which ultimately leads to a reduction in the amount of protein at it's target location due to increased degradation, impaired trafficking etc but **also due to variants that reduce the gene product level**. There has been considerable debate about whether Marfan syndrome is caused through haploinsufficiency, a dominant-negative mechanism or a combination of both.
MFS is mostly inherited in an autosomal-dominant manner, however rare cases with recessive homozygous or compound heterozygous FBN1 mutations have been described (Hogue et al., 2013). They exhibit a more severe phenotype compared with their heterozygous relatives.
According to a review by Verstraeten A et al in 2016, "a limited number of convincing genotype–phenotype correlations has emerged. 
**Cysteine-destroying or cysteine-creating mutations
are commonly associated with ectopia lentis (Schrijver et al., 1999). Haploinsufficient mutations, on the other hand, more frequently correlate with aortic events at young ages as well as pectus carinatum, dural ectasia, and skin striae (Baudhuin et al., 2015; Franken et al., 2015). Additionally, the vast majority of the mutations in exons 24–32 are linked to either neonatal MFS or other severe marfanoid presentations (Tiecke et al., 2001).** "
 
**Additional information related to ACMG evidence types**

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
>1%
estimated prevalence between 1.5 to 17.2 per 100 000 individuals

**BS1** (MAF too high for disease)
an allele frequency above 0.0002 (1:5000) was considered as a strong criterion for benignity

case–control studies were only considered to use as a strong criterion **(PS4)** if >1000 controls were tested.

**PM1** located in a mutational hotspot and critical established functional domain Cys substitution outside cb-EGF domain Introduction of a new Cys within cb-EGF domain Cb-site substitution in cb-EGF 

**PS3** well-established in vitro or in vivo studies show damaging effect Cys substitution within cb-EGF domain

**PP4**
The combination of ARD or dissection and EL seems to be more specific for MFS,17and we, therefore, only applied the PP4 criterion when a patient presented these 2 features combined.

**PVS1**
Because loss of expression or haploinsufficiency is known as a disease-causing mechanism in the FBN1 gene, the PSV1 criterion was used for all frameshift and nonsense variants not affecting the final exon 65 and for all splice-site variants in positions ±1 to 2. Frameshift and nonsense variants affecting exon 65 usually produce a protein which is shorter2 and therefore we used for these cases the PM4 criterion.

More info including additional criteria
*Muino-Mosquera L et al 2018 PMID: 29875124*

**List variant classes in this gene proven to cause this disease**

Splice region variant

Spice acceptor variant

Splice donor variant

Start lost

Frameshift variant

Stop gained

Stop gained predicted to undergo NMD

Missense

In frame insertion

In frame deletion

**Potential novel variant classes based on predicted functional
consequence**

stop_gained predicted to escape NMD

Splice acceptor predicted to undergo NMD

Splice acceptor predicted to escape NMD

Splice donor predicted to undergo NMD

Splice donor predicted to escape NMD

stop_lost

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[oORF\]

Frameshift \[oORF\]
**GLA -- Fabry Disease (MIM 301500)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:4296

The relationship between GLA and Fabry disease, an X-linked lysosomal storage disorder, was evaluated using the ClinGen Clinical Validity Framework as of March 3rd, 2017. Deficiency of the gene product, alpha-galactosidase A, was first reported in males with Fabry disease in 1967 (Brady et al; PMID 6023233), and variants in GLA were first associated with this disease in 1989 (Bernstein et al, PMID 2539398). Over 400 unique variants, including **missense, nonsense, splice site, frameshift, in-frame deletions, and complex rearrangements,** have been reported in humans (reviewed in Gal et al, 2006, PMID 21290673; Mehta,
2017, PMID 20301469). Evidence supporting this gene-disease relationship includes case-level data and experimental data. 

Twenty-five variants were curated in 25 probands from 3 publications (Bernstein et al, 1989, PMID 2539398; Topaloglu et al, 1989, PMID 10666480; Shimotori et al, 2008, PMID 18205205). More evidence is available in the literature, but the maximum score for genetic evidence (12 points) has been reached. The
mechanism for disease is **loss of function**. This gene-disease
association is supported by the function of the gene product (Brady et al, 1967, PMID 6020428) animal models and rescue (Oshima et al, 1997, PMID 9122231; Taguchi et al, 2013, PMID 24094090), and studies of the clinical impact of enzyme replacement therapy in humans (Beck et al, 2015, PMID 26937390). In summary, GLA is definitively associated with Fabry disease. This association has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over
time. \[CLINGEN GENE VALIDITY CURATION\]

**Literature review:**

*Sequence analysis of GLA followed by MLPA*

*Targeted analysis p.Ala143Pro in individuals from Nova Scotia
(incidence 1:15,000).*

*Targeted analysis IVS4+919G\>A in individuals of Chinese ancestry with atypical presentation \[Liu et al\].*

More than 800 GLA pathogenic variants have been identified, and most are family specific, occurring only in single pedigrees.
Affected males with **frameshift and nonsense variants** typically
present with classic Fabry disease; males with **missense pathogenic variants can present with either classic or atypical phenotypes** \[Pan et al 2016\].

GLA pathogenic variants that result in residual α-gal A activity of about 20% have been identified in individuals with atypical variants of Fabry disease. The clinical manifestations of atypical cases are not specific to Fabry disease (e.g., stroke, cardiomyopathy); therefore, the pathogenicity of some variants is unclear \[Lukas et al 2016\]. A number of variants including p.Ile91Thr, p.Arg112His, p.Phe113Leu, p.Asn215Ser, p.Met296Ile, p.Arg301Gln, and p.Gly328Arg are recurrent and associated
with late-onset cardiac disease \[Patel et al 2015\]. Most individuals with the **IVS4+919G\>A pathogenic variant** were not diagnosed until newborn screening identified this variant in their grandsons \[Liu et al 2015\].

The pathogenicity of some GLA variants is disputed. The p.Arg118Cys variant has been recurrently described in large Fabry disease screening studies of high-risk individuals; however, this variant does not always segregate with Fabry disease in a Mendelian fashion, and could be a modulator of cerebrovascular disease risk \[Ferreira et al 2015\]. The p.Ala143Thr variant has been associated with renal failure, stroke, and left ventricular hypertrophy which could potentially be the result of selection bias, as most individuals were detected in screening studies
\[Terryn et al 2013\].*

*Abnormal gene product. GLA pathogenic variants result in **mRNA
instability and/or severely truncated α-Gal A or an enzyme with markedly decreased activity**.*

Plasma globotriaosylsphingosine (lyso-Gb3) (the lyso derivative of the accumulated substrate) levels appear to correlate with disease severity and to decline with enzyme replacement therapy [Aerts et al 2008].
Urinary levels of lyso-Gb3 derivatives also correlate with disease severity [Auray-Blais et al 2015].
Plasma lyso-Gb3 levels are higher in affected males than females.

Identification of elevated plasma and urinary lyso-Gb3 can confirm the diagnosis in an individual with a GLA variant of uncertain significance identified by molecular genetic testing or late-onset disease manifestations. Niemann et al [2014] reported that individuals with a novel variant and organ involvement consistent with Fabry disease had lyso-Gb3 levels ≥2.7ng/mL; Individuals with a novel GLA variant and no organ involvement had lyso-Gb3 levels <2.7 ng/mL.

**GENEREVIEWS:**
https://www.ncbi.nlm.nih.gov/books/NBK1292/pdf/Bookshelf_NBK1292.pdf

**OMIM:** https://www.omim.org/entry/301500

"The genetic cause of Fabry's disease is well known. The GLA gene has been sequenced and hundreds of mutations identified. Point mutations (mis-sense or 20 non-sense mutations) are the most frequent, but small and large deletions or insertions are also seen.

**Mutations leading to complete loss of function of the gene product are associated with classic forms of the disease, whereas mutations resulting in amino acid substitutions might occasionally be associated with a mild phenotype and late variants.**
Attempts to correlate genotype with clinical presentation have been largely unsuccessful.

For women, **X inactivation probably has a role**, but that it alone explains all the clinical variability is unlikely. Symptomatic heterozygous women with Fabry's disease have skewed X inactivation in some cases, but not consistently"

*Zarate YA and Hopkin RJ, 2008 PMID: **18940466***

This study -paediatric group consisted of 15 male hemizygotes and 17 female heterozygotes with different types of mutations: **20 had missense mutations, 5 had nonsense (stop) mutations, 4 had splice-site defects, and 3 had frameshift** mutations (1 deletion, 2 duplications).
The adult group consisted of 36 male hemizygotes, 42 were female heterozygotes with also different types of mutations: 64 had missense mutations and 13 had nonsense mutations, 1 had a frameshift mutation, and no splice-site defects mutation were observed.

The GLA gene is approximately 12 kb and comprised of seven exons
(varying in size from 92 to 291 base pairs) each carrying a wide variety of molecular lesions \[11\]. The processed message is 1.45 kb long and encodes a 50-kDa precursor polypeptide of 429 amino acids \[3\]. Exon 1 contains the entire 5′ untranslated region, the sequence encoding the signal peptide, and the first 33 residues of the mature enzyme subunit \[12\]. Point mutations in the α-Gal protein may be sufficient to produce either a classic or severe Fabry phenotype, where no α-galactosidase activity is detectable, or a milder phenotype showing residual enzyme activity \[13\]. Different substitutions of the same codon may result in markedly different disease phenotypes \[14\], \[15\], \[16\], \[17\], \[18\], \[19\], \[20\]. Mutant alleles involving
nonsense codons or frameshifts, causing premature termination of
transcription, are usually associated with classic Fabry disease. The relationship between the genotype and phenotype in Fabry disease has been studied \[8\], \[21\]. Branton et al. \[21\] studied categories of missense mutations and found that patients with conservative single amino acid change have a significantly delayed onset of chronic insufficiency compared to those with non-conservative single amino acid change.

*Auray Blaise et al. 2008. PMID: **18023222***

**Pilot application of harmonised terms:**

**Inheritance:**

*X-linked* - Primarily recessive (with milder female expression)

**Allelic requirement:**

Monoallelic_X_hem    

**Disease associated variant consequences:**

*Dose change* -decreased gene product level

*Dose change* - absent gene product

*Altered gene product structure*

**Narrative summary of molecular mechanisms:**

Mechanism is loss of function of GLA due to reduction/absence of gene product or altered gene product structure. Variant classes include missense, nonsense, splice site, frameshift, in-frame deletions, and complex rearrangements. A recurrent intronic variant (c.640-801G\>A) is recognised as pathogenic and leads to aberrant mRNA splicing. GLA pathogenic variants result in mRNA instability and/or severely truncated α-Gal A or an enzyme with markedly decreased activity.
Many variants are unique however there are recognised recurrent variants also.

Mutations leading to complete loss of function of the gene product are usually associated with classic forms of the disease, whereas mutations resulting in amino acid substitutions and residual enzyme activity can present atypically with either symptoms not specific to Fabry's (e.g. cardiomyopathy) or a milder phenotype and later onset. Attempts to correlate genotype with clinical presentation have been largely unsuccessful.

Fabry disease is inherited in an X linked manner. Heterozygous females typically have milder symptoms at a later age of onset than males. Rarely, they may be relatively asymptomatic throughout a normal life span or may have symptoms as severe as those observed in males with the classic phenotype.

Identification of elevated plasma and urinary lyso-Gb3 can confirm the diagnosis in an individual with a GLA variant of uncertain significance identified by molecular genetic testing or late-onset disease manifestations.

**List variant classes in this gene proven to cause this disease:**

Missense

Frameshift variant

Frameshift variant predicted to undergo NMD

Inframe deletions

Inframe insertion

Stop gain

Stop gained predicted to undergo NMD

Spice acceptor variant

Splice donor variant

Splice acceptor variant predicted to undergo NMD

Splice donor variant predicted to undergo NMD

Intronic (?*IVS4+919G\>A aka c.640-801G\>A)* (Cryptic splice site)

Structural variants

**List other variant classes predicted to lead to the same functional consequence**

Splice acceptor variant predicted to escape NMD

Splice donor variant predicted to escape NMD

Frameshift variant predicted to escape NMD

start_lost

stop_gained predicted to escape NMD

stop_lost

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[oORF\]

Frameshift \[oORF\]
# **KCNH2 -- Long QT Syndrome**

**Review of source material:**
*Adler A et al 2020 (PMID: 31983240)*

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:6251

**ClinGen Evidence for Haploinsufficiency:**
There is a significant amount of evidence suggesting that loss of KCNH2 results in Long QT syndrome (LQTS). In addition to the evidence above, Itoh et al 2016 (PMID: 26669661) report 280 KCNH2 variants in families with LQTS; 95/280 were nonsense, frameshift or splice site variants (>100 families with multiple affected individuals) (Supplementary Table 2). Stattin et al 2012 (PMID: 23098067 ) reports 3 additional affected probands with truncating variants. HGMD also catalogs a number of putative loss of function variants, including: 70 disease causing nonsense variants,17 disease causing splice site variants (+/-2), 6 large deletions (multiple exons), and 141 disease causing frame shift variants.

Of note, large deletions involving KCNH2 in addition to other genes have also been observed in individuals diagnosed with Long QT syndrome. Barc et al (2011) (PMID: 21185499) studied 93 patients with LQTS who had experienced symptoms such as syncope, arrhythmia, and cardiac arrest. Two probands were found with deletion in KCNH2. The proband of Family 1 was 23 years old with a 650 kb deletion of exons 4-14 (+19 other genes centromeric to KCNH2). Six other carrier family members had LQTS but were asymptomatic. The proband of Family 2 was 28 years old with a 145 kb deletion of the entire KCNH2 and ABP1 genes. The deletion was also found in her affected mother and "healthy" brother.

**Literature review:**

"Class 1 mutations disrupt the synthesis/translation of Kv11.1 α-subunits (decrease n); Class 2 mutations reduce the intracellular transport or trafficking of Kv11.1 proteins to the cell membrane (decrease n); Class 3 mutations disrupt Kv11.1 channel gating (decrease Po); and Class 4 mutations negatively affect K+ permeation (decrease i).

**About 40% of LQT2-linked KCNH2 mutations are nonsense mutations, frameshift mutations, insertions, deletions, duplications, or involve a splice site that inhibits Kv11.1 protein synthesis/translation by generating incomplete proteins or causing nonsense-mediated RNA decay (NMD) (class 1 mechanism)** [18], [19], [20]. By provoking NMD, class 1 mutations are expected to cause haploinsufficiency. **The remaining ~60% of LQT2 mutations are missense,** where a single nucleotide change alters an amino acid codon to a different amino acid to cause a LOF by disrupting channel trafficking to the cell membrane (class 2 mechanism), gating (class 3 mechanism), and/or single channel current (class 4 mechanism) [16], [20], [21], [22], [23], [24], [25], [26], [27]. Over 150 suspected LQT2-causing missense mutations have been studied using heterologous expression systems and these studies demonstrate that ~90% of LQT2-linked missense mutations disrupt Kv11.1 channel function via a class 2 mechanism (Fig. 3) [20], [21], [22], [23], [24], [25], [26], [27], [28]. Class 2 LQT2 mutations decrease the folding efficiency of Kv11.1 proteins and increase their retention in the endoplasmic reticulum (ER) by cellular quality control mechanisms."

*Smith J et al.2016 (PMID: 27761161)*

"KCNH2 spans approximately 19 kb. The longest isoform consists of 16 exons and produces a protein of 1,159 amino acids (NM_000238.3). Two shorter isoforms of KCNH2 exist.

More than 700 pathogenic variants have been reported, including pathogenic missense, nonsense, splice site, and frameshift variants as well as large multiexon deletions...

Long QT syndrome (LQTS) associated with biallelic pathogenic variants or heterozygosity for pathogenic variants in two different genes (i.e., digenic pathogenic variants) is generally associated with a more severe phenotype with longer QTc interval and a higher incidence of cardiac events [Schwartz et al 2003, Westenskow et al 2004, Tester et al 2005, Itoh et al 2010]."

*Gene reviews LQT
https://www.ncbi.nlm.nih.gov/books/NBK1129/*

Compendium of Cardiac channel mutations: 29 (10% of genotype positive patients) had 2 mutations in either KCNQ1, KCNH2, SCN5A, KCNE1, KCNE2 9 with multiple KCNQ1 variants, 7 with a KCNQ1 and a KCNH2 variant, 5 with a KCNQ1 and a SCN5A variant, 2 with two KCNH2 variants, 4 with a KCNH2 and a SCN5A variant, and 2 with two SCN5A variants.

*Tester D et al. 2005. PMID: 15840476*

"...an ex6-14del of the KCNH2 gene [was found] in a 22-year-old woman misdiagnosed with epilepsy since age 9 years (QTc 560 ms) and a sibling with sudden death at age 13 years; and (3) an ex9-14dup of the KCNH2 gene [was found] in a 12 year-old boy (QTc 550 ms) following sudden nocturnal death of his 32-year-old mother."

*Eddy C-A et al 2008 PMID: 18774102*

"The major findings of the present study from 858 type-2 LQTS subjects with genetically confirmed KCNH2 mutations derived from 4 LQTS Registries are that 1) **there is a significant mutation type-location interaction;** specifically that the relative risk between C-terminus and the regions is different for missense versus non-missense locations, 2) patients with missense mutations in the transmembrane pore region have significantly higher cardiac event rates than those with missense mutations in either N-terminus, transmembrane non-pore or C-terminus regions, 3) patients with non-missense mutations were at significantly higher risk than those with missense mutations in the C-terminus region, 4) patients with mutations located in putative α-helical domains have significantly higher cardiac event rates than in those with mutations in either the β-sheet domains or other uncategorized locations, and these higher event rates are independent of traditional clinical risk factors and of β-blocker therapy..." 

*Shimizu W et al 2009 PMID: 19926013*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 
Incomplete penetrance;
Digenic (other LQT genes)

**Allelic requirement:**

Monoallelic_aut

(optional) modifiers 
Digenic (other LQT genes)

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is likely loss of function of KCNH2 due to **reduction/absence of gene product or altered gene product** due to a variety of mechanisms including disruption of synthesis of channel subunits, reduction in intracellular transport or trafficking, defects in ion permeation or channel gating. Both haploinsufficiency and a dominant negative effect are proposed mechanisms causing loss of function of KCNH2. About 60% of LQT2 mutations are missense variants, the remaining 40% are nonsense mutations, frameshift mutations, insertions, deletions, duplications, or involve a splice site. There has been a report of specific mutation type-location interaction. Patients with missense mutations in the transmembrane pore region have significantly higher cardiac event rates than those with missense mutations in either N-terminus, transmembrane non-pore or C-terminus regions. Digenic inheritance has been described with mutations in other LQT genes. Biallelic pathogenic variants or digenic pathogenic variants are generally associated with a more severe phenotype with longer QTc interval and a higher incidence of cardiac events. 

**List variant classes in this gene proven to cause this disease:**

- Spice acceptor variant
- Splice donor variant
- Frameshift variant
- Stop gained
- Stop gained predicted to undergo NMD
- Missense
- In frame insertion
- In frame deletion
- structural variants

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant predicted to escape NMD
- Splice acceptor variant predicted to undergo NMD
- Splice donor variant predicted to escape NMD
- Splice donor variant predicted to undergo NMD
- Frameshift predicted to escape NMD
- Frameshift variant predicted to undergo NMD
- Stop_gained predicted to escape NMD
- Stop_lost
- Start_lost
- gain of upstream Start [uORF]
- gain of upstream Start [oORF]
- Stop lost [oORF]
- Frameshift [oORF]
# **KCNQ1 -- Long QT Syndrome**

**Review of source material:**

*Adler A et al 2020 (PMID: 31983240)*

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:6294

**ClinGen Evidence for Haploinsufficiency**

PMID 18774102 Barc (2011): Describes a single 2 exon deletion in a patient with long QT syndrome type 1.
PMID 18774102 Eddy (2008): Describes a single 2 exon deletion resulting in a frameshift and premature stop codon in a patient with long QT syndrome type 1.
PMID 15840476 Tester (2005): Describes 88 patients with long QT syndrome and various mutations. Fourteen of these are either nonsense or frameshift mutations.

Heterozygous loss of function mutations can cause long QT syndrome type 1. Homozygous mutations result in the recessive condition Jervell and Lange-Nielsen syndrome.

**Literature review:**

"More than 500 pathogenic variants of KCNQ1 have been reported, including pathogenic missense, nonsense, splice site, and frameshift variants as well as large multiexon deletions.

**Normal gene product.** The potassium voltage-gated channel subfamily KQT member 1 is the alpha subunit forming the slowly activating potassium delayed rectifier IKs [Keating & Sanguinetti 2001].

**Abnormal gene product.** IKs channel with reduced function"

Long QT syndrome (LQTS) associated with biallelic pathogenic variants or heterozygosity for pathogenic variants in two different genes (i.e., digenic pathogenic variants) is generally associated with a more severe phenotype with longer QTc interval and a higher incidence of cardiac events [Schwartz et al 2003, Westenskow et al 2004, Tester et al 2005, Itoh et al 2010].

*Gene reviews LQT
https://www.ncbi.nlm.nih.gov/books/NBK1129/*

Compendium of Cardiac channel mutations: 29 (10% of genotype positive patients) had 2 mutations in either KCNQ1, KCNH2, SCN5A, KCNE1, KCNE2, 9 with multiple KCNQ1 variants, 7 with a KCNQ1 and a KCNH2 variant, 5 with a KCNQ1 and a SCN5A variant, 2 with two KCNH2 variants, 4 with a KCNH2 and a SCN5A variant, and 2 with two SCN5A variants.

*Tester D et al. 2005 PMID: 15840476*

**Missense mutations are responsible for the majority of LQT1 cases** and can cause channel loss of function through a variety of molecular mechanisms, including defects in ion permeation (altering the pathway through which ions flow through open channels), channel gating (mechanisms that regulate the opening and/or closing or channels), trafficking, KCNQ1-KCNE1 interaction, PKA-mediated signaling pathway, PIP2 binding, and calmodulin binding (Figure 4, C AND D) (Table 2). Non-missense mutations can also cause LQT1. Mutations belonging to certain groups may bear implications on patient phenotype, severity of arrhythmia, as well as response to therapy.

*Bohnen M et al. 2017 PMID: 27807201*

"Two distinct biophysical mechanisms mediate the reduced IKs current in patients with KCNQ1 mutations: (1) coassembly or trafficking defects in which mutant subunits are not transported properly to the cell membrane and fail to incorporate into the tetrameric channel, with the net effect being a ≤50% reduction in channel function (haploinsufficiency)5; and (2) formation of defective channels involving mutant subunits with the altered channel protein transported to the cell membrane, resulting in a dysfunctional channel having >50% reduction in channel current (dominant-negative effect)."

*Moss et al. 2007 (PMID: 17470695)*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

Incomplete penetrance 
Digenic (KCNH2, SCN5A, KCNE1)

**Allelic requirement:**

Monoallelic_aut

(optional) modifiers:
Digenic (KCNQ1, KCNH2, SCN5A, KCNE1)

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level
Dose Change: dose reduction: Absent gene product
Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is likely loss of function of KCNQ1 due to **reduction/absence of gene product or altered gene product** due to a variety of mechanisms including defects in ion permeation, channel gating, trafficking, KCNQ1-KCNE1 interaction, PKA-mediated signaling pathway, PIP2 binding, and calmodulin binding. Missense mutations are responsible for the majority of LQT1 cases. Digenic inheritance has been described with mutations in other LQT genes.

**List variant classes in this gene proven to cause this disease:**

- Spice acceptor variant
- Splice donor variant
- Frameshift variant
- Stop gained
- Stop gained predicted to undergo NMD
- Missense
- In frame insertion
- In frame deletion

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant predicted to escape NMD
- Splice donor variant predicted to escape NMD
- Frameshift predicted to escape NMD
- Stop_gained predicted to escape NMD
- Stop_lost
- Start_lost
- gain of upstream Start [uORF]
- gain of upstream Start [oORF]
- Stop lost [oORF]
- Frameshift [oORF]
# **LDLR -- Familial Hypercholesterolemia**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:6547

**ClinGen evidence for haploinsufficiency**
https://dosage.clinicalgenome.org/clingen_gene.cgi?sym=ldlr&subject

Five different classes of LDLR mutations have been identified, dependent on the effect on the phenotype. Class 1 mutations are null mutations that result in no detectable LDLR protein. In class 2 mutations, the transport of the LDLR from the endoplasmic reticulum to the Golgi apparatus is blocked completely (class 2a) or partially (class 2b). A class 3 mutation leads to expression of a non-functional LDLR. Class 4 mutations result in LDL binding but the LDLR-LDL complexes cannot be internalized, and in class 5 mutations, recycling of the LDLR is not efficient and therefore do not reach the cell surface (PMID: 26482752). LDLR is subject to Alu-mediated partial gene duplications. Multiple (~66) gross insertions and duplications (some publications: PMID: 22698793 (ex. 2-15), PMID: 23669246 (ex. 2-8), PMID: 20145306 (ex. 3-12), PMID: 11313767 (ex. 8-10), PMID: 23415438 (ex. 7-12), and PMID: 25461735 (prom.-ex. 6). These mutations lead to dysfunction of the LDLR protein

**Literature review:**

"FH is caused by a mutation in the gene encoding the LDLR in more than 90 % of the molecular diagnosed cases, and this mutation leads to absent or dysfunctional LDLR at the surface of the hepatocytes [7]. As a consequence, hepatic uptake of LDL-C is decreased which results in elevated plasma levels of LDL-C [1]. The LDLR gene is located on the short arm of chromosome 19, and to date, over 1700 mutations in the LDLR gene have been described (http://www.ucl.ac.uk/ldlr/Current/). Five different classes of LDLR mutations have been identified, dependent on the effect on the phenotype. Class 1 mutations are null mutations that result in no detectable LDLR protein. In class 2 mutations, the transport of the LDLR from the endoplasmic reticulum to the Golgi apparatus is blocked completely (class 2a) or partially (class 2b). A class 3 mutation leads to expression of a non-functional LDLR. Class 4 mutations result in LDL binding but the LDLR-LDL complexes cannot be internalized, and in class 5 mutations, recycling of the LDLR is not efficient and therefore do not reach the cell surface..."

*Hartgers M et al 2015 PMID: 26482752*

"The updated database (http://www.lovd.nl/LDLR) now includes 2925 curated variants, representing 1707 independent events. All 129 nonsense variants, 337 small frame-shifting and 117/118 large rearrangements were classified as 4 or 5. Of the 795 missense variants, 115 were in classes 1 and 2, 605 in class 4 and 75 in class 3. 111/181 intronic variants, 4/34 synonymous variants and 14/37 promoter variants were assigned to classes 4 or 5. Overall, 112 (7%) of reported variants were class 3."

*Leigh S et al 2017 PMID: 27821657*

"Pathogenic variants have been reported in the promoter, introns, and exons of LDLR. The majority of pathogenic variants fall within the ligand-binding (40%) or epidermal growth factor precursor-like (47%) domains, with the highest frequency of pathogenic variants reported in exon 4 (20%) [Leigh et al 2008, Usifo et al 2012]. More than 1,500 LDLR pathogenic variants have been reported in the University College London (UCL) database, highlighting the molecular heterogeneity of the disorder. See Table A, Locus-Specific Databases and ClinVar, for a list of reported variants.

Pathogenic variants in LDLR usually either reduce the number of LDL receptors produced within the cells or disrupt the ability of the receptor to bind LDL-C. Either way, heterozygous pathogenic variants in LDLR cause high levels of plasma LDL-C.

Complete loss-of-function variants in LDLR generally lead to more severe disease due to higher LDL-C levels [Khera et al 2016]. Partial loss-of-function variants in LDLR result in less severe disease due to lower LDL-C levels.

Recent findings suggest that only 73% of individuals with a heterozygous LDLR pathogenic variant have an LDL level >130 mg/dL, suggesting lower penetrance than previously proposed [Khera et al 2016]."

*Gene reviews https://www.ncbi.nlm.nih.gov/books/NBK174884 #hyperchol.Molecular_Genetics*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

incomplete penetrance

**Allelic requirement:**

Monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

FH is caused by a mutation in the gene encoding the LDLR in more than 90 % of the molecular diagnosed cases, and this mutation leads to absent or dysfunctional LDLR at the surface of the hepatocytes. As a consequence, hepatic uptake of LDL-C is decreased which results in elevated plasma levels of LDL-C.
The mechanism appears to be loss of function of LDLR resulting in **decreased/absent or altered gene product** which leads (through various mechanisms including null mutations, defective transport, defective receptor binding, inability to internalise LDLR-LDL complexes, inefficient LDLR recycling) to increased LDL-C particles accumulating in the blood. LDLR is genetically heterogeneous with more than 1500 pathogenic variants reported. Pathogenic variants have been reported in the promoter, introns, and exons of LDLR. The majority of pathogenic variants fall within the ligand-binding (40%) or epidermal growth factor precursor-like (47%) domains, with the highest frequency of pathogenic variants reported in exon 4 (20%)  Penetrance is incomplete.

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Splice region variant
- Missense
- In frame deletion
- In frame duplication
- synonymous
- intron_variant
- regulatory_region_variant

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- 5_prime_UTR_variant
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[uORF\]
- Stop lost \[oORF\]
- Start lost \[uORF\]
- Frameshift \[oORF\]
- Stop gained \[uORF\]
- intergenic_variant# **LMNA -- Dilated Cardiomyopathy**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:6636

**ClinGen evidence for haploinsufficiency**

LMNA encodes Lamins A and C, produced by alternative splicing of
transcripts encoded by this gene. Lamin A/C are intermediate filament proteins that localize to the inner nuclear membrane and nucleoplasm where they function in structural support, DNA organization and function. Mutations in LMNA are associated with a wide range of disorders (neuromuscular, cardiac, lipodystrophy and premature aging), some of which have overlapping and variably present clinical phenotypes. LMNA-associated phenotypes include both autosomal dominant and recessive inheritance patterns. A large number (\>400) of disease-associated LMNA variants have been reported; the vast majority are sequence-level alterations and are predominantly missense mutations. However frameshift, nonsense, and intragenic deletions and duplications, which are capable of resulting in a loss-of-function, are also reported. Thus LMNA haploinsufficiency is a possible mechanism of pathogenicity,
although it is not apparently the predominant mechanism. As yet, whole gene deletion of LMNA has not been reported in association with clinical phenotypes. Although the presence of \>2 independent publications with numerous loss-of-function-type variants, some of which include supportive functional data, provide evidence in support of LMNA haploinsufficiency, due to the extensive clinical heterogeneity,inheritance patterns, mutational spectrum, and lack of reports of whole gene deletions, the haploinsufficiency score given here is 2.

Gupta et al 2010 [[20127487]{.ul}](https://www.ncbi.nlm.nih.gov/pubmed/20127487) describe a deletion in LMNA in a patient with dilated cardiomyopathy, and the deletion caused less LMNA expression in cardiomyocytes. They identified a partial LMNA deletion (9 of 12 3? exons; exons 3-12, RefSeq NM_170707.3) in a patient with dilated cardiomyopathy by MLPA and qPCR. Functional studies on cardiomyocytes included immunostaining, which showed reduced lamin A/C levels, and electron microscopy, which showed
intracellular and nuclear abnormalities, supportive of a
loss-of-function due to the deletion. The authors propose
haploinsufficiency as the mechanism of pathogenicity based on the extent of the deletion, functional studies and supportive data from the literature.

MacLeod et al (2003) [12854972](https://www.ncbi.nlm.nih.gov/pubmed/12854972) identified a novel 2 base-pair deletion c.908_909delCT, causing a frameshift and truncated LMNA/C in a proband with history of paroxysmal atrial fibrillation, sick sinus syndrome, dilated cardiomyopathy. Per discussion: \"It is presently unknown whether autosomal dominant lamin A/C mutations impart their phenotype through dominant negative or
haploinsufficient mechanisms. The majority of reported LMNA mutations are missense. In this case, the mutant protein is thought to be expressed and to act through a dominant interfering mechanism. A smaller number of LMNA mutations are frameshifting and are expected to create truncated lamin A and C. In at least one case, a nonsense mutation at amino acid position 6 was described \[11\]. This mutation is effectively a null allele and therefore, dominant mutations, at least in this case, would be expected to arise from haploinsufficiency of lamin A/C.\"

Sebillion et al (2003) [12920062](https://www.ncbi.nlm.nih.gov/pubmed/12920062) identified an insertional mutation (28insA) that led to a stop codon and an aberrant truncated protein of 38 amino acids in a family affected by dilated cardiomyopathy with conduction defects\*. mRNA expression analysis was performed by signal intensity measurement of a RE-digested cDNA product
prepped from total RNA extraction from patient lymphoblastoid cell
lines; this analysis showed a reduction or loss of mutant fragment
compared to wild-type. In vitro cell transfection experiments showed a significant decrease in transfection efficiency for the 28insA cDNA, suggestive of reduced mRNA expression or stability leading to reduction of protein. The authors suggest this mutation may act via
haploinsufficiency. \*Note: the authors found no mutation in LMNA in
cases with isolated dilated cardiomyopathy.

**Literature review:**

*"LMNA*-related dilated cardiomyopathy (DCM) results
from [missense](https://www.ncbi.nlm.nih.gov/books/n/gene/glossary/def-item/missense/) variants,
with
occasional [nonsense](https://www.ncbi.nlm.nih.gov/books/n/gene/glossary/def-item/nonsense-variant/) or
splice-site variants and short insertions or deletions of *LMNA*.

The mechanism of cellular injury that causes *LMNA*-related DCM remains
incompletely understood. Because lamin A/C is a structural protein of
the nuclear membrane, it has been suggested that fragility of the
nuclear membrane in the setting of repetitive contraction of skeletal or
cardiac muscle may predispose to nuclear injury and cellular apoptosis.
An alternative hypothesis suggests that an abnormal lamin A/C protein
may disrupt the chromatin/lamin-associated protein complex, thereby
disturbing gene expression."

*Gene reviews <https://www.ncbi.nlm.nih.gov/books/NBK1674/>*

*"LMNA* missense and truncating mutations account for 5--8% of genetic
DCM
The mechanisms responsible for autosomal dominant DCM *LMNA* mutations
may be a mix of multiple defects including dominant-negative function as
well as
haploinsufficiency
Lamins A and C are implicated in many different cellular processes from
regulating gene expression, mechanosensing, DNA replication, and nuclear
to cytoplasmic transport."

*McNally et al. 2018 PMID: [28912180]*

Walsh et al found an excess of truncating and non truncating variants in LMNA associated with DCM in comparison to the reference dataset (ExAC)

*Walsh et al, 2016 (PMID 27532257)*

**From our in-house Atlas of DCM:**

**16/46 truncating**

Nonsense

Frameshift

Splicing

**30/46 non-truncating**

Missense

<https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=LMNA&icc=DCM>

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

Optional modifiers: 
Incomplete penetrance

**Allelic requirement:**

Monoallelic_aut

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The mechanism of cellular injury that causes *LMNA*-related DCM remains incompletely understood. Both haploinsufficiency and a dominant negative mechanism leading to **reduced or absent gene product or** **altered gene product structure** have been proposed. Because lamin A/C is a structural protein of the nuclear membrane, it has been suggested that fragility of the nuclear membrane in the setting of repetitive contraction of skeletal or cardiac muscle may predispose to nuclear injury and cellular apoptosis. An alternative hypothesis suggests that an abnormal lamin A/C protein may disrupt the chromatin/lamin-associated protein complex, thereby disturbing gene expression. LMNA*-related dilated cardiomyopathy (DCM) results from missense variants,
with occasional nonsense or splice-site variants and short insertions or deletions of *LMNA*. Penentrance is incomplete.

**List variant classes in this gene proven to cause this disease:**

Stop gained

Stop gained (predicted to undergo NMD)

Frameshift

Frameshift (predicted to undergo NMD)

Splice acceptor variant

Splice acceptor variant (predicted to undergo NMD)

Splice donor variant

Splice donor variant (predicted to undergo NMD)

Missense

In frame deletion

In frame duplication

**Potential novel variant classes based on predicted functional
consequence:**

Splice acceptor variant (predicted to escape NMD)

Splice donor variant (predicted to escape NMD)

Frameshift variant (predicted to escape NMD)

start_lost

stop_gained predicted to escape NMD

stop_lost

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[oORF\]

Frameshift \[oORF\]
# **MEN1 -- Multiple Endocrine Neoplasia Type 1**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:7010

Multiple endocrine neoplasia type 1 (MEN1) is a complex syndrome defined by the neoplastic transformation of at least two endocrine organs, most frequently parathyroid glands, pancreatic islets, anterior pituitary and endocrine pancreas. There is abundant evidence published associating the MEN1 gene with multiple endocrine neoplasia type 1, since the gene-disease relationship was first proposed by Chandrasekharappa SC, et al., 1997 (PMID: 9103196). Multiple case level studies have been performed with MEN1 patients that have variants in the MEN1 gene. The variants include single amino acid changes or deletions that affect the stability of the protein and nonsense or frameshift LOF variants. A significant amount of case-level data is available; the maximum points for genetic evidence has been reached (12 points). Multiple mouse models of MEN1 have been established to show development of tumors consistent with MEN1 syndrome. In summary, MEN1 is definitively associated with the autosomal autosomal dominant Multiple Endocrine Neoplasia Type 1 syndrome. This has been demonstrated in both the research and clinical diagnostic settings and has been upheld over time.
Gene Clinical Validity Standard Operating Procedures (SOP) - Version 7

**ClinGen Evidence for Haploinsufficiency**

**MEN1 sequence level variants (nonsense, frameshift, splice site, and exonic deletions) as well as non-focal MEN1 whole gene deletions are associated with the development of Multiple Endocrine Neoplasia type 1 (MEN1) syndrome (PMID: 9510467, 15105049, and 21763627 and Genereviews).** MEN1 syndrome is characterized by an increased susceptibility to develop endocrine tumors (i.e., parathryoid, pituitary, and well-differentiated endocrine tumors of the gastro-entero-pancreatic [GEP] tract), as well as non-endocrine tumors (e.g. facial angiofibromas, collagenomas, lipomas, meningiomas, ependymomas, and leiomyomas). The penetrance for the development of clinical features associated with MEN1 mutations is 50% by age 20 and 95% by age 40.

**Literature review:**

**More than 1,200 germline mutations in the MEN1 gene have been identified, which are scattered over the entire coding region of the gene without any significant hot spots or genotype-phenotype correlations (27, 29).** The majority of MEN1 germline mutations (69%) are predicted to be pathogenic due to either premature truncation of menin due to frame-shift mutations (42%) and nonsense mutations (14%), or exon region deletions which are attributed to splicing defects (10.5%) and large deletions (2.5%) (27, 29). Other MEN1 germline mutations include missense mutations (25.5%) and single or few amino acid in-frame deletions or insertions (5.5%), which require further investigation to determine their pathogenicity.

**Approximately 5–25% of patients with MEN1 may not have mutations in the MEN1 coding region. These individuals may have whole or partial gene deletions,** and it has been postulated that mutations may also occur in the promoter or untranslated regions (27, 30, 31). In addition, the occurrence of phenocopies, or patients that develop disease manifestations typically associated with mutations in the MEN1 gene but instead are due to another etiology, has been described in 5–10% of MEN1 kindreds (32–34). These phenocopies may occur in individuals with a family history of MEN1 and one MEN1-associated tumor or in patients with two MEN1-associated tumors with other gene involvement.

Germline MEN1 mutations have also been noted in families with a parathyroid only disorder, familial isolated primary hyperparathyroidism, where there is a higher frequency of missense mutations compared to patients with the MEN1 syndrome 

MEN1 acts as a tumor suppressor gene. Patients with germline inactivating mutations in MEN1 demonstrate loss of heterozygosity (LOH) in more than 90% of their tumors.

The protein product of MEN1, menin, is implicated in the regulation of transcription, genome stability, cell division, and cell proliferation, though the exact role of menin in tumorigenesis is yet to be elucidated (1, 27, 28, 42).

Two main nuclear localization signals (NLS), NLS1 and NLS2, as well as a third accessory NLS, NLSa, are harbored within the amino acid sequence of menin (43, 44). Mutations in MEN1 that lead to premature protein truncation may lead to functional inactivation of menin through loss of one or both main NLSs. Menin has not been demonstrated to have intrinsic enzymatic activity, but studies of protein-protein interaction by multiple groups have identified more than 50 proteins that could partner with menin

*Kamilaris C et al 2019 PMID: 31263451*

Frequency of types of MEN1 mutations reported in 1,091 MEN1 kindreds:

nonsense		23%
frameshift		41%
missense		20%
splice site		9%
in-frame
dels/ins		6%
gross del		1%

*Lemos M, Thakker R 2008 PMID: 17879353*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: incomplete penetrance

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

MEN1 acts as a tumor suppressor gene. The protein product of MEN1, menin, is implicated in the regulation of transcription, genome stability, cell division, and cell proliferation, though the exact role of menin in tumorigenesis is yet to be elucidated. Patients with germline inactivating mutations in MEN1 demonstrate loss of heterozygosity (LOH) in more than 90% of their tumors. Pathogenic variants are spread throughout the gene with no clear hot spot regions. The majority of variants are predicted to be pathogenic due to either premature truncation of menin due to frame-shift mutations (42%) and nonsense mutations (14%), or exon region deletions which are attributed to splicing defects (10.5%) and large deletions (2.5%). Approximately 5–25% of patients with MEN1 may not have mutations in the MEN1 coding region. These individuals may have whole or partial gene deletions, or mutations may occur in the promoter or untranslated regions.

**List variant classes in this gene proven to cause this disease:**

- Missense
- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- In frame deletions
- In frame insertions

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
# **MLH1 -- Lynch Syndrome**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:7127

**ClinGen Evidence for Haploinsufficiency**
PMID 14635101 - Taylor et al (2003) screen 215 subjects referred to a clinic for phenotype consistent with autosomal dominant familial nonpolyposis colon cancer (Lynch Syndrome) and identify 6 MLH1 deletions (as well as 10 novel MLH1 mutations.)

PMID 15942939 - van der Klift (2005) screened a large cohort of families with hereditary nonpolyposis colon cancer and identified 13 MLH1 deletions from 68 unrelated kindreds.

**Literature review:**

Lynch syndrome is caused by pathogenic variants in genes involved with the mismatch repair (MMR) pathway. This pathway functions to identify and remove single-nucleotide mismatches or insertions and deletion loops. Pathogenic variants in four of the MMR genes can cause Lynch syndrome [Peltomäki 2003]. The functions of the MMR genes can be disrupted by missense variants, truncating variants, splice site variants, large deletions, or genomic rearrangements.

More than 200 different pathogenic variants have been reported in MLH1 [Peltomäki 2003, Peltomäki & Vasen 2004]; see Table A. Deletions account for 5%-10% of germline MLH1 pathogenic variants.

Constitutional inactivation of MLH1 by methylation, along with somatic loss of heterozygosity of the functional allele, has been reported as a rare cause of Lynch syndrome (~0.6%) [Niessen et al 2009]. These individuals have silencing of one MLH1 allele, throughout their tissues, due to methylation of the promoter and a Lynch syndrome phenotype. Most of such cases are simplex (i.e., a single occurrence in a family), but a few families with inherited hypermethylation have been reported [Goel et al 2011]. MLH1 promoter methylation is not detectable by either sequence analysis or duplication/deletion analysis of MLH1.

MLH1 acts in a recessive manner at the cellular level where there is an absence of functional Mlh1 protein in the tumor cells. This results from inactivation of both MLH1 alleles in the tumor, which often occurs as a result of an inactivating variant or silencing of the MLH1 promoter by hypermethylation.

*Gene reviews
https://www.ncbi.nlm.nih.gov/books/NBK1211/*

Types of variant in MLH1:

Missense 40%
Nonsense or frameshift 40%
In-frame 2% (includes deletions, insertions or indels which do not affect the reading frame)
Splice  11%
Large rearrangement 10%

*Data from the InSiGHT database presented in Tamura K et al 2019 PMID: 31273487*

Variant types taken from InSiGHT database
*Plazzer JP et al 2013 PMID: 23443670*

Thirteen (27%) of the 48 genomic rearrangements reported from this study occurred in MLH1,
accounting for 14 patients (Fig. 1B). The exon 16
deletion was found in patients from 2 apparently
unrelated families. Notably, this deletion is different from the 3.5-kb MLH1 exon 16 deletion found
to represent a founder mutation in the Finnish
population (Nystrom-Lahti et al., 1995).

*van der Klift H et al 2005 PMID: 15942939*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: incomplete penetrance

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

A heterodimer of MLH1 and PMS2 coordinates the interplay between the mismatch recognition complex and other proteins necessary for mismatch repair. Loss of function variants including large genomic rearrangements can result in an altered/reduced or absent MLH1 protein. Inactivation of both MLH1 alleles (following a second hit) resulting in an absence of functional Mlh1 protein can be seen at a cellular level in tumour cells. Rarely (~0.6% of patients) constitutional MLH1 promoter methylation can cause Lynch syndrome.

**List variant classes in this gene proven to cause this disease:**

- Missense
- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- In frame deletions
- In frame insertions


**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
# **MSH2 -- Lynch Syndrome**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:7325

**ClinGen Evidence for Haploinsufficiency**
PMID 10850409 - Germline mutations within the MSH2 gene at 2p15 ( mismatch repair gene) have been seen in families Hereditary nonpolyposis colorectal cancer (HNPCC). HNPCC is an autosomal dominant disease characterized clinically by increased risk of early development of colorectal cancer as well as increased risk for other tumors. Charbonnier et al. analyzed a cohort of of families with Amsterdam criteria positive (AC+) fHNPCC and Amsterdam criteria negative (AC-) HNPCC. **A total of 12 genomic rearrangements and 12 point mutations were detected in the MSH2 gene (4 frameshift, 4 nonsense, 1 splice site, 2 missese).** One of the genomic rearrangments is a previously reported paracentric inversion of chromosome 2 (PMID 12203789) the remainder were exonic deletions of various lengths. The author also descibed a family that was negative for mutations within the MSH2 gene and other mis match repair (MMR) genes that had classical HNPCC and showed a high microsattelite insatbility phenotype and loss of MSH2 protein expression by IHC indicating the functional impairment of MSH2. Additional articles showing intragenic deletions include PMID 15949572, 9843200

PMID 12203789 - Wagner et al. describe a family with HNPCC (also known as Lynch syndrome) with a ~10 MB paracentric inversion that included exons 8-16 of the MSH2 gene. Inverse PCR showed the paracentric inversion breakpoints mapped within intron 7 and to a contig 10 Mb 3' of the MSH2 gene. Expression analysis was performed by a conversion analysis model which takes an Msh2 deficient rodent cellular background and using the patients lymphocytes to generate a somatic cell hybrid, analyzed the expression of the abnormal inverted MSH2 allele. Northern and Western blot analysis of the hybrid containing the abnormal inverted MSH2 show no detectable MSH2 mRNA or MSH2 protein, whereas the full length MSH2 allele was detected in the wild type allele hybrid.

PMID 9843200 - Wijnen et al. discussed the frequency of genomic deletion in HNPCC. A total of 137 families (51 Amsterdam criteria positive HNPCC and 86 Amsterdam negative with familial clustering of colorectal cancer reminiscent of HNPCC) were investigated by southern blot. Eight patients had aberrant restriction fragments indicating the presence of a genomic rearrangement. Four had aberrant restriction fragments indicative of an intragenic deletion in MSH2 encompassing various exons and a few of the deletions were confirmed by RT-PCR of the affected individuals. Author suggest that current technologies may not be able to detect all forms of mutations including genomic deletions (including complete gene deletions) and rearrangemnts and that mutation analysis of the MSH2 gene should include the examination of its genomic structure by Southern blot.

Although there have been no publications based on complete loss of the gene (hemizygous gene), there have been several reports of interstitial deletions, inversions, and point mutations (missense, framshift, nonsense) described in the literature that represent and autosomal dominant inheritance pattern. Functional studies have shown that deletions or disruptions result in a loss of function (inactivating mutations) (12203789, 9843200).

**Literature review:**

Lynch syndrome is caused by pathogenic variants in genes involved with the mismatch repair (MMR) pathway. This pathway functions to identify and remove single-nucleotide mismatches or insertions and deletion loops. Pathogenic variants in four of the MMR genes can cause Lynch syndrome [Peltomäki 2003]. The functions of the MMR genes can be disrupted by missense variants, truncating variants, splice site variants, large deletions, or genomic rearrangements.

More than 170 pathogenic variants have been identified in MSH2 [Peltomäki 2003, Peltomäki & Vasen 2004]. **The higher proportion of Alu repeats may contribute to the higher rate of genomic rearrangements in MSH2 than in MLH1 [van der Klift et al 2005]. At least 20% of germline MSH2 pathogenic variants are exon or multiexon deletions.**

MSH2 acts in a recessive manner at the cellular level where there is an absence of functional Msh2 protein in the tumor cells. This results from inactivation of both MSH2 alleles in the tumor, which often occurs by the mechanism of loss of heterozygosity. MSH2 promoter methylation has been shown to be the inactivating event that silences the normal allele in individuals with an MSH2-inactivating pathogenic variant. Of note, this is not a common cause of sporadic colon cancer.

**Heterozygosity for an MSH2 pathogenic variant is associated with the greatest risk for extracolonic cancers.**

MSH2 pathogenic variants have been reported more commonly than a pathogenic variant in the other three MMR genes in individuals with the Muir-Torre variant of Lynch syndrome [South et al 2008].

*Gene reviews
https://www.ncbi.nlm.nih.gov/books/NBK1211/*

Types of variant in MSH2:

Missense 31%
Nonsense or frameshift 49%
In-frame 2% (includes deletions, insertions or indels which do not affect the reading frame)
Splice  8%
Large rearrangement 10%

Apart from exons andintron/exon borders, a few point mutations have been identified in the promoter regions of MSH2

*Data from the InSiGHT database presented in Tamura K et al 2019 PMID: 31273487*

Among the 48 genomic rearrangements, the
majority (60%) affected the MSH2 gene.
A single deletion of MSH2 exons 1–6 is responsible for several North American HNPCC cases and has been shown to represent a
U.S. founder mutation (Wagner et al., 2003; Lynch
et al., 2004). 

*van der Klift H et al 2005 PMID: 15942939*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: incomplete penetrance

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The protein encoded by MSH2 forms a heterodimer with either DNA MMR protein MSH6 or MSH3 and functions to identify mismatches for mismatch repair. Loss of function variants including large genomic rearrangements (10% of pathogenic variants) can result in an altered/reduced or absent MSH2 protein. Inactivation of both MSH2 alleles (following a second hit) resulting in an absence of functional protein can be seen at a cellular level in tumour cells. The higher proportion of Alu repeats may contribute to the higher rate of genomic rearrangements in MSH2. MSH2 pathogenic variants are associated with the greatest risk for extracolonic cancers compared to the other MMR genes.

**List variant classes in this gene proven to cause this disease:**

- Missense
- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- In frame deletions
- In frame insertions


**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
# **MSH6 -- Lynch Syndrome**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:7329

**ClinGen Evidence for Haploinsufficiency**

PMID 14974087 - Plaschke et al 2004 identifies 7 truncating mutations in MSH6 associated with tumors displaying high levels of micro satellite instability.

PMID 2815724 - Baglietto et al (2010) assess the risk of Lynch syndrome in **113 MSH6 mutation carriers (truncation, loss, frameshift indel, or missense damaging) collected from multiple cancer centers.** "For MSH6 mutation carriers, the estimated cumulative risks to ages 70 and 80 years, respectively, were as follows: for colorectal cancer, 22% (95% confidence interval [CI]?=?14% to 32%) and 44% (95% CI?=?28% to 62%) for men and 10% (95% CI?=?5% to 17%) and 20% (95% CI?=?11% to 35%) for women; for endometrial cancer, 26% (95% CI?=?18% to 36%) and 44% (95% CI?=?30% to 58%); and for any cancer associated with Lynch syndrome, 24% (95% CI?=?16% to 37%) and 47% (95% CI?=?32% to 66%) for men and 40% (95% CI?=?32% to 52%) and 65% (95% CI?=?53% to 78%) for women. Compared with incidence for the general population, MSH6 mutation carriers had an eightfold increased incidence of colorectal cancer (HR?=?7.6, 95% CI?=?5.4 to 10.8), which was independent of sex and age. Women who were MSH6 mutation carriers had a 26-fold increased incidence of endometrial cancer (HR?=?25.5, 95% CI?=?16.8 to 38.7) and a sixfold increased incidence of other cancers associated with Lynch syndrome (HR?=?6.0, 95% CI?=?3.4 to 10.7)."

Gene Reviews (http://www.ncbi.nlm.nih.gov/books/NBK1211/) **estimates ~7-10% Lynch Syndrome subjects have germline mutation in MSH6 (deletions, frameshift, stops) that cause loss of MSH6. Throughout literature this estimate is repeated. While the mechanism is not explicit, overall it is thought heterozygous mutations in MSH6 impair mismatch repair.**

From Bellizzi et al (2009, PMID:19851131): "The MMR proteins function as heterodimers. The MSH2-MSH6 complex recognizes mispaired bases and insertion/deletion loops. It recruits MLH1-PMS2, which subsequently directs the remainder of the MMR machinery. MSH2 and MLH1 are the dominant (obligate) constituents of their respective pairs. In the absence of MSH6, MSH2 can pair with MSH3, and in the absence of PMS2, MLH1 can pair with PMS1. This may partially explain the somewhat attenuated Lynch phenotype attributed to MSH6 and PMS2 mutation, (reduced penetrance and older age of onset relative to MSH2 and MLH1 mutants?)"

**Literature review:**

Lynch syndrome is caused by pathogenic variants in genes involved with the mismatch repair (MMR) pathway. This pathway functions to identify and remove single-nucleotide mismatches or insertions and deletion loops. Pathogenic variants in four of the MMR genes can cause Lynch syndrome [Peltomäki 2003]. The functions of the MMR genes can be disrupted by missense variants, truncating variants, splice site variants, large deletions, or genomic rearrangements.

More than 30 pathogenic variants have been identified in MSH6 [Peltomäki & Vasen 2004]. Exon or multiexon deletions are a rare cause of germline MSH6 pathogenic variants.

MSH6 acts in a recessive manner at the cellular level where there is an absence of functional MSH6 protein in the tumor cells. This results from inactivation of both MSH6 alleles in the tumor, which often occurs by the mechanism of loss of heterozygosity.

Heterozygosity for a pathogenic variant in MSH6 is associated with MSI-low tumors. The colorectal cancers in families with an MSH6 pathogenic variant may be later in onset and more distally located than the cancers in families with Lynch syndrome resulting from a pathogenic variant in one of the other MMR genes; endometrial cancer is commonly observed in females with an MSH6 pathogenic variant [Wu et al 1999, Berends et al 2002]. Slightly lower risks for colorectal cancer and higher risks for endometrial cancer have been reported in families with an MSH6 pathogenic variant than in families with an MLH1 or MSH2 pathogenic variant [Berends et al 2002, Baglietto et al 2010].

*Gene reviews
https://www.ncbi.nlm.nih.gov/books/NBK1211/*

Types of variant in MSH6:

Missense 49%
Nonsense or frameshift 43%
In-frame 3% (includes deletions, insertions or indels which do not affect the reading frame)
Splice  3%
Large rearrangement 2%

*Data from the InSiGHT database presented in Tamura K et al 2019 PMID: 31273487*

Variant types taken from InSiGHT database
*Plazzer JP et al 2013 PMID: 23443670*


#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: incomplete penetrance

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The MSH2-MSH6 complex recognizes mispaired bases and insertion/deletion loops. It recruits MLH1-PMS2, which subsequently directs the remainder of the MMR machinery.
Loss of function of MSH6 results in dysfunction of this repair pathway. MSH2 and MLH1 are the dominant (obligate) constituents of their respective pairs. In the absence of MSH6, MSH2 can pair with MSH3, and in the absence of PMS2, MLH1 can pair with PMS1. This may partially explain the somewhat attenuated Lynch phenotype attributed to MSH6 and PMS2 mutation.
Truncating and non-truncating pathogenic variants in MSH6 have been reported; exon or multiexon deletions are a rare cause of germline MSH6 pathogenic variants.
Slightly lower risks for colorectal cancer and higher risks for endometrial cancer have been reported in families with an MSH6 pathogenic variant than in families with an MLH1 or MSH2 pathogenic variant.

**List variant classes in this gene proven to cause this disease:**

- Missense
- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- In frame deletions
- In frame insertions

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]

# **MUTYH -- Polyposis**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:7527

**ClinGen Evidence for Haploinsufficiency**

**Literature review:**

Autosomal recessive colorectal adenomatous polyposis is a disorder characterized by adult-onset of multiple colorectal adenomas and adenomatous polyposis. Affected individuals have a significantly increased risk of colorectal cancer.

Among 614 families recorded in 6 regional registers of polyposis in the UK, **Sampson et al. (2003) identified 111 with neither dominant transmission nor evidence of APC mutation. Molecular genetic analysis showed that 25 had biallelic mutations of the MYH gene. The data showed that MYH polyposis can be transmitted as an autosomal recessive trait**

Al-Tassan et al. (2002) studied a British family in which 3 sibs had multiple colorectal adenomas and carcinoma. There was no clear pathogenic change in the APC gene. Analysis of the MYH gene showed that the sibs were compound heterozygous for nonconservative missense variants.

Rouleau et al. (2011) reported a 45-year-old French man who was found to have 25 colorectal adenomas on colonoscopy. He had no family history of the disorder. Analysis of the APC gene was negative, and molecular analysis identified compound heterozygosity for mutations in the MUTYH gene: a missense mutation and a large rearrangement resulting in the deletion of exons 3 to 16. 

*Omim
https://omim.org/entry/608456*

Adenine DNA glycosylase (encoded by MUTYH): recognizes and excises misincorporated adenine bases to prevent G:C>T:A transversions from occurring.

A lack of functional adenine DNA glycosylase leads to accumulation of G:C>T:A transversions in daughter DNA strands post replication. Studies indicate that this transversion is common in colorectal tumor DNA from individuals with MAP. These pathogenic variants result in loss of adenine DNA glycosylase function [Lipton & Tomlinson 2004].

*Gene reviews https://www.ncbi.nlm.nih.gov/books/NBK107219/*

Thirty-six different mutations were found in 185 index patients (Table 1). The Y179C mutation represented 44% of all mutations (164/370), and the G396D represented 24% of all mutations (90/370). Truncating mutations (64/370) represented 17% of all changes.

We found that Y179C homozygotes presented earlier and had a significantly greater CRC hazard than G396D homozygotes and G396D/Y179C compound heterozygotes.

In contrast to the differences in phenotypic expression we observed between the most common MUTYH missense mutations, we found no significant difference between truncating and nontruncating mutations except for polyp count (ie, the number of truncating alleles was inversely correlated with the proportion of cases with <10 polyps).

*Nielsen M et al PMID: 19032956*

about 80 pathogenic mutations distributed throughout the gene and located at positions corresponding to different functional domains of the protein (see: Leiden Open Variation Database). **Although various types of alterations have been reported in MAP patients, including nonsense, small insertion/deletion, and splicing variants, missense mutations represent the great majority of the detected changes.**

**A number of variants appear recurrent in different populations, with Y179C (previously annotated as Y165C) and G396D (previously annotated as G382D) missense mutations accounting together for about 70% of germline alterations found in European patients** (reviewed by Cheadle and Sampson, 2007) however, in Asian populations, Y179C and G396D must be rare, since neither mutation has been found in MAP patients. On the other hand, other mutations have proven to be recurrent in patients from particular populations (reviewed by Poulsen and Bisgaard, 2008). Taken together, these findings indicate that sequencing of the entire MUTYH open reading frame has to be performed for the genetic testing, especially in populations of mixed ethnicity.

Recently, a large gene deletion spanning exons 4–16 has been found in two unrelated patients showing an attenuated phenotype; due to this observation, appropriate methods to detect gene rearrangements should be considered, at least for patients carrying either a single heterozygous mutation or a (apparently) homozygous disease-causing mutation (Rouleau et al., 2011; Torrezan et al., 2011). Besides rare mutations, a polymorphic allele (SNP rs3219468: G>C) associated with a significant reduction of a MUTYH transcription product has recently been implicated in CRC risk (Plotz et al., 2012).

*Venesio T et al 2012 PMID: 22876359*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal recessive

(optional) modifiers: 

**Allelic requirement:**

biallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Pathogenic variants in MUTYH result in loss of adenine DNA glycosylase function leading to an accumulation of G:C>T:A transversions in daughter DNA strands post replication. Studies indicate that this transversion is common in colorectal tumor DNA from individuals with MAP. Various types of alterations have been reported, including nonsense, small insertion/deletions, and splicing variants, however missense mutations represent the great majority of the detected changes. A number of variants appear recurrent in different populations, with Y179C (previously annotated as Y165C) and G396D (previously annotated as G382D) missense mutations accounting together for about 70% of germline alterations found in European patients; other mutations have proven to be recurrent in patients from particular populations (reviewed by Poulsen and Bisgaard, 2008). There is no clear genotype phenotype correlation however there appears to be some association between truncating variants and polyp number.

**List variant classes in this gene proven to cause this disease:**

- Missense
- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- In frame deletions


**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- In frame insertions
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]

# **MYBC3 -- Hypertrophic cardiomyopathy**


**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen:**

<https://search.clinicalgenome.org/kb/genes/HGNC:7551>

The MYBPC3 gene has been associated with autosomal dominant hypertrophic cardiomyopathy (HCM) using the ClinGen Clinical Validity Framework. This association was made using case-level data and case control data. MYBPC3 was first associated with HCM in 1995 (Watkins et al, 1995, PMID 7493025). There are over 290 variants asserted as pathogenic for MYBPC3 for HCM in ClinVar, and mutations in MYBPC3 are reported in 40% of the reported cases of HCM (Cirino and Ho, 2014, GeneReviews, PMID 20301725).
More evidence is available in the literature, but the maximum score for genetic evidence and/or experimental evidence (12 pts.) has been reached. Of note, MYBPC3 has been shown to cause HCM in an autosomal recessive fashion, with earlier and more severe presentation of phenotypes associated with HCM, and represents a semi-dominant condition. **The molecular mechanism for HCM is loss of function (LOF), and missense, nonsense, frameshift and splice site mutations in MYBPC3 have been shown to be pathogenic for cardiomyopathy.** Of note, this gene has been implicated in dilated cardiomyopathy and left ventricular noncompaction. This gene-disease association is supported by biochemical, expression, protein interaction, and animal models evidence. In summary, MYBPC3 is definitively associated with autosomal dominant HCM. This has been repeatedly demonstrated in both the research
and clinical diagnostic settings and has been upheld over time. This classification was approved by the ClinGen Hypertrophic Cardiomyopathy Expert Panel on September 5, 2017.

**Numerous frameshift, splicing site, exonic deletion and nonsense
mutations that all lead to premature termination support
haploinsufficiency as a pathogenetic mechanism. Complete deletion of MYBPC3 in patients with large CNV has not been reported with
cardiomyopathy phenotype.**

**Literature review:**

Walsh et al found an excess of truncating and non-truncating variants in MYBPC3 associated with HCM in comparison to the reference dataset (ExAC)


**From our in-house Atlas of HCM:**

*566/1176 truncating*

*610/1176 non-truncating*

<https://www.cardiodb.org/acgv/acgv_gene.php?gene=MYBPC3>

*Walsh et al, 2016 (PMID 27532257)*

"Truncating variants in MYBPC3, which we can estimate based on case and reference frequencies are causative in over 9% of HCM cases, have an EF>0.99 confirming that this variant class has a high likelihood of pathogenicity concordant with pedigree and functional studies."

*Walsh et al 2019 PMID: 30696458*

"90% of *MYBPC3* mutations are heterozygous frameshift, nonsense, or splice site mutations that result in premature termination codons (PTCs) on 1 allele. As such, these mutations are thought to result in HCM from an allelic loss of function via NMD of PTC-containing transcripts, leading to a reduction in MyBP-C content in the sarcomere. Alternatively, these mutations may result in production of truncated MyBP-C, though truncated MyBP-C has never been detected in human heart tissue."

*Helms et al.2020 PMID: 31877118*

"MYBPC3 mutations are the most common cause of hypertrophic
cardiomyopathy, accounting for about half of identified mutations...**many are within introns and are predicted to cause aberrant splicing leading to a frameshift and a premature chain termination, yet the truncated peptides have never been identified in human heart tissue carrying these mutations. Instead of expression of a poison peptide we consistently observe haploinsufficiency of MyBP-C in MYBPC3 mutant human heart
muscle."**

*Marston S et al. 2012 PMID: 22057632*

In a cohort of 114 Chinese patients with HCM, a total of 20 different mutations (8 novel and 12 known mutations) of *MYBPC3* were identified from 25 patients (21.9%). 

"Phenotype-genotype analyses showed that the patients with double
mutations (n = 2) or premature termination codon mutations (n = 12) showed more severe manifestations, compared with patients with missense mutations (n = 11). Particularly, we identified a recurrent truncation mutation (p.Y842X) in four unrelated cases (4/25, 16%), who showed severe phenotypes and suggest that the p.Y842X is a frequent mutation in Chinese HCM patients with severe phenotypes."

*Lui X et al. 2015 PMID: 26573135*

"Ten of the 18 probands with two or more P/LP variants had compound heterozygous or homozygous variants in the same gene, including 5 probands with two or more *MYBPC3* variants.

The average age (±SD) of probands referred for genetic testing tended to be younger among those with two or more P/LP variants (29 ± 3 years) compared with those with only one P/LP variant (39 ± 21 years; *P* = 0.29)."

*Alfares A et al. 2015 PMID: 25611685*

"We found 29 rare MYBPC3 splice-site variants in 56 of 557 (10%) unrelated HCM probands. Three variants were not predicted to alter RNA splicing, and 13 essential splice dinucleotide, nonsense, and short insertion or deletion variants were not further assessed. RNA analysis was performed on 9 variants (c.654+5G>C, c.772G>A, c.821+3G>T, c.927-9G>A, c.1090G>A, c.1624G>A, c.1624+4A>T, c.3190+5G>A, and c.3491-3C>G), and RNA splicing errors were confirmed for 7. Four variants in 4 families resulted in clinically meaningful reclassifications...

There were 180 unique variants [in ClinVar] in splice-site regions of MYBPC3, of which 56 (32%) were variants of uncertain significance, 98 (54%) were pathogenic or likely pathogenic, and 26 (14%) were benign or likely benign.

RNA sequence analysis can directly confirm the outcomes of RNA splicing. We found that 3 variants annotated as causing a missense change, in fact caused exon skipping and lead to frameshifts. MYBPC3 splice-site variants may cause other unexpected outcomes, such as splicing from alternative sites, as demonstrated with the c.654+5G>C variant and intron retention.."

*Singer ES et al 2019 PMID: 30645170*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

Optional modifiers:
Incomplete penetrance

**Allelic requirement:**

Monoallelic_aut

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is likely **loss of function** due to **reduced or absent gene product or** **altered gene product structure** leading to a reduction in MyBP-C content in the sarcomere. 90% of *MYBPC3* mutations are heterozygous frameshift, nonsense, or
splice site mutations that result in premature termination codons on 1 allele. There are reports of patients with double mutations or premature termination codon mutations showing more severe manifestations, compared with patients with missense mutations. Usual mode of inheritance is autosomal dominant but homozygous and compound heterozygous variants have been reported and appear to confer a more severe phenotype. It is characterised by incomplete penetrance. Hot spot regions for HCM in MYBPC3 include amino acid residues 485–502, 1248–1266.

**Additional information related to ACMG evidence types**

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
0.1% (het) 
3.16% (hom) 

**BS1** (MAF too high for disease) 0.02%
Assumptions
• Disease prevalence: 1/200 individuals (1/400 chromosomes)
• Penetrance: 30%
• Maximum pathogenic variant contribution: 2% based on MYBPC3 variant p.Arg502Trp (Walsh et al. 20175:6,000 probands)
• Note that the FAF (95% poisson) is available for each variant in ExAC (http://exac.broadinstitute.org/).

**PM2** A filtering allele frequency (FAF) less than 0.004% activates this rule
CAUTION: Population databases may contain presymptomatic individuals for diseases with reduced
penetrance/variable onset.

*Kelly MA et al 2018 PMID: 29300372* 

*Whiffin N et al 2018 PMID: 29369293*

**PM1** 

Walsh et al propose adaptation of ACMG/AMP guidelines for rule PM1 and HCM, relating to the relative frequencies of non-truncating variants in case cohorts and population controls. 
PM1_strong - EF >0.95
PM1_moderate - EF between 0.90 and 0.95 
PM1_supporting - EF between 0.80 and 0.90

Hot spot regions for HCM in MYBPC3 include amino acid residues 485–502, 1248–1266
Etiological fraction - 0.979 (0.971–0.987) for these HCM clusters. PM1_strong could be applied for variants in these regions.

**PVS1**
Can be applied for truncating variants in MYBPC3,

*Walsh et al 2019 PMID: 30696458*


**List variant classes in this gene proven to cause this disease:**

Stop gained

Stop gained (predicted to undergo NMD)

Frameshift

Frameshift (predicted to undergo NMD)

Splice acceptor variant

Splice acceptor variant (predicted to undergo NMD)

Splice donor variant

Splice donor variant (predicted to undergo NMD)

Splice region variant

Missense

In frame deletion

In frame insertion

**List other variant classes predicted to lead to the same functional consequence:**

Splice acceptor variant (predicted to escape NMD)

Splice donor variant (predicted to escape NMD)

Frameshift variant (predicted to escape NMD)

start_lost

stop_gained predicted to escape NMD

stop_lost

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[oORF\]

Frameshift \[oORF\]
# **MYH11 -- Familial Thoracic Aortic Aneurysm and Dissection**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:7569

Zhu et al (2006) (PMID 16444274) described two kindreds with thoracic aortic aneurysm and/or aortic dissections (TAAD) and patent ductus arteriosus (PDA). MYH11 was one of several candidate genes sequenced. MYH11 encodes for the smooth muscle myosin heavy chain. The French kindred was found to have two heterozygous mutations in cis: the first was a substitution at the splice-donor site of intron 32 (IVS32+1G>T) and the second was a missense mutation in exon 37 (G5361A) resulting in R1758Q. The IVS32+1G>T mutation on this allele results in an in-frame deletion of 71 amino acids (L1456_N1526del) and thus a deletion of exon 32. In the American kindred in this same study, an in-frame deletion of 72-nucleotides (3810_3881del) in exon 28 leads to the loss of 24 corresponding amino acids (R1241_L1264del). In the French kindred, all individuals with the mutation were found by imaging studies to have marked aortic stiffness, even in clinically asymptomatic. None of these variants was found in 340 normal chromosomes screened. Based on functional studies, the authors postulate a dominant negative mechanism.

Familial case of non-syndromic TAAD with a MYH11 L1264P mutation, in which PDA was not observed. (Authors performed genetic analyses of mutations of the TAAD related genes including FBN1, TGFBR1, TGFBR2, SMAD3, TGFB2, ACTA2 and MYH11) Co-segregation was shown in a 17 member three generation pedigree. This variant was recurrent ( also observed in PMID 17666408) Absent in ExAC. Variant is in a conserved residue of a functional domain.

ClinGen Evidence for Haploinsufficiency
**All described mutations in MYH11 currently support a dominant-negative model.** In addition, MYH11 is part of a recurrent deletion mediated by segmental duplications on 16p13.11 observed in patients with intellectual disability, epilepsy and additional phenotypic features. However, patients with large deletions including this gene have not been reported to have thoracic aortic aneurysm and/or aortic dissections (TAAD) or patent ductus arteriosus (PDA), providing evidence against haploinsufficiency causing this phenotype.

**Literature review:**

"We have recently described two kindreds presenting thoracic
aortic aneurysm and/or aortic dissection (TAAD) and patent
ductus arteriosus (PDA)1,2 and mapped the disease locus to
16p12.2-p13.13 (ref. 3). We now demonstrate that the disease
is caused by mutations in the MYH11 gene affecting the
C-terminal coiled-coil region of the smooth muscle myosin
heavy chain, a specific contractile protein of smooth muscle
cells (SMC). All individuals bearing the heterozygous
mutations, even if asymptomatic, showed marked aortic
stiffness. Examination of pathological aortas showed large
areas of medial degeneration with very low SMC content.
Abnormal immunological recognition of SM-MHC and the
colocalization of wild-type and mutant rod proteins in SMC,
in conjunction with differences in their coimmunoprecipitation
capacities, strongly suggest a dominant-negative effect."

*Zhu et al (2006) (PMID 16444274)*

"This study is the first to demonstrate that MYH11 mutations are a rare cause of familial TAAD. At the same time, we confirm that MYH11 are a common cause of familial TAAD associated with PDA. Two of three families with TAAD in conjunction with PDA were found to carry **novel missense point mutations in MYH11**, whereas none of the 93 families with familial TAAD alone was found to have mutations in this gene. By structural analysis, the mutations found in the TAAD/PDA families are predicted to be deleterious to protein function. Similar to two previously reported families with MYH11 mutations, **we observed variable expressivity of the mutant gene in these families**, with some family member presenting with TAAD in conjunction with PDA, some with PDA or some with TAAD alone. In addition, similar to our observations with TGFBR2 mutations (10), **decreased penetrance of the mutation was observed in some adults**, who had no known cardiovascular disease (Fig. 1A). Finally, the aneurysm in patients with MYH11 mutations involved the ascending aorta, a location where a majority of force is placed on the aortic wall with each cardiac contraction and spared the sinuses of Valsalva, which is the location of aneurysms in Marfan patients and TAAD patients with TGFBR2 mutations...

...These data suggest that MYH11 mutations are likely to be specific to the phenotype of TAAD/PDA and result in a distinct aortic and occlusive vascular pathology potentially driven by IGF-1 and Ang II."

*Pannu H et al. 2007 Oct 15 (PMID:17666408)*

"...The spectrum of MYH11 mutations identified for the familial TAAD/PDA phenotype is limited to four
mutations: a small deletion, a splice site mutation, and two missense mutations. Therefore,
MYH11 mutations can cause the disease in the
small subset of families with TAAD and PDA...

...A MYH11 missense mutation causing familial TAAD that
alters arginine 712, an invariant amino acid in
all type II myosins, is part of this α-helix. Other
MYH11 mutations are predicted to disrupt the coiled coil domain of the long C-terminal domain (66, 96). Therefore, disruption of either the motor domain or coiled-coil domain of
myosin is predicted to disrupt the structure or
function of myosin..."
*Milewicz D et al. 2008 PMID: 18544034*

"...At the molecular level, the MYH11 mutation (IVS32+1G>A) affects the canonical splice site sequence and is predicted to result in the loss of exon 32, and thus an in-frame deletion of 71 amino acids. This causes a conformational change of the α-helical coiled coil domain of the smooth muscle myosin heavy chain and impairs the motor function of the protein and its assembly with a homodimeric counterpart. This mutation is similar to the IVS32+1G>T mutation previously reported and further supports a dominant negative effect for MYH11 mutations (Zhu et al 2006 PMID 16444274)."

*Renard M et al. 2013 May 10 (PMID:21937134)*

ClinGen clinical validity working group decribed mutations in MYH11 as dominant negative missense mutations and in-frame deletions in C-terminal coiled coil domain.

*Renard et al 2018 PMID: 30071989*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 
incomplete penetrance

**Allelic requirement:**

Monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The majority of pathogenic variants are missense and in frame deletions. Functional evidence from Zhu et al from 2006 suggest the mechanism is likely dominant negative. An **Altered gene product structure** leads to a disruption to the structure or function of myosin. MYH11 pathogenic variants are most often associated with TAAD with patent ductus arteriosus but they are also a rare cause of FTAAD. Variable penetrance has been reported.
According to ClinGen the location of pathogenic variants is predominantly in the C-terminal coiled coil domain.

**List variant classes in this gene proven to cause this disease:**

- Missense
- In frame insertion
- In frame deletion

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant predicted to escape NMD
- Splice donor variant predicted to escape NMD
- Frameshift predicted to escape NMD
- Stop_gained predicted to escape NMD
- Stop_lost**[MYH7 DCM]{.ul}**

**Review of source material:**

**ClinGen**: 

Not yet curated by clinGen

**ClinGen Haploinsufficiency phenotype comments:**

To date only articles discussing missense mutations in the MYH7 gene
have been described in the literature as being associated with Familial
hypertrophic cardiomyopathy, **dilated cardiomyopathy**, Laing Distal
Myopathy (LDM), Myosin Storage Myopathy (PMID:23346452, 12788380,
12707239, 21846512, 25576864,24664454 ). PMID: 12788380 does however
describe a three base pair deletion leading to an amnio acid loss
glutamic acid in position 927. There was no disruption of the reading
frame downstream of the deletion.

**Literature review:**

**From our in-house Atlas of DCM:**

All Pathogenic/Likely pathogenic variants are missense

One nonsense variant is a 'hot' VUS

Good evidence MYH7 missense variants account for 5-6% of DCM

No evidence that truncating variants are pathogenic.

https://www.cardiodb.org/acgv/acgv_gene.php?gene=MYH7

Walsh R et al 2017
PMID: [27532257](https://www.ncbi.nlm.nih.gov/pubmed/27532257)

**[Pilot application of harmonised terms:]{.ul}**

**Inheritance:**

Autosomal dominant

Optional Modifiers: Incomplete penetrance

**Allelic requirement:**

Monoallelic_aut

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is primarily decrease in sarcomere force generation due to **altered protein product structure**

**List variant classes in this gene proven to cause this disease**

Missense

**Potential novel variant classes based on predicted functional
consequence**

splice_donor_variant

splice donor variant predicted to escape NMD

splice acceptor variant predicted to escape NMD

frameshift variant predicted to escape NMD

stop_gained predicted to escape NMD

stop_lost

inframe_insertion

inframe_deletion
**MYH7 - HCM**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen**: 

https://search.clinicalgenome.org/kb/genes/HGNC:7577

MYH7 HCM definitive (narrative not given)

**ClinGen Haploinsufficiency phenotype comments:**

To date only articles discussing missense mutations in the MYH7 gene have been described in the literature as being associated with Familial hypertrophic cardiomyopathy, dilated cardiomyopathy, Laing Distal Myopathy (LDM), Myosin Storage Myopathy (PMID:23346452, 12788380,
12707239, 21846512, 25576864,24664454 ). PMID: 12788380 does however describe a three base pair deletion leading to an amnio acid loss glutamic acid in position 927. There was no disruption of the reading frame downstream of the deletion.

**Literature review:**

"Missense variants in MYH7 are believed to cause HCM
through gain-of-function actions: variants produce an abnormal activated protein that incorporates into the sarcomere as a ‘poison peptide’ [2]. Haploinsufficiency in MYH7 is not
a recognised disease mechanism for HCM, and heterozygous variants that introduce premature termination codons (PTCs - i.e. nonsense, frameshift and splice variants) have not been demonstrated to be associated with this disease. 

Here we report a frameshift variant in MYH7, c.5769delG,
that is associated with HCM in an Egyptian cohort (3.3%) compared with ethnically-matched controls. This variant is absent from previously published large-scale Caucasian HCM cohorts. We further demonstrate strong evidence of co-segregation of c.5769delG with HCM in a large family (LOD score: 3.01). The predicted sequence of the variant MYH7 transcript shows that the frameshift results in a premature termination codon (PTC) downstream of the last exon-exon junction of the gene that is expected to escape nonsense-mediated decay (NMD). RNA sequencing of myocardial tissue obtained from a patient with the variant during surgical myectomy confirmed the expression of the variant MYH7 transcript."

*Allouba M et al 2020 submitted*

"MYH7 LOF variants are very rare and their contribution to inherited cardiomyopathy is incompletely understood. While there is currently no evidence for a disease-causing role in the heterozygous state, compound heterozygosity of LOF variants along with missense variants can lead to extremely severe presentations, mimicking recessive inheritance.25, 26"

*Kelly MA et al 2018 PMID: 29300372* 

**From our in-house Atlas of HCM:**

849/864  missense

10/864  in frame deletion

There are some unpublished truncating variants in the terminal exon which seem to be enriched in certain case cohorts and may escape NMD and also at least one splice region variant that turns out to cause an in-frame exon skipping event (at a relatively distal exon), leading to expression of an altered gene product structure.

https://www.cardiodb.org/acgv/acgv_gene.php?gene=MYH7

*Walsh R et al 2017 PMID:27532257*

**Pilot application of harmonised terms:**

**Inheritance:**

Autosomal dominant

Optional Modifiers: Incomplete penetrance

**Allelic requirement:**

Monoallelic_aut

Optional Modifiers:

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is primarily increase in sarcomere force generation due to **altered protein product structure**. Missense variants in MYH7 are believed to cause HCM "through gain-of-function actions: variants produce an abnormal activated protein that incorporates into the sarcomere as a ‘poison peptide’."

A novel frameshift variant has been identified in 3.3% of Egyptian HCM patients. It is predicted to result in a premature termination codon (PTC) downstream of the last exon-exon junction of the gene that is expected to escape nonsense-mediated decay (NMD).

MYH7 LOF variants are very rare and their contribution to inherited cardiomyopathy is incompletely understood. While there is currently no evidence for a disease-causing role in the heterozygous state, compound heterozygosity of LOF variants along with missense variants can lead to extremely severe presentations, mimicking recessive inheritance.

**Additional information related to ACMG evidence types**

**PM2** A filtering allele frequency (FAF) <0.004% activates this rule
CAUTION: Population databases may contain presymptomatic individuals for diseases with reduced
penetrance/variable onset.

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
0.1%
Assumptions (note: values deliberately set conservative to add “safety padding”)
Disease prevalence: 1/200 individuals (1/400 chromosomes) was used as a “safe” value.
Penetrance: Although MYH7 is generally regarded as a “penetrant” cardiomyopathy gene, this is not well characterized at the variant level and therefore, to accommodate all variants, a penetrance value of 30% was used.
Gene contribution was set at 10.6% based on the detection rate for HCM, which is the highest among MYH7
associated cardiomyopathies4

**BS1** (MAF too high for disease) 0.02%
Assumptions
• Disease prevalence: 1/200 individuals (1/400 chromosomes)
• Penetrance: 30%
• Maximum pathogenic variant contribution: 2% based on MYBPC3 variant p.Arg502Trp (Walsh et al. 20175:6,000 probands)
• Note that the FAF (95% poisson) is available for each variant in ExAC (http://exac.broadinstitute.org/).

**PM1** Applicable when missense variant is located within the head domain (codons 181-937, NM_000257)
*Kelly MA et al 2018 PMID: 29300372* 

Walsh et al propose adaptation of ACMG/AMP guidelines for rule PM1 and HCM, relating to the relative frequencies of non-truncating variants in case cohorts and population controls. 
PM1_strong - EF >0.95
PM1_moderate - EF between 0.90 and 0.95 
PM1_supporting - EF between 0.80 and 0.90

**MYH7 (residues 167–931) etiological fraction 0.976 (0.972–0.981)** so for this region PM1_strong could be applied

*Walsh et al 2019 PMID: 30696458*

**List variant classes in this gene proven to cause this disease**

Missense

**List other variant classes predicted to lead to the same functional consequence**

splice_donor_variant

splice donor variant predicted to escape NMD

splice acceptor variant predicted to escape NMD

frameshift variant predicted to escape NMD

stop_gained predicted to escape NMD

stop_lost

inframe_insertion

inframe_deletion
**MYH7 - HCM**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen**: 

https://search.clinicalgenome.org/kb/genes/HGNC:7577

MYH7 HCM definitive (narrative not given)

**ClinGen Haploinsufficiency phenotype comments:**

To date only articles discussing missense mutations in the MYH7 gene have been described in the literature as being associated with Familial hypertrophic cardiomyopathy, dilated cardiomyopathy, Laing Distal Myopathy (LDM), Myosin Storage Myopathy (PMID:23346452, 12788380,
12707239, 21846512, 25576864,24664454 ). PMID: 12788380 does however describe a three base pair deletion leading to an amnio acid loss glutamic acid in position 927. There was no disruption of the reading frame downstream of the deletion.

**Literature review:**

"Missense variants in MYH7 are believed to cause HCM
through gain-of-function actions: variants produce an abnormal activated protein that incorporates into the sarcomere as a ‘poison peptide’ [2]. Haploinsufficiency in MYH7 is not
a recognised disease mechanism for HCM, and heterozygous variants that introduce premature termination codons (PTCs - i.e. nonsense, frameshift and splice variants) have not been demonstrated to be associated with this disease. 

Here we report a frameshift variant in MYH7, c.5769delG,
that is associated with HCM in an Egyptian cohort (3.3%) compared with ethnically-matched controls. This variant is absent from previously published large-scale Caucasian HCM cohorts. We further demonstrate strong evidence of co-segregation of c.5769delG with HCM in a large family (LOD score: 3.01). The predicted sequence of the variant MYH7 transcript shows that the frameshift results in a premature termination codon (PTC) downstream of the last exon-exon junction of the gene that is expected to escape nonsense-mediated decay (NMD). RNA sequencing of myocardial tissue obtained from a patient with the variant during surgical myectomy confirmed the expression of the variant MYH7 transcript."

*Allouba M et al 2020 submitted*

"MYH7 LOF variants are very rare and their contribution to inherited cardiomyopathy is incompletely understood. While there is currently no evidence for a disease-causing role in the heterozygous state, compound heterozygosity of LOF variants along with missense variants can lead to extremely severe presentations, mimicking recessive inheritance.25, 26"

*Kelly MA et al 2018 PMID: 29300372* 

**From our in-house Atlas of HCM:**

849/864  missense

10/864  in frame deletion

There are some unpublished truncating variants in the terminal exon which seem to be enriched in certain case cohorts and may escape NMD and also at least one splice region variant that turns out to cause an in-frame exon skipping event (at a relatively distal exon), leading to expression of an altered gene product structure.

https://www.cardiodb.org/acgv/acgv_gene.php?gene=MYH7

*Walsh R et al 2017 PMID:27532257*

**Pilot application of harmonised terms:**

**Inheritance:**

Autosomal dominant

Optional Modifiers: Incomplete penetrance

**Allelic requirement:**

Monoallelic_aut

Optional Modifiers:

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is primarily increase in sarcomere force generation due to **altered protein product structure**. Missense variants in MYH7 are believed to cause HCM "through gain-of-function actions: variants produce an abnormal activated protein that incorporates into the sarcomere as a ‘poison peptide’."

A novel frameshift variant has been identified in 3.3% of Egyptian HCM patients. It is predicted to result in a premature termination codon (PTC) downstream of the last exon-exon junction of the gene that is expected to escape nonsense-mediated decay (NMD).

MYH7 LOF variants are very rare and their contribution to inherited cardiomyopathy is incompletely understood. While there is currently no evidence for a disease-causing role in the heterozygous state, compound heterozygosity of LOF variants along with missense variants can lead to extremely severe presentations, mimicking recessive inheritance.

**Additional information related to ACMG evidence types**

**PM2** A filtering allele frequency (FAF) <0.004% activates this rule
CAUTION: Population databases may contain presymptomatic individuals for diseases with reduced
penetrance/variable onset.

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
0.1%
Assumptions (note: values deliberately set conservative to add “safety padding”)
Disease prevalence: 1/200 individuals (1/400 chromosomes) was used as a “safe” value.
Penetrance: Although MYH7 is generally regarded as a “penetrant” cardiomyopathy gene, this is not well characterized at the variant level and therefore, to accommodate all variants, a penetrance value of 30% was used.
Gene contribution was set at 10.6% based on the detection rate for HCM, which is the highest among MYH7
associated cardiomyopathies4

**BS1** (MAF too high for disease) 0.02%
Assumptions
• Disease prevalence: 1/200 individuals (1/400 chromosomes)
• Penetrance: 30%
• Maximum pathogenic variant contribution: 2% based on MYBPC3 variant p.Arg502Trp (Walsh et al. 20175:6,000 probands)
• Note that the FAF (95% poisson) is available for each variant in ExAC (http://exac.broadinstitute.org/).

**PM1** Applicable when missense variant is located within the head domain (codons 181-937, NM_000257)
*Kelly MA et al 2018 PMID: 29300372* 

Walsh et al propose adaptation of ACMG/AMP guidelines for rule PM1 and HCM, relating to the relative frequencies of non-truncating variants in case cohorts and population controls. 
PM1_strong - EF >0.95
PM1_moderate - EF between 0.90 and 0.95 
PM1_supporting - EF between 0.80 and 0.90

**MYH7 (residues 167–931) etiological fraction 0.976 (0.972–0.981)** so for this region PM1_strong could be applied

*Walsh et al 2019 PMID: 30696458*

**List variant classes in this gene proven to cause this disease**

Missense

**List other variant classes predicted to lead to the same functional consequence**

splice_donor_variant

splice donor variant predicted to escape NMD

splice acceptor variant predicted to escape NMD

frameshift variant predicted to escape NMD

stop_gained predicted to escape NMD

stop_lost

inframe_insertion

inframe_deletion
# **MYL2 -- Hypertrophic Cardiomyopathy**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen:**

<https://search.clinicalgenome.org/kb/gene-validity/8768>

The MYL2 gene has been associated with autosomal dominant hypertrophic cardiomyopathy (HCM) in at least 40 probands in 10 publications (Poetter et al, 1996, PMID 8673105; Flavigny et al, 1998, PMID 9535554; Kabaeva et al, 2002, PMID 12404107; Richard et al, 2003, PMID 12707239; Morner et al, 2003, PMID 12818575; Garcia-Pavia et al, 2011, PMID 21896538; Santos et al, 2012, PMID 22429680; Berge et al, 2013, PMID 24111713; Lopes et al, 2015, PMID 25351510; Claes et al, 2015, PMID 26497160).
**More than 8 unique variants (missense, splice site, nonsense,
frameshift) have been identified in humans, and convincing segregation data has been reported** (Flavigny et al, 1998, PMID 9535554). In addition, at least 8 missense VUSs in MYL2 have been reported in patients with HCM. MYL2 was first associated with HCM in humans in 1996 (Poetter et al, 1996, PMID 8673105). The MYL2 gene was significantly enriched for missense variants in Walsh et al, 2016 (PMID 27532257) with and odds ratio of 6.74 (95% CI 4.69-9.70) for non-truncating variants and 3.10 (95% CI 0.89-10.7) for truncating variants. This gene-disease
association is supported by expression data (Price et al, 1980, PMID 7236212), interaction with other known HCM gene products (MYH7, MYL3) (Rayment et al, 1993, PMID 8316857), and animal models (Szczesna-Cordary et al, 2005, PMID 16076902; Wang et al, 2006, 16837010; Kerrick et al, 2009, PMID 18987303). In summary, MYL2 is definitively associated with autosomal dominant HCM. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This classification was approved by the ClinGen Hypertrophic
Cardiomyopathy Gene Curation Expert Panel on February 7th, 2017.

**ClinGen Haploinsufficiency phenotype comments:**
MYL2 is associated with cardiomyopathy by different pathogenic mechanisms with different modes of inheritance that can be recessive or dominant, depending on genotype. Homozygous or compound heterozygous loss-of -function variants have been reported to cause autosomal recessive form cardiomyopathy. Furthermore, the heterozygous carriers did not show evidence of the disorder (PMID: 24111713, 23365102). Other missense changes cause autosomal dominant hypertrophic cardiomyopathy presumably due to loss of the normal physiologic function, for instance, loss of the contraction power stoke or altered actin sliding velocities (PMID: 25825243; 25324513). One paper describes a founder missense pathogenic variant capable of causing HCM by itself or augmented by other factor such as arterial hypertension (PMID: 26497160). MYL2 variants, among others, identified as incidental findings in an exome cohort (PMID: 23861362). There are several heterozygous frameshifts, stop gains, and splice site disruptions listed in ExAc for MYL2 implying that haploinsufficiency is not a disease mechanism for this gene.

**Literature review:**

[[Poetter et al. (1996)]analyzed the MYL2 gene in 399 unrelated probands with hypertrophic cardiomyopathy
(see CMH10, [[608758]{.ul}](https://omim.org/entry/608758)), and
identified heterozygosity for 3 different missense mutations in 4
probands ([[160781.0001]{.ul}](https://omim.org/entry/160781?search=MYL2&highlight=myl2#0001)-[[160781.0003]{.ul}](https://omim.org/entry/160781?search=MYL2&highlight=myl2#0003)),
3 of whom had an unusual mid-left ventricular chamber thickening on echocardiography.

Flavigny et al. (1998) screened 42 probands from unrelated families with CMH for mutations in the MYL2 gene and identified 2 novel mutations, R58Q (160781.0004) and P18L (160781.0005), in 3 probands. The mutations were subsequently found in all affected family members, who were classified morphologically as Maron type 1, 2, or 3; none had the variant form of CMH described by Poetter et al. (1996). 

Szczesna et al. (2001) studied the effects of 5 mutations in the MYL2 gene on Ca(2+) binding and phosphorylation and found that both processes were significantly affected by all of the mutations. For example, the E22K mutation resulted in a 17-fold decrease in calcium binding compared with wildtype, and the R58Q mutant did not bind Ca(2+) at all. Ca(2+) binding to the R58Q mutant was restored upon phosphorylation, whereas the E22K mutant could not be phosphorylated. In addition, the alpha-helical content of phosphorylated R58Q greatly increased with Ca(2+) binding.

*OMIM https://www.omim.org/entry/160781#molecularGenetics*

*Poetter et al, 1996, PMID 8673105*
*Flavigny et al 1998 PMID 9535554*
*Szczesna et al 2001 PMID 11102452*

**From our in-house Atlas of HCM:**
There is a significant excess of non-truncating variants in MYL2 in HCM cases vs population controls. The excess of truncating variants is not statistically significant.

42/46 missense

1/46 inframe deletion (VUS)

2 nonsense (VUS)

1 frameshift (VUS)

The frameshift variant is described as 'hot VUS'. There are 2
submissions in ClinVar. It is expected to cause a premature truncatation codon and result in a disrupted or absent protein. As LoF is not an established mechanism of disease it has been classed as a VUS.

1 nonsense variant is also classed as a 'hot VUS' for similar reasons and has 4 submissions on ClinVar.

<https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=MYL2&icc=HCM>

*Walsh et al, 2016 PMID 27532257*

Homozygous or compound heterozygous mutations in the last exon of MYL2 have been reported as responsible for an autosomal recessive lethal myosinopathy characterised by rapidly progressive generalised muscle weakness and heart failure due to primarily dilated cardiomyopathy (but in some cases restrictive or septal hypertrophy).

*Weterman et al 2013. PMID 23365102*

**Inheritance**

Autosomal dominant

Optional modifiers: incomplete penetrance

**Allelic requirement**

Monoallelic_aut

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is primarily due to **altered gene product structure** as the main class of pathogenic variants described are missense. There have been a few reports of nonsense and frameshift variants but as loss of function does not appear to be the mechanism, their pathogenicity has not been confirmed. Inheritance is primarily autosomal dominant with incomplete penetrance.

**Additional information related to ACMG evidence types**

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
0.1% (het) 
3.16% (hom) 

**BS1** (MAF too high for disease) 0.02%
Assumptions
• Disease prevalence: 1/200 individuals (1/400 chromosomes)
• Penetrance: 30%
• Maximum pathogenic variant contribution: 2% based on MYBPC3 variant p.Arg502Trp (Walsh et al. 20175:6,000 probands)
• Note that the FAF (95% poisson) is available for each variant in ExAC (http://exac.broadinstitute.org/).

**PM2** A filtering allele frequency (FAF) less than 0.004% activates this rule
CAUTION: Population databases may contain presymptomatic individuals for diseases with reduced
penetrance/variable onset.

*Kelly MA et al 2018 PMID: 29300372* 

*Whiffin N et al 2018 PMID: 29369293*

Walsh et al propose adaptation of ACMG/AMP guidelines for rule PM1 and HCM, relating to the relative frequencies of non-truncating variants in case cohorts and population controls. 
PM1_strong - EF >0.95
PM1_moderate - EF between 0.90 and 0.95 
PM1_supporting - EF between 0.80 and 0.90

**MYL2 across whole gene etiological fraction is 0.890 (0.851–0.930)** so for this region PM1_supporting could be applied

*Walsh et al 2019 PMID: 30696458*

**List variant classes in this gene proven to cause this disease:**

- Missense

**List other variant classes predicted to lead to the same functional consequence**

- Frameshift predicted to escape NMD
- Stop_gained predicted to escape NMD
- Splice donor variant
- Splice donor variant predicted to escape NMD
- Splice acceptor variant predicted to escape NMD
- Stop_lost
- In frame_insertion
- In frame deletion
# **MYL3 -- Hypertrophic Cardiomyopathy**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen:**

<https://search.clinicalgenome.org/kb/gene-validity/8499>

The MYL3 gene has been associated with autosomal dominant hypertrophic cardiomyopathy (HCM) in at least 20 probands in 13 publications. More than 8 unique variants (mostly missense, 1 splice acceptor variant) have been reported in humans, and variants in this gene segregated with disease in 24 additional family members. MYL3 was first associated with HCM in humans in 1996 (Poetter et al, PMID 8673105). The MYL3 gene was
significantly enriched for missense variants in Walsh et al. 2016 (PMID 27532257), with an Odds Ratio of 5.00 (3.43-7.27) for HCM. This gene-disease association is supported by expression studies (Fujimoto et al, 1993, PMD 8417110), a mouse model (Vemuri et al, 1991, PMID 9927691), and evidence of interaction with MYH7 (Petzhold et al, 2011, PMID 21262909) as well as ACTC1 (Haase et al, 2006, PMID 16675844). In summary, MYL3 is definitively associated with autosomal dominant HCM. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This classification
was approved by the ClinGen Hypertrophic Cardiomyopathy Gene Curation Expert Panel on December 22, 2017.

**ClinGen Evidence for Haploinsufficiency**
**"There is no evidence at present to show that MYL3 variants have a loss of function effect.**

MYL3 is associated with dominant hypertrophic cardiomyopathy. Described pathogenic missense variants in the MYL3 gene cause the disorder by altering its normal physiologic function (e.g: impairing essential interaction with other components of the sarcomere) (PMID: 22131351 ). There is one report of a case with recessive inheritance due to autozygosity. The variant reported in recessive HCM was a homozygous missense change (p.Glu143Lys). Functional studies were not performed (PMID: 12021217, 21823217). ClinVar classifies the Glu143Lys variant as "uncertain" and reports it found in individuals with pathogenic changes in other genes. One report describes a splice site alteration predicted to cause exon skipping and no parental studies were done (PMID: 19035361). One frameshift variant in the middle of the gene has been described but it is unclear if it was causative of HCM or not (PMID: 25132132), and there are two instances of frameshift and one of an early canonical splice site change in ExAc (all at very low freq). There are three transcripts for MYL3 with different reading frames at the frameshift locations. The Database of Genomic Variants (DGV) lists a number of deletion CNVs including MYL3 and reported by multiple groups."

**Literature review:**

**Pathogenic missense variants are described in MYL3, likely causing disease by altering normal function and impairing protein protein interaction with components of the sarcomere. There are reports on ClinVar of frameshift, UTR and splice site variants but these are classified as uncertain significance or conflicting.
There is a splice acceptor variant classified as VUS-favour pathogenic but functional studies are unavailable.**

"Olson et al. (2002) reported a consanguineous family in which 3 sibs had presented with childhood-onset CMH characterized by midcavitary left-ventricular hypertrophy (CMH8; 608751). Both parents had completely normal hearts in their 40s. Mutation screening in a surviving affected sib revealed a homozygous missense G-to-A point mutation at codon 143 of the MYL3 gene, resulting in a glutamic acid-to-lysine (E143K) substitution. Heterozygotes had normal hearts. Sequence alignment of myosin essential light chains demonstrated high conservation of glutamic acid at position 143 across species. The E143K mutation was absent from 150 normal control DNA samples. **The authors concluded that this was a true autosomal recessive form of CMH8.**"
*Omim https://www.omim.org/entry/160790*

The same variant has been described in conjunction with pathogenic variants in other HCM genes. Additionally the same variant has been identified in the heterozygous state in \>35 individuals (primarily Latino ancestry) with HCM. See ClinVar for additional evidence and publications (*Gomez 2014,
McNamara 2017*) https://www.ncbi.nlm.nih.gov/clinvar/variation/14063/.

*Olson 2002 PMID 12021217; Caleshu 2011 PMID 21823217*

"[Poetter et al.(1996)]analyzed the MYL3 gene in 383 unrelated probands with hypertrophic cardiomyopathy and identified a
heterozygous missense mutation at a conserved residue
(M149V; [160790.0001] that segregated with disease in a large 3-generation family."

*Omim https://www.omim.org/entry/160790* 

**From our in-house Atlas of HCM:**

37/37 missense

[[https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=MYL3&icc=HCM]{.ul}](https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=MYL3&icc=HCM)

*Walsh et al. 2016 (PMID 27532257)*

**Inheritance**

Autosomal dominant

Optional modifiers: incomplete penetrance

**Allelic requirement**

Monoallelic_aut

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is primarily due to **altered gene product structure.** The pathogenic variants described are missense variants likely causing disease by altering normal function and impairing protein protein interaction with components of the sarcomere. There are reports on ClinVar of frameshift, UTR and splice site variants but these are classified as uncertain significance or conflicting. The MYL3 gene was significantly enriched for missense variants in Walsh et al. 2016 (PMID 27532257), with an Odds Ratio of 5.00 (3.43-7.27) for HCM. There was no enrichment in cases for truncating variants.

**Additional information related to ACMG evidence types**

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
0.1% (het) 
3.16% (hom) 

**BS1** (MAF too high for disease) 0.02%
Assumptions
• Disease prevalence: 1/200 individuals (1/400 chromosomes)
• Penetrance: 30%
• Maximum pathogenic variant contribution: 2% based on MYBPC3 variant p.Arg502Trp (Walsh et al. 20175:6,000 probands)
• Note that the FAF (95% poisson) is available for each variant in ExAC (http://exac.broadinstitute.org/).

**PM2** A filtering allele frequency (FAF) less than 0.004% activates this rule
CAUTION: Population databases may contain presymptomatic individuals for diseases with reduced
penetrance/variable onset.

*Kelly MA et al 2018 PMID: 29300372* 

*Whiffin N et al 2018 PMID: 29369293*

**PM1** 

Walsh et al propose adaptation of ACMG/AMP guidelines for rule PM1 and HCM, relating to the relative frequencies of non-truncating variants in case cohorts and population controls. 
PM1_strong - EF >0.95
PM1_moderate - EF between 0.90 and 0.95 
PM1_supporting - EF between 0.80 and 0.90

**MYL3 HCM cluster (amino acid residues 143–180) etiological fraction 0.925 (0.886–0.965)** so for this region PM1_moderate could be applied

EF across the whole gene is 0.833 (0.772–0.895)

*Walsh et al 2019 PMID: 30696458*

**List variant classes in this gene proven to cause this disease:**

- Missense

**List other variant classes predicted to lead to the same functional consequence**

- Frameshift predicted to escape NMD
- Stop_gained predicted to escape NMD
- Splice donor variant
- Splice donor variant predicted to escape NMD
- Splice acceptor variant predicted to escape NMD
- Stop_lost
- In frame_insertion
- In frame deletion
**NF2 -- Neurofibromatosis, type 2 (MIM 101000)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:7773

The first report indicating a relationship of the NF2 gene with
autosomal dominant Neurofibromatosis type 2 was reported by Rouleau et al., 1993 (PMID: 8379998). Numerous variants have been reported, in both ClinVar and in LOVD (https://databases.lovd.nl/shared/genes/NF2).
Evidence supporting this gene-disease relationship includes case-level data, segregation data, functional data, and model organisms. This gene-disease relationship has been studied for more than 20 years, therefore a significant amount of case-level data, segregation data, and experimental data is available and the maximum score for genetic evidence (12 points) and experimental evidence (6 points) has been
reached... The mechanism for the gene-disease relationship is **protein loss of function**, as the NF2 protein product, termed Merlin or Schwannomin, is a **tumor suppressor protein** (Trofatter et al., 1993;
PMID: 8453669). NF2 tumor suppressor function is regulated by the
protein confirmation, with the closed form acting as the active tumor
suppressor form (reviewed in Cooper and Giancotti, 2014 PMID:24726726).
NF2 has been associated with multiple disease entities and/or
phenotypes, including: (1) Meningioma, NF2-related, somatic (MIM:
607174) (2) Schwannomatosis, somatic Of note, these other phenotypes are
noted as somatic, and therefore are not represented/ counted in this
gene-disease relationship, and will be assessed separately. In summary,
**NF2 is DEFINITIVELY associated with autosomal dominant
Neurofibromatosis Type 2**. This has been repeatedly demonstrated in
both the research and clinical diagnostic settings, and has been upheld
over time. This classification was approved by the ClinGen General Gene
Curation Expert Panel on February 27, 2019.

**Literature Review:**

**OMIM**: https://www.omim.org/entry/101000

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1201/

Pathogenic variants. At least 400 NF2 pathogenic variants have been
described, with **missense, nonsense, and splicing variants and small deletions being the most common** (Table A, HGMD and Locus-Specific
Databases). A wide variety of pathogenic variants have been identified
in **all NF2 exons, except for the alternatively spliced exons.** 90% of
single-nucleotide variants are predicted to **truncate the protein** by
introduction of a **premature stop codon, a frameshift with premature
termination, or a splicing alteration**, supporting the view that loss
of the protein\'s normal function is necessary for the development of
tumors. C-to-T transitions in CGA codons causing pathogenic nonsense
variants are an especially frequent occurrence. Fewer than 10% of
detected pathogenic variants are **in-frame deletions and missense
variants**, which may indicate that alteration of particular functional
domains can abolish the NF2 **tumor suppressor activity** \[Baser et al
2006\].

Attempts to identify truncated protein product have been unsuccessful in the main, although the **non-truncated product from pathogenic missense variants may have partial function**. It is thought that nonsense-mediated decay may account for the lack of identifiable product from most variant types; however, this does not explain why **phenotypes are more severe for this type of variant than for whole-gene deletions**.

analyzed the **mosaic risk in de novo patients** with NF2 by age at the time of vestibular schwannoma diagnosis....... The risk of NF2 to an offspring of a patient presenting with bilateral vestibular schwannoma at less than 20 years of age was 29.3%, whereas the offspring risk for a
patient presenting with asymmetric disease after 40 years of age was
only 5.5%, as there is a 99% chance that they are mosaic.

***Evans and Wallace, 2009, PMID 19880713***

reported 5 NF2 patients with **constitutional rearrangements of
chromosome 22** and vestibular schwannomas, multiple intracranial
meningiomas, and spinal tumors.

***Tsilchorozidou et al , 2004, PMID 15235024***

Sixty-seven individuals (56.2%) from 41 of these kindreds revealed 36
different putative disease-causing mutations. These include 26 proposed
protein-truncating alterations (frameshift deletions/insertions and
nonsense mutations), 6 splice-site mutations, 2 missense mutations**, 1
base substitution in the 3\' UTR of the NF2 cDNA, and a single 3-bp
in-frame insertion.** Seventeen of these mutations are novel, whereas
the remaining 19 have been described previously in other NF2 individuals
or sporadic tumors.... Twenty-four of 28 patients with mutations that
cause premature truncation of the NF2 protein, schwannomin, present with
severe phenotypes. In contrast, all 16 cases from three families with
mutations that affect only a single amino acid have mild NF2. These data
provide conclusive evidence that a phenotype/genotype correlation exists
for certain NF2 mutations.

***Ruttledge, 1996, PMID: 8755919***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

Optional modifiers: mosaicism

**Disease associated variant consequences:**

*Dose change* -decreased gene product level

Altered protein product

**Narrative summary of molecular mechanisms:**

NF2 is a tumour suppression and mechanism of disease is principally loss of function, through protein truncation, haploinsufficiency and protein alteration. Classes of pathogenic variants include frameshift, deletions/insertions, nonsense, splice-site, missense mutations.
Less frequently, 3'UTR and inframe insertions have been reported.

**List variant classes in this gene proven to cause this disease:**

splice_region_variant

splice_acceptor_variant

(splice_acceptor_variant predicted to undergo NMD)

splice_donor_variant

(splice_donor_variant predicted to undergo NMD)

Frameshift_variant

(frameshift_variant predicted to undergo NMD)

Stop_gained

(stop_gained predicted to undergo NMD)

Inframe_deletion

Inframe_insertion

Missense

**Potential novel variant classes based on predicted functional
consequence**

start_lost

(stop_gained predicted to escape NMD)

(splice_acceptor_variant predicted to escape NMD)

(splice_donor_variant predicted to escape NMD)

(frameshift_variant predicted to escape NMD)

stop_lost

5_prime_UTR_variant

3_prime_UTR_variant

(gain of upstream Start \[uORF\])

(gain of upstream Start \[oORF\])

(Stop lost \[uORF\])

(Stop lost \[oORF\])

(Start lost \[uORF\])

(Frameshift \[uORF\])

(Frameshift \[oORF\])

(Stop gained \[uORF\])
**OTC - Ornithine carbamoyltransferase deficiency (MIM 311250)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:8512

The relationship between OTC and ornithine carbamoyltransferase
deficiency (X-linked) was evaluated using the ClinGen Clinical Validity
Framework as of February, 2019. The OTC gene is located on the X chromosome and encodes a mitochondrial matrix enzyme that catalyzes the second step of the urea cycle. **Most patients with OTC deficiency are
hemizygous males, but female carriers can also be symptomatic**.
Variants in OTC were first reported in humans with this disease as early
as 1985 (Rozen et al., PMID 2983225). More than 400 variants
(**missense, nonsense, frameshift, in-frame indels, and large
deletions**) have been reported in humans (Caldovic et al., 2015, PMID
26059767). Evidence supporting this gene-disease relationship includes
case-level and experimental data. More evidence is available in the
literature, but the maximum score for genetic evidence (12 pts.) has
been reached. This gene-disease relationship is supported by biochemical
assays, expression studies, and model systems. In summary, OTC is
definitively **associated with X-linked ornithine carbamoyltransferase
deficiency .** This has been repeatedly demonstrated in both the
research and clinical diagnostic settings, and has been upheld over
time.

Gene Clinical Validity Standard Operating Procedures (SOP) - Version 6

**Loss of function mutations result in OTC deficiency**, a common type
of urea cycle disorder. Males are always affected and 15% of carrier
females will have hyperammonemia at some point in their lifetimes and
may have cognitive symptoms even in the absence of hyperammonemia. See
GeneReviews for Urea Cycle Disorders.

The loss-of-function and triplosensitivity ratings for genes on the X
chromosome are made in the context of a male genome to account for the
effects of hemizygous duplications or nullizygous deletions. In
contrast, disruption of some genes on the X chromosome causes male
lethality and the ratings of dosage sensitivity instead take into
account the phenotype in female individuals. Factors that may affect the
severity of phenotypes associated with X-linked disorders include the
presence of variable copies of the X chromosome (i.e. 47,XXY or 45,X)
and skewed X-inactivation in females.

There have been no reports of focal duplications of OTC.

**Literature Review:**

**OMIM:** https://www.omim.org/entry/311250

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK154378/

*Pathogenic variants.*

Pathogenic variants. More than 400 pathogenic OTC variants have been
described in the literature \[Bailly et al 2015, Caldovic et al 2015,
Choi et al 2015, Gao et al 2015, Mohamed et al 2015, Prasun et al
2015\].

Additionally, 972 OTC variants, pathogenic and non-pathogenic, have been
collected and curated in the Leiden Open Variation Database.

Types of pathogenic variants:

About 17% of reported pathogenic variants are **nonsense variants, insertions, or deletions in the coding region that result in truncated protein and complete absence of functional OTC** \[Caldovic et al 2015,
Choi et al 2015, Gao et al 2015, Prasun et al 2015\]. An additional 2%
of reported pathogenic variants are **in-frame deletions and insertions
that disrupt structure of the OTC enzyme although they do not affect the
reading frame** of the OTC coding sequence \[Caldovic et al 2015\].

About **55% of reported pathogenic variants are missense variants**
\[Bailly et al 2015, Caldovic et al 2015, Choi et al 2015, Gao et al
2015\]. Although deleterious amino acid substitutions have been found
throughout the OTC protein coding region, exons 1 and 7 appear to have
fewer deleterious missense variants than other exons. This is possibly
because they encode the mitochondrial targeting peptide \[Horwich et al
1984\] and surface residues \[Shi et al 1998\], which tend to be less
conserved and more tolerant to amino acid substitutions.

About 13% of reported pathogenic variants affect **splicing of OTC
mRNA** \[Caldovic et al 2015, Mohamed et al 2015\].

An **OTC promoter variant** that disrupts interaction between promoter
and enhancer and results in reduced levels of functional mRNA and
protein has been reported \[Luksan et al 2010\].

About 12% of reported variants are **structural rearrangements
(deletions and duplications) within the OTC locus that result in
complete absence of functional OTC** \[Caldovic et al 2015, Choi et al
2015, Di Stefano et al 2015, Gallant et al 2015\].

There are no prevalent, population-specific OTC pathogenic variants
(i.e., no founder effect).

*Abnormal gene product.*

Pathogenic variants that affect mRNA splicing result either in **a
defective OTC transcript or reduced levels of functional transcript
leading to complete absence or reduced abundance of functional OTC
enzyme**.

Pathogenic variants of the consensus intron splice sequences and the
last base pair of exons 1-9 almost always result in the absence of
functional mRNA due to either absence of splicing or nonsense mediated
decay of aberrantly spliced OTC mRNA \[Tuchman et al 2002\].

Pathogenic variants that result in **creation of novel splice sites have
been found deep in OTC introns leading to aberrant splicing** and
reduced levels of functional OTC mRNA and protein \[Engel et al 2008\].

The effect of pathogenic missense variants on OTC folding and activity
depends on the chemical properties of the amino acid that is replacing
the original residue. Substitution of amino acids located either in the
active site or the protein\'s hydrophobic core result in absence of
functional enzyme due to either lack of enzymatic activity or inability
to fold \[Yamaguchi et al 2006\].

Substitution of amino acids located on the surface of the OTC protein or
remote from the active site result in partially functional enzyme due to
either reduced stability or enzymatic activity \[Yamaguchi et al 2006\].

Of the known point mutations, 27 are single base substitutions: **17
missense, 6 nonsense, 4 splice site, and the remaining 2 are single base
deletions**.

*Jang et al, 2018, PMID: 29282796*

More than 350 different mutations, including **missense, nonsense,
splice-site changes, small de-letions or insertions and gross
deletions**, have been describ-ed so far.

*Paprocka et al, 2013, PMID: 23821427*

Molecular analysis identified 37 different mutations (**22 missense, 5
large deletions, 4 small deletions, 1 insertion,3 nonsense and 2 splice
sites**) from all 49 patients; the mutations were dispersed throughout
all coding exons. In

*Choi et al, 2015, PMID: 25994866*

*Arn et al. (1989)* discussed phenotypic effects of heterozygosity for
mutations in the OTC gene. *Arn et al. (1990)* reported that **otherwise
normal women who are carriers of a mutant OTC allele are at increased
risk for hyperammonemic** coma, especially during puerperium.

Examined the genotype/phenotype correlations of 157 probands with OTC
deficiency, including 57 heterozygous females. In patients with
mutations that abolished enzyme activity, the severe clinical and
biochemical phenotype was homogeneous. The males in this group presented
within the first few days of life with high mortality and morbidity.
Most patients with the late-onset form had missense mutations in the OTC
gene, although a few had 3-bp deletions, and late-onset patients had
residual enzyme activity ranging from 26 to 74% of normal control
values. **Mutations in manifesting females were primarily of the
neonatal-onset type**. Substitutions occurring in CpG dinucleotides
accounted for approximately 31% of all mutations

*McCullough et al, 2000, PMID
[10946359])*

**Pilot application of harmonised terms:**

**Inheritance:**

*X-linked* - Primarily recessive (with milder female expression)

**Allelic requirement:**

Monoallelic_X_hem    

**Disease associated variant consequences:**

*Dose change* -decreased gene product level

*Dose change* - absent gene product

*Altered gene product structure*

**Narrative summary of molecular mechanisms:**

Pathogenic variants that affect mRNA splicing result either in a
defective OTC transcript or reduced levels of functional transcript
leading to complete absence or reduced abundance of functional OTC
enzyme. Variant classes include missense, nonsense, splice-site, small
indels and gross deletions. Smaller numbers of deeply intronic (creating
cryptic splice sites) and promotor variants have been described. Males
are always affected and 15% of carrier females will have hyperammonaemia
at some point in their lifetimes and may have cognitive symptoms even in
the absence of hyperammonaemia.

**List variant classes in this gene proven to cause this disease:**

Missense

Frameshift variant

Inframe deletions

Inframe insertion

Stop gain

Stop gained predicted to undergo NMD

Spice acceptor variant

Splice donor variant

Splice acceptor variant predicted to undergo NMD

Splice donor variant predicted to undergo NMD

**Potential novel variant classes based on predicted functional
consequence**

Splice region variant

Splice acceptor variant predicted to escape NMD

Splice donor variant predicted to escape NMD

Frameshift variant predicted to escape NMD

Frameshift variant predicted to undergo NMD

start_lost

stop_gained predicted to escape NMD

stop_lost

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[oORF\]

Frameshift \[oORF\]
# **PCSK9 -- Familial Hypercholesterolemia**

**Review of source material:**

**ClinGen:**

https://search.clinicalgenome.org/kb/genes/HGNC:20001

The relationship between PCSK9 and familial hypercholesterolemia (autosomal dominant) was evaluated using the ClinGen Clinical Validity Framework as of November 14, 2018. Variants in PCSK9 were first reported in humans with this disease as early as 2003 (Abifadel et al., PMID: 12730697). **At least 15 variants (missense) have been reported in humans.** Evidence supporting this gene-disease relationship includes case-level data, segregation data, and experimental data. **Variants in this gene have been reported in at least 17 probands in 6 publications (PMIDs: 12730697, 15772090, 22683120, 26541928, 29127338, 20006333). Variants in this gene segregated with disease in at least 31 family members. The mechanism for disease is heterozygous gain of function (Maxwell et al., 2005; PMID: 16577715), while heterozygous loss of function variants are associated with low levels of LDL cholesterol** (Cohen et al., 2005, PMID: 15654334). This gene-disease association is supported by animal models, expression studies, and in vitro functional assays. In summary, PCSK9 is definitively associated with autosomal dominant familial hypercholesterolemia. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time.
Gene Clinical Validity Standard Operating Procedures (SOP) - Version 7

**ClinGen evidence for haploinsufficiency**

Nonsense mutations in PCSK9 appear to be polymorphic and confer susceptibility to low LDL cholesterol (hypocholesterolemia) whereas dominant missense mutations predispose to high LDL cholesterol. There are also multiple reports of deletion CNVs at or near (and could be extending into depending on array resolution) PCSK9 in DGV. There are many PCSK9 frameshift and stop gain changes listed in ExAc (one at high frequency = 33 alleles = p.Tyr142Ter, rs67608943).

**Literature review:**

"we have shown that overexpression of PCSK9 in HepG2 cells leads to accelerated degradation of the LDLR by a nonproteasomal mechanism in a post-ER, pH-sensitive compartment...It is also not clear why PCSK9 overexpression in mice results in a similar phenotype to humans carrying PCSK9 missense mutations. The simplest interpretation is that these are gain-of-function mutations..."

*Maxwell KN et al. 2005 Feb 8 (PMID:15677715)*

**"In 2003, gain-of-function mutations in a third gene, encoding for PCSK9, were identified as a cause of FH *Abifadel M et al. 2003 Jun (PMID:12730697)*.** PCSK9, when forming a complex with the LDLR, is internalized by modification of the LDLR confirmation and interferes with LDLR recycling. This leads to LDLR degradation and therefore reduction of the amount of receptors available at the hepatocyte surface to bind circulating LDL particles"

*Hartgers M et al 2015 (PMID: 26482752)*

"Missense mutations in PCSK9 that cause a gain-of-function lead to a rare form of FH. Loss-of-function in certain ethnic populations has been shown to result in lower LDL-C levels and protect against CHD [28]."

*Henderson R et al 2016 (PMID: 27084339)*

The PCSK9 protein product binds to LDL lipid receptors and promotes their degradation in intracellular acidic compartments.

Pathogenic variants in this gene have been associated both with hypercholesterolemia and hypocholesterolemia.

**Gain-of-function pathogenic variants cause hypercholesterolemia by excessive degradation of LDLRs, reducing the amount of LDL-C removed from the blood.**

Loss-of-function pathogenic variants cause hypocholesterolemia (reduced blood cholesterol levels) by increasing the number of LDLRs on the surface of liver cells, resulting in a quicker than usual removal of LDL-C from the blood and reduced incidence of coronary artery disease [Cohen et al 2006, Pandit et al 2008].

Penetrance is approximately 90% in persons heterozygous for the c.381T>A (p.Ser127Arg) pathogenic variant.
Penetrance in persons heterozygous for the p.Asp374Tyr pathogenic variant is high, with FH manifesting at a young age [Naoumova et al 2005].
Penetrance for other heterozygous PCSK9 pathogenic variants remains largely unknown [Cariou et al 2011].

***Gene reviews** https://www.ncbi.nlm.nih.gov/books/NBK174884 #hyperchol.Molecular_Genetics*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

incomplete penetrance

**Allelic requirement:**

Monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The mechanism appears to be missense variants causing gain of function of PCSK9 resulting in an **altered gene product** which leads to excessive degradation of LDLRs, reducing the amount of LDL-C removed from the blood. Loss-of-function pathogenic variants cause hypocholesterolemia (reduced blood cholesterol levels) by increasing the number of LDLRs on the surface of liver cells. Penetrance is approximately 90% in persons heterozygous for the c.381T>A (p.Ser127Arg) pathogenic variant. Penetrance in persons heterozygous for the p.Asp374Tyr pathogenic variant is high, with FH manifesting at a young age [Naoumova et al 2005]. Penetrance for other heterozygous PCSK9 pathogenic variants remains largely unknown [Cariou et al 2011].

**List variant classes in this gene proven to cause this disease:**

- Missense

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant predicted to escape NMD
- Splice donor variant predicted to escape NMD
- Frameshift predicted to escape NMD
- Stop_gained predicted to escape NMD
- Stop_lost
- In frame deletion
- In frame insertion# **PKP2 -- Arrythmogenic Right Ventricular Cardiomyopathy**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:9024

The relationship between PKP2 and arrhythmogenic right ventricular
cardiomyopathy (autosomal dominant) was evaluated using the ClinGen Clinical Validity Framework as of September, 2018. Variants in PKP2 were first reported in humans with this disease as early as 2004 (Gerull et al., PMID: 15489853). PKP2 is the major causative gene for ARVC and accounts for 34%-74% of cases (McNAlly et al., 2005; PMID: 20301310).
**There are over 250 PKP2 variants listed in ClinVar for ARVC (missense, nonsense, frameshift, complex rearrangements,** etc) (Novelli et al., 2018; PMID: 30619891). This gene-disease relationship is well-known and therefore a significant amount of case-level data, segregation data and experimental data is available in the literature, therefore the maximum
score for both genetic evidence and experimental evidence has been
reached. Note, this curation effort may not be exhaustive of all
literature related to this gene-disease relationship. This gene-disease relationship is supported by animal models, in vitro assays, and protein interactions. In summary, PKP2 is definitively associated with autosomal dominant ARVC. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This classification was approved by the ClinGen Arrythmogenic Right Ventricular Cardiomyopathy Gene Curation Expert Panel on October,
26, 2018 (SOP Version 6).
(https://www.clinicalgenome.org/curation-activities/gene-disease-validity/educational-and-training-materials/standard-operating-procedures/)

**Literature review:**

"...More than 170 pathogenic variants have been
described, including gross deletions \[[Li Mura et al
2013], [Roberts et al 2013] 

Multiple instances of [digenic] inheritance
have been identified with one [pathogenic variant] in *PKP2* and
a second in another desmosomal [gene]\[[Cox
et al 2011] [Bao et al 2013][Bhonsale et al 2015][Groeneweg et al
2015].

Abnormalities in plakophilin are thought to perturb intercellular connections and lead to arrhythmia.

The estimate of digenic/biallelic inheritance varies from 4% to 47% [[Xu et al 2010] [Bao et al 2013] [Rigato et al 2013] [Bhonsale et al 2015] [Groeneweg et al 2015] 
In most studies, digenic/biallelic carriers have a more severe
arrhythmia [phenotype].

[Rigato et al [2013\] examined a cohort of 113 individuals of whom 84% had a single desmosomal [gene]variant and 16% had [compound
heterozygous]or [digenic]inheritance. Over an observation period that averaged 39 years, 16% had major arrhythmic events. Specific risk factors for having events were male sex and having multiple genetic variants."

*Gene reviews
<https://www.ncbi.nlm.nih.gov/books/NBK1131/>*

"Direct sequencing of 5 AC genes in 71 unrelated patients with AC identified 10 different PKP2 mutations in 12 index patients. One patient, heterozygous for a PKP2 nonsense mutation, developed severe heart failure and underwent cardiac transplantation. Western blotting and immunohistochemistry of the explanted heart showed a significant decrease in PKP2 protein expression without detectable amounts of truncated PKP2 protein. Cultured keratinocytes of the patient showed a similar reduction in PKP2 protein expression. Nine additional PKP2 mutations were investigated in both cultured keratinocytes and endomyocardial biopsies from affected individuals. **It was evident that PKP2 mutations introducing a premature termination codon in the reading frame were associated with PKP2 transcript and protein levels reduced to ≈50%, whereas a missense variant did not seem to affect the amount of PKP2 protein."**

*Rasmussen TB et al 2014 PMID: 24704780*

"...both truncating and missense mutations in the desmosome genes PKP2 (encoding plakophilin 2), DSG2 (encoding desmoglein 2) and DSC2 (encoding desmocollin 2) have been identified in patients with ACM.  PKP2 is the most commonly affected gene in adult cohorts...
**The molecular links between desmosome mutations and the pathological hallmarks of ACM --- cardiomyocyte loss, fibrosis, adipogenesis, inflammation and arrhythmogenesis --- remain poorly defined. Probable pathogenic mechanisms include loss of mechanical integrity at cell--cell junctions, altered signalling pathways at intercalated discs, disruption of ion channels and gap junctions, and aberrant protein trafficking."**

*Austin KM et al. 2019. PMID: 31028357*

**The ARVC variant database can be found at https://molgenis136.gcc.rug.nl/**

In 2019, Ye et al reevaluated ARVC variants using large population databases. "...more than 10% of variants previously
reported to cause ARVC were found unlikely to be associated with highly penetrant monogenic forms of ARVC."

There were some variants that were found in population databases but were nevertheless associated with serious cardiac phenotypes suggesting they could be disease-modifiers of ARVC. Updated classification of variants by gene is available in the supplementary data

*Ye JZ et al 2019 PMID: 31402444*

"Of 501 arrhythmogenic right ventricular cardiomyopathy probands, 322 (64.3%) carried 327 desmosomal P/LP variants. Most variants (n=247, 75.6%, in 245 patients) were identified in more than one proband and, therefore, considered nonunique. For 212/327 variants (64.8%) genetic cascade screening was performed extensively enough to identify the parental origin of the P/LP variant. **Only 3 variants were de novo, 2 of which were whole gene deletions.** For 24 nonunique P/LP PKP2 variants, haplotyping was conducted in 183 available families. For all 24 variants, multiple seemingly unrelated families sharing identical haplotypes were identified, suggesting that these variants originate from common founders.

Deletions encompassing one or more exons were seen in 13 patients (4.0%), including 3 whole gene deletions (2 PKP2 and one DSP)."

*van Lint FHM et al 2019 PMID: 31386562*

**In house atlas of cardiac variants**

There was a significant excess of truncating variants in ARVC cases vs population controls. This was not the case for non-truncating variants.

**95/101 truncating**

Nonsense

Splice donor variants

Splice acceptor variants

**6/101 non-truncating**

Missense (all VUS)

*Walsh R et al 2017 PMID: 27532257*

If one focuses on P/LP variants, only 5 of them (6%) are missense,
whereas most are nonsense (25%), frameshift variants (46%), splicing alterations ±1 or ±2 (17%) or large deletions (6%). In contrast with this finding, among the ones classified as variant of unknown significance (VUS) 59% are missense, 12% are intron variants located relatively far from the donor or acceptor sites, 3% are synonymous variants and 25% are nucleotide substitutions in the 3′UTR. **This data shows that the majority of *Pkp2* variants (\~77%) associated with ACM in ClinVar and annotated as P/LP are radical alterations,** hence considered at high probability of causing a disruption of the protein and resulting in extensive transcriptional and posttranslational alterations, **while single amino acid changes are of more complex interpretation, especially in light of the known "signal-to-noise ratio"** 

*Novelli et al., 2018; PMID: 30619891*

There are rare reports of recessive mutations in PKP2 causing non-syndromic ARVC.

*Soveizi M et al 2017 PMID: 28523642*

#### **Pilot application of harmonised terms:**

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Allelic requirement:**

Monoallelic_aut

Optional modifiers:

Digenic (DSG2, DSP)

**Inheritance:**

Autosomal dominant

Optional modifiers:
Digenic (DSG2, DSP)
Incomplete penetrance

**Narrative summary of molecular mechanisms:**

Mechanism likely loss of function of *PKP2* due to **reduction/absence of gene product or altered gene product structure** causing impaired desomsome function. Most pathogenic variants are nonsense, frameshift, splicing alterations or large deletions. A minority are missense variants (5%). The majority of missense variants are VUS. Truncating mutations have been associated with a decreased protein expression suggesting that PKP2 haploinsufficiency contributes to the pathogenesis. Multiple instances of digenic inheritance have been identified with one pathogenic variant in *PKP2* and a second in another desmosomal gene. In most studies, digenic/biallelic carriers have a more severe arrhythmia phenotype. The common mode of inheritance is autosomal dominant with incomplete penetrance, however there are 2 reports of rare recessive mutations in PKP2 causing non-syndromic ARVC. "The molecular links between desmosome mutations and the pathological hallmarks of ACM --- cardiomyocyte loss, fibrosis, adipogenesis, inflammation and arrhythmogenesis --- remain poorly defined. Probable pathogenic mechanisms include loss of mechanical integrity at cell--cell junctions, altered signalling pathways at intercalated discs, disruption of ion channels and gap junctions, and aberrant protein trafficking." A 2019 study found that the majority of ARVC variants were nonunique (low rate of de novo variants) and suggested that most originate from common founders.

**Additional information related to ACMG evidence types**

As per Cardioclassifier:

**PVS1** - null variant in a gene where Loss of Function (LoF) is a known mechanism of disease
Consider activating for DSG2 and ARVC as there is a significant burden of truncating variants in cases against controls from analysis of 7,855 cardiomyopathy cases and 60,706 controls 

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework). Adjusted for ICC from original ACMG guidelines
0.1% (het) 
3.16% (hom) 

**For BS1 and PM2:**

Estimated Prevalence: 1/1000
Max Allelic Contribution: 0.092	
Maximum population AF: 0.000092

*Whiffin N et al 2018 PMID: 29369293*

**List variant classes in this gene proven to cause this disease**

- Splice region variant
- Spice acceptor variant
- Splice acceptor variant predicted to undergo NMD
- Splice donor variant
- Splice donor variant predicted to undergo NMD
- Frameshift variant
- Frameshift variant predicted to undergo NMD
- Stop gained
- Stop gained predicted to undergo NMD
- Missense

**Potential novel variant classes based on predicted functional
consequence:**

- Start lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
??
- Splice acceptor variant predicted to escape NMD
- Splice donor variant predicted to escape NMD
- Frameshift variant predicted to escape NMD
- Stop gain variant predicted to escape NMD
- In frame insertion
- In frame deletion
- Stop lost ??# **PLN -- Intrinsic Cardiomyopathy**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen:**

https://search.clinicalgenome.org/kb/gene-validity/8772

The PLN gene is associated with hypertrophic cardiomyopathy, dilated cardiomyopathy, arrhythmogenic right ventricular cardiomyopathy (van der Zwaag, 2012 PMID: 22820313) and heart failure of unknown etiology. The typical inheritance for PLN related cardiomyopathy is autosomal dominant, albeit autosomal recessive inheritance has been noted and appears to follow a dosage effect or semidominance, as loss of both alleles results in an earlier and more severe phenotypic presentation (Haghighi, 2003 PMID: 12639993). PLN is encoded by one exon, and missense, nonsense, and frameshift mutations in the coding exon and the promoter region have been reported. Given that PLN only has one exon, many of the mutations (even LOF) result in a protein product. PLN encodes the 52 amino acid protein, phospholamban, that functions to regulate SERCA2 function in the sarcoplasmic reticulum. Phospholamban exists in both monomeric and homopentameric forms. The monomeric form is thought to inhibit SERCA2 activity. PLN-mediated SERCA2 inhibition is released upon phosphorylation of monomeric PLN by either PKA or CAMKII, and thus stabilization of the pentameric form (reviewed in Haghighi, 2014 PMID: 25451386, Young, 2015 PMID: 25563649). While the distinct genetic mechanism of PLN-mediated cardiomyopathy is unclear, the overall disease mechanism for PLN associated cardiomyopathy is dysregulation of SERCA2 function, Ca2+ handling, and disrupted relaxation and contraction of the heart. Multiple cases of PLN-mediated cardiomyopathy are reported in the literature, allowing and extending beyond the maximum score for genetic evidence (12 pts). This gene-disease association is supported by the function of the gene product, alteration of normal function in nonpatient cells expressing patient-derived mutant PLN, and animal models. In summary, PLN is definitively associated with intrinsic cardiomyopathy. This association has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time. This classification was approved by the Hypertrophic Cardiomyopathy Gene Curation committee on September 19, 2017.

Lumping and Splitting: Per criteria outlined by the ClinGen Lumping and Splitting Working Group, we found no difference in molecular mechanism(s) underlying the disease entities: (1) Cardiomyopathy, dilated, 1P (MIM: 609909) and (2) Cardiomyopathy, hypertrophic, 18 (MIM:613874). Evidence suggests that the mechanism of the disease is impaired SERCA2 regulation of Ca2+ handling for all conditions associated with PLN. Furthermore, a progressive cardiomyopathy beginning with hypertrophic and leading to dilated has been observed in a proband (Haghighi, 2003 PMID: 12639993). Both interfamilial and intrafamilial variability were observed between PLN variants (Haghighi, 2003 PMID: 12639993; van der Zwaag, 2012 PMID: 22820313). For clinical management, no striking differences should occur, as all of the phenotypes and conditions associated with PLN are of cardiovascular nature, and individuals should be monitored appropriately. Therefore, all of the disease entities have been lumped into one disease entity, Intrinsic cardiomyopathy.

**Literature review:**

"Schmitt et al. (2003) sequenced the PLN gene in 20 unrelated individuals with inherited dilated cardiomyopathy and heart failure (see CMD1P, 609909). In 1 sample, **an arginine-to-cysteine substitution at codon 9 in the cytosolic PLN domain was identified and segregated with disease in that 4-generation family** (R9C;172405.0001). Affected individuals had increased chamber dimensions and decreased contractile function at age 20 to 30 years, with progression to heart failure within 5 to 10 years after symptom onset. Congestive heart failure was severe in 12 individuals, necessitating cardiac transplantation in 4. The average age at death of affected individuals was 25.1 +/- 12.7 years. 

In 2 unrelated families with CMD1P (609909), Haghighi et al. (2003) identified **a truncating mutation in the PLN gene, leu39 to ter (L39X; 172405.0002). The 2 homozygous individuals developed dilated cardiomyopathy and heart failure requiring cardiac transplantation at ages 16 and 27 years, respectively; 11 heterozygous individuals exhibited variable clinical findings, indicating incomplete penetrance of the cardiomyopathy phenotype.** Haghighi et al. (2003) concluded that in contrast to mice in which Pln deficiency enhances myocardial inotropy and lusitropy without adverse effects, PLN is essential for cardiac health in humans, and its absence results in lethal heart failure. 

In affected members of a 7-generation family with CMD, Haghighi et al. (2006) identified heterozygosity for **a 3-bp deletion** in the PLN gene (172405.0003). 

Haghighi et al. (2008) analyzed the PLN gene in 381 CMD patients and 296 controls with no known cardiomyopathy history and **identified a heterozygous -36A-C variant in the 5-prime untranslated region (172405.0006) in 22 CMD patients and 1 control.** Functional analysis demonstrated that the -36A-C variant increased PLN activity by 24% compared to wildtype and that this alteration in the steroid receptor sequence for the glucocorticoid nuclear receptor/transcription factor resulted in enhanced binding. 

Minamisawa et al. (2003) analyzed the candidate gene phospholamban (PLN) in 87 patients with hypertrophic cardiomyopathy (see CMH18; 613874), 10 with dilated cardiomyopathy, and 2 patients with restricted cardiomyopathy (RCM; see 115210). In the proband of a 2-generation family with CMH, **Minamisawa et al. (2003) identified heterozygosity for a mutation in the promoter region (172405.0004) that increased transcriptional activity 1.5-fold compared to wildtype and was not found in 296 Japanese controls.** No PLN mutations were identified in the remaining 98 cardiomyopathy patients..."

*Omim https://www.omim.org/entry/172405*

"We sequenced known and putative HCM genes in a new large prospective HCM cohort (n = 804) and analysed data alongside the largest published series of clinically genotyped HCM patients (n = 6179), previously published HCM cohorts and reference population samples from the exome aggregation consortium (ExAC, n = 60 706) to assess variation in 31 genes implicated in HCM. **PLN (calcium signalling) had an excess of 0.28% with one variant, p.L39X, significantly enriched in cases (4/5435 cases versus 1/60671 ExAC samples, P = 2 × 10−4).**" 

*Walsh et al 2017 PMID 28082330*

"Three hundred and eighty-seven consecutive unrelated patients with HCM were screened for genetic variants in the 5 most frequent genes (MYBPC3, MYH7, TNNT2, TNNI3 and TPM1) using Sanger sequencing (N = 84) or NGS (N = 303). In the NGS cohort we analyzed 20 additional minor or candidate genes, and applied a proprietary bioinformatics algorithm for detecting CNVs. In the NGS cohort, **4 patients (1.3%) had pathogenic CNVs: 2 deletions in MYBPC3 and 2 deletions involving the complete coding region of PLN.**"

*Mademont-Soler I et al 2017 PMID: 28771489*

"...**promoter and coding region variants of PLN have been associated with DCM/heart failure and HCM**. For example, a C to G conversion at position −42 (C>G −42) promoter variant has been described in one HCM case in a study that included 186 HCM and DCM patients21. This variant, found in a female, diagnosed with HCM at 67 years of age with atrial fibrillation, had reduced penetrance in a small familial pedigree. A second promoter variant, an A>G −77 mutation, was identified in one out of 87 HCM patients22."

*Landstrom AP et al 2011 PMID: 21167350*

"Mutations in PLN cause cardiomyopathy in a distinct way, as they may not directly influence the structural or functional characteristics of cardiomyocytes, but disrupt the calcium homeostasis inside cardiomyocytes and interrupt the rhythm of myocardial contraction. The known mutations in human include R14del, R9C, R9L, R9H, Leu-39stop, and R25C. Each mutation has a unique mechanism to eventually induce cardiomyopathy. In brief, the presumed mechanisms are: (1) Completely or partially losing the function to inhibit SERCA; (2) Interfering the normal PLN to inhibit SERCA in a dominant negative way. (3) Failing to be phosphorylated and regulated by PKA, or further, disabling the normal function of PKA6,10,11.

R14del is a well-known PLN mutation in Dutch people, with 10–15% of both dilated cardiomyopathy and arrhythmogenic cardiomyopathy patients are claimed to be caused by PLN-R14del..."

*Jiang X et al 2020 PMID: 33020536*

**Inheritance**

Autosomal dominant

Optional modifiers: incomplete penetrance

**Allelic requirement**

Monoallelic_aut

**Disease associated variant consequences:**

Decreased gene product level
Absent gene product
Altered gene product structure

**Narrative summary of molecular mechanisms:**

PLN functions to regulate SERCA2 function in the sarcoplasmic reticulum. The mechanism is not fully understood but it is likely loss of function of PLN through either decreased/absent gene product or altered gene product structure. Jiang et al note "presumed mechanisms are: (1) Completely or partially losing the function to inhibit SERCA; (2) Interfering the normal PLN to inhibit SERCA in a dominant negative way. (3) Failing to be phosphorylated and regulated by PKA, or further, disabling the normal function of PKA". They also note that PLN gain of function mutations have the potential to induce disorder of intracellular calcium. 

PLN is encoded by one exon (52 amino acids), and missense, nonsense, and frameshift mutations in the coding exon and the promoter region have been reported. In addition deletions of the whole coding region have been reported. In the Netherlands there is a founder mutation R14del. Up to 10–15% of both dilated cardiomyopathy and arrhythmogenic cardiomyopathy patients are claimed to be caused by PLN-R14del. As per ClinGen: "The typical inheritance for PLN related cardiomyopathy is autosomal dominant, albeit autosomal recessive inheritance has been noted and appears to follow a dosage effect or semidominance, as loss of both alleles results in an earlier and more severe phenotypic presentation. Given that PLN only has one exon, many of the mutations (even LOF) result in a protein product."

**List variant classes in this gene proven to cause this disease:**

- Missense
- In frame deletion
- Stop_gain
- Frameshift
- Promoter (regulatory_region_variant)
- Whole exon deletion

**Potential novel variant classes based on predicted functional
consequence**

- Frameshift predicted to escape NMD
- stop_gained predicted to escape NMD
- Frameshift predicted to undergo NMD
- stop_gained predicted to undergo NMD
- stop_lost
- inframe_insertion
- start lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
- ?Structural variants/transcript ablation


# **PMS2 -- Lynch Syndrome**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:9122

**ClinGen Evidence for Haploinsufficiency**

PMID 18602922 - Authors discuss results of 99 probands who had a Lynch syndrome associated tumor and absence of PMS2 protein by IHC. Total of 55 patients found to have deleterious mutations in PMS2. Most notable was an insertion/deletion mutation in exon 7. Several patients (10 patients) in this study showed deletions of one or more exons , two patients (Patient 28, 29) showed complete gene deletions.

PMID 23837913 - Authors described the use of long range PCR and MLPA analysis in the identification of deletions within the PMS2 gene. Six patients with suspected Lynch syndrome were identified to have mutations (5 with deletions of one or more exons with loss of PMS2 protein expression by IHC and 1 frameshift mutation). In addition 4 novel VOUS were also identified in this study.

PMID 22585707 - 1) a large (?125 kb) deletion containing the entire PMS2 gene, three other genes (ANKRD61, AIMP2 [JTV1], and EIF2AK1), and a portion of the coding region of RSPH10B. 2) a genomic deletion involving PMS2 exon 8. Special note from this article: Because of the high prevalence of nondeleterious hybrid PMS2 and PMS2CL alleles, the distribution of PMS2 and PMS2CL-specific sequences downstream of exon 12 displays high interindividual variability (Hayward et al., 2007;Ganster et al., 2010; van der Klift et al., 2010). Randomly chosen reference samples of DNA are thus likely to differ widely in terms of the distribution of gene- and pseudogene-derived sequences in this region. This is reflected by high standard deviations for the reference DNA signals generated with all the paralog-discriminating probes located downstream of exon 12 (Supporting Information Fig. S1). It is important to note that an unequal distribution of gene-derived and pseudogene-derived sequences in the reference DNA set will reduce the accuracy of copy number assessments at these loci in patient DNA samples. For this reason, reference DNAs must harbor two PMS2-specific copies and two PMS2CL-specific copies of each sequence bound by paralog-discriminating probes for exons 11-15.

**Literature review:**

PMS2 comprises 15 exons encoding a protein of 862 amino acids. Multiple pseudogenes have been identified at 7p22, 7p12-13, 7q11, and 7q22 [Nicolaides et al 1995]. For a detailed summary of gene and protein information, see Table A, Gene.

Pathogenic variants. Germline pathogenic variants in PMS2 are rare [Hendriks et al 2006]. Single-nucleotide variants and large gene rearrangements have been reported. **Studies that have included large deletion testing have found that up to 20% of pathogenic variants may be large deletions. Large deletion testing of PMS2 is technically difficult due to the numerous pseudogenes,** and it presents significant challenges to laboratories trying to provide comprehensive large deletion testing for the whole gene. The currently available MLPA (multiplex ligation-dependent probe amplification) kit can detect deletions but does not clarify whether the deletion may be in one of the pseudogenes. Testing in coordination with a panel of reference samples can help determine whether deletions are clinically significant [Vaughn et al 2011].

PMS2 acts in a recessive manner at the cellular level where there is an absence of functional PMS2 protein in the tumor cells. This results from inactivation of both PMS2 alleles in the tumor, which often occurs by the mechanism of loss of heterozygosity.

Heterozygosity for a PMS2 pathogenic variant is associated with the lowest risk (25%-32% risk) for any Lynch syndrome-related cancer [Senter et al 2008]. However, while the overall risk of CRC is lower, age of onset may still be early. A review of 234 PMS2 pathogenic variant carriers found that 8% were diagnosed before age 30 [Goodenberger et al 2016].

*Gene reviews
https://www.ncbi.nlm.nih.gov/books/NBK1211/*

Types of variant in PMS2:

Missense 62%
Nonsense or frameshift 24%
In-frame 1% (includes deletions, insertions or indels which do not affect the reading frame)
Splice  3%
Large rearrangement 10%

*Data from the InSiGHT database presented in Tamura K et al 2019 PMID: 31273487*

Of the approximately 3,300 unique MMR gene variants listed in the International Society for Gastrointestinal Hereditary Tumors (InSiGHT) database, only 9% are reported in PMS2, compared to 39% in MLH1, 33% in MSH2 and approximately 19% reported in MSH6 (http://www.insight-database.org/genes). 

Analysis of the PMS2 gene has been complicated by the presence of a large family of pseudogenes that are highly homologous to PMS2. Fourteen pseudogenes (ψ1 – 14) overlap with some or all of exons 1 – 5 of PMS2 and vary in length (Figure 1A) whereas the PMS2CL pseudogene (formerly ψ0) possesses high sequence similarity to exons 9 and 11–15

*Blount J et al 2018 PMID: 29286535*

Variant types taken from InSiGHT database
*Plazzer JP et al 2013 PMID: 23443670*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: incomplete penetrance

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

A heterodimer of MLH1 and PMS2 coordinates the interplay between the mismatch recognition complex and other proteins necessary for mismatch repair. In the absence of PMS2, MLH1 can pair with PMS1. This may partially explain the somewhat attenuated Lynch phenotype attributed to PMS2 mutations.
Single-nucleotide variants and large gene rearrangements (up to 20% in some studies) have been reported. Large deletions are difficult to identify due to the presence of a large family of pseudogenes highly homologous to PMS2.
Heterozygosity for a PMS2 pathogenic variant is associated with the lowest risk (25%-32% risk) for any Lynch syndrome-related cancer [Senter et al 2008]. However, while the overall risk of CRC is lower, age of onset may still be early. A review of 234 PMS2 pathogenic variant carriers found that 8% were diagnosed before age 30 [Goodenberger et al 2016].

**List variant classes in this gene proven to cause this disease:**

- Missense
- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- In frame deletions
- In frame insertions

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]

# **PRKAG2 -- Hypertrophic Cardiomyopathy**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen:**

[[https://search.clinicalgenome.org/kb/gene-validity/10051]{.ul}](https://search.clinicalgenome.org/kb/gene-validity/10051)

To date only missense mutations have been described in the PRKAG2 gene and associated with Wolf-Parkinson-White syndrome, glycogen storage disease of heart and hypertrophic cardiomyopathy.

**Literature review:**

"PRKAG2 syndrome (PS) is a rare, early-onset autosomal dominant inherited disease, characterized by ventricular pre-excitation, supraventricular arrhythmias and cardiac hypertrophy. It is frequently accompanied by chronotropic incompetence and advanced heart blocks, leading to premature pacemaker (PM) implantation.

The syndrome is caused by mutations in the gene encoding for the 5’ Adenosine Monophosphate-Activated Protein Kinase (AMPK), specifically for its γ2 regulatory subunit (PRKAG2).

AMPK is an enzyme deeply involved in cellular ATP metabolic regulation.6 PRKAG2 genetic mutations are rare and have been recognized mainly in the context of patients with non-sarcomeric familial hypertrophic cardiomyopathy associated with Wolff-Parkinson-White (WPW) syndrome.7

PS can show different expressivity both of ventricular hypertrophy and arrhythmic features, ranging from an asymptomatic condition to sudden cardiac death (SCD). PS can occasionally lead to heart failure (HF), or demonstrate systemic involvement.7-9

Almost all studies report missense mutations. Only
Blair et al. PMID: 11371514 documented an insertion mutation (Exon
5:InsLeu).[^8^](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4704128/#R8) The most commonly reported mutation were C.905G\>A (Arg302Gln) and c.1463A\>T (Asn488Ile), with 110 and 40 cases (respectively 57% and 21%)

*PRKAG2* mutations are suspected to modify the tridimensional structure of AMPK, altering its affinity for AMP and modifying the enzyme activity. Studies on transgenic mice have showed an enhanced enzymatic activity during the early stage of
PS and a decreased activity during the advanced
phase;  a recent study demonstrated an impaired myocardial glycogen uptake in adult patients with PS."
*Porto A et al. 2016 PMID: [26729852]*

"Defects in *PRKAG2* are associated with a recently described cardiac syndrome triad consisting of familial ventricular preexcitation (Wolff-Parkinson-White syndrome, WPW), conduction system disease and cardiac hypertrophy mimicking HCM. Histological studies of myocardial tissue from affected
individuals and transgenic mice expressing mutant forms of the *PRKAG2* gene confirmed glycogen storage as the
pathologic basis for this cardiac syndrome. Because of similar
echocardiographic features, *PRKAG2* disease could be misdiagnosed as HCM. Unlike HCM, however, individuals with the *PRKAG2* mutations have a higher incidence of progressive cardiac conduction disease requiring implantation of a
pacemaker. It is therefore important to distinguish hypertrophy associated with *PRKAG2* mutations from that due to sarcomere protein defects"

*Liu Y et al. 2013 (PMID:23741347)*

"Five missense mutations and one in-frame insertion have been identified in the *PRKAG2* coding sequence. The most common mutation is Arg302Gln, now identified in seven families. Variable expressivity of the clinical phenotype appears to exist and may be mutation specific. For example, in three families with the Arg302Gln mutation, symptomatic onset of disease occurs in late adolescence and cardiac hypertrophy is detected in 30% to 50% of patients. A smaller kindred with the Arg531Gly mutation has presented with severe arrhythmogenic disease as early as age 2, but exhibited no evidence of cardiac hypertrophy or LV dysfunction despite survival into their fifth decade \[[31]. However, this does not exclude the possibility of undetected myopathic disease. Interestingly, the Arg302Gln and Arg531Gly mutations occur at the identical positions within cystathionine \[beta\]-synthase domains 1 and 4, respectively. This finding may provide a basis for structure-function studies of cystathionine \[beta\]-synthase domains within *PRKAG2*.

*Gollob M et al 2002 PMID: 12015471*

Missense mutations in the regulatory subunit, PRKAG2, activate AMPK and cause left ventricular hypertrophy, glycogen accumulation and ventricular pre-excitation

*In vitro* studies indicate that PRKAG2 mutations decrease the
nucleotide-dependence of AMPK catalytic activity ([Scott et al.,
2004](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5193246/#R41)),
resulting in gain of function. Once activated, AMPK regulates multiple metabolic pathways including increased glucose uptake by GLUT4 translocation ([Kurth-Kraczek et al.,
1999](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5193246/#R24)) and glycolysis by phosphofructokinase-2 regulation ([Marsin et al., 2000](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5193246/#R34))

*Hinson JT et al. 2016 Dec 20 (PMID:28009297)*

"Since the identification of PRKAG2 as a cause of this condition, only around 200 patients have been reported in the literature. Of these cases, **22 distinct heterozygous variants have been described and all of them are located with, or in close proximity to, the adenine nucleotide binding CBS domains (Fig. 3A), which are recognized as functional regions interacting with AMP, ADP or ATP. The most frequently identified mutations are R302Q (135 cases from 14 families, ∼57%), followed by N488I (40 cases from 2 families, ∼21%) [15].** This study has added 22 more PRKAG2 cases and 4 novel mutations in the exploration journey of the disease.
All mutants here lead to disruption of the ligand-binding interfaces and/or break communication with the alpha and beta subunits which directly impacts the catalytic portion of the kinase...

**The probands bearing a PRKAG2 mutation present typically with palpitations, syncope, chest pain, or features of HF. Only 1 is identified after an abnormal ECG (Proband 7). While penetrance is 100%, as prescribed previously [14], the disease progression in our objectives for the same mutation have interfamilial variability, even heterogeneity..."**

*Hu D et al 2020 PMID: 32259713*

"Clinical, electrocardiographic, and echocardiographic data from 90 subjects with PRKAG2 variants (53% men; median age 33 years; interquartile range [IQR]: 15 to 50 years) recruited from 27 centers were retrospectively studied.

Compared with previously published series (5,6,13), our cohort displayed considerable genetic heterogeneity, with a total of 26 rare unique genetic variants, most of which were missense."

There is 1 intron variant and 1 frameshift variant reported in this study. These variants are not reported on ClinVar but are predicted to be pathogenic.

"2 of the rare genetic variants (p.Arg302Gln and p.Asn488Ile) were present in 44% of the patients included in the cohort. Patients with these 2 genetic variants exhibited pre-excitation more frequently and had a lower prevalence of syncope but otherwise showed a very similar clinical profile. 

At the end of follow-up, 76% of patients (68 of 90) had signs and symptoms of PS, but penetrance of PS was only 31% at 40 years of age or less."

*Lopez-Sainz A et al 2020 PMID: 32646569*

**From our in-house Atlas of HCM:**

Walsh et al found an excess of non-truncating variants in PRKAG2
associated with HCM in comparison to the reference dataset (ExAC)

*40/42 non-truncating*

*2/42 truncating (1 nonsense, 1 fs -- both VUS)*

*Walsh et al, 2016 (PMID 27532257)*
<https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=PRKAG2&icc=HCM>

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

**Allelic requirement:**

Monoallelic_aut

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is likely due to modified enzyme activity due to **altered gene product structure**. PRKAG2 genetic mutations are rare and have been recognized mainly in the context of patients with non-sarcomeric familial hypertrophic cardiomyopathy associated with Wolff-Parkinson-White (WPW) syndrome. The syndrome is caused by mutations in the gene encoding for the 5’ Adenosine Monophosphate-Activated Protein Kinase (AMPK), specifically for its γ2 regulatory subunit (PRKAG2). Hu et al report that pathogenic variants are located with, or in close proximity to, the adenine nucleotide binding CBS domains. *PRKAG2* mutations are suspected to modify the tridimensional structure of AMPK, altering its affinity for AMP and modifying the enzyme activity.
Almost all studies report missense mutations. Only
Blair et al. documented an insertion mutation (Exon
5:InsLeu). There are at least 2 reports of frameshift variants being identified in PRKAG2 but the significance of these is not known. The most commonly reported mutations were C.905G\>A (Arg302Gln) and c.1463A\>T (Asn488Ile), with 110 and 40 cases (respectively 57% and 21%). There is a debate about penetrance but certainly variable expressivity of the clinical phenotype appears to exist and may be mutation specific.

**List variant classes in this gene proven to cause this disease:**

- Missense
- In frame insertion

**List other variant classes predicted to lead to the same functional consequence:**

- Frameshift predicted to escape NMD
- Stop_gained predicted to escape NMD
- Splice donor variant
- Splice acceptor variant predicted to escape NMD
- Stop_lost
- In frame deletion
**PTEN - PTEN hamartoma tumor syndrome (MIM 153480)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:9588

no single gene duplications of the entire PTEN locus have been reported.

**Literature Review:**

**OMIM:** https://www.omim.org/entry/158350

The chromosomal region containing the Cowden syndrome locus was known to
contain the **tumor suppressor gene PTEN**, which had been found to be
mutated in sporadic brain, breast, and prostate cancer. Liaw et al.
(1997) found germline mutations in the PTEN gene in 4 of 5 families with
Cowden syndrome. **Missense (601728.0001) and nonsense mutations were
predicted to disrupt the protein tyrosine/dual-specificity phosphatase
domai**n of the protein. All affected individuals of the 5 families
studied manifested trichilemmomas, regardless of whether their mutation
was a missense or nonsense mutation. Nonsense mutations were associated
with macrocephaly in 2 families. In 1 of these families, a **premature
stop codon at position** 157 (601728.0003) was also associated with
Lhermitte-Duclos disease (LDD), manifested by ataxia and dysplastic
cerebellar gangliocytomatosis. In the other family, a stop codon at
position 233 (601728.0002) was not associated with cerebellar
manifestation. Liaw et al. (1997) speculated that the larger N-terminal
truncation may be responsible for the more severe LDD phenotype.

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1488/

Pathogenic variants. Germline pathogenic variants have been found
throughout PTEN (with the exception of exon 9) and include **missense,
nonsense, and splice-site variants, small deletions, insertions, and
large deletions.** More than 150 unique pathogenic variants are
currently listed in the Human Gene Mutation Database (see Table A).
**Nearly 40% of pathogenic variants are found in exon 5, which encodes
the phosphate core motif** \[Eng 2003\]. Most pathogenic variants are
unique, although a number of recurrent pathogenic variants have been
reported, particularly those in Table 3.

Abnormal gene product. The **majority (76%) of germline pathogenic
variants in PTEN predict either truncated PTEN protein, lack of protein
(haploinsufficiency), or dysfunctional protein**. Many **missense
variants are functionally null and several act as dominant negatives**
\[Weng et al 2001a, Weng et al 2001b\]. When PTEN is absent, decreased,
or dysfunctional, phosphorylation of AKT1 is uninhibited, leading to the
inability to activate cell cycle arrest and/or to undergo apoptosis. In
addition, through lack of protein phosphatase activity, the
mitogen-activated protein kinase (MAPK) pathway is dysregulated, leading
to abnormal cell survival \[Eng 2003\].

...report a study of 3042 individuals who met relaxed criteria for
Cowden Syndrome and report 290 (9.5%) with pathogenic mutations in PTEN.
The majority were **presumed loss of function mutations - 92 (32%)
nonsense mutations, 42 (14%) small deletions, 24 (8%) small insertions,
3 (1%) indels, 8 (3%) large deletions, 19 (7%) splice-site donor
mutations, 9 (3%) splice-site acceptor mutations.**

***Tan et al, PMID 21194675***

***...***report on 481 patients with pathogenic germline PTEN variants.
This cohort includes those reported in Tan et al (PMID 21194675). The
mutation spectrum includes **242 truncating mutations, as well as 12
exon-level deletions.**

***Yahia et al, PMID 32003824***

Supplementary information 16 patients, no genotype:phenotype
correlation. **Missense, nonsense, splice donor, 2kb upstream variant
(-1084), deletion, insertion**

***Ciaccio et al, 2019, PMID: 30528446***

The mutational spectra of CS and BRRS overlap, with many of the
mutations occurring in exons 5, 7, and 8...... two-thirds of these
mutations were found in exons 5, 7,and 8 \[Marsh et al., 1998b\]. About
40% of all CS germline mutations are located in exon 5.

..Tabulated variant in **5'UTR, stop gain, frameshift, splice
acceptor/donor (truncating), missense null, insertions, deletions
(mostly out of frame), -8 splice region**

***Eng, 2003,PMID: 12938083***

... studied 34 different **germline PTEN intronic variants from 61 CS
patients**, characterized their PTEN mRNA processing, and analyzed PTEN
expression and downstream readouts of P-AKT and P-ERK1/2. While we found
that many **mutations near splice junctions result in exon skipping, we
also identified the presence of cryptic splicing that resulted in
premature termination or a shift in isoform usage.** PTEN protein
expression is significantly lower in the group with splicing changes
while P-AKT, but not P-ERK1/2, is significantly increased. Our
observations of these PTEN intronic variants should contribute to the
determination of pathogenicity of PTEN intronic variants and aid in
genetic counseling.

***Chen et al, 2017, PMID: 28677221***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is predominantly loss of function and the majority of variants
predict either truncated PTEN protein, lack of protein
(haploinsufficiency), or dysfunctional protein. Variant classes include
missense (many functionally null), nonsense, and splice-site variants,
small deletions, insertions, and large deletions. Nearly 40% of
pathogenic variants are found in exon 5 (encoding the phosphate core
motif) and several are recurrent. No pathogenic mutations have been
identified in exon 9. Variants in the 5'UTR and deeply intronic (exon
skipping and cryptic splice sites) have been reported.

**List variant classes in this gene proven to cause this disease:**

splice_region_variant

splice_acceptor_variant

(splice_acceptor_variant predicted to undergo NMD)

splice_donor_variant

(splice_donor_variant predicted to undergo NMD)

(splice_acceptor_variant predicted to escape NMD)

(splice_donor_variant predicted to escape NMD)

Frameshift_variant

(frameshift_variant predicted to undergo NMD)

(frameshift_variant predicted to escape NMD)

Stop_gained

(stop_gained predicted to undergo NMD)

Inframe_deletion

Inframe_insertion

Missense

5_prime_UTR_variant

intron_variant

**Potential novel variant classes based on predicted functional
consequence**

start_lost

(stop_gained predicted to escape NMD)

stop_lost

3_prime_UTR_variant

(gain of upstream Start \[uORF\])

(gain of upstream Start \[oORF\])

(Stop lost \[uORF\])

(Stop lost \[oORF\])

(Start lost \[uORF\])

(Frameshift \[uORF\])

(Frameshift \[oORF\])

(Stop gained \[uORF\])
---
output:
  word_document: default
  html_document: default
---
**RB1 -- Retinoblastoma (MIM 180200)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:9884

Retinoblastoma (RB) is an embryonic malignant neoplasm of retinal
origin. It almost always presents in early childhood and is often
bilateral. The retinoblastoma gene (RB1) was the first **tumor
suppressor gene** identified based on location in the region of 13q14
germline deletion in some patients with retinoblastoma (PMID:
3480530). Oncogenic **point mutations** in RB1 were first reported by
Yandell DW, et al., 1989 (PMID: 2594029). RB1 encodes a negative
regulator of the cell cycle through its ability to bind the
transcription factor E2F and repress transcription of genes required for
S phase (PMID: 10647931). There is abundant evidence published
associating the RB1 gene with retinoblastoma. Multiple case level
studies (PMID: 2594029, 25928201) and a case control study (PMID: 28193182) have been performed with retinoblastoma patients that have
variants in the RB1 gene. The variants include **single amino acid
changes, the whole gene deletion (about 3-5%) and nonsense or frameshift
LOF variants**. Many de novo variants were reported. A significant
amount of case-level data is available, but the maximum points for
genetic evidence has been reached (12 points). A zebrafish model of RB1
has been established to show development of retinoblastoma consistent
with the disease phenotype. Tumor formation was observed when the
IRBP-Cre:Rb1-/- mice were crossed with Tp53 mice to heterozygosity. In
summary, RB1 is definitively associated with autosomal dominant
retinoblastoma. This has been repeatedly demonstrated in both the
research and clinical diagnostic settings, and has been upheld over
time.

**Literature Review:**

**OMIM:** https://www.omim.org/entry/180200

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1452/

Pathogenic variants. More than 2,500 nucleotide variants have been
observed in white blood cell DNA of individuals with retinoblastoma or
in tumors; more than 1,700 are archived (see Table A, Locus Specific).
The majority of RB1 pathogenic variants result in a **premature
termination codon, usually through single-base substitutions, frameshift
variants, or out-of-frame exon skipping caused by splice site
variants**. Pathogenic variants have been found scattered throughout
**exon 1 to exon 25 of RB1 and its promoter region**. In a single
family, a possible pathogenic variant in exon 27 was identified [Mitter et al 2009]. Recurrent **pathogenic variants are observed at methylated
CpG dinucleotides that are part of CGA codons or the splice donor site
of intron 12**. Other important types of pathogenic variants are
**complex rearrangements and deletions** [Albrecht et al 2005, Rushlow et al 2009, Castéra et al 2013].

Abnormal gene product. Pathogenic variants in RB1 lead to the expression
of proteins that have **lost cell cycle-regulating functions**.
Retention of partial activity has been observed in proteins resulting
from **pathogenic variants that are associated with low-penetrance**
retinoblastoma

......analyzed 19 patients with RB. RB1 mutations were identified in 13
tumors, including the following germline mutations: **55 bp duplication
within exon 10 (truncated protein) and a 10 bp deletion in exon 18
(truncated protein).**

***Dunn et al, 1989, PMID 2601691***

....analyzed tumors from 7 patients with RB. RB1 mutations were
identified in all tumors, including a de novo germline mutation
(ARG445TER) -- stop gain.

***Yandell, 1989, PMID 2594029***

...reported on 119 patients with RB and found RB1 mutations in 99
patients (83%). The mutation spectrum included **42% base substitutions,
26% small length alterations, and 15% large deletions.**

***Lohmann, 1996, PMID 8651278***

....intronic variants affecting less conserved splice motifs require
additional analysis to ascertain whether splicing is altered. Although
the frequency of these variations is low, their impact on genetic
counselling is high, since they are **usually associated with low
penetrance phenotypes** and unaffected carriers. In this work, we used
minigene assays to study infrequent germline intronic variations for
which functional data were not available. Using this approach, the
aberrant splicing and the resulting oncogenic nature of **three intronic
RB1 mutations was established (c.501-15T\>G, c.719-9C\>G,
c.2326-8T\>A).**

***Gamez-Pozo, 2007, PMID: 18000883***

....**Mosaicism was evident in 5.5% of bilateral probands (23 of 421),
in 3.8% of unilateral probands (22 of 572), and in one unaffected mother
of a unilateral proband.** Half of the mosaic mutations were only
detectable by AS-PCR for the 11 recurrent CGA\>TGA mutations, and not by
standard sequencing. This suggests that significant numbers of low-level
mosaics with other classes of RB1 mutations remain unidentified by
current technology.

***Rushlow, 2009, PMID: 19280657***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

Optional modifiers: incomplete penetrance

Optional modifiers: mosaicism

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

RB1 is a tumour suppressor gene. Mechanism of disease is loss of
function secondary to a truncated RB1 protein or lack of protein
(haploinsufficiency). Most variants are missense, frameshift variants or
splice site throughout exons 1-25 and the promotor region. Large complex
rearrangements and deletions are noted and recurrent pathogenic variants
are observed, including the splice donor site of intron 12. Mosaicism was evident in 5.5% of bilateral probands and
in 3.8% of unilateral probands. This suggests that significant numbers of low-level mosaics with other classes of RB1 mutations remain unidentified by current technology.

**List variant classes in this gene proven to cause this disease:**

splice_region_variant

splice_acceptor_variant

(splice_acceptor_variant predicted to undergo NMD)

splice_donor_variant

(splice_donor_variant predicted to undergo NMD)

(splice_acceptor_variant predicted to escape NMD)

(splice_donor_variant predicted to escape NMD)

Frameshift_variant

(frameshift_variant predicted to undergo NMD)

(frameshift_variant predicted to escape NMD)

Stop_gained

(stop_gained predicted to undergo NMD)

Inframe_deletion

Inframe_insertion

Missense

intron_variant

**Potential novel variant classes based on predicted functional
consequence**

start_lost

(stop_gained predicted to escape NMD)

stop_lost

5_prime_UTR_variant

3_prime_UTR_variant

(gain of upstream Start \[uORF\])

(gain of upstream Start \[oORF\])

(Stop lost \[uORF\])

(Stop lost \[oORF\])

(Start lost \[uORF\])

(Frameshift \[uORF\])

(Frameshift \[oORF\])

(Stop gained \[uORF\])
# **RB1 -- Retinoblastoma**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:9884

Retinoblastoma (RB) is an embryonic malignant neoplasm of retinal origin. It almost always presents in early childhood and is often bilateral. The retinoblastoma gene (RB1) was the first tumor suppressor gene identified based on location in the region of 13q14 germline deletion in some patients with retinoblastoma (PMID: 3480530). Oncogenic point mutations in RB1 were first reported by Yandell DW, et al., 1989 (PMID: 2594029). RB1 encodes a negative regulator of the cell cycle through its ability to bind the transcription factor E2F and repress transcription of genes required for S phase (PMID: 10647931). There is abundant evidence published associating the RB1 gene with retinoblastoma. Multiple case level studies (PMID: 2594029, 25928201) and a case control study (PMID: 28193182) have been performed with retinoblastoma patients that have variants in the RB1 gene. The variants include single amino acid changes, the whole gene deletion (about 3-5%) and nonsense or frameshift LOF variants. Many de novo variants were reported. A significant amount of case-level data is available, but the maximum points for genetic evidence has been reached (12 points). A zebrafish model of RB1 has been established to show development of retinoblastoma consistent with the disease phenotype. Tumor formation was observed when the IRBP-Cre:Rb1-/- mice were crossed with Tp53 mice to heterozygosity. In summary, RB1 is definitively associated with autosomal dominant retinoblastoma. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time.
Gene Clinical Validity Standard Operating Procedures (SOP) - Version 7

**ClinGen Evidence for Haploinsufficiency**


**Literature review:**

More than 2,500 nucleotide variants have been observed in white blood cell DNA of individuals with retinoblastoma or in tumors; more than 1,700 are archived (see Table A, Locus Specific). **The majority of RB1 pathogenic variants result in a premature termination codon, usually through single-base substitutions, frameshift variants, or out-of-frame exon skipping caused by splice site variants.** Pathogenic variants have been found scattered throughout exon 1 to exon 25 of RB1 and its promoter region. In a single family, a possible pathogenic variant in exon 27 was identified [Mitter et al 2009]. Recurrent pathogenic variants are observed at methylated CpG dinucleotides that are part of CGA codons or the splice donor site of intron 12. Other important types of pathogenic variants are complex rearrangements and deletions [Albrecht et al 2005, Rushlow et al 2009, Castéra et al 2013].

Pathogenic variants in RB1 lead to the expression of proteins that have lost cell cycle-regulating functions. Retention of partial activity has been observed in proteins resulting from pathogenic variants that are associated with low-penetrance retinoblastoma [Bremner et al 1997, Otterson et al 1997, Sánchez-Sánchez et al 2007].

In the majority of families with heritable retinoblastoma, all members who have inherited the germline pathogenic variant develop multiple tumors in both eyes. It is not unusual to find, however, that the founder (i.e., the first person in the family to have retinoblastoma) has only unilateral retinoblastoma. Most of these families segregate RB1 null alleles that are altered by frameshift or nonsense variants. **With few exceptions, RB1 null alleles show nearly complete penetrance (>99%)** [Taylor et al 2007, Dommering et al 2014, Frenkel et al 2016].

**Fewer than 10% of families show a "low-penetrance" phenotype with reduced expressivity** (i.e., increased prevalence of unilateral retinoblastoma) and incomplete penetrance (i.e., ≤25%). This low-penetrance phenotype is usually associated with RB1 in-frame, missense, or distinct splice site variants, certain indel variants in exon 1, or pathogenic variants in the promoter region.

A third category of families shows differential penetrance depending on the parental origin of the pathogenic allele (parent-of-origin effect) [Klutz et al 2002, Eloy et al 2016, Imperatore et al 2018].

Cytogenetically visible deletions involving 13q14 that also result in deletions of additional genes in the same chromosome region as RB1 may cause developmental delay [Castéra et al 2013] and mild-to-moderate facial dysmorphism. As sizeable deletions of 13q14 show reduced expressivity, a considerable proportion of individuals with such deletions show unilateral retinoblastoma only; some of these children do not develop any tumors [Mitter et al 2011]. Contiguous loss of MED4, which is located centromeric to RB1, explains reduced expressivity in individuals with large deletions that include both RB1 and MED4 [Dehainault et al 2014].

**Gene reviews**
*https://www.ncbi.nlm.nih.gov/books/NBK1452/#retinoblastoma*

A searchable database (RBGMdb) has been constructed with 932 published RB1 mutations. The spectrum of these mutations has been analyzed with the following results: 1) the retinoblastoma protein is frequently inactivated by deletions and nonsense mutations while missense mutations are the main inactivating event in most genetic diseases. 2) Near 40% of RB1 gene mutations are recurrent and gather in sixteen hot points, including twelve nonsense, two missense and three splicing mutations. The remainder mutations are scattered along RB1, being most frequent in exons 9, 10, 14, 17, 18, 20, and 23. 3) The analysis of RB1 mutations by country of origin of the patients identifies two groups in which the incidence of nonsense and splicing mutations show differences extremely significant, and suggest the involvement of predisposing ethnic backgrounds. 4) A significant association between late age at diagnosis and splicing mutations in bilateral retinoblastoma patients suggests the occurrence of a delayed-onset genotype. 5) Most of the reported mutations in low-penetrance families fall in three groups: a) Mutations in regulatory sequences at the promoter resulting in low expression of a normal Rb; b) Missense and in-frame deletions affecting non-essential sequence motifs which result in a partial inactivation of Rb functions; c) Splicing mutations leading to the reduction of normal mRNA splicing or to alternative splicing involving either true oncogenic or defective (weak) alleles.

*Valverde JR et al 2005 PMID:16269091*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The mechanism appears to be loss of function of TSC2 leading to **decreased/absent or altered gene product** which results in uncontrolled cell growth and proliferation. Missense variants account for approximately 26% of all TSC2 pathogenic variants with approximately 50% concentrated in the carboxy domain. Variants are distributed throughout the coding regions of TSC2, except for the alternatively spliced exons (25 and 31). Approximately 33% of TSC2 pathogenic variants are located in exons 32-41 (and associated splice sites) that encode the carboxy domain of tuberin consisting of several important functional motifs (e.g., GAP domain, estrogen receptor- and calmodulin-binding domains, and multiple signal pathway kinase targets).
Penetrance is nearly 100% but there is variable expressivity.


**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Missense

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- In frame deletion
- In frame duplication
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[uORF\]
- Frameshift \[uORF\]
- Stop gained \[uORF\]
**RET -- Familial Medullary Thyroid Cancer**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:9967

Inherited medullary thyroid carcinoma (MTC) is primarily caused by RET
mutations that are commonly localized in exons 5, 8, 10, 11, and 13-16.
In this study, we report pedigrees for individuals with MTC that harbor
a germline S409Y variant within exon 6 of the RET proto-oncogene
.....novel germline RET S409Y variant is likely pathogenic and is
associated with lower penetrance of MTC than that for the C618Y and
C634Y mutations.

*Qi et al, 2019, PMID: 31364476*

More than 90% of the cases of MEN 2A and FMTC are caused by mutations in
one of the six highly conserved cysteine residues that are encoded by
exons 10 (codons 609, 611,618, and 620) and 11 (codons 630 and 634) of
the RET gene. In a few MEN 2A or FMTC families, mutations have been
reported involving non-cysteine domains in exons 11 (codon 631),
13(codons 768, 790, and 791), 14 (codon 804 and844), 15 (codon 891), or
a 9-bp duplication in exon 8 (8--12) ....Codon 804 mutations of the RET
exon 14,which have been detected in a few MEN 2A orFMTC families,
include single-base substitutionsresulting in a GTG !TTG (V804L),
GTG!CTG(V804L), or GTG!ATG (V804M) missensechange (11, 15--21). It has
been assumed that mutation in non-cysteine domains, including codon 804
in exon 14, results in a weaker onco-genic activation of the RET protein
and an attenuated form of the MEN 2A or FMTC phenotype compared to that
observed in patientswith cysteine domain mutation at codon 634 ofexon 11
(20, 22, 23). This proposal is supportedby clinical observations showing
a late onset andpossibly a reduced penetrance and expression ofthe
disease phenotype in families with codon 804mutations of the RET exon 14
(11, 20)

*Patocs et al, 2003, PMID: 12694233*

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

*Optional modifier -- incomplete penetrance*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

Altered protein product

**Narrative summary of molecular mechanisms:**

The molecular mechanism of disease in MEN2a (including the isolated
familial medullary thyroid cancer) is gain of function, resulting from
an activating missense germline mutation of the RET proto-oncogene (a
tyrosine kinase). Most MEN2A/FMTC cases result from missense mutations
that lead to ligand-independent dimerization (constitutive activation),
located in one of six cysteine codons in the extracellular domain of the
encoded protein (codons 609, 611, 618, and 620 in exon 10 and codons 630
and 634 in exon 11. In a few families a small in frame duplication and
small 2bp indel have been found. Attenuated phenotypes may be associated
with cysteine domain mutations at codon 634 of exon 11 and at
non-cysteine residues (including codon 804 in exon 14)

**List variant classes in this gene proven to cause this disease:**

Missense

Inframe_deletion

Inframe_insertion

**Potential novel variant classes based on predicted functional
consequence**

splice_acceptor_variant predicted to escape NMD

splice_donor_variant predicted to escape NMD

frameshift_variant predicted to escape NMD

stop_gained predicted to escape NMD

Stop lost

**Not included**

splice_region_variant

splice_acceptor_variant

(splice_acceptor_variant predicted to undergo NMD)

splice_donor_variant

(splice_donor_variant predicted to undergo NMD)

Frameshift_variant

(frameshift_variant predicted to undergo NMD)

Stop_gained

(stop_gained predicted to undergo NMD)

start_lost

5_prime_UTR_variant

3_prime_UTR_variant

(gain of upstream Start \[uORF\])

(gain of upstream Start \[oORF\])

(Stop lost \[uORF\])

(Stop lost \[oORF\])

(Start lost \[uORF\])

(Frameshift \[uORF\])

(Frameshift \[oORF\])

(Stop gained \[uORF\])
**RET -- Multiple endocrine neoplasia, type 2a (MIM 171400**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:9967

Numerous variants have been reported in RET in relation to multiple
endocrine neoplasia type A , the majority being **missense mutation in
conserved cysteine amino acid residues within the dimerization and
activation domain, including p.C609, p.C611, p.C618, p. C620, p. C630,
p.C634 (reviewed Plaza-Menaho et al., 2006; PMID: 16979782). The
molecular mechanism for the RET-multiple endocrine neoplasia type 2A is
gain of function (**GOF**), as the missense mutations result in
ligand-independent dimerization that induces activation of the RET
protein that is a receptor tyrosine kinase (Plaza-Menaho et al., 2006;
PMID: 16979782; Drilon et al., 2017, PMID 29134959). There are variant
databases specific to the RET gene including the Multiple Endocrine
Neoplasia type 2 (MEN2) and RET database
(http://www.arup.utah.edu/database/MEN2/MEN2_welcome.php) and the RET
gene LOVD database (https://databases.lovd.nl/shared/genes/RET).
Substantial evidence supports this gene-disease relationship includes
case-level data, segregation and experimental data with the maximum
score (12) for genetic evidence reached. This gene-disease relationship
is supported by functional studies including expression, cell assays and
animal models, several that were patient derived pathogenic variants
expressed in the mouse (reviewed in Wiedmann et al, 2016; PMID:
26184857). These animal models developed MEN2A related tumors including
medullary thyroid carcinomas. Of note, **Multiple endocrine neoplasia
IIB (MIM:162300) follows a different GOF mechanism and has variants
specific to this disease** entity and has been curated separately. Due
to variants asserted in Medullary thyroid carcinoma (MIM:155240),
Pheochromocytoma (MIM:171300) with both multiple endocrine neoplasia
type 2A and type 2B, the evidence for each of these was lumped into the
appropriate gene-disease relationship based on the variant asserted for
the evidence.

**Literature Review:**

**OMIM**: https://www.omim.org/entry/171400 and 162300

**GENEREVIEWS:** <https://www.ncbi.nlm.nih.gov/books/NBK1257/>

Pathogenic variants. The most common pathogenic variants are
**non-conservative substitutions located in one of six cysteine codons
in the extracellular domain of the encoded protein.** They include
codons **609, 611, 618, and 620 in exon 10 and codons 630 and 634 in
exon 11** \[Takahashi et al 1998\]. All of these variants have been
identified in families with MEN 2A and some have been identified in
families with FMTC. Pathogenic variants in these sites have been
detected in **98% of families with MEN 2A** \[Eng et al 1996\]. See
Table A for a database of RET variants \[Margraf et al 2009\].

The risk for aggressive MTC, pheochromocytoma, and hyperparathyroidism
can be estimated based on genotype. See Table 3 for management
recommendations.

In addition to the pathogenic variants in the cysteine residues in exons
10 and 11 that have been found in families with **MEN 2A, pathogenic
variants in codons 631, 768, 790, 804, 844, and 891, and others in exons
5, 8, 10, 11, and 13-16, have been identified in a small number** of
families \[Hofstra et al 1997, Berndt et al 1998, Kloos et al 2009,
Wells et al 2015\].

**Small, in-frame duplications have been reported in four** families
\[Höppner & Ritter 1997, Höppner et al 1998, Pigny et al 1999,
Niccoli-Sire et al 2003\].

Rare families with **two pathogenic variants in cis configuration** have
been reported; for example, alteration of both codons 634 and 635 in one
family with MEN 2A; alteration of both codons 804 and 844 in one family
with FMTC \[Bartsch et al 2000\]; and alteration of codons 804 and 806
in an individual with MEN 2B \[Miyauchi et al 1999\].

Distribution of RET Mutations in Multiple Endocrine

Neoplasia 2 in Denmark 1994--2014: A Nationwide StudyAccounting for 36%
of all families, RET germline mutations of codon 611 were the most
frequent. Subsequently, mutations of codons 634 (17%), 918 (14%), 618
(11%), 620, (8%), 631 (3%), 790 (3%), 804 (3%), 852 (3%), and 883 (3%)
followed (Table 3). No mutations of codons 292, 515, 533,609, 630, 666,
750, 768, 891, 904, or 912 were identified.

***Mathieson et al, 2017, PMID: 27809725***

In 1994 a study performed by the International RET Consortium (IRC)
\[35\] analyzed 477 MEN 2 kindred and demonstrated that about 94% of
cases presented with a RET germline mutation affecting one of the
following codons: 609, 611, 618, 620, 634, 768, 804 and 918. In the
present series, the percentage of RET positive families (193/195, 98.9%)
was higher than that reported in the IRC study.

***Eng et al 1996 PMID PMID: 8918855 Elisei et al, 2019,
PMID: 31510104***

More than 90% of the cases of MEN 2A and FMTC are caused by mutations in
one of the six highly conserved cysteine residues that are encoded by
exons 10 (codons 609, 611,618, and 620) and 11 (codons 630 and 634) of
the RET gene. In a few MEN 2A or FMTC families, mutations have been
reported involving non-cysteine domains in exons 11 (codon 631),
13(codons 768, 790, and 791), 14 (codon 804 and844), 15 (codon 891), or
a 9-bp duplication in exon 8 (8--12) ....Codon 804 mutations of the RET
exon 14,which have been detected in a few MEN 2A orFMTC families,
include single-base substitutionsresulting in a GTG !TTG (V804L),
GTG!CTG(V804L), or GTG!ATG (V804M) missensechange (11, 15--21). It has
been assumed thatmutation in non-cysteine domains, includingcodon 804 in
exon 14, results in a weaker onco-genic activation of the RET protein
and anattenuated form of the MEN 2A or FMTCphenotype compared to that
observed in patientswith cysteine domain mutation at codon 634 ofexon 11
(20, 22, 23). This proposal is supportedby clinical observations showing
a late onset andpossibly a reduced penetrance and expression ofthe
disease phenotype in families with codon 804mutations of the RET exon 14
(11, 20)

***Patocs et al, 2003, PMID: 12694233***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

*Optional modifier -- incomplete penetrance*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

Altered protein product

**Narrative summary of molecular mechanisms:**

The molecular mechanism of disease in MEN2a/b is gain of function,
resulting from an activating missense germline mutation of the RET
proto-oncogene (a tyrosine kinase). MEN2A results from missense mutations that lead to
ligand-independent dimerization (constitutive activation), located in
one of six cysteine codons in the extracellular domain of the encoded
protein (codons 609, 611, 618, and 620 in exon 10 and codons 630 and 634
in exon 11. In a few families a small in frame duplication and small 2bp
indel have been found. LOF mutations in RET are associated with
Hirschsprung disease.

**List variant classes in this gene proven to cause this disease:**

Missense

Inframe_deletion

Inframe_insertion

**Potential novel variant classes based on predicted functional
consequence**

splice_acceptor_variant predicted to escape NMD

splice_donor_variant predicted to escape NMD

frameshift_variant predicted to escape NMD

stop_gained predicted to escape NMD

Stop lost

**Not included**

splice_region_variant

splice_acceptor_variant

(splice_acceptor_variant predicted to undergo NMD)

splice_donor_variant

(splice_donor_variant predicted to undergo NMD)

Frameshift_variant

(frameshift_variant predicted to undergo NMD)

Stop_gained

(stop_gained predicted to undergo NMD)

start_lost

5_prime_UTR_variant

3_prime_UTR_variant

(gain of upstream Start \[uORF\])

(gain of upstream Start \[oORF\])

(Stop lost \[uORF\])

(Stop lost \[oORF\])

(Start lost \[uORF\])

(Frameshift \[uORF\])

(Frameshift \[oORF\])

(Stop gained \[uORF\])
**RET -- Multiple endocrine neoplasia, type 2b (MIM
162300)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:9967

The gene RET was first reported in relation to autosomal dominant
multiple endocrine neoplasia type 2B (MEN2B) characterized by very early
onset medullary thyroid cancer, pheochromocytoma, parathyroid tumors and
dysmorphic features first reported in in 1994 (Hofstra et al, 1994
PMID:7906866; Carlson et al. 1994 PMID:7977365). Earlier evidence
suggested that the disease MEN2B mapped to chromosome 10 (where the RET
gene is located) as early as 1990 (Norum et al., 1990 PMID:1979053).
**One variant, p.Met918Thr, is responsible for nearly all cases of
MEN2B, and it is estimated that around 50% of the reported cases of
MEN2B are de novo** (Carlson et al. 1994 PMID:7977365; Plaza-Menaho et
al., 2006; PMID: 16979782). Another RET variant (**p. Ala883Phe) has
been asserted to be responsible for the development of \~5% of MEN2B**
(Smith et al., 1997 PMID:9294615). The molecular mechanism for the gene
RET in the disease entity MEN2B is **gain of function**. While similar
to RET-MEN2A (described in a separate curation), in that the mechanism
is GOF, the mechanism(s) responsible for the two disease entities are
distinct. In MEN2B the p.Met918Thr variation occurs in exon 16 of RET in
the **tyrosine kinase domain which causes autophosphorylation and
activation of RET.** There is a significant amount of case-level data
with the maximum points for genetic evidence reached (12 points). This
gene-disease relationship is supported by functional studies including
expression, cell assays and animal models, several that were patient
derived pathogenic variants expressed in the mouse (reviewed in Wiedmann
et al, 2016 PMID: 26184857). These animal models developed carcinomas
consistent with the disease multiple endocrine neoplasia type 2B.

**Literature Review:**

**OMIM**: https://www.omim.org/entry/171400 and 162300

**GENEREVIEWS:** <https://www.ncbi.nlm.nih.gov/books/NBK1257/>

Approximately 95% of all individuals with the **MEN 2B phenotype have a
pathogenic variant in the tyrosine kinase domain of RET at codon 918 in
exon 16,** which substitutes a threonine for methionine \[Eng et al
1996\]. A second pathogenic variant, **p.Ala883Phe, resulting from a
two-nucleotide inde**l, has been found in 2%-3% of individuals with MEN
2B \[Gimm et al 1997, Smith et al 1997\]. **Two variants in cis
configuration on one RET allele** have been reported in individuals with
MEN 2B (see Table 5 for codon 804 in combination with 778, 805, 806, and
904) \[Miyauchi et al 1999, Menko et al 2002, Cranston et al 2006, Kloos
et al 2009\].


Rare families with **two pathogenic variants in cis configuration** have
been reported; for example, alteration of both codons 634 and 635 in one
family with MEN 2A; alteration of both codons 804 and 844 in one family
with FMTC \[Bartsch et al 2000\]; and alteration of codons 804 and 806
in an individual with MEN 2B \[Miyauchi et al 1999\].

Distribution of RET Mutations in Multiple Endocrine

Neoplasia 2 in Denmark 1994--2014: A Nationwide StudyAccounting for 36%
of all families, RET germline mutations of codon 611 were the most
frequent. Subsequently, mutations of codons 634 (17%), 918 (14%), 618
(11%), 620, (8%), 631 (3%), 790 (3%), 804 (3%), 852 (3%), and 883 (3%)
followed (Table 3). No mutations of codons 292, 515, 533,609, 630, 666,
750, 768, 891, 904, or 912 were identified.

***Mathieson et al, 2017, PMID: 27809725***

In 1994 a study performed by the International RET Consortium (IRC)
\[35\] analyzed 477 MEN 2 kindred and demonstrated that about 94% of
cases presented with a RET germline mutation affecting one of the
following codons: 609, 611, 618, 620, 634, 768, 804 and 918. In the
present series, the percentage of RET positive families (193/195, 98.9%)
was higher than that reported in the IRC study.

***Eng et al 1996 PMID PMID: 8918855 Elisei et al, 2019,
PMID: 31510104***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

*Optional modifier -- incomplete penetrance*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

Altered protein product

**Narrative summary of molecular mechanisms:**

The molecular mechanism of disease in MEN2b is gain of function,
resulting from an activating missense germline mutation of the RET
proto-oncogene (a tyrosine kinase).  While similar
to RET-MEN2A (described in a separate curation), in that the mechanism
is GOF, the mechanism(s) responsible for the two disease entities are
distinct. In MEN2B the p.Met918Thr variation occurs in exon 16 of RET in
the tyrosine kinase domain which causes autophosphorylation and
activation of RET. In MEN2B nearly all cases are caused
by one variant, p.Met918Thr (50% *de novo*) with 5% caused by
p.Ala883Phe. LOF mutations in RET are associated with
Hirschsprung disease.

**List variant classes in this gene proven to cause this disease:**

Missense

Inframe_deletion

Inframe_insertion

**Potential novel variant classes based on predicted functional
consequence**

splice_acceptor_variant predicted to escape NMD

splice_donor_variant predicted to escape NMD

splice donor variant

frameshift_variant predicted to escape NMD

stop_gained predicted to escape NMD

Stop lost

**Not included**

splice_region_variant

splice_acceptor_variant

(splice_acceptor_variant predicted to undergo NMD)

(splice_donor_variant predicted to undergo NMD)

Frameshift_variant

(frameshift_variant predicted to undergo NMD)

Stop_gained

(stop_gained predicted to undergo NMD)

start_lost

5_prime_UTR_variant

3_prime_UTR_variant

(gain of upstream Start \[uORF\])

(gain of upstream Start \[oORF\])

(Stop lost \[uORF\])

(Stop lost \[oORF\])

(Start lost \[uORF\])

(Frameshift \[uORF\])

(Frameshift \[oORF\])

(Stop gained \[uORF\])
**RYR1 -- Malignant hyperthermia (MIM 145600)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:10483

The **penetrance of MHS is unknown**. What is known is that up to 50% of
individuals with MHS have undergone anesthesia uneventfully despite use
of one of the agents known to trigger MH. (Tier 4)

**Loss of function variants in RYR1 have been implicated in autosomal
recessive and autosomal dominant** diseases. The autosomal recessive
conditions include congenital fiber-type disproportion (CFTD) (10-20% of
cases) and Multiminicore disease (MmD), especially cases associated with
ophthalmoplegia (in a small minority of cases). The variants in these
diseases are **mostly missense mutations, not large deletions or
duplications**. Autosomal dominantly-inherited conditions include
Central Core Disease (CCD) and malignant hyperthermia susceptibility
(MHS). Most autosomal dominant cases of CCD are also associated with
variants in RYR1, but notably there also have been reported occurrences
of autosomal recessive variants in RYR1 for this condition. Rossi et al
2006 (PMID 17293538) reported a family with a single-nucleotide
deletion in exon 100 that led to a deletion of the last 202 amino
acids (p.R4837fsX4839). Although the father and his two daughters were
all confirmed to have this mutation, only he was clinically affected
based upon the standard in vitro contracture test (IVCT). Of note,
however, the daughters did have the same histologic changes in the
skeletal muscle fibers so their lack of clinical findings could be due
to reduced penetrance. Monnier et al 2001 (PMID 11709545) identified
three in-frame amino acid deletions (c.12640 del9nt, c.13938 del6nt, and
c.14578 delttc) that led to CCD in three multi-generation families
meeting clinical criteria and histologic findings in skeletal muscles.
None of the mutations was found in 100 unrelated chromosomes from the
general population. In up to 70-80% of cases of MHS, variants in RYR1
are believed to be causative. Most mutations associated with MHS are
**single nucleotide substitutions, but a single amino acid deletion
resulting in deletion of a conserved glutamic acid** at position 2347
(p.G2347del) was reported as the first deletion mutation. Sambuughin et
al 2001 (PMID 11389482) identified this in two unrelated families with
this mutation in multiple affected family members. Interestingly, the
same variant was observed in autosomal dominant MHS and autosomal
recessive lethal neonatal hypotonia in the same family. Monnier et al
2009 (PMID 19734047) described a child with lethal neonatal hypotonia
who had compound heterozygosity for mutations in RYR1 including
p.Lys929_Ser3713delinsAsn (leading to an in-frame large deletion
extending from exon 23 to exon 76) and a non-synonymous p.Ser2279dup,
exon 42. Given that both parents were confirmed carriers, each for one
of the mutations, and were unaffected, these were presumed to be
recessive mutations for the neonatal hypotonia. However, the father who
was the confirmed p.Ser2279dup carrier was found by IVCT to meet
diagnostic criteria for autosomal dominantly inherited MHS trait.

**Literature Review:**

**OMIM:** https://www.omim.org/entry/145600

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1146/

Note that due to the **gain-of-function disease** mechanism, genetic
heterogeneity, and variable expressivity of this disorder, data from
functional studies are critical in reaching a likely pathogenic or
pathogenic classification using ACMG criteria.

**Missense variants** in the ryanodine receptor 1 gene (RYR1) are
associated with malignant hyperthermia but only a minority of these have
met the criteria for use in predictive DNA diagnosis.....Segregation
analysis is of limited value in assessing pathogenicity of RYR1 variants
in malignant hyperthermia. Functional analyses in HEK293 cells provided
evidence to support the use of p.R2336H, p.R2355W, p.E3104K, p.G3990V
and p.V4849I for diagnostic purposes but not p.D3986E.

*Merritt A et al, 2017, PMID: 28403410*

The majority of mutations (96%) so far detected in the RYR1gene in
association with MH and/or CCD are **missense changes**(Table 2 and
Supplementary Table S1, which is available online
athttp://www.interscience.wiley.com/jpages/1059-7794/suppmat). Approximately
40% of RYR1 mutations occur at CpG dinucleotide sequences, with almost
all mutations occurring in the heterozygous state and homozygotes being
reported on rare occasions, e.g.,mutation p.C35R described by Lynch et
al. \[1997\]. In cases of **MH41% of reported mutations are
recurrent**, accounting for MH susceptibility in 80% of genotyped
families.

*Robinson, 2006, PMID: 16917943*

It is recognized that the lack of penetrance of the clinical MH
phenotype can be for non-genetic reasons, as some people who develop MH
are known to have had previous exposure to triggering anesthetics with
no apparent problem.

*Riazi, 2018, PMID: 28902675*

A recent study compared the predicted and actual consequences of
missense mutations and found that half of the de novo or low-frequency
missense mutations found by genome sequencing and inferred as
deleterious, correspond to nearly neutral variants that have little
impact on the clinical phenotype of individual cases.48 Similarly, a
significant proportion of RYR1 sequence variants in the human gene
mutation database classified as "disease-causing mutations" was found to
be benign, probably benign, or as being of unknown pathogenicity.37

The present report documents, in exon 44 of RYR1 in two unrelated,
MH-susceptible families, a 3-bp deletion that results in deletion of
a conserved glutamic acid at position 2347. This is the first deletion,
in RYR1, found to be associated with MH susceptibility.

*Sambuughin, 2003, PMID: 11389482*

Exome sequencing was performed on 870 volunteers not ascertained for
MHS. Variants in RYR1 and CACNA1S were annotated using an algorithm that
filtered results based on mutation type, frequency, and information in
mutation databases. .....In RYR1, the authors identified 65 **missense
mutations, one nonsense, two that affected splicing, and one
non-frameshift indel**

*Gonslaes, 2013, PMID: 24195946*

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

*Incomplete penetrance*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

*Altered gene product structure*

*(Decreased gene product level)*

**Narrative summary of molecular mechanisms:**

Mechanism is predominantly thought to be gain-of-function disease (with
genetic heterogeneity and variable expressivity), though some Loss of
function variants in RYR1 have been implicated in autosomal dominant
disease. The penetrance is unknown but recognised to be incomplete. Most
variants are missense (some recurrent), though a small number of
nonsense, splicing, and non-frameshift deletions are noted.

**List variant classes in this gene proven to cause this disease:**

Missense

Nonsense

Splice_acceptor_variant

Splice_donor_variant

Inframe_deletion

**Potential novel variant classes based on predicted functional
consequence**

Splice acceptor variant predicted to escape NMD

Splice donor variant predicted to escape NMD

Frameshift variant predicted to escape NMD

Stop gained predicted to escape NMD

Stop lost

Inframe_insertion


*?? If consider haploinsufficiency as a potential MOA*

Frameshift

Splice region variant

Stop_gained

start_lost

Splice acceptor predicted to escape NMD

Splice donor predicted to escape NMD

stop_gained predicted to escape NMD

stop_lost

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[oORF\]

Frameshift \[oORF\]
# **RYR2 -- CPVT**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:10484

**ClinGen Haploinsufficiency comments:**

The RYR2 gene encodes the cardiac ryanodine receptor that regulates contraction by the intracellular release of calcium and plays and essential rolein excitation-contraction in the heart (PMID: 24743769) . **Pathogenic changes that result in gain-of-function of the RYR2 gene have reported in individuals with several types of tachyarrhythmias, including catecholaminergic polymorphic ventricular tachycardia (CPVT)**, catecholaminergic idiopathic ventricular fibrillation, atrial fibrillation and a structurally normal heart (Napolitano et al., 2014). Loss of function variants have not been associated with RYR2-related disorders. An in-frame deletion of exon 3 has been reported in patients with CPVT and other arrhythmias, however the pathogenetic mechanism has not been well defined (PMID: 24743769).

**Literature review:**

The main gene responsible for up to 50% of CPVT cases is RyR2, which encodes for the cardiac isoform of the ryanodine receptor. RyR2 mediates the release of calcium from the sarcoplasmic reticulum, required for sarcomere contraction. **Genetic alterations in the RyR2 gene increases calcium release and can trigger life-threatening ventricular arrhythmias under catecholaminergic stimulation**

*Bosch C et al. 2017 Jan (PMID:27988446)*

Four different single nucleotide substitutions leading to hRyR2 missense mutations (nonconservative amino acid changes) were identified in the 4 probands with catecholaminergic VT.

*Priori SG et al. 2001 Jan 16 (PMID:11208676)*

"RyR2 mutations appear to be preferentially located in four regions (Figure 3). The term “domains” indicates those regions, according to this classification, domain I includes amino acids 77 to 466, domain II amino acids 2246–2534 , domain III amino acids 3778–4201 and domain IV amino acids 4497–4959 (Figure 4). These regions are highly conserved in RyR across species and are superimposable (except for region III) to the localization of RyR1 mutations associated with central core disease and malignant hyperthermia (47)...analysis of our CPVT cohort by direct ORF sequencing indicates that 24% of RyR2 mutations identified in CPVT patients are located outside the four canonical domains."

"...observations indicate that CPVT RyR2 mutations preferentially sensitize the channel to luminal Ca2+ activation, while only a few CPVT RyR2 mutations sensitize the channel to both cytosolic and luminal Ca2+ activation."

"Premature stop codons, frameshifts, and out-of-frame insertions or deletions **have not** been identified in CPVT patients screened for mutation on the RYR2 gene."

**"We have identified 82 RYR2 mutations in our CPVT probands: 79 were point mutations and 3 (3.6%) were small deletions/insertions (one in frame insertion and two in frame deletions).** Small in-frame duplications or in-frame insertions (48,53) are present in patients with the CPVT phenotype, suggesting that their functional consequences are similar to that of point mutations, rendering the channel more prone to spontaneous SR Ca2+ release during adrenergic stimulation."

*Priori SG et al.2011 (PMID: 21454795)*

More than 150 RYR2 pathogenic variants causing CPVT have been reported to date [Priori & Chen 2011]. Sequencing of the entire coding region and flanking intronic regions is optimal, as 24% of pathogenic variants are located outside of the regions encoding the KBP12.6-binding region, the calcium-binding domain, and the transmembrane domain (C-terminus) [Priori & Chen 2011].

No mutation hot spots have been reported to date.

Exon-spanning deletions have been reported [Marjamaa et al 2009, Medeiros-Domingo et al 2009, Campbell et al 2015, Leong et al 2015].

The mean penetrance of RYR2 pathogenic variants is 83% [Author, unpublished data]. Therefore, asymptomatic individuals with RYR2-related CPVT are a minority.

*Gene reviews https://www.ncbi.nlm.nih.gov/books/NBK1289/*

#### **Pilot application of harmonised terms:**

**Inheritance:**

Autosomal dominant

(optional) modifiers

**Allelic requirement:**

Monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is likely gain of function through **altered gene product structure** causing increased RYR2 channel sensitivity to Ca2+ activation and increased spontaneous calcium release. The majority of the pathogenic variants are missense. Premature stop codons, frameshifts, and out-of-frame insertions or deletions **have not** been identified in CPVT patients. Exon-spanning deletions have been reported. Sequencing of the entire coding region and flanking intronic regions is optimal, as 24% of pathogenic variants are located outside of the regions encoding the KBP12.6-binding region, the calcium-binding domain, and the transmembrane domain (C-terminus). Penetrance is high, estimated by the authors of gene reviews at approximately 80%.

**List variant classes in this gene proven to cause this disease:**

- Missense
- In frame deletion
- In frame insertion

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant predicted to escape NMD
- Splice donor variant
- Splice donor variant predicted to escape NMD
- Frameshift predicted to escape NMD
- Stop_gained predicted to escape NMD
- Stop_lost# **SCN5A -- Brugada Syndrome**

**Review of source material:**

Hosseini SM et al. 2018 Sep. Circulation. PMID: 29959160
<https://doi.org/10.1161/CIRCULATIONAHA.118.035070>

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:10593

**Genetic evidence:**

-   Autosomal dominant disorder

-   Case level data with evidence of probands with predicted or proven
    null variants or some evidence of gene impact

> *Vatta M et al. 2002 Apr (PMID:12051963); Deschênes I et al. 2000 Apr
> (PMID:10727653); Chen Q et al. 1998 Mar 19 (PMID:9521325); Bezzina C
> et al. 1999 Dec 3-17 (PMID:10590249)*

-   Evidence of segregation in one or more families

*Chen Q et al. 1998 Mar 19 (PMID:9521325)*

-   Case control data (aggregate variant analysis)

> *Kapplinger JD et al. 2010 Jan (PMID:20129283); Schulze-Bahr E et al.
> 2003 Jun (PMID:14961552)*

**Experimental evidence:**

-   Cell culture model

*Liang P et al. 2016 Nov 8 (PMID:27810048);*

**Conclusion:**

-   Convincing evidence with replication in \>2 publications over time
    (\>3yrs)

-   No valid contradictory evidence

**Outcome:**

-   Gene-disease relationship - DEFINITIVE

**ClinGen Haploinsufficiency Comments:**

SCN5A encodes a subunit (type V, alpha) of a voltage-gated sodium channel protein that functions in cardiac muscle to regulate conduction. More than 600 SCN5A variants have been reported in association with cardiac channelopathy phenotypes.

Heterozygous sequence-level mutations in SCN5A are associated with a number of cardiovascular abnormalities including Brugada syndrome, cardiac conduction disease (CCD), long QT syndrome type 3 (LQT3), atrial fibrillation, and sick sinus syndrome (SSS). Functional studies (patch clamp assays, cellular localization studies, and western blots) have been performed on a number of the reported mutations; however, these studies have been done in in vitro models and it is unknown whether these mutations would have the same functional consequence in vivo. To this point, one study on SCN5A variants stated that ?caution should be taken when extrapolating the findings from the in vitro study of HEK293 cells and oocytes to the more complex in vivo conditions? (Gui et al. 2010).

**Based on the functional studies mentioned above, SCN5A loss-of-function-type mutations have been associated with Brugada syndrome,** CCD, and SSS, while SCN5A gain-of-function-type mutations have been associated with LQT3 (Hong et al., 2005; Makiyama et al., 2005; Zimmer and Surber, 2008; Kapplinger et al. 2010; Eastaugh et al., 2011; Remme 2013). Both loss-of-function and gain-of-function-type mutations of SCN5A have been identified in association with atrial fibrillation (Remme, 2013). Incomplete penetrance for some of the SCN5A mutant phenotypes has also been observed (Cordeiro et al., 2006; Makita et al., 2007; Eastaugh et al., 2011).

Although there are a number of SCN5A mutations that are reported to be associated with loss-of-function-type mechanism (Tfelt-Hansen et al. 2009), the SCN5A genotype/phenotype association is still not well understood as there are several mutations that have been associated with both Brugada syndrome (loss-of-function) and LQT3 (gain-of-function) (Zimmer and Surber et al., 2008; Blich et al. 2015). Due to the clinical heterogeneity, mutational spectrum, and lack of reports of whole gene deletions, the haploinsufficiency score is a 1.

**Literature review:**

The *SCN5A* gene encodes the alpha subunit of the main cardiac sodium channel Na~v~1.5. This channel predominates inward sodium current (INa) and plays a critical role in regulation of cardiac electrophysiological function. INa mediated by Nav1.5 can be classified into peak and late sodium currents (INa-P and INa-L). Mutations of SCN5A can impair Nav1.5 function and change the magnitude and duration of INa-P and INa-L, consequently leading to different types of fatal arrhythmias.

*Han et al 2018 (PMID: 29806494)*

"Original Brugada paper identified a missense mutation, an insertion of AA disrupting splice donor sequence, and single base deletion causing an in frame stop codon in exon 1. Missense mutation appeared to lead to altered SCN5A function whilst the latter 2 mutations were predicted to cause loss of function."

*Chen Q et al. 1998 Mar 19 (PMID:9521325)*

**Another study described 293 distinct mutations in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21 splice-site, and 9 in-frame deletions/insertions.** The 4 most frequent BrS1-associated mutations were E1784K (14), F861WfsX90 (11), D356N (8), and G1408R (7). Most mutations localized to the transmembrane-spanning regions.

*Kapplinger JD et al 2010 (PMID: 20129283)*

Over 300 SCN5A loss-of-function mutations have been identified in
connection with BrS. Misfolded channels, trafficking defects, and
negatively shifted steady-state inactivation curves contribute to a reduced availability of functional Nav1.5 channels on the plasma
membrane.

*Han et al 2018
(PMID: [29806494]; Schulze-Bahr et al 2003 (PMID: 14961552); Mizusawa et al 2012 (PMID: 22715240)*

To note, gain of function mutations in SCN5A have been linked to other cardiac phenotypes and loss of function and gain of function mutations can co-exist and have been linked to an overlapping phenotype.

*Han et al 2018 (PMID: 29806494)*

#### **Pilot application of harmonised terms:**

**Allelic requirement:**

Monoallelic_het

**Inheritance:**

Autosomal dominant

Optional modifiers: incomplete penetrance

**Disease associated variant consequences:**

Dose Change: dose reduction: Decreased gene product level

Dose Change: dose reduction: Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism likely loss of function of SCN5A due to **reduction/absence of gene product or altered gene product** due to a variety of mechanisms (e.g. null alleles, misfolded channels, trafficking defects, and negatively shifted steady-state inactivation curves contribute to a reduced availability of functional Nav1.5 channels on the plasma membrane.) One study described 293 distinct mutations in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21 splice-site, and 9 in-frame deletions/insertions. The 4 most frequent BrS1-associated mutations were E1784K, F861WfsX90, D356N, and G1408R. Most mutations localized to the transmembrane-spanning regions. To note gain of function mutations have been reported in association with Long QT syndrome. Although loss of function appears to be the mechanism in Brugada the SCN5A genotype/phenotype association is still not completely understood.

**List variant classes in this gene proven to cause this disease**

- Splice region variant
- Spice acceptor variant
- Splice donor variant
- Start lost
- Frameshift variant
- Stop gained
- Stop gained predicted to undergo NMD
- Missense
- In frame insertion
- In frame deletion

**Potential novel variant classes based on predicted functional
consequence:**

- Splice acceptor variant predicted to escape NMD
- Splice donor variant predicted to escape NMD
- Frameshift variant predicted to escape NMD
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
### **SCN5A -- Long QT Syndrome**



**Review of source material:**
*Adler A et al 2020 (PMID: 31983240)*

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:10593

**ClinGen Evidence for Haploinsufficiency:**
SCN5A loss-of-function-type mutations have been associated with Brugada syndrome, CCD, and SSS, **while SCN5A gain-of-function-type mutations have been associated with LQT3** (Hong et al., 2005; Makiyama et al., 2005; Zimmer and Surber, 2008; Kapplinger et al. 2010; Eastaugh et al., 2011; Remme 2013). Both loss-of-function and gain-of-function-type mutations of SCN5A have been identified in association with atrial fibrillation (Remme, 2013). Incomplete penetrance for some of the SCN5A mutant phenotypes has also been observed (Cordeiro et al., 2006; Makita et al., 2007; Eastaugh et al., 2011).

**Literature review:**

"...we identified **identical intragenic deletions of SCNSA in affected members of two LQT families**. This deletion was not identified in more than 500 control individuals. 
....SCNSA is the cardiac sodium channel gene. Subtle mutations of this gene would be expected to cause an LQT phenotype. Finally, the type (an in-frame deletion of three amino acids) and location (a region of known importance for sodium channel inactivation) of the deletions support the conclusion that SCN5A is LQT3." 

*Wang Q et al 1995 PMID: 7889574*

"**Radical pathogenic variants such as frameshift, nonsense, and splice site types are relatively more frequent in KCNQ1 and KCNH2 and are not present in SCN5A in individuals with LQTS** (such pathogenic variants in SCN5A cause Brugada syndrome rather than LQTS)...

SCN5A consists of 28 exons, spans approximately 80 kb; it encodes a protein of 2,016 amino acids (NM_198056.2). An isoform lacking amino acid Gln1077 exists.

**More than 200 pathogenic variants are known; they include pathogenic missense variants and in-frame deletions or insertions**...

**Long QT syndrome (LQTS) associated with biallelic pathogenic variants or heterozygosity for pathogenic variants in two different genes (i.e., digenic pathogenic variants) is generally associated with a more severe phenotype** with longer QTc interval and a higher incidence of cardiac events [Schwartz et al 2003, Westenskow et al 2004, Tester et al 2005, Itoh et al 2010]."

*Gene reviews LQT
https://www.ncbi.nlm.nih.gov/books/NBK1129/*

"LQTS3 is caused by gain-of-function mutations of SCN5A. Approximately 8–10% of patients with SCN5A mutations are positively phenotypic as having LQTS. The first SCN5A mutation related to LQT3, the deletion of amino acids 1505–1507 (ΔKPQ), was identified by Wang et al.27 According to previous reports, cardiac events primarily occurred during sleep in LQT3 patients, and 18% died suddenly.28 **The gain-of-function SCN5A mutation leads to enhanced INa-P and INa-L, which finally triggers life-threating arrhythmias primarily in LQT3 patients...**

The underlying mechanisms of SCN5A mutations that lead to the gain-of-function of INa-P are mainly due to abnormalities in mutation-induced kinetic properties, including augmented INa-P amplitudes, negative shifts in the voltage-dependence of activation, and an increased speed of recovery from inactivation...

An increase in INa-L due to acquired conditions or inherited SCN5A mutations in favor of intracellular Ca2+ loading,47,48 the occurrence of early and delayed after depolarization (EAD and DAD),49,50 triggered activities,51 and spontaneous diastolic depolarization52 that promotes the spatial and temporal dispersion of ventricular repolarization can lead to reentrant arrhythmias...

SCN5A mutations that present with an overlapped phenotype of LQT3 and BrS were also described.124 In vitro studies suggested that these uncommon SCN5A mutations cause a mixed phenotype by altering the amplitude of INa-P and INa-L through enhanced sodium channel inactivation, a negative shift in steady‐state sodium channel inactivation, and enhanced tonic block in response to sodium channel blockers."

*Han D et al. 2018 PMID: 29806494*

"LQTS3 differs from LQTS1 and LQTS2 in various aspects (5). **LQTS3 patients present more often with marked resting bradycardia, and QT interval prolongation is more pronounced during slow heart rate (with in fact normalization at faster heart rates). This behavior may explain why arrhythmic events occur more frequently at rest and are less likely to be triggered by adrenergic stressors (6). Furthermore, the first cardiac event is more likely to be lethal and seems to occur during or after puberty (compared with much earlier in childhood in for example LQTS1)...**

Nav1.5 mutations in LQTS3 display a gain-of-function either by a pathological increase in INaL or window current or both...

...Besides a retarded repolarization process, a sustained influx of Na+ ions (through increased INaL) or more Na+ influx (through augmented window current) may lead to higher intracellular Na+ concentrations within the cardiomyocyte. Subsequently, this may cause intracellular Ca2+ overload (through reverse-mode operation of the Na+/Ca2+ exchanger) with potential adverse effects on myocardial contraction, relaxation, and oxygen consumption"

*Wilde A et al.2018 PMID: 29798782*

"nearly 2% of healthy Caucasians and 5% of healthy nonwhite subjects also host rare missense SCN5A variants"

*Kapplinger JD et al 2010 PMID: 20129283*

"Although clustering in the three interdomain linkers (IDL), control mutations were scattered throughout all gene regions, highlighting a significantly greater degree of genetic background “noise” in SCN5A than in either KCNQ1 or KCNH2. Moreover, given that the case frequency of missense mutations is lowest in SCN5A, the relative frequency of missense mutations found in cases compared to controls was quite low...

The region in SCN5A with the greatest association between mutation discovery and disease causation was the transmembrane/linker regions..."

*Kapa S et al 2009 PMID: 19841300*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

Incomplete penetrance 

Digenic (other LQT genes)

**Allelic requirement:**

Monoallelic_aut

(optional) modifiers:

Digenic (other LQT genes)

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is likely gain of function of SCN5A due to **altered gene product structure**. 

More than 200 pathogenic variants are known, they include pathogenic missense variants and in-frame deletions or insertions. The sodium current mediated by Nav1.5 consists of peak and late components (INa-P and INa-L). It is thought that gain-of-function SCN5A mutations lead to enhanced INa-P and INa-L, which can trigger life-threating arrhythmias. 

Rare missense variants are estimated to occur in around 2% of healthy Caucasians and 5% of healthy nonwhite subjects so collectively missense variants are not rare in the healthy population.

Can present with specific features: patients may have marked resting bradycardia, QT interval prolongation more pronounced during slow heart rate (which might explain why arrhythmic events occur more frequently at rest), a first cardiac event that is lethal, and onset after puberty. 

Digenic inheritance has been reported with variants in SCN5A reported alongside pathogenic variants in other LQT genes. Biallelic pathogenic variants or digenic pathogenic variants are generally associated with a more severe phenotype with longer QTc interval and a higher incidence of cardiac events. 

Note loss of function variants in SCN5A are associated with Brugada syndrome and loss of function and gain of function variants can co-exist causing a mixed phenotype.

**List variant classes in this gene proven to cause this disease:**

- Missense
- In frame insertion
- In frame deletion

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant predicted to escape NMD
- Splice donor variant
- Splice donor variant predicted to escape NMD
- Frameshift predicted to escape NMD
- Stop_gained predicted to escape NMD
- Stop_lost

# **SDHAF2 -- Paraganglioma**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:26034

**ClinGen Evidence for Haploinsufficiency**

PMID 22241717 - SDHAF2 (also known SDH5) is a tumor suppressor gene which encodes for a protein required for flavination of a succinate dehydrogenase subunit. A novel frameshift mutation was reported by Piccini et al 2012 (PMID: 22241717) in a 44-year-old woman diagnosed with paraganglioma, but with a negative family history. In exon 4, she had c.357-358insT (p.Tyr119LeufsX7) mutation which has not been observed in other patients and there was no indication that other family members were tested.

PMID 26096992 - Zhu et al 2015 (PMID 26096992) assessed the frequency of germline mutations in Chinese patients with head and neck paragangliomas (HNPGL) without family history. They identified one female patient who had early age of onset of HNPGL who had a novel truncation mutation c.130C>T (p.Gln44Ter).

In a Dutch kindred and a Spanish family (proven to be unrelated by high-density SNP array of 610,000 SNPs) with early onset of head and neck paragangliomas (Hao et al (2009), PMID 19628817 and Bayley et al (2010), PMID 20071235), all affected family members were found to inherit a c.232G>A change in exon 2, which caused a Gly78Arg (G78R) mutation. Of note, this variant was not identified in 400 unaffected controls. **There is notable parent-of-origin effect given that the phenotype is only expressed when paternally inherited.** In both families, it is not expressed when the mutation is maternally inherited. Mulitplex-PCR for large deletions of SDHAF2 in 200 patients did not detect any deletions. (Bayley et al (2010), PMID 20071235).

A novel frameshift mutation has been reported in a patient with nonsyndromic paraganglioma and a novel truncation mutation has been reported in a patient with benign HNPGLs. Their phenotype is consistent with that of affected patients from 2 large families with a recurrent missense mutation in the same SDHAF2 gene.

**Literature review:**

SDHA, SDHAF2, SDHB, SDHC, and SDHD are tumor suppressor genes. Somatic second-hit variants in tumors include gross chromosomal rearrangements, recombination, single-nucleotide variants, or epigenetic changes that result in allelic inactivation.

Affected individuals from a Dutch family described by van Baars et al [1982] were found to have a single-nucleotide change (c.232G>A) in exon 2 in SDHAF2. This resulted in a p.Gly78Arg alteration in the most conserved region of the protein and is believed to be a founder variant [Hensen et al 2012]. Of note, c.232G>C (p.Gly78Arg) has also been reported [Piccini et al 2012]. Additional loss-of-function variants have been reported in SDHAF2.

Pathogenic variants in SDHD and SDHAF2 (and possibly MAX) demonstrate parent-of-origin effects and cause disease almost exclusively when they are paternally inherited.

*Gene reviews
https://www.ncbi.nlm.nih.gov/books/NBK1548/*

A study of 200 patients did not "identify any germline or somatic mutations of SDHAF2, and no gross gene deletions were noted in the subset of apparently sporadic patients analysed. Investigation of the Spanish family identified a pathogenic germline DNA mutation of SDHAF2, 232G→A (Gly78Arg), identical to the Dutch kindred."

"SDHAF2 mutations do not have an important role in phaeochromocytoma and are rare in head and neck paraganglioma. Identification of a second family with the Gly78Arg mutation suggests that this is a crucial residue for the function of SDHAF2. We conclude that SDHAF2 mutation analysis is justified in very young patients with isolated head and neck paraganglioma without mutations in SDHD, SDHC, or SDHB, and in individuals with familial antecedents who are negative for mutations in all other risk genes."

*Bayley JP et al 2010 PMID: 20071235*

Of 972 participants in the European-American-Asian Pheochromocytoma-Paraganglioma Registry without mutations in the classic pheochromocytoma/paraganglioma susceptibility genes, 58 probands (6.0%) carried certain or likely pathogenic germline mutations that included 29 in SDHA, 20 in TMEM127, 8 in MAX, and 1 in SDHAF2.

*Bausch B et al 2017 PMID 28384794*

In a large Dutch family, originally described by van Baars et al. (1982), segregating autosomal dominant paraganglioma (PGL2; 601650), Hao et al. (2009) identified a G-to-A transition at nucleotide 232 of exon 2 of the SDHAF2 gene, resulting in a gly-to-arg substitution at codon 78 (G78R). This mutation was not found in 400 unaffected control individuals and segregated with the phenotype in the family. Thirty-three individuals with the mutation had developed the disease, but not 7 individuals (median age 74 years) who had inherited the mutation from their mothers. This suggested an SDHD (602690)-like parent of origin-specific inheritance. Only 5 individuals (median age 42 years) with a paternal mutation had not developed overt paraganglioma. This reduced penetrance was thought to relate to young age and/or presence of undetected tumors.

Hensen et al. (2012) identified the Dutch founder G78R mutation in 46 cases from 4 Dutch families out of a larger cohort of 1,045 patients from 340 Dutch families with paraganglioma and pheochromocytoma.
*Omim
https://www.omim.org/entry/613019?search=SDHAF2&highlight=sdhaf2*

Functional assays of SDHAF2 Gly78Arg have shown this variant results in a destabilized SDHAF2 protein and impairs SDHAF2-SDHA interaction (Hao 2009, Bezawork-Gelata 2014).
*ClinVar
https://www.ncbi.nlm.nih.gov/clinvar/variation/401/*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

**Allelic requirement:**

monoallelic_aut

(optional) modifiers: imprinting effect - parent of origin effect (paternal)

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Although the mechanim appears to be loss of function, much of the literature on SDHAF2 is centred around a missense variant decribed in a large Dutch family. A single-nucleotide change (c.232G>A) in exon 2 in SDHAF2. From functionl studies this founder variant appears to cause destabilisation of the SDHAF2 protein and impaired SDHAF2-SDHA interaction. A study of 200 patients with paragangliomas did not identify any patients with deletions in SDHAF2 however a few loss of function variants (nonsense and frameshift) have been reported. There are 2 splicing variants on ClinVar which have been classified as likely pathogenic.
Pathogenic variants in SDHD and SDHAF2 (and possibly MAX) demonstrate parent-of-origin effects and cause disease almost exclusively when they are paternally inherited.

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Missense


**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant
- Splice acceptor variant (predicted to escape NMD)
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- In frame insertions
- In frame deletions
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
# **SDHB -- Paraganglioma**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:10681

**ClinGen Evidence for Haploinsufficiency**

In addition to the Iberian founder mutation, a Dutch founder deletion has also been described, involving exon 3 (c.201-4429_287-933del) (PMID: 25827221). A deletion of exons 1 and 2 has also been reported in an individual of Japanese descent (PMID:20379037).

PMID: 29386252
Andrews et al (2017) performed a retrospective survey of 1832 individuals with personal or family history of pheochromocytoma/paraganglioma. **A total of 673 had germline SDHB variants detected, of which 36 had single or multi-exon deletions and duplications. The most common copy number variants were exon 1 deletion (n=18, 49%), exon 3 deletion (n=5, 14%) and whole gene deletion (n=2, 5.6%). No whole gene duplications were identified.**

PMID: 27485256
Hoekstra et al (2016) utilized MLPA analysis and breakpoint mapping of index patients with paraganglioma/pheochromocytoma who had tested negative for SDH gene sequence variants to identify 7 unique intragenic deletions within SDHB in 8 cases (no other genes involved). Two cases with whole gene deletion were also detected (contiguous multi-gene deletion confirmed for one, other unknown). Inheritance was not assessed.

PMID: 19454582
Burnichon et al (2009) tested a cohort of 445 patients with paragangliomas. Germline SHDB variants were identified in 96 patients, of which four unique large deletions, 12 frameshift, 7 splice site, 12 nonsense and 23 missense. Of the large deletions, one was the ~20kb deletion described by Cascon et al.

PMID: 12618761
Benn et al (2003) identified novel SDHB variants in the probands from four families and two apparently sporadic cases (six of seven probands studied), including two missense mutations, a single nonsense and frameshift mutation, as well as two splice site mutations, one of which was shown to have partial penetrance resulting in 'leaky' splicing.

PMID:19351833
A patient with paraganglioma in Neumann et al (2009) showed an exon 3 duplication, but there are no reports of whole gene duplication.

Note on Penetrance of SDHB variants:
PMID: 26113606
Baysal and Maher (2015) note that for "germline SDHB mutations, the increased use of presymptomatic genetic testing in extended families has resulted in recognition that the penetrance of SDHB mutations is lower than initially thought. Thus initial estimates of the penetrance of germline SDHB mutations were in excess of 70% but have progressively fallen to 25?40% (Benn et al. 2006, Solis et al. 2009, Hes et al. 2010, Ricketts et al. 2010, Schiavi et al. 2010)."

**Literature review:**

SDHA, SDHB, SDHC, and SDHD are four nuclear genes responsible for hereditary PGL/PCC syndromes. They encode the four subunits of the mitochondrial enzyme succinate dehydrogenase (SDH).

SDHA, SDHAF2, SDHB, SDHC, and SDHD are tumor suppressor genes. Somatic second-hit variants in tumors include gross chromosomal rearrangements, recombination, single-nucleotide variants, or epigenetic changes that result in allelic inactivation.

**Nonsense, missense, splice site variants, intragenic deletions and insertions, and whole-gene SDHB deletions have been reported in individuals/pedigrees affected with hereditary PGL/PCC syndromes. More than 100 pathogenic sequence variants have been described for SDHB.** A database of normal and pathogenic variants for the SDH subunit genes is maintained by the Leiden University Medical Center (see Table A). SDHB variants are predominantly found in exons 1-7.

**Large SDHB deletions have been reported, most commonly involving SDHB exon 1, but also other multiexon deletions and whole-gene deletions** [Cascón et al 2006, Burnichon et al 2009, Neumann et al 2009, Solis et al 2009, Buffet et al 2012, Rattenberry et al 2013].

Pathogenic variants in SDHB result in reduced or absent succinate dehydrogenase function because of loss or dysfunction of the affected subunit, or failure of the SDH heterotetramer to assemble.

*Gene reviews
https://www.ncbi.nlm.nih.gov/books/NBK1548/*

Three hundred and forty-four probands and 436 of their relatives harboured an intragenic mutation in SDHB/SDHC/SDHD
(table 1). Forty-five intragenic mutations in 134 probands were
reported previously.10 The ratio of mutation classes among
probands was similar to that reported previously10 (44%
missense, 15% nonsense, 13% splice-site, 15% frameshift,
0.5% inframe deletions and 12% large CNAs). There were a number of recurrent mutations, for example, SDHB splice-site
c.72+1G>Tand SDHD missense c.242C>T (p.Pro81Leu)
mutations accounted for 20% of probands, and the 10 most
common mutations accounted for 48% (see online supplementary figure 3).

We unequivocally confirmed that SDHD mutation carriers had a
higher overall penetrance for symptomatic tumours and a higher
risk of head and neck paraganglioma (HNPGL) compared with SDHB, whereas SDHB mutation carriers had a higher risk of malignancy and were significantly more likely to develop phaeochromocytoma
and paraganglioma (PPGL).

Male SDHB mutation carriers have a higher age-related penetrance of PPGL/HNPGL (P=0.0034) and PPGL (P=0.0079),
compared with women

*Andrews KA et al 2018 PMID: 29386252*

A comprehensive database of germline SDHB and SDHD
mutations is maintained at http://chromium.liacs.nl/
LOVD2/SDH/home.php (Bayley et al. 2005). A wide variety
of intragenic mutations have been described and, more
recently, single or multiple exon deletions (and, occasionally, intragenic duplications; McWhinney et al. 2004,
Cascon et al. 2008, Neumann et al. 2009). 

A number of frequent SDHB and SDHD mutations were observed and
these may result from a high mutation rate or to founder
effects. 

Particularly for germline SDHB mutations, the
increased use of presymptomatic genetic testing in
extended families has resulted in recognition that the
penetrance of SDHB mutations is lower than initially
thought. Thus initial estimates of the penetrance of
germline SDHB mutations were in excess of 70% but
have progressively fallen to 25–40% (Benn et al. 2006, Solis
et al. 2009, Hes et al. 2010, Ricketts et al. 2010, Schiavi et al.
2010). The relatively low penetrance of SDHB mutations is
consistent with the observation of a low de novo mutation
rate, frequent founder mutations and the relatively high
number of mutations detected in apparently isolated cases
(Baysal et al. 2002, Neumann et al. 2002, Cascon et al.
2009, Jafri et al. 2013)

**For SDHB mutations, extra-adrenal and non-HNPGL is more often
the presenting feature than HNPGL or pheochromocytoma, and there is a significantly higher risk of malignant
paraganglioma and poor prognosis** (w25% lifetime risk;
Gimenez-Roqueplo et al. 2003, Amar et al. 2007, Ricketts
et al. 2010). 

SDH-mutated PPGLs show robust expression of hypoxia
induced genes, and genomic and histone hypermethylation. These effects occur in part through succinate-mediated inhibition of a-ketoglutarate-dependent dioxygenases.
Mutations in SDH subunits account for most familial and
sporadic HNPGLs and PPGLs, and have also been linked
to other neoplasms including GISTs, renal cancer, and
pituitary adenomas. Abundant evidence suggests that
constitutive hypoxic stimulation plays an important role
in development of SDH-mutated paraganglioma tumors.
However, mechanisms by which SDH regulates oxygen
sensing and signaling are poorly understood.

*Baysal B et al 2015 PMID: 26113606*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: Incomplete penetrance

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Deletions and loss of function mutations in SDHB are associated with paraganglioms. Pathogenic variants in SDHB result in reduced or absent succinate dehydrogenase function because of loss or dysfunction of the affected subunit or failure of the SDH heterotetramer to assemble. Details of mechanisms by which SDH mutations activate hypoxic pathways and trigger subsequent neoplastic transformation remain poorly understood. Nonsense, missense, and splice site variants, intragenic insertions and deletions, and a whole-gene deletion have been reported in SDHB. 
Large SDHB deletions have been reported, most commonly involving SDHB exon 1, but also other multiexon deletions and whole-gene deletions.For SDHB mutations, extra-adrenal and non-HNPGL is more often the presenting feature than HNPGL or pheochromocytoma, and there is a significantly higher risk of malignant
paraganglioma and poor prognosis (25% lifetime risk).
Initial estimates of the penetrance of
germline SDHB mutations were in excess of 70% but
have progressively fallen to 25–40%. 


**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Missense
- In frame deletions

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- In frame insertions
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]

# **SDHC -- Paraganglioma**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:10682

**ClinGen Evidence for Haploinsufficiency**

PMID 19546167 - Bayley et al (2009) describe patients with multi-exon deletions with paraganglioma, including one individual with a deletion of exons 5 and 6 of the SDHC.

PMID 15342702 - Baysal et al. (2004) described a multiplex "family with head and neck paragangliomas and...an 8.37 kb SDHC deletion, which spans two AluY elements and removes exon 6." They identified the same deletion in a reportedly unrelated sporadic case; further investigation identified "allele sharing with the familial cases at seven polymorphic markers near SDHC, suggesting a common ancestral origin."

PMID 17667967 - Pasini et al. (2008) described germline SDHC variants in individuals with paraganglioma and gastrointestinal stromal tumors. The authors identified one novel nonsense variant (c.43C>T; p. Arg15X) and one splice-site substitution (IVS5+1G>A or c.405+1G>A), which resulted in "exon 5 [being] spliced out, causing a frameshift and a stop codon in the 3? untranslated region of the gene."

**Literature review:**

SDHA, SDHB, SDHC, and SDHD are four nuclear genes responsible for hereditary PGL/PCC syndromes. They encode the four subunits of the mitochondrial enzyme succinate dehydrogenase (SDH).

SDHA, SDHAF2, SDHB, SDHC, and SDHD are tumor suppressor genes. Somatic second-hit variants in tumors include gross chromosomal rearrangements, recombination, single-nucleotide variants, or epigenetic changes that result in allelic inactivation.

**Nonsense, missense, splice site, regulatory, and whole-exon-deletion SDHC pathogenic variants have been reported in individuals** and pedigrees affected with hereditary PGL/PCC syndromes.

SDHC encodes the succinate dehydrogenase cytochrome b560 subunit, a 169-amino-acid protein.
Pathogenic variants in SDHC result in reduced or absent succinate dehydrogenase function because of loss or dysfunction of the affected subunit or failure of the SDH heterotetramer to assemble.

**Germline SDHC pathogenic variants appear to be primarily (but not exclusively) associated with HNPGL. However, up to 10% of SDHC-related tumors are observed in the thoracic cavity** [Peczkowska et al 2008, Else et al 2014].

The majority of GISTs associated with PGL (Carney Stratakis syndrome; OMIM 606864) occur in individuals with a germline pathogenic variant in SDHA or SDHC.

*Gene reviews
https://www.ncbi.nlm.nih.gov/books/NBK1548/*

Three hundred and forty-four probands and 436 of their relatives harboured an intragenic mutation in SDHB/SDHC/SDHD
(table 1). Forty-five intragenic mutations in 134 probands were
reported previously.10 **The ratio of mutation classes among
probands was similar to that reported previously10 (44%
missense, 15% nonsense, 13% splice-site, 15% frameshift,
0.5% inframe deletions and 12% large CNAs).**

**We found 83% (5/6) of SDHC CNAs were exon 6 deletions,** and all three SDHD CNAs in our series were exon 4 deletions.

In 34 SDHC mutation carriers with detailed clinical information, 19 were clinically affected (15 HNPGL, 3 PPGL and 1 had both HNPGL and PPGL). One patient with HNPGL had local spread and malignant features. Age-related risks of symptomatic PPGL/HNPGL in SDHC were similar to that of SDHD. Compared with SDHB mutation carriers, SDHC carriers had a lower risk of PPGL (P=0.02 and P=0.06 before and after Bonferroni correction) and a higher risk of HNPGL (P<0.001 after Bonferroni correction for three comparisons, log-rank test) (figure 1).

*Andrews KA et al 2018 PMID: 29386252*

The spectrum of mutations in our series encompassed
partial deletions, missense, and nonsense mutations. We
report two new mutations not previously described in patients with SDHC-associated PGL: a novel missense mutation (c.214CG; p.Arg72Gly), and a splice site mutation (c.4051 GC). Both mutations affect highly
conserved sites, and mutations of the same codon and
splice junction have been described previously (10, 11).
The c.4051GC mutation has been described in a patient with a gastrointestinal stromal tumor (GIST) (11).

**SDHC mutations appear to be associated with a lower disease penetrance of PGL/PC when
compared with SDHB or SDHD mutations.** Mediastinal
and multiple PGLs might be more common in PGL3 than
was appreciated in prior studies. Patients with SDHC mutations should undergo screening with biochemical evaluation and imaging. Furthermore, up to one in 10 patients
with PGL3 may develop a mediastinal PGL, and imaging
of this area should be considered for surveillance in individuals with SDHC mutations.

*Else T et al 2014 PMID: 24758179*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: incomplete penetrance

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Deletions and loss of function mutations in SDHC are associated with paraganglioms. Pathogenic variants in SDHC result in reduced or absent succinate dehydrogenase function because of loss or dysfunction of the affected subunit or failure of the SDH heterotetramer to assemble. Details of mechanisms by which SDH mutations activate hypoxic pathways and trigger subsequent neoplastic transformation remain poorly understood. Nonsense, missense, splice site variants, intragenic insertions and deletions, and a whole-gene deletion have been reported in SDHC. Approximately 10% of pathogenic variants appear to be larger deletions. One study found 5/6 copy number variants were exon 6 deletions. SDHC mutations appear to be associated with a lower disease penetrance of PGL/PC when compared with SDHB or SDHD mutations. 

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Missense
- In frame deletions

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- In frame insertions
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
# **SDHD -- Paraganglioma**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:10683

**ClinGen Evidence for Haploinsufficiency**

PMID 10657297 - Baysal et al. 2000 describe SDHD variants in five families with hereditary paraganglioma, including two nonsense variants. The authors report that " none of the mutations has been observed in more than 200 normal control chromosomes...[and that] the mutations cosegregate with the disease phenotype in all affected individuals. "

PMID 12111639 - Cascon et al. (2002) report variants in SDHD identified in individuals with paraganglioma and/or pheochromocytoma, including one nonsense variant and one "4-bp frameshift deletion in codon 112 (13732delGACT)" that reportedly "gave rise to a 132-amino acid-truncated protein by creating a premature stop codon."

Several deletions involving SDHD have also been reported, though many of these often include adjacent genes (either in part or in their entirety). For example, McWhinney (2004) et al describe one family with a 96 kb deletion including the entire SDHD gene with a history of paraganglioma (PMID:15531530) . The authors note that this region may also include part of the adjacent TIMM8B gene. Bayley et al (2009) describe two patients with paraganglioma who are deleted for exons 1 and 2 of SDHD as well as the promoter. These deletions also included varying amounts of TIMM8B and other genes (PMID:19546167). Additionally, Caninanos et al (2011) describe one family with paraganglioma and a 25 kb deletion involving the promotor and exons 1 and 2 of SDHD, as well as 5 other genes (DLAT, PIH1D2, C11orf57, TIMM8B, SDHD) (PMID: 20310044)..

**Literature review:**

SDHA, SDHB, SDHC, and SDHD are four nuclear genes responsible for hereditary PGL/PCC syndromes. They encode the four subunits of the mitochondrial enzyme succinate dehydrogenase (SDH).

SDHA, SDHAF2, SDHB, SDHC, and SDHD are tumor suppressor genes. Somatic second-hit variants in tumors include gross chromosomal rearrangements, recombination, single-nucleotide variants, or epigenetic changes that result in allelic inactivation.

**Nonsense, missense, and splice site variants, intragenic insertions and deletions, and a whole-gene deletion have been reported in SDHD in individuals and pedigrees with hereditary PGL/PCC syndromes. More than 70 pathogenic sequence variants have been described for SDHD (see Table A). SDHD pathogenic variants are distributed throughout the four exons of the gene.**

Pathogenic variants in SDHD result in reduced or absent succinate dehydrogenase function because of loss or dysfunction of the affected subunit or failure of the SDH heterotetramer to assemble.

No consistent genotype-phenotype correlations have been identified.

Pathogenic variants in SDHD and SDHAF2 (and possibly MAX) demonstrate parent-of-origin effects and cause disease almost exclusively when they are paternally inherited. However, a thorough family history and risk assessment should be used in determining surveillance strategies in these families regardless of suspected parent-of-origin effects.

*Gene reviews
https://www.ncbi.nlm.nih.gov/books/NBK1548/*

Three hundred and forty-four probands and 436 of their relatives harboured an intragenic mutation in SDHB/SDHC/SDHD
(table 1). Forty-five intragenic mutations in 134 probands were
reported previously.10 The ratio of mutation classes among
probands was similar to that reported previously10 (44%
missense, 15% nonsense, 13% splice-site, 15% frameshift,
0.5% inframe deletions and 12% large CNAs). **There were a number of recurrent mutations, for example, SDHB splice-site
c.72+1G>Tand SDHD missense c.242C>T (p.Pro81Leu)
mutations accounted for 20% of probands,** and the 10 most
common mutations accounted for 48% (see online supplementary figure 3).

We unequivocally confirmed that SDHD mutation carriers had a
higher overall penetrance for symptomatic tumours and a higher
risk of head and neck paraganglioma (HNPGL) compared with SDHB, whereas SDHB mutation carriers had a higher risk of malignancy and were significantly more likely to develop phaeochromocytoma
and paraganglioma (PPGL).

We replicated our previous finding that SDHD p.Pro81Leu
mutation carriers manifest almost exclusively with HNPGL,
while other SDHD mutation types predispose to both HNPGLs
and PPGLs. From a clinical perspective, it can be proposed
that SDHD p.Pro81Leu mutation carriers do not need intensive imaging for PPGL

*Andrews KA et al 2018 PMID: 29386252*

A comprehensive database of germline SDHB and SDHD
mutations is maintained at http://chromium.liacs.nl/
LOVD2/SDH/home.php (Bayley et al. 2005). A wide variety
of intragenic mutations have been described and, more
recently, single or multiple exon deletions (and, occasionally, intragenic duplications; McWhinney et al. 2004,
Cascon et al. 2008, Neumann et al. 2009). 

A number of frequent SDHB and SDHD mutations were observed and
these may result from a high mutation rate or to founder
effects. Thus the relative frequency of some mutations can
vary with geographical location. In the Netherlands, two
major SDHD founder mutations have been identified
(c.274GOT (p.Asp92Tyr) and c.416TOC (p.Leu139Pro)),
and these account for O90% of SDHD mutation carriers. A SDHD c.33C/A (p.Cys11X) founder mutation has been reported in central Europe (Poland; Peczkowska et al. 2008). The common SDHD
c.242COT (p.Pro81Leu) mutation has been reported as
both a recurrent and a founder mutation (Baysal et al.
2002). 

Homozygous SDHD mutations have been associated
with recessively inherited encephalomyopathy and mitochondrial complex II deficiency (Jackson et al. 2014).
Tumorigenesis in SDH-mutated neoplasia appears to follow
a ‘two hit’ (retinoblastoma-like) model and it has been
proposed that the parent-of-origin effects may reflect the
tendency for the ‘second hit’ causing inactivation of the WT
allele in SDHD-related tumorigenesis to be loss of the whole
chromosome 11. The imprinted gene cluster at 11p15.5
contains the maternally expressed growth suppressor
CDKN2B and the paternally expressed IGF2 growth factor
(Lim & Maher 2010). In cases of a paternally inherited
germline SDHD mutation, loss of the maternally-derived
chromosome 11 would, in a single event, result in biallelic
SDHD inactivation and loss of CDKN1C expression but preservation of IGF2 expression from the paternal allele.

SDH-mutated PPGLs show robust expression of hypoxia
induced genes, and genomic and histone hypermethylation. These effects occur in part through succinate-mediated inhibition of a-ketoglutarate-dependent dioxygenases.
However, details of mechanisms by which SDH mutations activate hypoxic pathways and trigger subsequent neoplastic transformation remain poorly understood.

*Baysal B et al 2015 PMID: 26113606*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

**Allelic requirement:**

monoallelic_aut

(optional) modifiers: imprinting effect - parent of origin effect (paternal) 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Deletions and loss of function mutations in SDHD are associated with paraganglioms. Pathogenic variants in SDHD result in reduced or absent succinate dehydrogenase function because of loss or dysfunction of the affected subunit or failure of the SDH heterotetramer to assemble. Details of mechanisms by which SDH mutations activate hypoxic pathways and trigger subsequent neoplastic transformation remain poorly understood. Nonsense, missense, and splice site variants, intragenic insertions and deletions, and a whole-gene deletion have been reported in SDHD. Pathogenic variants in SDHD demonstrate parent-of-origin effects and cause disease almost exclusively when they are paternally inherited. SDHD mutation carriers had a higher overall penetrance for symptomatic tumours and a higher risk of head and neck paraganglioma (HNPGL) compared with SDHB. A number of frequent SDHD mutations were observed and these may result from a high mutation rate or to founder effects. The common SDHD
c.242COT (p.Pro81Leu) mutation has been reported as
both a recurrent and a founder mutation (Baysal et al.
2002). Carriers of this particular mutation appear to manifest almost exclusively with HNPGL, while other SDHD mutation types predispose to both HNPGLs and PPGLs. Homozygous SDHD mutations have been associated with recessively inherited encephalomyopathy and mitochondrial complex II deficiency.

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Missense
- In frame deletions

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- In frame insertions
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
**SMAD3 -- Loeys-Dietz syndrome type 3 (MIM 613795)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:6769

**Literature Review:**

**OMIM:** https://www.omim.org/entry/613795

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1133/

*Pathogenic variants.* Most variants in SMAD3 are predicted to lead to
loss of function. This is supported by the fact that about half of the
currently reported pathogenic variants lead to nonsense or out-of-frame
frameshift variants \[Wischmeijer et al 2013\]

This protein functions as a transcriptional modulator activated by
transforming growth factor-beta (provided by RefSeq, Apr 2009).

*Abnormal gene product.* Despite the predicted loss-of-function nature
of most SMAD3 pathogenic variants, a paradoxic gain of function on the
overall TGFβ signaling pathway in aortic walls of affected individuals
has been observed \[van de Laar et al 2011\].

212 individuals with 51 SMAD3 variants, including m**ore than 10
nonsense, partial gene deletions and frame shift variants, as well as
many missense and in-frame variants** in MH2 and MH1 domain of the gene.
The median age at first aortic event was significantly lower in
individuals with SMAD3 MH2 missense variants than those with
haploinsufficient variants but there was no difference in frequency of
aortic events by variant type.

*Hostetler et al, 2019, PMID 30661052*

11 patients (11.7%) had SMAD3 variants (**3 frame shift, 1 splicing, an
in-frame deletion in exon 6 (part of MH2 domain) and a loss of
initiation codon**). Regarding the in-frame deletion of exon 6 was
likely familial since the Proband\'s father and paternal grandmother
were both affected and died in their 30s.

*Overwater et al, 2018, PMID* 29907982

Novel SMAD2/3 and TGFB2/3 variants - multiple **nonsense, frame shift
and splicing variants** in SMAD3 gene.

*Schepers et al, 2018, PMID 29392890*

SMAD3 mutations cause a **frameshift and premature stop of translation,
or substitution of highly conserved amino acids**, which are predicted
in silico to have a pathologic effect. As such, the pathogenic mechanism
underlying these mutations is **probably loss of function**. Comparable
to several other conditions presenting with arterial aneurysms, such as
MFS, LDS and ATS, loss of function mutations in SMAD3 seem to lead to a
**paradoxical increase in TGFbeta signaling** in the aortic wall.

*Wischmeijer et al, 2012, PMID 23554019*

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is likely to be predominantly loss of function due to
pathogenic splice site, frameshift, premature stop, or substitution of
highly conserved amino acids variants that lead to a paradoxical
increase in TGF-beta signaling in the aortic wall. In frame variants and
partial gene deletions are also reported.

**List variant classes in this gene proven to cause this disease:**

Nonsense

Splice_acceptor_variant

Splice_donor_variant

Frameshift_variant

Inframe_deletion

Missense

**Potential novel variant classes based on predicted functional
consequence**

Splice_region_variant

Splice_acceptor_variant predicted to escape NMD

Splice_donor_variant predicted to escape NMD

Frameshift_variant predicted to escape NMD

Start_lost

Stop_gained predicted to escape NMD

Stop_lost

In frame insertion

5_prime_UTR_variant

Gain of upstream Start \[uORF\]

Gain of upstream Start \[oORF\]

Stop lost \[uORF\]

Stop lost \[oORF\]

Start lost \[uORF\]

Frameshift \[uORF\]

Frameshift \[oORF\]

Stop gained \[uORF\]
**SMAD4 -- Juvenile polyposis syndrome, (MIM 174900)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:6770

Mutation and phenotype analysis was used in 80 unrelated patients of
whom 65 met the clinical criteria for juvenile polyposis syndrome
(typical JPS) and 15 were suspected to have JPS. Using MLPA, **large
genomic deletions involving the SMAD4 were found in six** (9%) patients
with typical JPS. Four exhibited a heterozygous deletion of all SMAD4
probes encompassing the entire **SMAD4 gene and the promoter** region.
One patient had a deletion of **coding exons 5-11** and another had a
deletion of **coding exons 6-11**. All deletions were confirmed in a
second independent MLPA test.

***Aretz et al, 2007, PMID 17873119***

Archival material of 29 patients with JPS from 27 families was
collected. MLPA identified a germline **hemizygous large genomic
deletion involving SMAD4** in a one patient.

***Van Hattem et al, 2008, PMID 18178612***

DNA was extracted from 102 JPS probands. By MLPA, one **proband had
deletion of most of SMAD4 and another deletion of the 5\' end of
SMAD4**.

***Calva-Cerqueira et al, 2009, PMID 18823382***

**Literature Review:**

**OMIM:** https://www.omim.org/entry/174900 (and 600993)

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1469/

*Mechanism*

Although SMAD4 is a **tumor suppressor gene**, loss of heterozygosity
has not been demonstrated definitively as causal in the development of
polyps. Furthermore, whether such changes would affect cells in the
epithelium, the lamina propria, or both is also not known. SMAD4 is
**the common intracellular mediator of the TGF-β superfamily signaling**
pathways. BMPR1A is a type I cell surface receptor for the BMP pathway.
Ligands, such as TGF-β or BMP, bind to a receptor and **activate
signaling pathways** leading to protein complexes that migrate to the
nucleus and bind directly to DNA sequences **to regulate transcription**
\[Heldin et al 1997\]. The downstream genes under the control of these
signaling pathways are still being actively investigated.

*Pathogenic variants.*

. See Table 5. Germline pathogenic variants have been described in all
eleven coding exons. Changes include **small deletions, insertions, and
missense and nonsense** pathogenic variants. **Two splice site
variants** have been reported. Most pathogenic variants are unique, but
three have been reported in multiple unrelated families:
c.1244_1247delACAG, c.1162C\>T, and p.Arg361Cys. See Howe et al \[2004\]
and Calva-Cerqueira et al \[2009\] for a comprehensive list of the
pathogenic variants reported in SMAD4 (previously known as MADH4).
**Larger deletions of SMAD4 may also occur in up to 4% of affected**
individuals \[Aretz et al 2007, van Hattem et al 2008, Calva-Cerqueira
et al 2009\]. Deletions and pathogenic missense variants have also been
reported in **the promoter region** \[Calva-Cerqueira et al 2010\].

*Abnormal gene product.*

The **MH1** **domain** of the SMAD4 protein can directly bind to the DNA
of target genes. Pathogenic variants in this domain can **significantly
reduce the DNA binding activity** of SMAD4. Most pathogenic variants,
including the three recurrent pathogenic variants in Table 5, occur in
**the MH2 domain**, which plays an important role for nuclear
localization, interaction with other MAD proteins, and transcriptional
activation. In vitro studies demonstrate that **pathogenic nonsense
variants lead to significantly reduced bone morphogenetic protein
signaling**, with less of an effect for missense variants \[Carr et al
2012\].

Overall, **frameshift, nonsense, and missense** variants accounted for
the majority of pathogenic SMAD4 (72.9%) and BMPR1A (61.8%) alterations
in the ECS as well as the LBSB group (SMAD4: 79.9%; BMPR1A: 70.8%; Supplementary Table 3). Only
**large genomic, i.e., single or multiexon deletions** in SMAD4 were
significantly overrepresented in the ECS compared with the LBSB group.
**Splice site variants were noted** in 4-6% of SMAD4 and 10-16% of
BMPAR1A

***Blatter et al, 2020, PMID: 32398773***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

*Decreased gene product level

Absent gene product

*Altered gene product structure*

**Narrative summary of molecular mechanisms:**

SMAD4 is a tumour suppression gene and has a role in common
intracellular mediator of the TGF-β superfamily signalling. The
molecular mechanism of disease in relation to Juvenile polyposis is felt
to be loss of function principally due to reduced or absent DNA binding
activity in SMAD4. Variant classes include small and large deletions,
insertions, missense, nonsense, splice site and deletion/missense in the
promotor region. Of note, activating (gain of function) heterozygous de
novo mutations in the MH2 domain (principally reported to affect
residues 496 and 500 in the SMAD4 gene) cause Myhre syndrome.

**List variant classes in this gene proven to cause this disease:**

Stop_gained

missense

Frameshift_variant

Splice_acceptor_variant

Splice_donor_variant

regulatory_region_variant (deletions/missense in promotor region)

**Potential novel variant classes based on predicted functional
consequence**

**splice_region_variant**

**(splice_acceptor_variant predicted to undergo NMD)**

**(splice_acceptor_variant predicted to escape NMD)**

**(splice_donor_variant predicted to undergo NMD)**

**(splice_donor_variant predicted to escape NMD)**

**start_lost**

**(frameshift_variant predicted to undergo NMD)**

**(frameshift_variant predicted to escape NMD)**

**(stop_gained predicted to undergo NMD)**

**(stop_gained predicted to escape NMD)**

**stop_lost**

**inframe_insertion**

**inframe_deletion**

**(gain of upstream Start \[uORF\])**

**(gain of upstream Start \[oORF\])**

**(Stop lost \[uORF\])**

**(Stop lost \[oORF\])**

**(Start lost \[uORF\])**

**(Frameshift \[uORF\])**

**(Frameshift \[oORF\])**

**(Stop gained \[uORF\])**

**Not included**

**synonymous_variant**

**intron_variant**

**intergenic_variant**

**3_prime_UTR_variant**

**5_prime_UTR_variant**
# **STK11 -- Peutz-Jeghers Syndrome**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:11389

**ClinGen Evidence for Haploinsufficiency**

PMID 2563227 - Hearle et al 2006 report multiple (38) unrelated probands with Peutz-Jeghers syndrome assessed for STK11 mutations (exon deletions, nonsense, missense and loss of function from in-frame deletions in kinase domain). 16% (6/38) carry exon deletions and total 50% (19/38) had mutations in STK11: "four nonsense mutations, six deletions, and two insertions predicted to lead to truncation of the expressed protein, four missense mutations, and three splice site mutations".

PMID 20623358 - Resta et al 2010 reports multiple unrelated probands with Peutz-Jeghers syndrome carrying large deletions detected by MLPA (15/51 ~29%)

Haploinsufficiency phenotype comments:
Deletions (whole gene and exonic) and loss-of-function mutations in STK11 are associated with Peutz-Jeghers syndrome (PJS).

**Literature review:**

STK11 acts as a suppressor by activating TSC2 through an AMP-dependent protein kinase [Corradetti & Guan 2006] leading to accumulation of mTOR, which is critical for protein translation.

**More than 300 STK11 pathogenic variants have been reported in persons with Peutz-Jeghers syndrome. All types of variants have been reported, from missense variants to whole-gene deletion.**

Intragenic homologous recombination has been noted as a mechanism that can lead to deletion of exons 4-7 of STK11 [Ankala et al 2012].

It was also noted that recombination among Alu elements is a frequent cause of deletions of exons 2 and 3 [Borun et al 2015].

Penetrance - To date all reported individuals with pathogenic variants in STK11 have shown clinical manifestations.

Genotype - phenotype correlation - evidence is conflicting. Hearle et al found that the variant type and site within the functional domains of the expressed protein did not affect cancer risk. Other studies have reported that truncating variants are associated with earlier onset polyps or more GI surgeries. Amos et al 2004; Salloch et al 2010.

*Gene reviews
https://www.ncbi.nlm.nih.gov/books/NBK1266/*

Jenne et al. (1998) performed mutation analysis in 5 unrelated PJS patients and found mutations in STK11 in each. In a 3-generation PJS family affected members carried an STK11 allele with a deletion of exons 4 and 5 and an inversion of exons 6 and 7. In 4 other unrelated PJS patients, they found 3 nonsense mutations and 1 acceptor splice site mutation. All 5 germline mutations were predicted to disrupt the function of the kinase domain. Jenne et al. (1998) concluded that germline mutations in STK11, probably in conjunction with acquired genetic defects of the second allele in somatic cells according to the Knudson model, caused the manifestations of PJS.

*Omim
https://www.omim.org/entry/602216*

Among STK11 mutations recorded in the Human Genome Mutation Database (HGMD), more than 300 point mutations and small range mutations are recorded, and 89 gross deletions or insertions

*Yu-Liang J et al 2018 PMID: 30528796*

A total of 419 Peutz-Jeghers syndrome patients (193 males
and 226 females) ascertained through 225 probands were
available for analysis

Eighty-five (83%) of the germ-line STK11/LKB1 mutations identified represent unique sequence changes. A diagrammatic representation of the coding sequence of STK11/LKB1 and the corresponding functional domains of the expressed protein is shown in Fig. 1A.

**Mutations were scattered throughout the gene, but no mutation
in exon 9 was identified in any patient. Over 85% of both
truncating and missense mutations localize to regions of
STK11/LKB1 encoding the kinase domain of the expressed
protein** (Fig. 1B). Sixty-one of the mutations resulted in the
truncation of the protein by the creation of premature
transcription termination signals, 16 occurred within highly
conserved splice sites (4 were in-frame deletions predicted to
lead to loss of kinase activity), and 15 were missense mutations.
All of the missense mutations led to nonconservative amino
acid changes that altered amino acids highly conserved in
evolution among human, mouse, and Xenopus homologues of
STK11/LKB1 and resided within the kinase domain of the
protein encoded by exons 1 to 8. Other mutations included one
large-scale genomic deletion and four exonic deletions. Seven
families had uncharacterized mutations.

*Hearle N et al 2006 PMID: 16707622*

The kinase domain of the human 433 amino acid protein is localised between residues 49 and 309,7 and shows homology to the conserved catalytic core of the kinase domain common to both serine/threonine and tyrosine protein kinase family members.10 Most mutations found in PJS patients are small deletions/insertions or single base substitutions leading to an abnormal truncated/kinase inactive protein.

*Schumacher V et al 2005 PMID: 15863673*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Deletions and loss of function mutations in STK11 are associated with Peutz-Jeghers syndrome. "STK11 is a multi-tasking tumor suppressor that has a role in apoptosis, cell cycle arrest, cell proliferation, cell polarity, and energy metabolism." All types of variants have been reported, from missense variants to whole-gene deletions. Mutations are seen throughout the gene but one study noted that no mutation in exon 9 was identified in any patient. Over 85% of both truncating and missense mutations localize to regions of STK11/LKB1 encoding the kinase domain.
Intragenic homologous recombination has been noted as a mechanism that can lead to deletion of exons 4-7 of STK11 [Ankala et al 2012]. It was also noted that recombination among Alu elements is a frequent cause of deletions of exons 2 and 3 [Borun et al 2015].
There are conflicting reports on whether there are genotype phenotype correlations. Hearle et al found that the variant type and site within the functional domains of the expressed protein did not affect cancer risk. Other studies have reported that truncating variants are associated with earlier onset polyps or more GI surgeries. Penetrance appears to be complete. 

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Missense
- In frame deletions

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- In frame insertions
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
**TGFBR1 -- Loeys-Dietz syndrome type 1A/2A (MIM 609192/608967)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:11772

The first report indicating a relationship between the TGFBR1 gene and
autosomal dominant Loeys Dietz syndrome was reported by Dietz et al in
2005 (PMID: 15731757). Loeys Dietz syndrome is a multisystemic disorder
with craniofacial, skeletal, integumental and ocular involvement
(reviewed in Van Laer et al., 2014 PMID: 24443023 ). Loeys Dietz
syndrome comprises a broad phenotypic spectrum of severity including
seemingly isolated aortic aneurysm and/or dissection to the full
syndrome and thus represents a continuum of disease (reviewed in Van
Laer et al., 2014 PMID: 24443023; Verstraeten et al., 2016 PMID:
26919284). Aortic aneurysm and tortuosity represent the major morbidity
and mortality of Loeys Dietz syndrome. **Over 120 variants have been
identified in TGBFR1** (Verstraeten et al., 2016 PMID: 26919284)
according to the UMD-TGFBR1 database (http://www.umd.be/TGFBR1/), with
the **majority being missense and more rarely, nonsense, splice site and
small deletions.** The mechanism for the gene-disease relationship is
**perturbation of the TGF beta signaling pathway, however the exact
molecular mechanism remains unclear and could include gain of function
and dominant negative** (reviewed in Van Laer et al., 2014 PMID:
24443023; Verstraeten et al., 2016 PMID: 26919284). The majority of the
mutations result in **aberrant kinase activity of TGFBR1**. Evidence
supporting this gene-disease relationship includes case-level data,
segregation data, functional data, and model organisms. This
gene-disease relationship has been studied for more than 10 years and a
significant amount of case-level data, segregation data, and
experimental data is available and the maximum score for genetic
evidence (12 points) has been reached. Note, this curation effort may
not be exhaustive of all literature related to this gene-disease
relationship. In summary, TGFBR1 is definitively associated with
autosomal dominant Loeys Dietz syndrome. This has been repeatedly
demonstrated in both the research and clinical diagnostic settings, and
has been upheld over time. This classification was approved by the
ClinGen General GCEP on March 27, 2019 (SOP Version 6)

**Literature Review:**

**OMIM:** https://www.omim.org/entry/609192

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1133/

Pathogenic variants. The large majority of pathogenic variants
identified so far are located in the exons coding for the intracellular
serine-threonine kinase domain of both receptors. They most commonly
involve **pathogenic missense variants; only a few nonsense** variants
have been described.

*Loeys et al, 2015, PMID: 15731757*

LDS is induced by dominant-negative missense variants within or near STK
domain,

*Fujiwara et al, 2018, PMID: 29706644*

a Japanese familial case of LDS with a **novel splice donor site variant
in TGFBR1** gene (c.973 + 1G\>A; NG_007461.1). The intronic variant was
predicted to mediate in-frame exon 5 skipping within the
serine/threonine kinase(STK) domain... .... LDS variant generated two
types of in-frame transcription products, r.\[806_973del,965_973 del\],
and produced two functionally inactivated proteins

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

*Altered gene product structure*

(Decreased gene product level)

**Narrative summary of molecular mechanisms:**

Mechanism is predominantly due to an abnormal gene product leading to
perturbation of the TGF beta signaling pathway, however the exact
molecular mechanism remains unclear and could include gain of function
and dominant negative. Some haploinsufficient variants (frameshift,
nonsense, splice and small deletions have been noted)

**List variant classes in this gene proven to cause this disease:**

Missense (most)

Stop_gained

Stop_gained predicted to undergo NMD

Splice_acceptor_variant

Small deletions (few)

Frameshift_variant

**Potential novel variant classes based on predicted functional
consequence**

Splice_donor_variant

Splice acceptor variant predicted to escape NMD

Splice acceptor variant predicted to undergo NMD

Splice donor variant predicted to escape NMD

Splice donor variant predicted to undergo NMD

Frameshift variant predicted to escape NMD

Frameshift variant predicted to undergo NMD

start_lost

stop_gained predicted to escape NMD

stop_lost

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[oORF\]

Frameshift \[oORF\]

Inframe_insertion

Inframe_deletion
**TGFBR2 -- Loeys-Dietz syndrome (type 1B and 2B MIM 610168/610380)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:11773

Assertion made by the Aortopathy working group. Considerable phenotypic
variability humans and mice with different genetic background.
Definitive for both LDS and nonsyndromic HTAAD.

**Numerous heterozygous missense and nonsense** mutations in TGFBR2 are
linked to Loeys-Dietz syndrome (LDS) types 1B (MIM \#610168) and
2B/Marfan (MIM \#610380). The **nonsense mutations are clustered in the
C-terminus**. The mechanism appears to be **dominant negative rather
than haploinsufficiency** (PMID 21098638). One paper describes a
**896-kb deletion including TGFBR2 but the patient had not signs of LDS
but instead exhibited microcephaly and developmental delay (PMID
21567932**). Due to the uncertainty regarding the exact mechanism by
which mutations in TGFBR2 cause LDS, the **haploinsufficiency rating is
deemed a 2.**

**Literature Review:**

**OMIM:** https://www.omim.org/entry/609192

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1133/

Pathogenic variants. The large majority of pathogenic variants
identified so far are located in the exons coding for the intracellular
serine-threonine kinase domain of both receptors. They most commonly
involve **pathogenic missense variants; only a few nonsense** variants
have been described.

A **splice mutation that disrupts reading frame, causing loss of
function**-type alteration, removing amino acids

*Loeys et al, 2015, PMID: 15731757*

Describes **missense** mutations and one **nonsense** mutation in
patients with /Loeys-Dietz 2B

*Loeys et al, PMID 16928994*

Screened 457 probands for mutations in TGFBR2 and found 23 mutations,
including 3 **nonsense** variants. PMID *18781618*

A **Loss-of-function splice mutation (synonymous** but causes abnormal
splicing) and a translocation disrupting TGFBR2 in Marfan type
2/Loeys-Dietz 2B. PMID *15235604*

**Missense mutations, one nonsense** mutation in patients with Marfan
type 2/Loeys-Dietz 2B. PMID *15235604*

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

*Altered gene product structure*

(Decreased Gene product)

**Narrative summary of molecular mechanisms:**

Mechanism is likely to be due to a dominant negative model with an
abnormal gene product leading to perturbation of the TGF beta signaling
pathway. There is uncertainty whether the phenotype may additionally be
caused as a result of haploinsufficiency. A few nonsense variants are
reported and one paper describes a 896-kb deletion including TGFBR2 but
the patient had not signs of LDS but instead exhibited microcephaly and
developmental delay (PMID 21567932).

**List variant classes in this gene proven to cause this disease:**

Missense (most)

Frameshift (few nonsense)

Splice_acceptor_variant

**Potential novel variant classes based on predicted functional
consequence**

Splice_donor_variant

Splice acceptor variant predicted to escape NMD

Splice donor variant predicted to escape NMD

Frameshift variant predicted to escape NMD

Stop gained predicted to escape NMD

Stop lost

Inframe_insertion

Inframe_deletion

*?? If consider haploinsufficiency as a potential MOA*

Frameshift

Splice region variant

Stop_gained

start_lost

Splice acceptor predicted to undergo NMD

Splice donor predicted to undergo NMD

stop_gained predicted to undergo NMD

stop_lost

gain of upstream Start \[uORF\]

gain of upstream Start \[oORF\]

Stop lost \[oORF\]

Frameshift \[oORF\]
# **TMEM43 -- Arrythmogenic Right Ventricular Cardiomyopathy**

**Review of source material:**

James CA et al PMID: 33831308

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:28472

The relationship between TMEM43 and arrhythmogenic right ventricular dysplasia (autosomal dominant) was evaluated using the ClinGen Clinical Validity Framework as of July 10th, 2019. Variants in TMEM43 were first reported in humans with this disease as early as 2008 (Merner et al., PMID 18313022). **At least 9 variants (mostly missense) have been reported in humans.** However, the pathogenicity of most of the variants is unknown. **The majority of genetic evidence comes from case-level data and
segregation data for one founder variant, p.Ser358Leu, which has been reported in more than 20 families with ARVC** and occurred de novo in one individual (Merner et al., 2008, PMID 18313022; Christensen et al. 2011, PMID 21214875; Baskin et al., 2013, PMID 23812740; Hodgkinson et al., 2013, PMID 22725725; Milting et al., 2014, PMID 24598986). This gene-disease relationship is also supported by an animal model, expression stuies, and in vitro assays. In summary, TMEM43 is definitively associated with autosomal dominant arrhythmogenic right ventricular dysplasia . This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over
time. This classification was approved by the ClinGen Arrythmogenic Right Ventricular Cardiomyopathy Gene Curation Expert Panel on October
26, 2018 (SOP Version 6).

**ClinGen Evidence for Haploinsufficiency**

Only heterozygous point mutations within the gene have been described in families with arrhythmogenic right ventricular cardiomyopathy (ARVC)(PMID:21214875, 23812740, 24598986,18313022). Gene function is unknown. PMID 21391237 described two patients with TMEM43 heterozygous missense mutations in Emery Dreifuss Muscular Dystrophy Related Myophathy. No familial follow-up was available. Functional studies suggest that mutant TMEM43 may be involved in the nuclear localization of emrin and SUN2.

**Literature review:**

A rare form of ARVC is caused by a missense mutation within the gene of transmembrane protein 43 (*TMEM43*) on chromosome 3p25 (ARVC-5). In a total of 15 Canadian families, a heterozygous amino acid substitution (p.S358L) in *TMEM43* fully cosegregated with autosomal dominant ARVC and was proposed to be a founder mutation on the island of Newfoundland (Canada).***TMEM43*-p.S358L is a fully penetrant mutation**.

However, of most variants the **molecular disease mechanism is not yet known** and the pathogenic role of some missense variants is still a matter of debate.

*Milting H et al 2015 Apr 7 (PMID:24598986)*

we find that the TMEM43 S358L mutation hyperactivated NF-κB signal as expected. However, this activation does not promote typically
inflammatory responses. Instead, it induces another signal TGFβ in fibrosis progress

*Zheng G et al. 2019 Feb (PMID:29980933)*

In 83 affected individuals with arrhythmogenic right ventricular
dysplasia-5 from 15 unrelated Newfoundland families, [Merner et al.(2008)PubMed: 18313022] identified heterozygosity
for a missense mutation (S358L) in the
TMEM43 gene that was not found in 47 spouses or 161 controls. 

In an analysis of the TMEM43 gene in 55 Danish probands who fulfilled the criteria for ARVD and 10 patients with only some features of ARVD, [Christensen et al.
(2011)](https://www.omim.org/entry/612048#3) identified 1 woman
fulfilling the criteria who carried the S358L variant. 

In DNA samples from 195 unrelated individuals with suspected
ARVD, [Baskin et al. (2013)]PubMed: 23812740] identified 6 patients who carried the S358L \'Newfoundland\' mutation in TMEM43, including a 43-year-old New Zealand man who was not of Newfoundland descent. In addition, 5 patients carried 5 different rare sequence variants in the TMEM43 (see, e.g., [612048.0004](https://www.omim.org/entry/612048#0004)), 2 of whom also carried a variant in the PKP2 and DSP genes, respectively.

**Omim** https://www.omim.org/entry/612048
*Baskin et al.(2013)]PubMed: 23812740; Merner et al. (2008)PubMed: 18313022*

One putative pathogenic variant, the missense change p.Ser358Leu, was identified in a number of families, the majority of which were of Newfoundland ancestry [Merner et al 2008, Christensen et al 2011, Baskin et al 2013]. This variant may relocate proteins essential for cardiac conduction, thereby altering gap junction function to reduce cardiac conduction velocity [Siragam et al 2014]. TMEM43 interacts with emerin and lamins A and B and may be a binding partner in the LINC complex (linker of the nucleoskeleton and cytoskeleton). The pathogenic mechanism of the abnormal gene product is unknown.

*Gene reviews https://www.ncbi.nlm.nih.gov/books/NBK1131/*

#### **Pilot application of harmonised terms:**

**Inheritance:**

Autosomal dominant

Optional modifiers:

**Allelic requirement:**

Monoallelic_autosomal

Crosscutting modifiers: 

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The majority of genetic evidence comes from one founder missense
mutation, S358L. Molecular mechanism is largely unknown but a proposed mechanism is hyperactivation of NF-kB signal due to **altered gene product structure**. Although ARVC is known to display variable penetrance, this particular founder mutation has been described as fully penetrant. Other missense mutations have been reported but their pathogenicity is debated. There is no evidence currently for haploinsufficiency as a mechanism.

**List variant classes in this gene proven to cause this disease**

- Missense

**Potential novel variant classes based on predicted functional
consequence:**

???  
Perhaps if evidence is only robust for one missense variant, then all other variants should be treated with caution even other missense variants.??

# **TNNI3 -- Hypertrophic Cardiomyopathy**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen:**

https://search.clinicalgenome.org/kb/gene-validity/8770

The TNNI3 gene has been associated with hypertrophic cardiomyopathy (HCM). TNNI3 was first associated with this disease in humans in 1997 (Kimura et al, PMID 9241277). At least 60 unique variants, with varying levels of evidence to support their pathogenicity, have been reported in humans (reviewed in Mogensen et al, 2015, PMID 26440512). Variants in this gene segregated with disease in at least 6 families (Kimura et al, 1997, PMID 9241277; Rani et al, 2012, PMID 22876777; Choi et al, 2010, PMID 20641121; Mogensen et al, 2004, PMID 15607392). More evidence is available in the literature, but the maximum score for genetic evidence
(12 pts) was reached. **The mechanism for disease is likely dominant negative, as 91% of mutations reported are missense variants** (Mogensen et al, 2015, PMID 26440512). The gene-disease association is supported by the function of the gene product, animal models, and in vitro assays.
In summary, TNNI3 is definitively associated with hypertrophic
cardiomyopathy. This has been repeatedly demonstrated in both research and clinical diagnostic settings, and has been upheld over time. This classification was approved by the ClinGen Hypertrophic Cardiomyopathy Gene Curation Expert Panel on September 5, 2017.

**ClinGen Evidence for Haploinsufficiency**

TNNI3 encodes a cardiac muscle isoform of Troponin I. The majority of reported mutations are missense mutations that affect the TNNI3 coding sequence; however, these mutations are not reported to result in a loss of function. Instead, functional studies on these mutations have shown that they affect Ca2+ binding to myofilaments containing the mutant TNNI3 (PMIDs: 16531415 and 22675533) or result in an increased myofilament response to Ca2+ (PMID: 11735257).

At this time there is limited evidence to support haploinsufficiency of this gene. The only evidence supporting the haploinsufficiency of TNNI3 is a single report of a patient with a single nucleotide deletion in TNNI3 that resulted in decreased TNNI3 protein levels (PMID: 18006163).
There are currently no reported TNNI3 whole gene deletions.

Please note that there are additional indel/splice site mutations that have been reported in this gene. These mutations either do not have functional data to support a loss of function disease mechanism (PMID: 20474083, 25524337, 24111713, 12707239, 25940119, and 18467357), are thought to be associated with a mechanism other than loss of function (PMID: 11735257 and 21533915), or have a complex genotype (PMID: 21835320 and 25132132).

**Literature review:**

Mutations in the troponin complex introduce alterations in
Ca^2+^ affinity and protein-protein interactions which may ultimately lead to the development of cardiomyopathy

...a total of 37 studies of patients with HCM have reported 66 different cTnI mutations in 256 probands.
Ninety-one percent (n = 60) of all mutations reported were missense variants, 85% of which (n = 51) have been identified in exons 7 and 8 of cTnI
Only 2 splice-site mutations have been reported. Six of the mutations reported (Arg141Gln, Arg145Trp, Arg157Val, Arg162Gln, Ser166Phe, and Lys183Del) appeared with a particularly high frequency and were identified in 116 of the 256 probands (45%).

*Mogensen et al, 2015, PMID 26440512*

"A multivariable model stratified by evaluation with CMR demonstrated the independent predictive value of male sex and an abnormal ECG before the end of follow-up. Compared with patients with MYBPC3, individuals with TNNI3 variants had a lower penetrance of HCM. Older age at first evaluation was also associated with an increased risk"

*Lorenzini M et al 2020 PMID: 32731933*

**From our in-house Atlas of HCM:**

128/135 missense

1/135 inframe deletion

There was no statistically significant excess in truncating variants in HCM cases vs population controls

2/135 nonsense (both VUS)

4/135 frameshift (2 VUS, 2 pathogenic)

Both of the pathogenic fs variants are classified as uncertain
significance in ClinVar.

c.538del is a 'hot VUS'. It is predicted to lead to a premature
termination codon within the last 50 bases of the second to last exon and escape NMD

*Walsh et al, 2016 (PMID 27532257)*

<https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=TNNI3&icc=HCM>

<https://www.ncbi.nlm.nih.gov/clinvar/variation/229332/>

**Inheritance**

Autosomal dominant

Optional modifiers: incomplete penetrance

**Allelic requirement**

Monoallelic_aut

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The mechanism is likely dominant negative due to **altered gene product structure.** rather than haploinsufficiency as the majority of pathogenic variants in TNNI3 are missense variants and there are no reported gene deletions. There are reports of nonsense and frameshift variants but there is conflicting evidence for pathogenecity. Functional studies on missense mutations have shown that they affect Ca2+ binding to myofilaments containing the mutant TNNI3 (PMIDs: 16531415 and 22675533) or result in an increased myofilament response to Ca2+ (PMID: 11735257). Mogensen et al reported "ninety-one percent (n = 60) of all mutations reported were missense variants, 85% of which (n = 51) have been identified in exons 7 and 8 of cTnI. Six of the mutations
reported (Arg141Gln, Arg145Trp, Arg157Val, Arg162Gln, Ser166Phe, and Lys183Del) appeared with a particularly high frequency and were identified in 116 of the 256 probands (45%)". When compared to MYPBC3 variants, penetrance appears to be lower.

**Additional information related to ACMG evidence types**

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
0.1% (het) 
3.16% (hom) 

**BS1** (MAF too high for disease) 0.02%
Assumptions
• Disease prevalence: 1/200 individuals (1/400 chromosomes)
• Penetrance: 30%
• Maximum pathogenic variant contribution: 2% based on MYBPC3 variant p.Arg502Trp (Walsh et al. 20175:6,000 probands)
• Note that the FAF (95% poisson) is available for each variant in ExAC (http://exac.broadinstitute.org/).

**PM2** A filtering allele frequency (FAF) less than 0.004% activates this rule
CAUTION: Population databases may contain presymptomatic individuals for diseases with reduced
penetrance/variable onset.

*Kelly MA et al 2018 PMID: 29300372* 

*Whiffin N et al 2018 PMID: 29369293*

**PM1** 

Walsh et al propose adaptation of ACMG/AMP guidelines for rule PM1 and HCM, relating to the relative frequencies of non-truncating variants in case cohorts and population controls. 
PM1_strong - EF >0.95
PM1_moderate - EF between 0.90 and 0.95 
PM1_supporting - EF between 0.80 and 0.90

There is a cluster of HCM cases in the troponin C and actin-binding domains in TNNI3, amino acid residues 141–209. Etiological fraction for this cluster is 0.974 (0.963–0.984) so PM1_strong could be activated.

*Walsh et al 2019 PMID: 30696458*

**List variant classes in this gene proven to cause this disease:**

- Missense
- In frame deletion

**Potential novel variant classes based on predicted functional
consequence**

- Frameshift predicted to escape NMD
- stop_gained predicted to escape NMD
- Splice acceptor variant predicted to escape NMD 
- Splice donor variant
- Splice donor variant predicted to escape NMD
- stop_lost
- inframe_insertion

# **TNNT2 -- Dilated Cardiomyopathy**

**Review of source material:**

**ClinGen:**

Not curated by ClinGen for DCM yet

**Literature review:**

Troponin T is a regulatory protein found in striated muscles that forms a complex with troponin I (TnI) and troponin C (TnC) that, together with tropomyosin (TM), is required for Ca^2+^-dependent regulation of muscle contraction.

Kamisago et al. (2000) identified a mutation in the TNNT2 gene as the cause of familial dilated cardiomyopathy. 

Mirza et al. (2005) studied all 8 published mutations causing dilated cardiomyopathy (CMD), including 5 in the TNNT2 gene (lys210del, R141W, R131W, R205L, and D270N; 191045.0006-191045.0010, respectively), 2 in the TPM1 gene (E54K, 191010.0004; and E40K, 191010.0005), and 1 in the TNNC1 gene (G159D; 191040.0001). Thin filaments, reconstituted with a 1:1 ratio of mutant:wildtype proteins, all showed reduced Ca(2+) sensitivity of activation in ATPase and motility assays, and, except for the E54K alpha-tropomyosin mutant which showed no effect, all showed lower maximum Ca(2+) activation. Incorporation of the TNNT2 mutations R141W and R205L into skinned guinea pig cardiac trabeculae also decreased Ca(2+) sensitivity of force generation. Thus, diverse thin filament CMD mutations appeared to affect different aspects of regulatory function yet change contractility in a consistent manner. Mirza et al. (2005) stated that the CMD mutations depressed myofibrillar function, an effect opposite to that of CMH-causing thin filament mutations, and suggested that decreased contractility might trigger pathways that ultimately lead to the clinical phenotype. 

Mogensen et al. (2004) analyzed the TNNT2 gene in 235 consecutive unrelated probands with dilated cardiomyopathy and identified 4 different mutations in 4 families, respectively. The mutations segregated with the disease in each family and were absent in 200 ethnically matched control chromosomes and 1,520 chromosomes from patients with hypertrophic cardiomyopathy. Functional studies showed significant impairment of mutated troponin interaction compared with wildtype control, indicating an altered regulation of myocardial contractility. 

***Omim** https://www.omim.org/entry/191045*

We propose that the variant classes significantly enriched in 1 or both DCM cohorts are likely to be most relevant for genetic testing in DCM. These were truncating variants in TTN, DSP, LMNA, BAG3, and VCL, and nontruncating variants in MYH7, TNNT2, TNNC1, LMNA, and TPM1.

*Mazzarotto F et al 2020 PMID: 31983221*

**From our in-house Atlas of HCM:**

36/36 non-truncating  (missense, in frame deletions)

0/36 truncating

*Walsh et al, 2016 (PMID 27532257)*
*Pugh TJ et al, 2014 (PMID: 24503780)*

We recently undertook bidirectional resequencing of TNNT2, the cardiac troponin T gene, in 313 probands with DCM. We identified six TNNT2 protein-altering variants in nine probands, all who had early onset, aggressive disease. Additional family members of mutation carriers were then studied when available. Four of the nine probands had DCM without a family history, and five had familial DCM. Only one mutation (Lys210del) could be attributed as definitively causative from prior reports. Four of the five missense mutations were novel (Arg134Gly, Arg151Cys, Arg159Gln, Arg205Trp), and one was previously reported with hypertrophic cardiomyopathy (Glu244Asp). Based on the clinical, pedigree and molecular genetic data these five mutations were considered possibly or likely disease causing. To further clarify their potential pathophysiologic impact, we undertook functional studies of these mutations in cardiac myocytes reconstituted with mutant troponin T proteins. We observed decreased Ca2+ sensitivity of force development, a hallmark of DCM, in support of the conclusion that these mutations are disease-causing.

*Hershberger RE et al 2010 PMID: 20031601*

#### **Pilot application of harmonised terms**

**Inheritance**

Autosomal dominant

(optional) modifiers: incomplete penetrance

**Allelic requirement**

Monoallelic_aut

(optional) modifiers:

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Troponin T is a regulatory protein found in striated muscles that forms a complex with troponin I (TnI) and troponin C (TnC) that, together with tropomyosin (TM), is required for Ca^2+^-dependent regulation of muscle contraction. Non-truncating variants (missense, in-frame deletions) are enriched in TNNT2 in DCM patients compared to controls (this is not the case for truncating variants). The mechanism appears to be dominant negative rather than haploinsufficiency. Functional studies have shown pathogenic missense variants in TNNT2 causing decreased Ca2+ sensitivity of force development.

**List variant classes in this gene proven to cause this disease:**

- Missense
- In frame deletion

**Potential novel variant classes based on predicted functional
consequence**

- Splice acceptor variant predicted to escape NMD
- Splice donor variant predicted to escape NMD
- Frameshift predicted to escape NMD
- Stop gained predicted to escape NMD
- stop_lost
- inframe_insertion
# **TNNT2 -- Hypertrophic Cardiomyopathy**

**Review of source material:**

*Ingles J et al. 2019 PMID: 30681346*

**ClinGen:**

https://search.clinicalgenome.org/kb/gene-validity/8770

The TNNT2 gene has been associated with hypertrophic cardiomyopathy.
TNNT2 was first associated with the disease in humans in 1994
(Thierfelder et al, 1994, PMID 8205619; Watkins et al, 1993, PMID
7981753; Watkins et al, 1995, PMID 7898523). Many unique variants, with varying levels of evidence to support their pathogenicity, have been reported in humans, as variants in TNNT2 account for \~5% of HCM cases (Cirino and Ho, 2014, PMID 20301725). Variants in this gene segregated with disease in at least 3 families (Thierfelder et al, 1994, PMID8205619). More evidence is available in the literature, but the maximum score for genetic evidence (12 pts) was reached. **The mechanism for disease is likely dominant negative, as most mutations reported are missense variants**. The gene-disease association is supported by the function of the gene product, animal models, and in vitro assays. 
In summary, **TNNT2 is definitively associated with hypertrophic
cardiomyopathy**. This has been repeatedly demonstrated in both research and clinical diagnostic settings, and has been upheld over time. This classification was by the ClinGen Hypertrophic Cardiomyopathy Gene Curation Expert Panel on November 7th, 2017.

It is unknown if haploinsufficiency results in a phenotype. In PMID 19666196, MLPA was used to screen TNNT2 for large rearrangements and none were found. There are a few isolated reports of splice site mutations and stop mutations but pathogenicity remains unproven.

**Literature review:**

Troponin T is a regulatory protein found in striated muscles that forms a complex with troponin I (TnI) and troponin C (TnC) that, together with tropomyosin (TM), is required for Ca^2+^-dependent regulation of muscle contraction.

*Thierfelder et al, 1994, PMID 8205619* demonstrated that affected
individuals from 3 unrelated families with the form of familial
hypertrophic cardiomyopathy linked to 1q
(CMH2; [[115195]{.ul}](https://www.omim.org/entry/115195)) contained point mutations: missense mutations (ile79-to-asn, 
arg92-to-gln) in 2 of them and a mutation in the splice donor sequence of intron 15 in the third. The
abnormalities were demonstrated by screening by RNase A protection
assays followed by sequencing.

***Omim** https://www.omim.org/entry/191045*

*Tardiff JC et al. 1998, PMID:9637714* stated that 9 mutations had been described in the TNNT2 gene that cause familial hypertrophic
cardiomyopathy, including 7 missense mutations, a deletion of an
internal amino acid, and a splice site mutation that would result in the loss of either the 14 or 28 C-terminal residues with the addition of 7 non-TNNT2 amino acids in the latter case.

*Tardiff JC et al. 1998, PMID:9637714* 

Twelve different mutations in exons 8, 9, 11, 15, and 16 were identified in 20 families: Arg278Cys in 3 families; Arg92Leu in 2 families; Arg92Trp in 3 families; ΔGlu163 in 3 families; IVS15+1G\>A in 2 families; and Ala104Val, Arg278His, Arg92Gln, Arg94Leu, Glu163Lys, Glu83Lys, Ile79Asn in single families.

*Pasquale et al.2011 PMID: 22144547*

The entire coding sequences of 9 genes
(*MYH7*, *MYBPC3*, *TNNI3*, *TNNT2*, *MYL2*, *MYL3*, *TPM1*, *ACTC*, and *TNNC1*) were analyzed in 197 unrelated index cases with familial or sporadic hypertrophic cardiomyopathy. Analysis of *TNNT2* showed 5 missense mutations, one codon deletion (Del E160), and one nonsense mutation (W287ter). *TNNT2* mutations are located in regions essential for anchoring the troponin-tropomyosin complex onto the thin filament.[**^[18]{.ul}^**](https://www.ahajournals.org/doi/full/10.1161/01.CIR.0000066323.15244.54?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed#R18-126913) 
In 2 unrelated patients, a termination codon (W287X) involving the last residue of the protein was identified.

*Richard P et al. 2003 PMID: 12707239*

"Residues 80--180 of TnT, which bind directly to Tm, harbor most of the known disease-causing single-residue substitutions, with two hotspots at residues 92 and 160--163.

....suggests that the primary mechanism underlying disease for mutants in the TNT1 region may be the altering of the affinity of TnT for Tm."

*Gangadharan B et al. 2017 PMID: [28973951]*

**From our in-house Atlas of HCM:**

[[https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=TNNT2&icc=HCM]{.ul}](https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=TNNT2&icc=HCM)

92/119  missense

16/119  inframe deletion

4/119 nonsense

*Walsh et al, 2016 (PMID 27532257)*

These 4 reported are all the same variant - W287X -- leads to a
premature termination codon at position 287. This alteration occurs within the last exon and may therefore escape nonsense mediated decay (NMD), resulting in a truncated protein that is lacking the last 2 amino acids.

Other nonsense mutations on ClinVar have conflicting evidence.

7/119 essential splice site

Variant c.821 + 1G\>C is reported as pathogenic. A variant at same
position c.821 + 1G\>A is predicted to result in skipping of exon 15.
(Int. J. Mol. Sci. 2016, 17, 1883; doi:10.3390/ijms17111883
supplementary data).

#### **Pilot application of harmonised terms**

**Inheritance**

Autosomal dominant

(optional) modifiers: Incomplete penetrance

**Allelic requirement**

Monoallelic_aut

(optional) modifiers:

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Troponin T is a regulatory protein found in striated muscles that forms a complex with troponin I (TnI) and troponin C (TnC) that, together with tropomyosin (TM), is required for Ca^2+^-dependent regulation of muscle contraction. Mechanism may be an alteration in the affinity of troponin T for tropomyosin due to **altered gene product structure.** The mechanism appears to be dominant negative rather than haploinsufficiency. Most of the pathogenic variants in TNNT2 are missense variants. Several splicing variants have been reported but depending on the location of these there is conflicting evidence in the literature regarding pathogenecity. Nonsense variants have been identified in the penultimate and final exon. It is possible these will lead to a truncated protein escaping nonsense mediated decay. Variants in HCM cases cluster in the tropomyosin-binding domain, amino acid residues 79--179, which bind directly to Tm. Gangadharan et al suggest there are two hotspots within that region at residues 92 and 160--163.

**Additional information related to ACMG evidence types**

**BA1** (MAF above which a variant can be classified as BENIGN assuming a MENDELIAN framework)
0.1% (het) 
3.16% (hom) 

**BS1** (MAF too high for disease) 0.02%
Assumptions
• Disease prevalence: 1/200 individuals (1/400 chromosomes)
• Penetrance: 30%
• Maximum pathogenic variant contribution: 2% based on MYBPC3 variant p.Arg502Trp (Walsh et al. 20175:6,000 probands)
• Note that the FAF (95% poisson) is available for each variant in ExAC (http://exac.broadinstitute.org/).

**PM2** A filtering allele frequency (FAF) less than 0.004% activates this rule
CAUTION: Population databases may contain presymptomatic individuals for diseases with reduced
penetrance/variable onset.

*Kelly MA et al 2018 PMID: 29300372* 

*Whiffin N et al 2018 PMID: 29369293*

**PM1** 

Walsh et al propose adaptation of ACMG/AMP guidelines for rule PM1 and HCM, relating to the relative frequencies of non-truncating variants in case cohorts and population controls. 
PM1_strong - EF >0.95
PM1_moderate - EF between 0.90 and 0.95 
PM1_supporting - EF between 0.80 and 0.90

The tropomyosin-binding domain in TNNT2 is a cluster for HCM non-truncating variants, amino acid residues 79-179
Etiological fraction - 0.958 (0.941–0.974). PM1_strong could be applied for this cluster.

**PVS1**
??

*Walsh et al 2019 PMID: 30696458*

**List variant classes in this gene proven to cause this disease:**

- Missense
- In frame deletion
- stop_gained predicted to escape NMD
- ?? Splice donor variant

**List other variant classes predicted to lead to the same functional consequence**

- Splice acceptor variant predicted to escape NMD
- Splice donor variant predicted to escape NMD
- Frameshift predicted to escape NMD
- stop_lost
- inframe_insertion
**TP53 -- Li-Fraumeni syndrome 1 (MIM 151623)**

**ClinGen:** https://search.clinicalgenome.org/kb/genes/HGNC:11998

TP53 was first reported in relation to autosomal dominant Li-Fraumeni syndrome in 1991 (Malkin et al., 1991, PMID: 1978757; Li et al., 1998,
PMID: 3409256). Li Fraumeni is associated with increased risk of multiple pediatric and adult malignancies (see discussion below).
Numerous variants have been reported in TP53 in relation to the development of Li-Fraumeni syndrome and include **missense, small duplications, small deletions, frameshift, and nonsense mutations** have been reported in humans. De novo inheritance has been noted in
Li-Fraumeni syndrome in 7-20% of cases (Schneider et al., OMID: 20301488; Gene Reviews). There are several databases describing TP53 variants of interest in Li-Fraumeni Syndrome, including: (1) the TP53
database in LOVD (https://databases.lovd.nl/shared/genes/TP53) ; (2) The TP53 Website (http://p53.fr/) (Leroy et al, 2013; PMID: 23161690) which
houses the IARC TP53 database (http://p53.iarc.fr/) (Olivier et al.,
2002, PMID: 12007217); (3) Database of germline p53 mutations
(http://stary.lf2.cuni.cz/projects/germline_mut_p53.htm); (4) The UMD
TP53 website (http://www.umd.be:2072/IFAMTP53A.shtml). There is
significant genetic evidence supporting this gene-disease relationship
includes case-level data and segregation data and the maximum score for
genetic evidence (12 points) has been reached. This gene-disease
relationship is also supported by expression studies and multiple animal
models that get LFS associated tumors...... The molecular mechanism of
TP53 dysfunction for all the associated disease entities listed above
are either **loss of function or dominant negative**; both of which
result in **reduced (or absent) transcriptional activity and loss of the
tumor suppressive function of the p53 protein** **function**. Per the
criteria outlined by the ClinGen Lumping and Splitting Working Group, we
have found no difference in molecular mechanism, inheritance pattern, or
phenotypic expressivity, and therefore have lumped the above listed
disease entities into the curation for TP53 in Li-Fraumeni Syndrome. In
summary, TP53 is definitively associated with autosomal dominant
Li-Fraumeni syndrome. This has been repeatedly demonstrated in both the
research and clinical diagnostic settings, and has been upheld over
time.

Gene Clinical Validity Standard Operating Procedures (SOP) - Version 7

**Literature Review:**

**OMIM:** https://www.omim.org/entry/151623

**GENEREVIEWS:** https://www.ncbi.nlm.nih.gov/books/NBK1311/

Mechanism of disease causation. Germline TP53 pathogenic variants create
a **constitutive defect of p53 DNA binding and transcriptional response
to DNA damage**. According to Zerdoumi et al \[2017\], \"germline TP53
mutations represent a genetic permissive context facilitating malignant
transformation of cells in which DNA damage has occurred.\"

TP53-specific laboratory technical considerations. TP53 missense
variants are the variants most commonly identified in tumors and they
present challenges in germline interpretation.

A recent study reported that individuals with germline TP53 pathogenic
variants resulting in p53 loss of function appeared to have a more
severe phenotype than individuals with pathogenic variants that caused
partial deficiency of p53. ......in contract another report showed
dominant-negative pathogenic variants (in which the mutated p53 protein
interferes with the function of the wild type p53 protein) appeared to
have more clinically severe phenotypes than did individuals with other
TP53 pathogenic variants \[Bougeard et al 2015\]. A laboratory study
also reported that **dominant-negative pathogenic variants appear to
cause a more profound alteration of p53** DNA binding than other
pathogenic variants \[Zerdoumi et al 2017\].

However, the **penetrance of LFS may be overestimated** as more
individuals recently identified with a germline TP53 pathogenic variant
do not meet classic LFS or Chompret criteria due to a less striking
family and personal history of cancer \[Rana et al 2018\]. **Individuals
with TP53 pathogenic variant p.Arg337His appear to have a lower
lifetime risk of cancer than those with other TP53 pathogenic variants**
\[Ferreira et al 2019\].

...study of 214 families harboring 133 distinct TP53 alterations..... Of
these 133 alterations, 124 were **point mutations. Nine families
carried distinct TP53 genomic rearrangements (two whole gene
deletions, six partial deletions and one partial duplication**).
Functional comparison studies indicated that patients with
dominant-negative-acting missense mutations had significantly
earlier tumor onset, non-dominant-negative missense mutations were
intermediate in severity, and loss-of-function type mutations
(including nonsense, frameshift and genomic alterations) are
associated with later tumor onset.... A deletion segregated with LFS
in four subjects but was also identified in an unaffected 70-year old
relative, indicating incomplete penetrance.

***Bougeard et al, 2015, PMID 26014290***

....eight patients with **CNVs affecting TP53** from a cohort of 4524
patients with diverse phenotypes tested across multiple diagnostic
laboratories in Canada and the US. Four patients with focal TP53
deletions **(intragenic/exonic or 3? partial**) were affected with
early-onset cancer while four patients with non-focal deletions
encompassing TP53 and additional genes in 17p13.1 (size range 543 kb
to 2.4 Mb) had overlapping developmental phenotypes but did not have
cancer at the age of ascertainment (Ages 3.4, 5.75, 7.58, 33.67).

**Shlien et al, 2010, PMID 21056402**

**The spectrum of mutations shows that the majority of all germline
mutations cluster in exons 5-8,** however this is undoubtedly skewed
because most studies only examine these exons. In our own study, of 19
mutations found to date, five fall outside these exons (26%;

Varley et al, 1997a). This finding has profound implications for genetic
testing within Li-Fraumeni families, and we recommend that all exons,
both coding and non-coding, plus all splice junctions and the promoter
region are analysed. We have analysed all the reported germline
mutations to determine the mutation type. There is no difference in the
frequency of **missense and nonsense** mutations between Li-Fraumeni
families,other types of families and individuals with no family history.
In addition, there are no differences between LFS and LFL families.
There are two reports of **large germline deletions, one involving the
whole of exon 10 (Plummer et al, 1994) and the other removing 167 bp,
including part of exon 1 and intron 1** (Varley et al, 1997a). Both of
these deletions were flanked by short direct repeat sequences. Thirteen
other mutations involve **short insertion or deletion** events
(Sameshima et al, 1992; Toguchida et al, 1992; Felix et al, 1993; Birch
et al, 1994a; Hamelin et al, 1994; Mazoyer et al, 1994; Stolzenberg et
al, 1994; Lubbe et al, 1995; Strauss et al, 1995; Felix et al, 1996;
Varley et al, 1996a, 1997a).

***Varley, Evans and Birch, 1997, PMID: 9218725***

Rare germline variant (rs78378222) in the TP53 3\' UTR - Evidence for a
new mechanism of cancer predisposition in Li-Fraumeni syndrome. .... a
rare variant that is located in the 3\' untranslated region
(3\' UTR) of TP53, in 7 probands (5.4%) of a cohort from LFS/LFL
patients without TP53 germline mutations in the coding regions.

***Macedo et al, 2016, PMID: 26823150***

**Pilot application of harmonised terms:**

**Inheritance:**

*Autosomal Dominant*

Optional Modifier -- Incomplete penetrance

**Allelic requirement:**

Monoallelic_aut    

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Molecular mechanism is loss of function or dominant negative; both of
which result in reduced (or absent) transcriptional activity and loss of
the tumor suppressive function of the p53 protein. Variant classes
include missense, small duplications, small deletions, frameshift, and
nonsense mutations. Variants are distributed throughout the coding
sequence and whole gene deletions are also documented. Some variant have
been shown to have incomplete penetrance (p.Arg337His).

**List variant classes in this gene proven to cause this disease:**

Splice_acceptor_variant

Splice_acceptor_variant predicted to undergo NMD

Splice_donor_variant

Splice_donor_variant predicted to undergo NMD

Frameshift_variant

Frameshift_variant predicted to undergo NMD

Stop_gained

Stop_gained predicted to undergo NMD

Inframe_deletion

Inframe_insertion

Missense

3_prime_UTR_variant

**Potential novel variant classes based on predicted functional
consequence**

Splice_region_variant

Splice_acceptor_variant predicted to escape NMD

Splice_donor_variant predicted to escape NMD

Frameshift_variant predicted to escape NMD

Start_lost

Stop_gained predicted to escape NMD

Stop_lost

5_prime_UTR_variant

Gain of upstream Start \[uORF\]

Gain of upstream Start \[oORF\]

Stop lost \[uORF\]

Stop lost \[oORF\]

Start lost \[uORF\]

Frameshift \[uORF\]

Frameshift \[oORF\]

Stop gained \[uORF\]

**Not included**

intron_variant
# **TPM1 -- Dilated Cardiomyopathy**

**Review of source material:**

**ClinGen:**

Not yet curated by ClinGen for DCM

**Literature review:**

Recent study compared the burden of rare protein-altering variation in DCM-associated genes in up to 2538 patients with DCM.  The variant classes significantly enriched in 1 or both DCM cohorts were truncating variants in TTN, DSP, LMNA, BAG3, and VCL,
and nontruncating variants in MYH7, TNNT2, TNNC1, LMNA, and TPM1. 

*Mazzarotto F et al 2020 PMID: 31983221*

Using depletion and reconstitution of TPM1 and troponin in porcine cardiac myofibrils, Chang et al. (2005) studied 3 hypertrophic cardiomyopathy (CMH) -associated TPM1 mutations (E62Q; E180G, 190010.0001; and L185R) and 2 dilated cardiomyopathy (CMD) -associated mutations (E54K and E40K) and found that all CMH-associated mutations increased the Ca(2+) sensitivity of ATPase activity and had decreased abilities to inhibit ATPase activity, whereas **the CMD-associated mutations decreased the Ca(2+) sensitivity of ATPase activity and had no effect on the inhibition of ATPase activity.** Chang et al. (2005) concluded that mutations that cause CMH and CMD disrupt discrete mechanisms, and suggested that this may explain the manifestation of distinct cardiomyopathic phenotypes. 

Mirza et al. (2005) studied all 8 published mutations causing dilated cardiomyopathy (CMD), including 5 in the TNNT2 gene (lys210del, R141W, R131W, R205L, and D270N; 191045.0006-191045.0010, respectively), 2 in the TPM1 gene (E54K and E40K), and 1 in the TNNC1 gene (G159D, 191040.0001). Thin filaments, reconstituted with a 1:1 ratio of mutant:wildtype proteins, all showed reduced Ca(2+) sensitivity of activation in ATPase and motility assays, and, except for the E54K alpha-tropomyosin mutant which showed no effect, all showed lower maximum Ca(2+) activation

*Omim <https://omim.org/entry/191010>*

**From our in-house Atlas of HCM:**

19/21 missense
1/21 frameshift (VUS)
1/21 nonsense (VUS)

<https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=TPM1&icc=HCM>

*Walsh et al, 2016 (PMID 27532257)*
*Pugh TJ et al, 2014 (PMID: 24503780)*

DCM is in general characterised by reduced penetrance and variable expressivity.

*Hershberger RE et al 2013 PMID: 23900355*

"...studies revealed that mutations in thin filament proteins cause HCM and DCM via an opposing functional defect. HCM mutations cause increased myofilament Ca2+ sensitivity, whereas DCM mutations have been shown to reduce myofilament Ca2+ sensitivity.
In inherited cardiomyopathies, the evidence presented in TABLES 2–5 indicates a direct link between the effect of thin filament gene mutations on Ca2+ sensitivity of force and the prevailing phenotype: an association between HCM and DCM with an increase or decrease in Ca2+ sensitivity of force, respectively."

*van der Velden J et al 2019 PMID: 30379622* 

"One of these sarcomere genes is TPM1, which encodes alpha-tropomyosin and causes HCM (10 TPM1 gene variants), DCM (11 TPM1 variants), cardiomyopathy (2 TPM1 variants) and left-ventricular non-compaction (3 TPM1 variants) according to ClinVar database (January 2020) [4]. **All reported TPM1 variants are missense variants that result in a single amino acid substitution, and they account for 1.5% of all likely pathogenic and pathogenic gene variants in HCM and 1.9% in DCM [5]."**

Our study confirms this clinical heterogeneity in two families with TPM1 variants and different cardiomyopathy subtypes, namely TPM1T201M (DCM) and TPM1E62Q and TPM1M281T (HCM; and in combination RCM). We also describe, for the first time, compound heterozygote TPM1E62Q/M281T variants in a child with RCM. Overall, the genotypes and diverse clinical characteristics of the family members of the DCM and RCM probands suggest that additional causal factors are necessary for the development of cardiac abnormalities in carriers of a heterozygous TPM1 variant, whereas compound heterozygosity for TPM1 variants causes severe cardiomyopathy in childhood.

"...we observed a decrease in Ca transients for both HCM- and DCM-related TPM1 variants. The previous study and our current data indicate that TPM1 variant-related changes in Ca2+ affinity and CaT are independent of the type of cardiac remodeling, which suggests that the diverse clinical cardiac phenotypes seen in affected subjects may rather involve modifier genes and/or environmental factors [17]."

*Dorsch LM et al 2020 PMID: 32882290*

The aim of the present study is to investigate the correlation between the 3' untranslated region (3'UTR) single nucleotide polymorphisms (SNPs) of the TPM1 gene and dilated cardiomyopathy (DCM). A total of 245 patients with DCM and 245 healthy controls were recruited to analyze the TPM1 gene rs12148828, rs11558748, rs707602, rs6738, rs7178040 loci genotypes. The risk of DCM development in the rs6738 locus G allele carriers were 1.69 times more than A allele carriers (95% CI: 1.22-2.33, P = .001). Age and gender had no effect on the association of TPM1 gene SNPs with DCM risk (P > .05).

*Yao Q et al 2019 PMID: 31689804*

#### **Pilot application of harmonised terms**

**Inheritance**

Autosomal dominant

Optional modifiers: incomplete penetrance

**Allelic requirement**

Monoallelic_aut

Optional modifiers:

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is primarily due to **altered gene product structure.** The majority of pathogenic variants described are missense variants. Non-truncating variants in TPM1 were enriched in a large cohort of DCM patients (2538) in comparison to controls. Pathogenic variants in TPM1 have also been associated with HCM. None of the variants associated with DCM have been seen to cause HCM in other individuals or families. Pathogenic variants associated with DCM appear to decrease the Ca(2+) sensitivity of ATPase activity and had no effect on the inhibition of ATPase activity. This was in contrast to variants that caused HCM which appeared to increase the Ca(2+) sensitivity of ATPase activity and had decreased abilities to inhibit ATPase activity. More recently Dorsch et al observed "a decrease in Ca transients for **both** HCM- and DCM-related TPM1 variants" suggesting "that TPM1 variant-related changes in Ca2+ affinity and CaT are independent of the type of cardiac remodeling, which suggests that the diverse clinical cardiac phenotypes seen in affected subjects may rather involve modifier genes and/or environmental factors..."  DCM is in general characterised by reduced penetrance and variable expressivity.

**List variant classes in this gene proven to cause this disease:**

- Missense

**Potential novel variant classes based on predicted functional
consequence**

- Frameshift predicted to escape NMD
- Stop_gained predicted to escape NMD
- Splice donor variant
- Splice donor variant predicted to escape NMD
- Splice acceptor variant predicted to escape NMD
- Stop_lost
- In frame_insertion
- In frame deletion

# **TPM1 -- Hypertrophic Cardiomyopathy**

**Review of source material:**

<https://www.ahajournals.org/doi/10.1161/CIRCGEN.119.002460>

**ClinGen:**

<https://search.clinicalgenome.org/kb/gene-validity/8771>

The TPM1 gene has been associated with hypertrophic cardiomyopathy
(HCM). TPM1 was first associated with this disease in humans in 1994
(Thierfelder et al, PMID 8205619). At least 15 unique heterozygous
variants (missense), with varying levels of evidence to support their
pathogenicity, have been reported in humans (reviewed in Redwood and
Robinson, 2013, PMID 24005378). Variants in this gene segregated with
disease in at least 7 families (Thierfelder et al, 1994, PMID 8205619;
Jääskeläinen et al, 1998, PMID 9822100; Karibe et al, 2001, PMID
11136687; Jongbloed et al, 2003, PMID 12651045). More evidence is
available in the literature, but the maximum score for genetic evidence
(12 pts) has been reached. The mechanism for disease is likely dominant
negative (Redwood and Robinson, 2013, PMID 24005378). The gene-disease
association is supported the function of the gene product, animal
models, and in vitro assays. In summary, TMP1 is definitively associated
with HCM. This has been repeatedly demonstrated in both research and
clinical diagnostic settings, and has been upheld over time. This
classification was approved by the ClinGen Hypertrophic Cardiomyopathy
Gene Curation Expert Panel on December 20, 2016.

**Literature review:**

Redwood and Robinson reviewed the literature in 2013 and reported that there were at least 15 unique missense mutations reported in humans with HCM. Mutations in TPM1 have also been associated with DCM. None of the mutations associated with DCM have been seen to cause HCM in other individuals or families.

***"TPM1* mutations most likely act via a dominant negative, poison polypeptide mechanism rather than via haploinsufficiency.**

Analysis of vastus lateralis muscles from two HCM patients carrying the Asp175Asn mutation revealed equal expression of wild type and mutant α-tropomyosin proteins (Bottinelli et
al. [1998])

*Redwood and Robinson, 2013, PMID 24005378*

"Of the 4 human tropomyosin genes, TPM1 is the most versatile and
encodes at least 10 tissue-specific variants via alternative splicing and/or the use of 2 promoters."

Mutations in TPM1 "like those in the cardiac troponin T gene
(TNNT2; [191045](https://omim.org/entry/191045)), are characterized by relatively mild and sometimes subclinical hypertrophy but a high incidence of sudden death. Genetic testing may therefore be especially important in this group"

*Omim <https://omim.org/entry/191010>*

"...studies revealed that mutations in thin filament proteins cause HCM and DCM via an opposing functional defect. HCM mutations cause increased myofilament Ca2+ sensitivity, whereas DCM mutations have been shown to reduce myofilament Ca2+ sensitivity.
In inherited cardiomyopathies, the evidence presented in TABLES 2–5 indicates a direct link between the effect of thin filament gene mutations on Ca2+ sensitivity of force and the prevailing phenotype: an association between HCM and DCM with an increase or decrease in Ca2+ sensitivity of force, respectively."

*van der Velden J et al 2019 PMID: 30379622* 

"One of these sarcomere genes is TPM1, which encodes alpha-tropomyosin and causes HCM (10 TPM1 gene variants), DCM (11 TPM1 variants), cardiomyopathy (2 TPM1 variants) and left-ventricular non-compaction (3 TPM1 variants) according to ClinVar database (January 2020) [4]. **All reported TPM1 variants are missense variants that result in a single amino acid substitution, and they account for 1.5% of all likely pathogenic and pathogenic gene variants in HCM and 1.9% in DCM [5]."**

Our study confirms this clinical heterogeneity in two families with TPM1 variants and different cardiomyopathy subtypes, namely TPM1T201M (DCM) and TPM1E62Q and TPM1M281T (HCM; and in combination RCM). We also describe, for the first time, compound heterozygote TPM1E62Q/M281T variants in a child with RCM. Overall, the genotypes and diverse clinical characteristics of the family members of the DCM and RCM probands suggest that additional causal factors are necessary for the development of cardiac abnormalities in carriers of a heterozygous TPM1 variant, whereas compound heterozygosity for TPM1 variants causes severe cardiomyopathy in childhood.

"...we observed a decrease in Ca transients for both HCM- and DCM-related TPM1 variants...Our current data indicate that TPM1 variant-related changes in Ca2+ affinity and CaT are independent of the type of cardiac remodeling, which suggests that the diverse clinical cardiac phenotypes seen in affected subjects may rather involve modifier genes and/or environmental factors [17]."

*Dorsch LM et al 2020 PMID: 32882290*

**From our in-house Atlas of HCM:**

66/66 missense

<https://www.cardiodb.org/acgv/acgv_gene_disease.php?gene=TPM1&icc=HCM>

*Walsh et al, 2016 (PMID 27532257)*

#### **Pilot application of harmonised terms**

**Inheritance**

Autosomal dominant

Optional modifiers: ?age related penetrance

**Allelic requirement**

Monoallelic_aut

Optional modifiers:

**Disease associated variant consequences:**

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Mechanism is primarily due to **altered gene product structure.** All reported TPM1 variants are missense variants that result in a single amino acid substitution, and they account for 1.5% of all likely pathogenic and pathogenic gene variants in HCM and 1.9% in DCM. In the 2013 Redwood et al review 15 unique missense mutations had been reported in humans with HCM. None of the mutations associated with DCM have been seen to cause HCM in other individuals or families. Redwood et al propose the most likely mechanism to be dominant negative, there is no evidence currently for haploinsufficiency. 
A review by van der Velden J et al proposed "that mutations in thin filament proteins cause HCM and DCM via an opposing functional defect. HCM mutations cause increased myofilament Ca2+ sensitivity, whereas DCM mutations have been shown to reduce myofilament Ca2+ sensitivity." More recently Dorsch et al observed "a decrease in Ca transients for both HCM- and DCM-related TPM1 variants" suggesting "that TPM1 variant-related changes in Ca2+ affinity and CaT are independent of the type of cardiac remodeling, which suggests that the diverse clinical cardiac phenotypes seen in affected subjects may rather involve modifier genes and/or environmental factors..." 
There is significant clinical heterogeneity even within the same family.  There is one report of compound heterozygote TPM1E62Q/M281T variants in a child with RCM. The authors suggest that compound heterozygosity for TPM1 variants causes severe cardiomyopathy in childhood.

**List variant classes in this gene proven to cause this disease:**

- Missense

**Potential novel variant classes based on predicted functional
consequence**

- Frameshift predicted to escape NMD
- Stop_gained predicted to escape NMD
- Splice donor variant
- Splice donor variant predicted to escape NMD
- Splice acceptor variant predicted to escape NMD
- Stop_lost
- In frame_insertion
- In frame deletion

# **TSC1 -- Tuberous sclerosis**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:12362

TSC1, which encodes the protein hamartin, was first reported in relation to autosomal dominant tuberous sclerosis complex (TSC) in 1997 (van Slegtenhorst et al., 1997 PMID: 9242607). Numerous variants have been reported in TSC1 in relation to tuberous sclerosis complex, **including missense, nonsense, indels and small deletions that result in frameshift, as well as intragenic deletions (Hasbani and Crino, 2018 PMID: 29478616).** The TSC1 database in LOVD (http://chromium.lovd.nl/LOVD2/TSC/home.php?select_db=TSC1) shows a total of 928 unique variants associated with TSC. Evidence supporting this gene-disease relationship includes case-level data, segregation and experimental data. This gene-disease relationship has been studied for 20 years, therefore a significant amount of case-level data and segregation is available, however the maximum score for genetic evidence (12 points) has been reached. This gene-disease relationship is supported by functional studies including expression and protein interaction, as well as animal models and rescue experiments, thus the maximum score for experimental evidence (6 points) has been reached. TP53 has been associated with several disease entities and/or phenotypes through germline inheritance, including: (1) Tuberous sclerosis-1 (MIM:191100) and (2) Lymphangioleiomyomatosis (MIM:606690) Evidence in the literature supports that Lymphangioleiomyomatosis occurs in individuals with TSC and mutations in TSC1. Additionally, a sporadic form of Lymphangioleiomyomatosis (S-LAM) can occur through by the two hit model, in which the wildtype allele of TSC1 incurs loss of heterozygosity (LOH) specifically in the lung (Murphy et al., 2017 PMID: 28643793). Of note, the involvement of TSC1 in Lymphangioleiomyomatosis (sporadic) is rare, and the TSC2 gene is more frequently shown to have a relationship with S-LAM. Hamartin, the protein product of the TSC1 gene, is a known tumor suppressor, therefore loss of function results in uncontrolled cell proliferation. Consistent with this model the **molecular mechanism for TSC1 is loss of function, as indicated by deletion and nonsense variation, as well as biochemical loss of tumor suppressor activity (Hasbani and Crino, 2018 PMID: 29478616).** Per the criteria outlined by the ClinGen Lumping and Splitting Working Group, we have found no difference in molecular mechanism, inheritance pattern, or phenotypic expressivity, and therefore Lymphangioleiomyomatosis due to germline mutations in TSC1 alone that occur in TSC patients has been lumped into the disease entity, Tuberous sclerosis complex. However, the somatically occurring Lymphangioleiomyomatosis due to one (or two) somatic mutations in TSC1 is not covered under the current Gene Disease Validity Classification matrix, asit is restricted to germline inheritance, therefore no evidence for somatic mutations in TSC1 were included in the current curation. In summary, TSC1 is DEFINITIVELY associated with autosomal dominant tuberous sclerosis complex. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time.

**ClinGen Evidence for Haploinsufficiency**
PMID 9242607 van Slegtenhorst et al. (1997): Reports numerous nonsense mutations identified within TSC1 amongst individuals with a clinical diagnosis of tuberous sclerosis.

PMID 9924605 Kwiatkowska et al. (1998): Reports two additional nonsense mutations (not previously reported in van Slegtenhorst et al. 1997) identified within TSC1 amongst individuals with a clinical diagnosis of tuberous sclerosis

PMID 9803264 Young et al. (1999): Reports several other nonsense mutations identified amongst individuals with a clinical diagnosis of tuberous sclerosis not previously reported in van Slegtenhorst et al.

**Literature review:**

**"Almost all TSC1 pathogenic variants are predicted to cause truncation of the hamartin protein; the location of the TSC1 pathogenic variant does not appear to associate with disease severity. Approximately 650 unique TSC1 pathogenic variants have been identified in more than 1,950 individuals/families with TSC1-related TSC** (Table A). Most pathogenic variants are unique, but a few are known to recur, including those in specific codons of exon 15. Other pathogenic variants are scattered throughout the exons and splice sites. **A small percentage of pathogenic missense variants have been identified and located mostly in the region encoding the N-terminal of hamartin** 

TSC1 is approximately 50 kb in size and the longest transcript variant (NM_000368.4), comprising 23 exons. The first two exons are noncoding. Exon 5 and exon 12 are alternatively spliced, producing shorter transcript variants.

Tuberin and hamartin were shown to be key regulators of the AKT/mTOR signaling pathway and to participate in several other signaling pathways including the MAPK, AMPK, b-catenin, calmodulin, CDK, autophagy, and cell cycle pathways [Kozma & Thomas 2002, Astrinidis et al 2003, El-Hashemite et al 2003, Harris & Lawrence 2003, Yeung 2003, Au et al 2004, Birchenall-Roberts et al 2004, Li et al 2004, Mak & Yeung 2004, Zhang et al 2013]. The hamartin tuberin complex can also regulate mTORC2 complex activity that affects cytoskeleton formation and AKT activation [Han & Sahin 2011].

Most TSC1 pathogenic variants and 70% of TSC2 pathogenic variants are predicted to result in a loss of functional protein products. Subsequent loss of function leads to uncontrolled cell growth and cell proliferation resulting in the formation of hamartias (a focal malformation consisting of disorganized arrangement of tissue types that are normally present in the anatomic area) and hamartomas...

...because tuberin and hamartin are subjected to multiple cell signaling pathway regulation, the quantity and quality of both somatic pathogenic variants and environmental factors targeting these pathways are expected to modify disease expression in individuals who have only one normal germline copy of TSC1 or TSC2.

A pathogenic variant is defined as a variant that clearly inactivates the function of the TSC1 or TSC2 proteins (i.e., out-of-frame indel or nonsense variant), prevents protein synthesis (i.e., large genomic deletion), or is a pathogenic missense variant whose effect on protein function has been established by functional assessment (see LOVD Database – TSC1, LOVD Database – TSC2, Hoogeveen-Westerveld et al [2012], and Hoogeveen-Westerveld et al [2013])."

**Gene reviews**
*https://www.ncbi.nlm.nih.gov/books/NBK1220/#tuberous-sclerosis.Molecular_Genetics*

"Over 1,800 different small TSC-causing mutations
have been defined and these are distributed throughout
the coding regions of both genes, except for the final
exon (23) of TSC1 and the alternatively spliced exons
(25 and 31) of TSC2. Mutations are catalogued at http://
chromium.lovd.nl/LOVD2/TSC. The great majority
of TSC1 mutations are small truncating nonsense and
insertion or deletion (indel) mutations with only a small
number of functionally confirmed missense mutations
identified, all of which occur in the 5ʹ region of the
gene15,16. 

Penetrance is almost complete, but some individuals carrying ‘mild mutations’ may not fulfil clinical criteria for definite TSC"

*Henske E et al. 2016 PMID: 27226234*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The mechanism appears to be loss of function of TSC1 leading to **decreased/absent or altered gene product** which results in uncontrolled cell growth and proliferation. TSC1 is approximately 50 kb in size and the longest transcript (NM_000368.4), comprises 23 exons. The first two exons are noncoding. Exon 5 and exon 12 are alternatively spliced, producing shorter transcript variants. Most variants are unique with only a few recurrent mutations reported in specific regions including certain codons of exon 15. Mutations are distributed throughout the coding regions of the gene, except for the final exon (23). The majority of mutations are small truncating nonsense and insertion or deletion mutations predicted to cause truncation of the hamartin protein with only a small number of functionally confirmed missense mutations
identified, all of which occur in the 5ʹ region of the gene. Hamartin is a key regulator of the mTOR-AKT pathway. Penetrance is nearly 100% but there is variable expressivity.

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Missense

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- In frame deletion
- In frame duplication
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
# **TSC2 -- Tuberous sclerosis**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:12363

TSC2, which encodes the protein tuberin, was first reported in relation to autosomal dominant tuberous sclerosis complex (TSC) in 1993 (European Chromosome 16 tuberous Sclerosis Consortium PMID: 8269512). **Numerous variants have been reported in TSC2 in relation to tuberous sclerosis complex, including missense, nonsense, indels and small deletions that result in frameshift, as well as intragenic deletions** (Hasbani and Crino, 2018 PMID: 29478616). The TSC2 database in LOVD (http://chromium.lovd.nl/LOVD2/TSC/home.php?select_db=TSC2) shows a total of 2689 unique variants associated with TSC2 (as of Jan 07, 2019). Evidence supporting this gene-disease relationship includes case-level data and experimental data. This gene-disease relationship has been studied for 20 years, therefore a significant amount of case-level data is available, however the maximum score for genetic evidence (12 points) has been reached. This gene-disease relationship is supported by functional studies including expression and protein interaction and animal models. The molecular mechanism for TSC2 in tuberous sclerosis is loss of function, as indicated by deletion and nonsense variation, as well as biochemical loss of tumor suppressor activity (Hasbani and Crino, 2018 PMID: 29478616). In summary, TSC2 is DEFINITIVELY associated with autosomal dominant tuberous sclerosis complex. This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time.

**ClinGen Evidence for Haploinsufficiency**

PMID 31525612 Published in 2019, This study found 3 nonsense variants, 6 splicing variants, a total of 18 frameshift variants, and 2 large deletions in TSC2. Additionally, 51.9% of the cohort (39 individuals) had de novo variants in TSC2.

PMID 10205261 Published in 1999, Jones et al. used PCR and single-strand confirmation polymorphism (SSCP) and Heteroduplex analysis on 150 Tuberous sclerosis complex patients and their families to assess variants in TSC1 and TSC2. The methodology found that 9 cases of variants in TSC2 were familial; 88 other cases involving TSC2 variants were sporadic. This study identified 20 nonsense variants, 21 small insertions or deletions, 8 splice variants, 5 in-frame deletions, and 8 large intragenic deletions in TSC2.

PMID 29932062 Published in 2014, Glushkova et al. used Sanger sequencing and multiplex ligation-dependent probe amplification (MLPA) on 17 unrelated individuals who met the clinical description for Tuberous sclerosis complex (TSC) to search for variants in TSC1 and TSC2. This study identified 2 nonsense variants, 3 frame-shift variants, 1 splice variant, and 1 large deletion in TSC2. In terms of inheritance, 4 individuals had a de novo variant.

Haploinsufficiency phenotype comments:
Variants involving the TSC2 gene, including intragenic and whole-gene heterozygous deletions, cause tuberous sclerosis complex (TSC). TSC is an autosomal dominant multi-system disorder, which affects 1 in 6000 people. About half of these patients are affected by intellectual disability. **PMID 11030407: Reviewed 222 TSC2 mutations that had been reported in the literature. Large genomic deletions (intragenic and whole-gene) accounted for 16% of all variants reported.**

**Literature review:**

Fifty percent (145/292) of TSC2 mutations were point
mutations (see online tables 2, 3). In contrast to TSC1,
nonsense mutations in TSC2 made up only 38% (55 of
145) of this class. TSC2 has seven CpG sites that can undergo transition to nonsense codons; however, mutations
have only been seen at five of these and they comprise
only 36% of the nonsense mutations. TSC2 missense mutations (35 verified and 24 probable) were also common
and accounted for 41% (59 of 145). **TSC2 has two CpG
sites that appear to be mutational hotspots for missense
mutations: 1831–2 (611R→W and 611R→Q) and 5024–5
(1675P→L). These two CpG sites contain 22 of the 59
(37%) verified or probable missense mutations. Other
than the CpG sites, missense mutations were clustered
(8/574 versus 23/4850, P<0.02) in the region with
rap1GAP homology (exons 34–38).** Splice mutations (of
which three were seen recurrently) made up the remaining
21% of point mutations in TSC2.

*Cheadle J 2000 PMID 11030407*

70% of TSC2 pathogenic variants are predicted to result in a loss of functional protein products. Subsequent loss of function leads to uncontrolled cell growth and cell proliferation resulting in the formation of hamartias (a focal malformation consisting of disorganized arrangement of tissue types that are normally present in the anatomic area) and hamartomas.

More than 1,900 unique TSC2 pathogenic variants distributed throughout the gene have been identified in more than 5,800 individuals/families with TSC2-related TSC. **Approximately 33% of TSC2 pathogenic variants are located in exons 32-41 (and associated splice sites) that encode the carboxy domain of tuberin** consisting of several important functional motifs (e.g., GAP domain, estrogen receptor- and calmodulin-binding domains, and multiple signal pathway kinase targets).

Missense variants account for approximately 26% of all TSC2 pathogenic variants with approximately 50% concentrated in the carboxy domain. Missense variants are rarely the direct target of kinases: only two missense variants at the Tyr1571 residue are the predicted target of tyrosine kinase [Hoogeveen-Westerveld et al 2013].

A higher percentage of individuals with more severe features of TSC have a de novo TSC2 pathogenic variant versus a de novo TSC1 pathogenic variant [Au et al 2007]. Individuals representing simplex cases (i.e., a single occurrence in a family) are more likely to have a TSC2 pathogenic variant, while those with familial TSC have an almost equal proportion of TSC1 and TSC2 pathogenic variants [Au et al 2007].

Individuals with a TSC2 pathogenic variant are at greater risk for:
Renal malignancy [Yang et al 2014]
Intellectual disability[Kothare et al 2014]
Autistic disorder, low IQ, and infantile spasms [Numis et al 2011]

**Gene reviews**
*https://www.ncbi.nlm.nih.gov/books/NBK1220/#tuberous-sclerosis.Molecular_Genetics*

"Over 1,800 different small TSC-causing mutations
have been defined and these are distributed throughout
the coding regions of both genes, except for the final
exon (23) of TSC1 and the alternatively spliced exons
(25 and 31) of TSC2. Mutations are catalogued at http://
chromium.lovd.nl/LOVD2/TSC. TSC2 mutations include frequent
missense mutations (30% of cases) and large deletions
and other rearrangements (5% of cases).

Penetrance is almost complete, but some individuals carrying ‘mild mutations’ may not fulfil clinical criteria for definite TSC"

*Henske E et al. 2016 PMID: 27226234*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The mechanism appears to be loss of function of TSC2 leading to **decreased/absent or altered gene product** which results in uncontrolled cell growth and proliferation. Missense variants account for approximately 30% of all TSC2 pathogenic variants with approximately 50% concentrated in the carboxy domain. Variants are distributed throughout the coding regions of TSC2, except for the alternatively spliced exons (25 and 31). Approximately 33% of TSC2 pathogenic variants are located in exons 32-41 (and associated splice sites) that encode the carboxy domain of tuberin. PMID 11030407: Reviewed 222 TSC2 mutations that had been reported in the literature. Large genomic deletions (intragenic and whole-gene) accounted for 16% of all variants reported.
Penetrance is nearly 100% but there is variable expressivity.

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Missense

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- In frame deletion
- In frame duplication
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
# **VHL -- Von Hippel-Lindau**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:12687

The VHL gene is associated with the autosomal dominant cancer syndrome von Hippel Lindau disease which includes retinal hemangiomas, cerebellar and spinal hemangioblastomas, renal cell carcinoma and pheochromocytomas https://www.ncbi.nlm.nih.gov/books/NBK1463/. . Following multiple reports of segregation and linkage analysis from large pedigrees of VHL-affected families (PMID: 6582782, PMID: 2642584, PMID: 1982450, PMID: 1680799), the VHL gene was identified by Latif et al in 1993 (PMID: 8493574). Numerous variants have been reported in VHL in relation to the development of von Hippel Lindau disease, and this gene curation relies on a cohort of 114 cases with 12 unique VHL truncating and missense variants from Chen et al 1995 (PMID:7728151) . Although the present curation does not distinguish between VHL Type 1 and Type 2, generally, truncating and grossly damaging missense are associated with VHL Type 1 while less damaging missense are associated with VHL Type 2. The present curation does not include Chuvash Polycythemia or associated variants. Other databases describing germline VHL variants include: Clinical Interpretations of Variants in Cancer (https://civicdb.org), VHLdb (http://vhldb.bio.unipd.it/home), and large compendiums of genetic variants (PMID: 20151405). There is extensive genetic evidence supporting this gene-disease relationship including both case-level and familial data. The VHL gene is a tumor-suppressor with a wide variety of cellular functions (PMID: 25533676), most notably in regulation of Hypoxia Inducible Factor A (HIFa) through binding to hydroxylated HIFa, which induces subsequent ubiquitination and proteasomal degradation (PMID: 25533676). To bind hydroxylated HIFa, VHL is assembled into a complex with CUL2, RBX1 and Elongin B and C (VCB complex). The molecular mechanism of VHL dysfunction in von Hippel Lindau disease involves mutations which result in absent or reduced assembly of the VCB complex, or absent/reduced binding of the VCB complex (VHL) to HIFa. This gene-disease relationship is supported experimentally by expression studies, biochemical function, functional alterations and animal models. In summary, VHL is definitively associated with the autosomal autosomal dominant von Hippel Lindau syndrome. This has been demonstrated in both the research and clinical diagnostic settings and has been upheld over time.
Gene Clinical Validity Standard Operating Procedures (SOP) - Version 7

**ClinGen Evidence for Haploinsufficiency**
Loss-of-function-type mutations (including whole gene deletions) in the gene VHL are associated with autosomal dominant von Hippel-Lindau syndrome (VHL, OMIM: 193300). VHL is a cancer predisposition syndrome characterized by the development of hemangioblastomas of the brain, spine, and retina, renal lesions (including renal cysts and renal cell carcinoma), pheochromocytoma, paragangliomas, pancreatic lesions (including pancreatic cysts and neuroendocrine tumors), endolymphatic sac tumors, epididymal or papillary cystadenomas, and additional tumor types. The mean age-of-onset for VHL is ~26 years and the penetrance is approximately 87-97% by age 60 (see GeneReviews, OMIM, and PMIDs: 27966541 and 2274658).

Of note, homozygous or trans-heterozygous (germline) mutations in VHL are associated with the autosomal recessive condition, Familial Erythrocytosis-2 (OMIM: 263400)

**Literature review:**

**Our analysis of all VHL families found 52% had missense, 13%
had frameshift, 11% had nonsense, 6% had in-frame deletions/
insertions, 11% had large/complete deletions, and 7% had splice
mutations** (Tables 2–4, Fig. 3). Of the VHL Type 1 families 43, 17,13, 9, 8, and 10 were missense, frameshift, nonsense, splice, inframe deletion/insertions, and partial/complete deletions, respectively (Fig. 3). As expected, VHL Type 2 families mainly had
missense mutations (83.5%). However, this is not as high as some
studies, reporting up to 96% of those with pheochromocytomas
to have missense mutations [Zbar et al., 1996]. We also found
VHL Type 2 families had 7, 5, 4, 0.5, and 0.5% nonsense,
frameshift, splice, in-frame deletion/insertions, and partial deletions, respectively (Fig. 3). The small percentage of nonsense and partial deletions along with the absence of complete deletions
supports theories that an intact though altered pVHL is associated
with pheochromocytomas.

*Nordstrom-O'Brien M et al 2010 PMID: 20151405*

**Gene reviews**
More than 500 germline pathogenic variants have been identified in families with von Hippel-Lindau (VHL) syndrome (see Table A) [Nordstrom-O'Brien et al 2010]. The spectrum of pathogenic variants reported includes 52% missense, 13% frameshift, 11% nonsense, 6% in-frame deletions/insertions, 11% large/complete deletions, and 7% splice-site variants. Single-nucleotide variants have been identified in all three exons. The arginine codon 167 is considered a mutational "hot spot."

Synonymous pathogenic variants in VHL exon 2 that alter splicing through exon 2-skipping are associated with erythrocytosis or VHL disease in five families [Lenglet et al 2018].

Pathogenic variants in VHL either prevent its expression (i.e., deletions, frameshifts, nonsense variants, and splice site variants) or lead to the expression of an abnormal protein (i.e., pathogenic missense variants). The type of VHL that results from a pathogenic missense variant depends on its effect on the three-dimensional structure of the protein [Stebbins et al 1999]. Pathogenic variants in VHL cause misfolding and subsequent chaperonin-mediated breakdown [Feldman et al 2003]. Pathogenic missense variants that destabilize packing of the alpha-helical domains, decrease the stability of the alpha-beta domain interface, interfere with binding of elongin C and HIF1α, or disrupt hydrophobic core residues result in loss of HIF regulation and are more likely to result in VHL type 1. Pathogenic missense variants that result in pVHL that is normal with respect to HIF regulation are more likely to be associated with VHL type 2 (see Genotype-Phenotype Correlations). Furthermore, VHL pathogenic variants affect vessel branching and maturation via the Notch signaling pathway [Arreola et al 2018].

VHL pathogenic variants are highly penetrant. Almost all individuals who have a pathogenic variant in VHL are symptomatic by age 65 years [Maher et al 1991].

*https://www.ncbi.nlm.nih.gov/books/NBK1463/#vhl*

#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

Loss-of-function-type mutations (including whole gene deletions) in the gene VHL are associated with autosomal dominant von Hippel-Lindau syndrome. "The spectrum of pathogenic variants reported includes 52% missense, 13% frameshift, 11% nonsense, 6% in-frame deletions/insertions, 11% large/complete deletions, and 7% splice-site variants. Single-nucleotide variants have been identified in all three exons. The arginine codon 167 is considered a mutational "hot spot." Decreased/absent or altered gene product causes absent or reduced assembly of the VCB complex, or absent/reduced binding of the VCB complex (VHL) to HIFa. The mean age-of-onset for VHL is ~26 years. VHL pathogenic variants are highly penetrant. Almost all individuals who have a pathogenic variant in VHL are symptomatic by age 65 years
As per ClinGen, truncating variants and grossly damaging missense are associated with VHL Type 1 while less damaging missense are associated with VHL Type 2. Of note, homozygous or trans-heterozygous (germline) mutations in VHL are associated with the autosomal recessive condition, Familial Erythrocytosis-2 (OMIM: 263400)

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Missense
- In frame deletion
- In frame duplication
- Synonymous (alter splicing)
- Structural variant
- transcript ablation

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
# **WT1 -- Wilms Tumour**

**Review of source material:**

**ClinGen:**
https://search.clinicalgenome.org/kb/genes/HGNC:12796

Wilms tumor is the most common renal tumor of childhood, occurring with an incidence of 1 in 10,000 and with a median age of diagnosis between 3 and 4 years of age. Wilms tumours are thought to develop from abnormally persistent embryonal cells within nephrogenic rests (PMID: 19948536). The WT1 gene encodes a zinc finger DNA-binding protein that acts as a transcriptional activator or repressor depending on the cellular or chromosomal context (PMID: 1127846). The relationship between WT1 and Wilms tumor type 1 (autosomal dominant) was first reported in 1991 (Pelletier et al., PMID: 1654525). Evidence supporting this gene-disease relationship includes case-level data and experimental data. Seven variants (nonsense, frameshift) in this gene reported in at least 8 probands in 3 publications (PMIDs: 15483024, 1654525, 18688870) are included in this curation. More evidence is available in the literature, but the maximum score for genetic evidence (12 pts.) has been reached. This gene-disease association is also supported by expression studies, cell culture and mouse models (PMIDs: 2164159, 26035382, 21123950, 14645201). Per criteria outlined by the ClinGen Lumping and Splitting Working Group, **we found differences in molecular mechanism and phenotypic variability between Wilms tumor type 1 and congenital malformation syndromes associated with WT1 variants (Denys-Drash syndrome, Frasier syndrome, Meacham syndrome, and nephrotic syndrome, type 4). For example, many individuals with Denys-Drash syndrome harbor heterozygous missense variants in exon 8 or 9 and may act in a dominant-negative manner (Royer-Pokora et al., 2004; PMID: 15150775), while the majority of variants reported for Wilms tumor type 1 are loss-of-function variants.** Therefore, we have split curations for WT1 into Wilms tumor type 1 and congenital malformation syndromes associated with variants in WT1. **In summary, loss of function variants in WT1 are definitively associated with autosomal dominant Wilms tumor type 1.** This has been repeatedly demonstrated in both the research and clinical diagnostic settings, and has been upheld over time.
Gene Clinical Validity Standard Operating Procedures (SOP) - Version 7

**ClinGen Evidence for Haploinsufficiency**
Deletions and loss of function mutations are associated with increased risk for Wilms tumor with or without genitourinary malformations and/or renal failure. See Gene Reviews and Royer-Pokora (2004) PMID:15150775 for review. Large deletions that also encompass the PAX6 gene lead to WAGR syndrome (Wilms tumor-aniridia-genital anomalies-retardation). There are also many heterozygous missense mutations described that may act in a dominant negative manner that lead to a spectrum of clinical phenotypes including Denys-Drash syndrome, Frasier syndrome, Meacham syndrome, and Nephrotic syndrome type 4.

**Literature review:**

Only patients with exon mutations developed WT. Twelve of 36 (33%) with missense mutation and seven of eight (88%) with truncating mutations developed WT; no significant difference between mutations in DNA-binding or non–DNA-binding regions was observed. Patients with missense mutation were diagnosed at a median age of 1.3 (IQR, 1–1.8) years, patients with a truncating mutation at 1 (IQR, 0.8–1.3 months) (P=0.2). Bilateral WT occurred in five of seven (71%) patients with truncating mutations and two of 12 (17%) patients with missense mutations (P=0.04).

*Lehnhardt A et al 2015; PMID: 25818337*

**we observed a much higher frequency (52%) of bilateral tumors in patients carrying truncation mutations...**
The reduced frequency of bilateral tumors in patients carrying missense mutations compared with truncation mutations cannot be due to a difference in the mechanism of inactivating the second allele, however, and may be due to the function of the mutant protein. Most patients with missense mutations were described as having DDS, consistent withprevious reports of a predominance of WT1 missense muta-tions in DDS patients [Huff, 1996; Jeanpierre et al., 1998;Schumacher et al., 1998]. 

There were only three patients with characterized splice site mutations and all developed unilateral tumors. In all three cases no normal transcript was seen in the tumors[Royer-Pokora and Schumacher, in press; Schneider et al.,1993; Sakamoto et al., 2001].

*Royer-Pokora (2004) (PMID:15150775)*

Schumacher et al. (1997) identified 19 hemizygous WT1 gene mutations/deletions in tissue samples from 64 patients...Of the patients with germline alterations, 6 had associated genitourinary (GU) tract malformations and a unilateral tumor, 2 had a bilateral tumor and normal GU tracts, and 2 had a unilateral tumor and normal GU tracts. Three mutations were tumor-specific and were found in patients with unilateral tumors without genital tract abnormalities. These data demonstrated the correlation of WT1 mutations with stromal-predominant histology, suggesting that a germline mutation in WT1 predisposes to the development of tumors with this histology. Twelve mutations were nonsense mutations resulting in truncation at different positions in the WT1 protein, and only 2 were missense mutations. Of the stromal-predominant tumors, 67% showed loss of heterozygosity, and in 1 tumor a different somatic mutation in addition to the germline mutation was identified. **Thus, in a large proportion of a histopathologically distinct subset of Wilms tumors, the classic 2-hit inactivation model, with loss of a functional WT1 protein, is the underlying cause of tumor development.**

***Omim** https://www.omim.org/entry/194070*


#### **Pilot application of harmonised terms**

**Inheritance:**

Autosomal dominant

(optional) modifiers: 

**Allelic requirement:**

monoallelic_aut

(optional) modifiers 

**Disease associated variant consequences:**

Decreased gene product level

Absent gene product

Altered gene product structure

**Narrative summary of molecular mechanisms:**

The mechanism appears to be loss of function of WT1 resulting in **decreased/absent or altered gene product** which leads to a predisposition to Wilms tumour. The majority of variants reported in association with Wilms tumour type 1 are loss-of-function variants whereas many individuals with congenital malformation syndromes (e.g. Denys-Drash syndrome) harbor heterozygous missense variants in exon 8 or 9 which may act in a dominant-negative manner. Truncating mutations appear to cause a higher frequency of bilateral tumours than missense mutations.

**List variant classes in this gene proven to cause this disease:**

- Stop gained
- Stop gained (predicted to undergo NMD)
- Frameshift
- Frameshift (predicted to undergo NMD)
- Splice acceptor variant
- Splice acceptor variant (predicted to undergo NMD)
- Splice donor variant
- Splice donor variant (predicted to undergo NMD)
- Missense

**List potential novel variant classes based on predicted functional consequence:**

- Splice acceptor variant (predicted to escape NMD)
- Splice donor variant (predicted to escape NMD)
- Frameshift variant (predicted to escape NMD)
- start_lost
- stop_gained predicted to escape NMD
- stop_lost
- In frame deletion
- In frame duplication
- gain of upstream Start \[uORF\]
- gain of upstream Start \[oORF\]
- Stop lost \[oORF\]
- Frameshift \[oORF\]
